---
title: "DE Analysis"
author: "Ha Tran"
date: "22/08/2021"
output: workflowr::wflow_html
editor_options:
  chunk_output_type: console
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE,
                      message = FALSE,
                      warning = FALSE,
                      eval = TRUE,
                      fig.width = 11)
```

# Data Setup

### Load Library
```{r load libraries}
# working with data
library(dplyr)
library(magrittr)
library(readr)
library(tibble)
library(reshape2)
library(tidyverse)

# Visualisation:
library(kableExtra)
library(ggplot2)
library(grid)
library(pander)
library(cowplot)
library(pheatmap)
library(RColorBrewer)

# Custom ggplot 
library(ggpubr)
library(ggbiplot)
library(ggrepel)

# Bioconductor packages:
library(edgeR)
library(limma)
library(Glimma)

# Set ggplot theme
theme_set(theme_minimal())


# Adjust ggplot theme for poster and report
# theme_update(
#   plot.title = element_text(color = "gray20", size = 7, hjust = 0, vjust = .5, face = "bold"),
#   plot.subtitle = element_text(color = "gray25", size = 6, angle = 0, hjust = 0, vjust = .5, face = "plain"),
#   legend.title = element_text(color = "gray25", size = 5, angle = 0, hjust = 0, vjust = .5, face = "plain"),
#   legend.text = element_text(color = "gray25", size = 5, angle = 0, hjust = 0, vjust = .5, face = "plain"),
#   axis.text.x = element_text(color = "gray30", size = 5, angle = 0, hjust = .5, vjust = .5, face = "plain"),
#   axis.text.y = element_text(color = "gray30", size = 5, angle = 0, hjust = 1, vjust = 0, face = "plain"),
#   axis.title.x = element_text(color = "gray30", size = 6, angle = 0, hjust = .5, vjust = 0, face = "plain"),
#   axis.title.y = element_text(color = "gray30", size = 6, angle = 90, hjust = .5, vjust = .5, face = "plain"))
```


### Import DGElist Data

DGElist object containing the raw feature count, sample metadata, and gene metadata, created in the Set Up stage. 

```{r importData}
# load DGElist previously created in the set up
dge <- readRDS(here::here("0_data/rds_object/dge.rds"))
```


# Differential Gene Expression Analysis

Here multiple methods of identifying differential expression will be compared. These methods include

1. Generalised Linear Model (GLM) with Likelihood Ratio Test (LRT),
2. Generalised Linear Model (GLM) with Quasi-likelihood F-tests (QLF), and
3. Limma-voom 

**These methods have been internally assessed and only the GLM:QLF + TREAT method will be run**

## Initial Parameterisation

The varying methods used to identify differential expression all rely on similar initial parameters. These include:

1. The Design Matrix,
2. Estimation of Dispersion, and
3. Contrast Matrix

### Design Matrix

The experimental design can be parameterised in a one-way layout where one coefficient is assigned to each group. The design matrix formulated below contains the predictors of each sample

```{r designMatrix}
#setup desgin matrix with sample_group
designMatrix <- model.matrix(~ 0 + sample_group,
                             data = dge$samples)

#remove "sample_group" from each column names
colnames(designMatrix) <- gsub("sample_group", "", colnames(designMatrix))

#remove "S48h_" from row names
rownames(designMatrix) <- gsub("S48h_", "", rownames(designMatrix))

#display the design matrix
designMatrix %>% as.data.frame()
```

### Contrast Matrix
The contrast matrix is required to provide a coefficient to each comparison and later used to test for significant differential expression with each comparison group

```{r constrastMatrix}
# make specific contrasts
contrast <- limma::makeContrasts(
  Mac_vs_CKI = Mac - CKI,
  Nme_vs_CKI = Nme - CKI,
  Omt_vs_CKI = Omt - CKI,
  Tri_vs_CKI = Tri - CKI,
  UT_vs_CKI = UT - CKI,
  levels = designMatrix)

#rename the five comparisons
colnames(contrast) <- c("CKI-Mac vs CKI", "CKI-Nme vs CKI", "CKI-Omt vs CKI", "CKI-Tri vs CKI", "UT vs CKI")

contrast %>% as.data.frame()

#generate data frame of contrast colname for downstream analysis
comparison_group=colnames(contrast) %>% as.data.frame()
```

### Estimating Dispersion
The negative binomial dispersion is calculated using the `edgeR` User guide recommended method of using the `estimateDisp` function. This method calculates the common dispersion, trended dispersion and tagwise dispersions in one run. This function also returns the formal `DGEList` object but with additional entries for the negative binomial dispersion for all genes.

The dispersion can then be visualised through the `plotBCV` function which shows the biological coefficient of variance (BCV) of each gene

```{r NBDispersion}
# setting robust to T is highly recommneded by the edgeR user guide
NB_dispersion <- estimateDisp(y = dge, design = designMatrix, robust = TRUE)

## Display the common dispersion parameter of NB model
# NB_dispersion$common.dispersion

#visualisation of dispersion estimate thorugh the BCV plot which is just the square root of the dispersion parameter under the NB model
edgeR::plotBCV(NB_dispersion)

#save
invisible(dev.print(svg, here::here("2_plots/2_dge/NB_dispersion.svg")))
```

## Generalised Linear Model (GLM) with Quasi-likelihood F-Test (QLF)

The negative binomial model can be extended with quasi-likelihood methods to account for gene specific variability from both biological and technical sources.

*While the likelihood ratio test is a more obvious choice for inferences with GLMs, the QL F-test is preferred as it reflects the uncertainty in estimating the dispersion for each gene. It provides more robust and reliable error rate control when the number of replicates is small. The QL dispersion estimation and hypothesis testing can be done by using the functions `glmQLFit()` and `glmQLFTest()`.*

### Extending the NB dispersion with the QL method

*For the QL dispersions, estimation can be performed using the `glmQLFit` function. This returns a DGEGLM object containing the estimated values of the GLM coefficients for each gene, as well as the fitted mean-QL dispersion trend, the squeezed QL estimates and the prior degrees of freedom (df). These can be visualized with the `plotQLDisp` function.*

```{r qlf_dispersion}
#extend the NB dispersion with QL method
qlf_dispersion <- edgeR::glmQLFit(y = NB_dispersion, design = designMatrix, robust = TRUE)

## view gene coefficients
# head(qlf_dispersion$coefficients)

#visualise the disperion parameters after apply QL method to NB dispersion estimate 
plotQLDisp(glmfit = qlf_dispersion)

#save
invisible(dev.print(svg, here::here("2_plots/2_dge/qlf_dispersion.svg")))
```

### Apply QL F-test

Following the application of the QL method to NB dispersion estimate, the actually testing for DE genes can be performed. This can be achieved through the implementation of QL F-test. Unlike the LRT test, the QLF enable a greater control of type 1 errors.

```{r QL F-test}
# Create list object that for loop will populate
qlf=list()
qlf_decideTest=list()
qlf_all=list()
qlf_sig=list()

for (i in 1:ncol(contrast)){
  #at each iteration, let x = name of each contrast group
  x=comparison_group[i,]

  #populate list with DGELRT object for every comparison
  qlf[[x]] <-
    edgeR::glmQLFTest(glmfit = qlf_dispersion, contrast = contrast[,x])

  #populate significant list with decide test
  qlf_decideTest[[x]] <-
    decideTests(qlf[[x]], p.value = 0.05, adjust.method = "fdr") %>% summary() %>% as.data.frame()

  #populate all list with list of all DE genes
  qlf_all[[x]] <-
    edgeR::topTags(object = qlf[[x]], n = Inf) %>% as.data.frame()

  #populate sig list with list of significant DE genes
  qlf_sig[[x]] <-
    edgeR::topTags(object = qlf[[x]], n = Inf, adjust.method = "fdr", p.value = 0.05, sort.by = "PValue") %>% as.data.frame()
}
```


#### Visualisation {.tabset}

**Visualisation of QL F-test has been disable, the code, however, remains.**

##### P Value histogram

```{r pVal_qlf, eval=FALSE}
#generate p value histogram for all comparison
lapply(1:length(qlf),
       function(x){
         hist(x = qlf[[x]]$table$PValue,
              breaks = 50,
              main = paste0("P-Values QL F-test ", names(qlf[x])),
              xlab = "P-Value",
              col = "gray50")

#save
# invisible(dev.print(pdf, here::here(paste0("2_plots/2_dge/qlf_pValue_histogram_", names(qlf[x]), ".svg"))))
})
```

##### Mean-Difference plot
The differential expression visualised through `plotMD` function, where the `logFC` is plotted against the relative abundance of the gene `logCPM`. Significant genes with FDR of 0.05 or less are highlighted.

```{r MA_qlf, eval=FALSE}
lapply(1:length(qlf_sig),
       function(x) {

         #create data.frame specific for the custome MA plot
         MAplotData_qlf <- dplyr::select(as.data.frame(qlf_sig[[x]]), logCPM, logFC, FDR)
         colnames(MAplotData_qlf) <- c("baseMeanLog2", "log2FoldChange", "padj")

         #create custom MA plot
         MAplot_qlf <- ggpubr::ggmaplot(
           data = MAplotData_qlf,
           fdr = 0.05,
           fc = 1.5,
           genenames = as.vector(rownames(MAplotData_qlf)),
           size = 1.5,
           alpha = 0.8,
           label.rectangle = TRUE,
           palette = c("firebrick3", "dodgerblue3", "gray50"),
           top = 5,
           select.top.method = c("padj", "fc"),
           main = "MA Plot from GLM-QLF ",
           legend = "bottom",
           submain = names(qlf_sig[x]),
           xlab = expression("log"[2]*"CPM"),
           ylab = expression("log"[2]*"FC"),
           ylim = c(-8, 8),
           font.label = c(13, "italic", "gray30"),
           ggtheme = theme_minimal())

         #display plot
         MAplot_qlf

         #save MA plot
         ggsave(paste0("MA_qlf_", names(qlf_sig[x]), ".svg"), plot = MAplot_qlf, path = here::here(("2_plots/2_dge/"))
         })
```

##### Volcano Plot

Here the `ggplot` function can also be used to illustrate the DE genes significant `pValue` and `logFC`. In this instance, the significance is determined as genes with `PValue < 0.05` **and** `abs(logFC)>log(2)`.

```{r volcano_qlf, eval=FALSE}
lapply(1:length(qlf_sig),
       function(x) {

         #add an extra column and determine whether the DE genes are significant
         qlf_sig[[x]] <- qlf_sig[[x]] %>% as.data.frame() %>%
           dplyr::mutate(Expression = case_when
                         (FDR <= 0.05 & logFC >= 1.5 ~ "Up-regulated",
                          FDR <= 0.05 & logFC <= -1.5 ~ "Down-regulated",
                          TRUE ~ "Insignificant"))

         #adding labels to top genes
         top <- 5
         top_genes_qlf <- bind_rows(
           qlf_sig[[x]] %>%
             filter(Expression == 'Up-regulated') %>%
             arrange(FDR, desc(abs(logFC))) %>%
             head(top),
           qlf_sig[[x]] %>%
             filter(Expression == 'Down-regulated') %>%
             arrange(FDR, desc(abs(logFC))) %>%
             head(top)
           )
         invisible(top_genes_qlf %>% as.data.frame())

         #generate volcano plot with the allDEgene data.frame
         volcano_qlf <- qlf_sig[[x]] %>%
           ggplot(aes(x = logFC,
                      y = -log(FDR, 10))) +
           geom_point(aes(colour = Expression),
                      size = 1.5,
                      alpha = 0.8,
                      show.legend = FALSE) +
           geom_label_repel(data = top_genes_qlf,
                            mapping = aes(logFC, -log(FDR,10), label = gene),
                            size = 3) +
           xlab(expression("log"[2]*"FC")) +
           ylab(expression("-log"[10]*"FDR")) +
           xlim(-8,8)+
           ylim(0,30)+
           scale_color_manual(values = c("dodgerblue3", "gray50", "firebrick3")) +
           guides(colour = guide_legend(override.aes = list(size=1.5))) +
           labs(title = "Volcano Plot: GLM-QLF",
                subtitle = paste0(names(qlf_sig[x])),
                colour = "Expression")

         #save to directory
         ggsave(paste0("volcano_qlf_", names(qlf_sig[x]), ".svg"), plot = volcano_qlf, path = here::here("2_plots/2_dge/"))

         #display
         volcano_qlf
         })
```

### Apply TREAT

Following the application of the QL method to NB dispersion estimate, the actual testing for DE genes can be performed. This can be achieved through the implementation of TREAT. Unlike the QL-F test, TREAT enables identificaiton of DE genes relative to a `logFC` threshold. The current `logFC` threshold is `|logFC>=1.5|`

```{r applyTreat}
# Create list object
treat=list()
treat_decideTest=list()
treat_all=list()
treat_sig=list()

for (i in 1:ncol(contrast)){
  #at each iteration, let x = name of each contrast group
  x=comparison_group[i,]

  #populate list with DGELRT object for every comparison
  treat[[x]] <-
    edgeR::glmTreat(glmfit =  qlf_dispersion, contrast = contrast[,x], lfc = 1.5)

  #populate significant list with decide test
  treat_decideTest[[x]] <- decideTests(treat[[x]], p.value = 0.05, adjust.method = "fdr") %>% summary() %>% as.data.frame()


  #populate all list with list of all DE genes
  treat_all[[x]] <-
    edgeR::topTags(object = treat[[x]], n = Inf) %>% as.data.frame()

  #populate sig list with list of significant DE genes
  treat_sig[[x]] <-
    edgeR::topTags(object = treat[[x]], n = Inf, adjust.method = "fdr", p.value = 0.05, sort.by = "PValue") %>% as.data.frame()
}
```

#### Visualisation {.tabset}

##### P Value histogram

```{r pValueHistogram_treat}
lapply(1:length(treat),
       function(x){
         hist(x = treat[[x]]$table$PValue,
              breaks = 50,
              main = paste0("P-Values treat ", names(treat[x])),
              xlab = "P-Value",
              col = "gray50")

invisible(dev.print(pdf, here::here(paste0("2_plots/2_dge/pValue_treat_", names(treat[x]), ".svg"))))
})
```

##### Mean-Difference plot
The differential expression can initial visualised through `plotMD` function, where the `logFC` is plotted against the relative abundance of the gene `logCPM`. Significant genes with FDR of 0.05 or less are highlighted.

```{r MAplotCustom_treat}
MAplot=list()
lapply(1:length(treat_all),
       function(x) {

         #create data.frame specific for the custome MA plot
         MAplotData_treat <- dplyr::select(as.data.frame(treat_all[[x]]), logCPM, logFC, FDR)
         colnames(MAplotData_treat) <- c("baseMeanLog2", "log2FoldChange", "padj")

         #create custom MA plot
         MAplot_treat <- ggpubr::ggmaplot(
           data = MAplotData_treat,
           fdr = 0.05,
           fc = 2.828,
           genenames = as.vector(rownames(MAplotData_treat)),
           size = 1.5,
           alpha = 0.8,
           label.rectangle = TRUE,
           palette = c("firebrick3", "dodgerblue3", "gray50"),
           top = 5,
           ylim = c(-8, 8),
           select.top.method = c("padj", "fc"),
           main = "MA Plot from GLM:QLF-TREAT",
           submain = names(treat_all[x]),
           legend = "NA",
           legend.title = "Expression",
           xlab = expression("log"[2]*"CPM"),
           ylab = expression("log"[2]*"FC"),
           font.label = c(13, "italic", "gray30"), 
           ggtheme = theme_minimal())
         
         #save MA plot
         ggsave(filename = paste0("MA_treat_", names(treat_all[x]), ".svg"),
                plot = MAplot_treat,
                path = here::here("2_plots/2_dge"))

         #display plot
         MAplot_treat
         })
```

##### Volcano Plot

Here the `ggplot` function can also be used to illustrate the DE genes significant `pValue` and `logFC`. In this instance, the significance is determined as genes with `PValue < 0.05` **and** `abs(logFC)>log(2)`.
```{r volcano_treat}
lapply(1:length(treat_all),
       function(x) {

         #add an extra column and determine whether the DE genes are significant
         treat_all[[x]] <- treat_all[[x]] %>% as.data.frame() %>%
           dplyr::mutate(Expression = case_when
                         (FDR <= 0.05 & logFC >= 1.5 ~ "Up-regulated",
                          FDR <= 0.05 & logFC <= -1.5 ~ "Down-regulated",
                          TRUE ~ "Insignificant"))

         #adding labels to top genes
         top <- 5
         top_genes_treat <- bind_rows(
           treat_all[[x]] %>%
             filter(Expression == 'Up-regulated') %>%
             arrange(FDR, desc(abs(logFC))) %>%
             head(top),
           treat_all[[x]] %>%
             filter(Expression == 'Down-regulated') %>%
             arrange(FDR, desc(abs(logFC))) %>%
             head(top)
           )
         invisible(top_genes_treat %>% as.data.frame())

         #generate volcano plot with the allDEgene data.frame
         volcano_treat <- treat_all[[x]] %>%
           ggplot(aes(x = logFC,
                      y = -log(FDR, 10))) +
           geom_point(aes(colour = Expression),
                      size = 1.5,
                      alpha = 0.8,
                      show.legend = FALSE) +
           geom_label_repel(data = top_genes_treat,
                            mapping = aes(logFC, -log(FDR,10), label = gene),
                            size = 3) +
           xlab(expression("log"[2]*"FC")) +
           ylab(expression("-log"[10]*"FDR")) +
           xlim(-8,8)+
           ylim(0,30)+
           scale_color_manual(values = c("dodgerblue3", "gray50", "firebrick3")) +
           guides(colour = guide_legend(override.aes = list(size=1.5))) +
  
           labs(title = "Volcano Plot: GLM:QLF-TREAT",
                subtitle = paste0(names(treat_all[x])),
                colour = "Expression")

         #save to directory
         ggsave(paste0("volcano_treat_", names(treat_all[x]), ".svg"),
                plot = volcano_treat,
                path = here::here("2_plots/2_dge/"))

         #display
         volcano_treat
         })
```

#### Heatmaps {.tabset}

##### Top Common

This heatmap displays the expression of the most significant DE genes common among all five comparisons

```{r heatmap_common, fig.width=8, fig.height=10}
#create matrix with log cpm counts
logCPM <- cpm(dge, prior.count=3, log=TRUE)
rownames(logCPM) <- dge$genes$gene
colnames(logCPM) <- paste(dge$samples$pretty_name, sep="-")

#join common significant DE genes into df
common <- join_all(list(treat_sig$`UT vs CKI`,
                        treat_sig$`CKI-Mac vs CKI`,
                        treat_sig$`CKI-Nme vs CKI`,
                        treat_sig$`CKI-Omt vs CKI`,
                        treat_sig$`CKI-Tri vs CKI`), by = 'gene', type = 'inner')

kable(common, digits = Inf) %>%
  kable_styling(bootstrap_options = c("striped", "hover")) %>%
  scroll_box(height = "600px")

#merge the log cpm counts with the top 30 common de genes
logCPM <- logCPM[common$gene[1:30],]

#set colour palette
my_palette <- colorRampPalette(c(
  rgb(32,121,226, maxColorValue = 255),
  # rgb(144,203,180, maxColorValue = 255), 
  rgb(254,238,229, maxColorValue = 255), 
  # rgb(251,192,52, maxColorValue = 255), 
  rgb(226,46,45, maxColorValue = 255)))(n = 201)

#df for heatmap annotation of sample type
anno <- as.factor(dge$samples$sample_type) %>% as.data.frame 
colnames(anno) <- "Sample Type"
anno$`Sample Type` <-  gsub("single_deletion", "Single Deletion", anno$`Sample Type`)
anno$`Sample Type` <-  gsub("UT", "Untreated", anno$`Sample Type`)
rownames(anno) <- colnames(logCPM) 

#setting colour of sample type annotation
anno_colour <- RColorBrewer::brewer.pal(3, "Set3")
names(anno_colour) <- c("CKI", "Single Deletion", "Untreated")
anno_colours <- list("Sample Type" = anno_colour) 


pheatmap(mat = logCPM,
         scale = "row",
         clustering_distance_rows = "euclidean",
         treeheight_row = 60,
         treeheight_col = 30,
         cutree_rows = 4,
         cutree_cols = 5,
         main = "Heatmap of top 30 common DE genes",
         legend_breaks = c(seq(-2, 10, by = 2), max(logCPM)),
         legend_labels = c(seq(-2, 10, by = 2), "logCPM\n"),
         angle_col =90,
         
         annotation_legend = T,
         annotation_col = anno,
         annotation_colors = list("Sample Type" = anno_colour),
         annotation_names_col = FALSE,
         annotation = TRUE,
         
         # border_color = F,
         
         border_color = "gray70",
         color = my_palette
         )

invisible(dev.print(device = svg, here::here("2_plots/2_dge/heatmap_logCPM_common.svg")))
```

##### Top

This heatmap displays the expression of the most significant DE genes among all five comparisons. top 10 most significant DE genes from each comparison were combined into a single data frame. NA values were assigned to genes that were not commonly in the top 10. Since only the gene column is important for filtering the logCPM dataframe, the values within this table are the `FDR` values

```{r unique_table}
#create matrix with log cpm counts
logCPM <- cpm(dge, prior.count=3, log=TRUE)
rownames(logCPM) <- dge$genes$gene
colnames(logCPM) <- paste(dge$samples$pretty_name, sep="-")

top_unique=list()
for (i in 1:ncol(contrast)){
  #at each iteration, let x = name of each contrast group
  x=comparison_group[i,]
  
top_unique[[x]] <- treat_sig[[x]][1:10,]
}

unique <- as.data.frame(do.call(rbind, lapply(top_unique, as.data.frame))) %>% rownames_to_column("group") %>% dplyr::select("gene", "group", "FDR") 
unique$group <- gsub("\\..*", "", unique$group)
unique <- unique %>% pivot_wider(names_from = "group", values_from = "FDR") %>% column_to_rownames("gene")
kable(unique, digits = Inf) %>%
  kable_styling(bootstrap_options = c("striped", "hover")) %>%
  scroll_box(height = "600px")

#merge the log cpm counts with the top 30 common de genes
logCPM <- logCPM[rownames(unique),]
```

The gene list (rowname) was then used to filter the logCPM transformed dataframe. This heatmap illustrates the expression of the those `r nrow(logCPM)` genes. 

```{r heatmap_unique, fig.width=8, fig.height=10}

#set colour palette
my_palette <- colorRampPalette(c(
  rgb(32,121,226, maxColorValue = 255),
  # rgb(144,203,180, maxColorValue = 255), 
  rgb(254,238,229, maxColorValue = 255), 
  # rgb(251,192,52, maxColorValue = 255), 
  rgb(226,46,45, maxColorValue = 255)))(n = 201)

#df for heatmap annotation of sample type
anno <- as.factor(dge$samples$sample_type) %>% as.data.frame 
colnames(anno) <- "Sample Type"
anno$`Sample Type` <-  gsub("single_deletion", "Single Deletion", anno$`Sample Type`)
anno$`Sample Type` <-  gsub("UT", "Untreated", anno$`Sample Type`)
rownames(anno) <- colnames(logCPM) 

#setting colour of sample type annotation
anno_colour <- RColorBrewer::brewer.pal(3, "Set3")
names(anno_colour) <- c("CKI", "Single Deletion", "Untreated")
anno_colours <- list("Sample Type" = anno_colour) 

pheatmap(mat = logCPM,
         # scale = "row",
         clustering_distance_rows = "euclidean",
         treeheight_row = 60,
         treeheight_col = 30,
         cutree_rows = 4,
         cutree_cols = 4,
         main = "Heatmap of top 10 DE genes",
         legend_breaks = c(seq(-2, 10, by = 2), max(logCPM)),
         legend_labels = c(seq(-2, 10, by = 2), "logCPM\n"),
         angle_col =90,
         
         annotation_legend = T,
         annotation_col = anno,
         annotation_colors = list("Sample Type" = anno_colour),
         annotation_names_col = FALSE,
         annotation = TRUE,
         
         # border_color = F,
         
         border_color = "gray70",
         color = my_palette
         )

invisible(dev.print(device = svg, here::here("2_plots/2_dge/heatmap_logCPM_unique.svg")))

```

## Export Data
```{r exportData}
designMatrix %>% saveRDS(here::here("0_data/rds_object/designMatrix.rds"))
contrast %>% saveRDS(here::here("0_data/rds_object/contrastMatrix.rds"))

# save rds object for use in downstream GO and KEGG analysis
qlf %>% saveRDS(file = here::here("0_data/rds_object/qlf.rds"))
qlf_all %>% saveRDS(file = here::here("0_data/rds_object/qlf_all.rds"))
qlf_sig %>% saveRDS(file = here::here("0_data/rds_object/qlf_sig.rds"))
treat %>% saveRDS(file = here::here("0_data/rds_object/treat.rds"))
treat_all %>% saveRDS(file = here::here("0_data/rds_object/treat_all.rds"))
treat_sig %>% saveRDS(file = here::here("0_data/rds_object/treat_sig.rds"))

#save each all comparison group in the output directory
writexl::write_xlsx(x = qlf_decideTest, here::here("3_output/qlf_decideTest.xlsx"))
writexl::write_xlsx(x = qlf_sig, here::here("3_output/qlf_sig.xlsx"))
writexl::write_xlsx(x = treat_decideTest, here::here("3_output/treat_decideTest.xlsx"))
writexl::write_xlsx(x = treat_sig, here::here("3_output/treat_sig.xlsx"))
```



