---
title: "Introduction"
---

## Background

The introduction of next-generation sequencing (NGS) and recent advancements in omics-related techniques represent a new approach for identifying novel oncogenes and cancer regulators. However, the primary challenge that remains is translating this new information into developing novel cancer therapeutics (Simon & Roychowdhury, 2013). Serendipitously, the sensitivity of omics technologies have also been applied to identify and validate potentially novel cancer therapeutics derived from natural drugs such as Traditional Chinese Medicine (TCM) (Harvey, Edrada-Ebel, & Quinn, 2015). With the emphasis on homeopathy, most TCMs are formulated from various plant species, delivering a broad-spectrum treatment through multiple bioactive compounds with diverse targets and pathways (Thazin Nwe Aung, Qu, Kortschak, & Adelson, 2017; Hsiao & Liu, 2010). Therefore, if a TCM treatment is effective against cancer, the molecular mechanism of its diverse and active compounds warrants further investigation. 

Compound Kushen Injection (CKI) is a Chinese State Food and Drug Administration (SFDA) approved TCM formula extensively prescribed in combination with chemotherapy and radiotherapy to treat various forms of solid tumours (Wang et al., 2015). CKI is prepared from the roots of Kushen (Sophora flavescentis) and Baituling (Heterosmilacis japonicea), two medicinal herbs with long ethnomedicinal significance in China. Chromatography and mass spectroscopy experiments have revealed 21 significant compounds in CKI. Nine of these compounds were recognised as the main constituents of CKI, with oxymatrine (Omt), oxysophocarpine (Ospc),  matrine (Mt) being the most abundant (Ma et al., 2014). Matrine and oxymatrine, in particular, have been implicated with various anti-tumour activities such as inhibition of proliferation and induction of apoptosis against various solid tumours, including breast, cervical, and prostate cancer (T. N. Aung et al., 2019; Gao et al., 2018; Qu et al., 2016). Conversely, the potential hepatotoxicity and lack of bioavailability of Mt and Omt demonstrated in rats may pose a potential drawback for clinical applications of CKI (Yuan et al., 2010). This disparity highlights the anti-cancer potential of CKI but also the need to understand its molecular mechanisms for the improvement and optimisation of CKI and potentially other experience-based TCMs. 

The complex heterogeneity of herbal medicine like CKI poses a challenge for standard pharmacodynamic analysis, whereby a single bioactive compound is typically isolated and assessed for biological activity. This conventional approach is limited to identifying the molecular function of a single component and ignores the potential effect of the whole heterogeneous mixture. Furthermore, past attempts at exploring the functional mechanisms of CKI have primarily relied on investigating one or few pathways through quantification of canonical genes and proteins, thus, potentially ignoring crucial targets of CKI.  In this regard, high-throughput RNA sequencing and subsequent whole transcriptome analysis may identify novel pathways targeted by CKI (Qu et al., 2016).
Regarding the aforementioned challenges, Aung et al. (2019) have utilised high-performance liquid chromatography (HPLC) to isolate the major compounds within CKI then reconstituted all but a few selected compounds back together. Breast cancer and non-cancerous cell lines were then treated with these reconstituted CKI samples, followed by cell-based assays and whole transcriptome analyses. These transcriptomes were then compared to identify the differentially expressed genes and reveal the potential mode of action associated with a single or multiple CKI compound. 

The bioassays demonstrated that deletions of a major compound from CKI (CKI-1) did not significantly affect the anti-tumour activities of the remaining CKI mixture, compared to the unfractionated CKI treatment (T. N. Aung et al., 2019). Of these nine single deletions, four were selected for the subsequent RNA-seq and differential gene expression (DGE) analysis due to their significant structural differences. Interestingly, despite the lack of phenotypic effects, treatment with CKI-1 lead to significant perturbation of genes compared to the unfractionated CKI sample (T. N. Aung et al., 2019). Further examination of these DE genes shared between all samples revealed 86 significantly perturbed pathways. However, this consideration of other treatment groups means that many pathways only perturbed by CKI-1 treatment are potentially excluded.


## Project Objectives

The objective of this project will be to identify the differentially expressed gene across the four CKI single deletions and unfractionated CKI. The raw read count data of CKI-1, CKI, and UT will be obtained from the National Center for Biotechnology Information (NCBI) deposited by T. N. Aung et al. (2019). Gene ontology and pathway enrichment analysis will then identify the potential pathways associated with the DEG. These pathways can then be visualised through various plots and used to infer the molecular mechanisms associated with some major compounds in CKI. Furthermore, if time permissible, a complementary co-expression analysis will also be performed to generate a functional network, describing the interactions between differentially expressed gene sets (Langfelder & Horvath, 2008).

The culmination of the aforementioned transcriptomic analyses may reveal potential molecular mechanisms associated with CKI major compound within breast cancer cell line. Moreover, the findings may provide a critical starting point for subsequent in vitro and in vivo functional validations, thus, further improving the therapeutic potential of Compound Kushen Injection.

