<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Ha Tran" />


<title>DE Analysis</title>

<script src="site_libs/header-attrs-2.11/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/lightable-0.0.1/lightable.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">CKI_DGE</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Setting Up
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="obtainData.html">Obtaining Data</a>
    </li>
    <li>
      <a href="setUp.html">QC and Pre-processing</a>
    </li>
  </ul>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Analysis
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="dge.html">DE Analysis</a>
    </li>
    <li>
      <a href="go.html">GO Term Enrichment Analysis</a>
    </li>
    <li>
      <a href="kegg.html">KEGG Pathway Enrichment Analysis</a>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">DE Analysis</h1>
<h4 class="author">Ha Tran</h4>
<h4 class="date">22/08/2021</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2021-11-13
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>01_CKI_DGE/1_analysis/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges" class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown file has unstaged changes. To know which version of the R Markdown file created these results, you’ll want to first commit it to the Git repo. If you’re still working on the analysis, you can ignore this warning. When you’re finished, you can run <code>wflow_publish</code> to commit the R Markdown file and build the HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed12345code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(12345)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed12345code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(12345)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomtranmanhha13501CKIDGEtree848e971e9ccbf102898eeb089076953d68a0817ftargetblank848e971a"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/tranmanhha135/01_CKI_DGE/tree/848e971e9ccbf102898eeb089076953d68a0817f" target="_blank">848e971</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomtranmanhha13501CKIDGEtree848e971e9ccbf102898eeb089076953d68a0817ftargetblank848e971a" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/tranmanhha135/01_CKI_DGE/tree/848e971e9ccbf102898eeb089076953d68a0817f" target="_blank">848e971</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/

Untracked files:
    Untracked:  0_data/rds_object/qlf_all.rds
    Untracked:  0_data/rds_object/qlf_sig.rds
    Untracked:  0_data/rds_object/treat_all.rds
    Untracked:  0_data/rds_object/treat_sig.rds
    Untracked:  1_analysis/about.html
    Untracked:  1_analysis/figure/dge.Rmd/NBDispersion-1.png
    Untracked:  1_analysis/figure/dge.Rmd/coolmap-1.png
    Untracked:  1_analysis/figure/dge.Rmd/heatmap_common-1.png
    Untracked:  1_analysis/figure/dge.Rmd/heatmap_unique-1.png
    Untracked:  1_analysis/figure/dge.Rmd/qlf_dispersion-1.png
    Untracked:  1_analysis/figure/dge.Rmd/volcano_treat-1.png
    Untracked:  1_analysis/figure/dge.Rmd/volcano_treat-2.png
    Untracked:  1_analysis/figure/dge.Rmd/volcano_treat-3.png
    Untracked:  1_analysis/figure/dge.Rmd/volcano_treat-4.png
    Untracked:  1_analysis/figure/dge.Rmd/volcano_treat-5.png
    Untracked:  1_analysis/figure/go.Rmd/
    Untracked:  1_analysis/figure/kegg.Rmd/completePathway-1.png
    Untracked:  1_analysis/figure/kegg.Rmd/dotPlot-1.png
    Untracked:  1_analysis/figure/kegg.Rmd/partialPathway, -1.png
    Untracked:  1_analysis/figure/kegg.Rmd/partialPathway-1.png
    Untracked:  1_analysis/figure/setUp.Rmd/afterFiltering-2.png
    Untracked:  1_analysis/site_libs/
    Untracked:  2_plots/1_setup/after_normalisation.svg
    Untracked:  2_plots/2_dge/MA_treat_CKI-Mac vs CKI.svg
    Untracked:  2_plots/2_dge/MA_treat_CKI-Nme vs CKI.svg
    Untracked:  2_plots/2_dge/MA_treat_CKI-Omt vs CKI.svg
    Untracked:  2_plots/2_dge/MA_treat_CKI-Tri vs CKI.svg
    Untracked:  2_plots/2_dge/MA_treat_UT vs CKI.svg
    Untracked:  2_plots/2_dge/NB_dispersion.svg
    Untracked:  2_plots/2_dge/heatmap_logCPM_common.svg
    Untracked:  2_plots/2_dge/heatmap_logCPM_unique.svg
    Untracked:  2_plots/2_dge/pValue_treat_CKI-Mac vs CKI.svg
    Untracked:  2_plots/2_dge/pValue_treat_CKI-Nme vs CKI.svg
    Untracked:  2_plots/2_dge/pValue_treat_CKI-Omt vs CKI.svg
    Untracked:  2_plots/2_dge/pValue_treat_CKI-Tri vs CKI.svg
    Untracked:  2_plots/2_dge/pValue_treat_UT vs CKI.svg
    Untracked:  2_plots/2_dge/volcano_treat_CKI-Mac vs CKI.svg
    Untracked:  2_plots/2_dge/volcano_treat_CKI-Nme vs CKI.svg
    Untracked:  2_plots/2_dge/volcano_treat_CKI-Omt vs CKI.svg
    Untracked:  2_plots/2_dge/volcano_treat_CKI-Tri vs CKI.svg
    Untracked:  2_plots/2_dge/volcano_treat_UT vs CKI.svg
    Untracked:  2_plots/3_go/
    Untracked:  2_plots/4_kegg/
    Untracked:  3_output/enrichKEGG_treat_sig.xlsx
    Untracked:  3_output/goana_treat.xlsx
    Untracked:  3_output/kegga_treat_sig.xlsx
    Untracked:  3_output/qlf_decideTest.xlsx
    Untracked:  3_output/qlf_sig.xlsx
    Untracked:  3_output/treat_decideTest.xlsx
    Untracked:  3_output/treat_sig.xlsx
    Untracked:  Meeting Notes.pdf

Unstaged changes:
    Modified:   0_data/rds_object/dge.rds
    Deleted:    0_data/rds_object/dge_treat_unfiltered.rds
    Deleted:    0_data/rds_object/enrichGO_treat.rds
    Deleted:    0_data/rds_object/enrichKEGG_treat_filtered.rds
    Modified:   0_data/rds_object/qlf.rds
    Deleted:    0_data/rds_object/qlf_filtered.rds
    Modified:   0_data/rds_object/treat.rds
    Deleted:    0_data/rds_object/treat_filtered.rds
    Deleted:    0_data/rds_object/treat_unfiltered.rds
    Modified:   1_analysis/dge.Rmd
    Modified:   1_analysis/figure/dge.Rmd/MAplotCustom_treat-1.png
    Modified:   1_analysis/figure/dge.Rmd/MAplotCustom_treat-2.png
    Modified:   1_analysis/figure/dge.Rmd/MAplotCustom_treat-3.png
    Modified:   1_analysis/figure/dge.Rmd/MAplotCustom_treat-4.png
    Modified:   1_analysis/figure/dge.Rmd/MAplotCustom_treat-5.png
    Modified:   1_analysis/figure/dge.Rmd/pValueHistogram_treat-1.png
    Modified:   1_analysis/figure/dge.Rmd/pValueHistogram_treat-2.png
    Modified:   1_analysis/figure/dge.Rmd/pValueHistogram_treat-3.png
    Modified:   1_analysis/figure/dge.Rmd/pValueHistogram_treat-4.png
    Modified:   1_analysis/figure/dge.Rmd/pValueHistogram_treat-5.png
    Modified:   1_analysis/figure/setUp.Rmd/MDSplot-1.png
    Modified:   1_analysis/figure/setUp.Rmd/afterFiltering-1.png
    Modified:   1_analysis/figure/setUp.Rmd/libraryChecks-1.png
    Modified:   1_analysis/figure/setUp.Rmd/pcaSampleGroup-1.png
    Modified:   1_analysis/figure/setUp.Rmd/pcaSampleType-1.png
    Modified:   1_analysis/figure/setUp.Rmd/visualisingNormalisation-1.png
    Modified:   1_analysis/go.Rmd
    Modified:   1_analysis/kegg.Rmd
    Modified:   1_analysis/setUp.Rmd
    Modified:   2_plots/1_setup/counts_before_and_after_filtering.svg
    Modified:   2_plots/1_setup/library_size.svg
    Modified:   2_plots/1_setup/sample_group_PCA_plot.svg
    Modified:   2_plots/1_setup/sample_type_PCA_plot.svg
    Deleted:    2_plots/2_dge/MA_plot_treat_CKI-Mac vs CKI.svg
    Deleted:    2_plots/2_dge/MA_plot_treat_CKI-Nme vs CKI.svg
    Deleted:    2_plots/2_dge/MA_plot_treat_CKI-Omt vs CKI.svg
    Deleted:    2_plots/2_dge/MA_plot_treat_CKI-Tri vs CKI.svg
    Deleted:    2_plots/2_dge/MA_plot_treat_UT vs CKI.svg
    Deleted:    2_plots/2_dge/estimate_dispersion.svg
    Modified:   2_plots/2_dge/qlf_dispersion.svg
    Deleted:    2_plots/2_dge/qlf_pValue_histogram_CKI-Mac vs CKI.svg
    Deleted:    2_plots/2_dge/qlf_pValue_histogram_CKI-Nme vs CKI.svg
    Deleted:    2_plots/2_dge/qlf_pValue_histogram_CKI-Omt vs CKI.svg
    Deleted:    2_plots/2_dge/qlf_pValue_histogram_CKI-Tri vs CKI.svg
    Deleted:    2_plots/2_dge/qlf_pValue_histogram_UT vs CKI.svg
    Deleted:    2_plots/2_dge/treat_pValue_histogram_CKI-Mac vs CKI.svg
    Deleted:    2_plots/2_dge/treat_pValue_histogram_CKI-Nme vs CKI.svg
    Deleted:    2_plots/2_dge/treat_pValue_histogram_CKI-Omt vs CKI.svg
    Deleted:    2_plots/2_dge/treat_pValue_histogram_CKI-Tri vs CKI.svg
    Deleted:    2_plots/2_dge/treat_pValue_histogram_UT vs CKI.svg
    Deleted:    2_plots/2_dge/volcano_plot_treat_CKI-Mac vs CKI.svg
    Deleted:    2_plots/2_dge/volcano_plot_treat_CKI-Nme vs CKI.svg
    Deleted:    2_plots/2_dge/volcano_plot_treat_CKI-Omt vs CKI.svg
    Deleted:    2_plots/2_dge/volcano_plot_treat_CKI-Tri vs CKI.svg
    Deleted:    2_plots/2_dge/volcano_plot_treat_UT vs CKI.svg
    Deleted:    3_output/enrichGO_treat.xlsx
    Deleted:    3_output/enrichKEGG_treat_unfiltered.xlsx
    Deleted:    3_output/qlf_filtered.xlsx
    Deleted:    3_output/treat_filtered.xlsx

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>1_analysis/dge.Rmd</code>) and HTML (<code>docs/dge.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/01_CKI_DGE/blob/848e971e9ccbf102898eeb089076953d68a0817f/1_analysis/dge.Rmd" target="_blank">848e971</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-11-08
</td>
<td>
adjust plot size, added dot plot for GO and KEGG, added heatmap for DE genes, minor cosmetic changes
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/01_CKI_DGE/848e971e9ccbf102898eeb089076953d68a0817f/docs/dge.html" target="_blank">848e971</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-11-08
</td>
<td>
adjust plot size, added dot plot for GO and KEGG, added heatmap for DE genes, minor cosmetic changes
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/01_CKI_DGE/1a47b4569a1da246294ec62c8e47e2f6e2638e10/docs/dge.html" target="_blank">1a47b45</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-31
</td>
<td>
changed heatmap size
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/01_CKI_DGE/blob/70049b22b9a79841a2368454293f8825f08980dd/1_analysis/dge.Rmd" target="_blank">70049b2</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-31
</td>
<td>
Remove old GO and KEGG code, adjusted directory paths, updated KEGG, minor cosmetic changes
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/01_CKI_DGE/70049b22b9a79841a2368454293f8825f08980dd/docs/dge.html" target="_blank">70049b2</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-31
</td>
<td>
Remove old GO and KEGG code, adjusted directory paths, updated KEGG, minor cosmetic changes
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/01_CKI_DGE/blob/6823b3c881eddd07176bce86153e0543293d7f02/1_analysis/dge.Rmd" target="_blank">6823b3c</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-30
</td>
<td>
Add minor cosmetic changes
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/01_CKI_DGE/6823b3c881eddd07176bce86153e0543293d7f02/docs/dge.html" target="_blank">6823b3c</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-30
</td>
<td>
Add minor cosmetic changes
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/01_CKI_DGE/blob/7a6571ba908cf21e984811a75b8123b1e28a4d69/1_analysis/dge.Rmd" target="_blank">7a6571b</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-30
</td>
<td>
minor changes
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/01_CKI_DGE/7a6571ba908cf21e984811a75b8123b1e28a4d69/docs/dge.html" target="_blank">7a6571b</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-30
</td>
<td>
minor changes
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/01_CKI_DGE/f5f4f74bfd77b68627ec8414b040c66a60ed7344/docs/dge.html" target="_blank">f5f4f74</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-30
</td>
<td>
please please please
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/01_CKI_DGE/blob/f94a382136e6b166de00c046a8ac9df415ff15e0/1_analysis/dge.Rmd" target="_blank">f94a382</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-30
</td>
<td>
rerun website building to no avail. will it still work on github?
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/01_CKI_DGE/blob/a6d8f8a05c71e8da432f180ed252874e35ca0ae1/1_analysis/dge.Rmd" target="_blank">a6d8f8a</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-30
</td>
<td>
what is happening
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/01_CKI_DGE/a6d8f8a05c71e8da432f180ed252874e35ca0ae1/docs/dge.html" target="_blank">a6d8f8a</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-30
</td>
<td>
what is happening
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/01_CKI_DGE/blob/d769a7cbf44fe4841bdf220ed214b5f73c00b47a/1_analysis/dge.Rmd" target="_blank">d769a7c</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-30
</td>
<td>
Upload GO and KEGG analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/01_CKI_DGE/d769a7cbf44fe4841bdf220ed214b5f73c00b47a/docs/dge.html" target="_blank">d769a7c</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-30
</td>
<td>
Upload GO and KEGG analysis
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/01_CKI_DGE/blob/67e2d670268223ebb4a53e199040aa1830496a7f/1_analysis/dge.Rmd" target="_blank">67e2d67</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-29
</td>
<td>
copied code from old workflow for setup and dge
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/01_CKI_DGE/67e2d670268223ebb4a53e199040aa1830496a7f/docs/dge.html" target="_blank">67e2d67</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-29
</td>
<td>
copied code from old workflow for setup and dge
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="data-setup" class="section level1">
<h1>Data Setup</h1>
<div id="load-library" class="section level3">
<h3>Load Library</h3>
<pre class="r"><code># working with data
library(dplyr)
library(magrittr)
library(readr)
library(tibble)
library(reshape2)
library(tidyverse)

# Visualisation:
library(kableExtra)
library(ggplot2)
library(grid)
library(pander)
library(cowplot)
library(pheatmap)
library(RColorBrewer)

# Custom ggplot 
library(ggpubr)
library(ggbiplot)
library(ggrepel)

# Bioconductor packages:
library(edgeR)
library(limma)
library(Glimma)

# Set ggplot theme
theme_set(theme_minimal())


# Adjust ggplot theme for poster and report
# theme_update(
#   plot.title = element_text(color = &quot;gray20&quot;, size = 7, hjust = 0, vjust = .5, face = &quot;bold&quot;),
#   plot.subtitle = element_text(color = &quot;gray25&quot;, size = 6, angle = 0, hjust = 0, vjust = .5, face = &quot;plain&quot;),
#   legend.title = element_text(color = &quot;gray25&quot;, size = 5, angle = 0, hjust = 0, vjust = .5, face = &quot;plain&quot;),
#   legend.text = element_text(color = &quot;gray25&quot;, size = 5, angle = 0, hjust = 0, vjust = .5, face = &quot;plain&quot;),
#   axis.text.x = element_text(color = &quot;gray30&quot;, size = 5, angle = 0, hjust = .5, vjust = .5, face = &quot;plain&quot;),
#   axis.text.y = element_text(color = &quot;gray30&quot;, size = 5, angle = 0, hjust = 1, vjust = 0, face = &quot;plain&quot;),
#   axis.title.x = element_text(color = &quot;gray30&quot;, size = 6, angle = 0, hjust = .5, vjust = 0, face = &quot;plain&quot;),
#   axis.title.y = element_text(color = &quot;gray30&quot;, size = 6, angle = 90, hjust = .5, vjust = .5, face = &quot;plain&quot;))</code></pre>
</div>
<div id="import-dgelist-data" class="section level3">
<h3>Import DGElist Data</h3>
<p>DGElist object containing the raw feature count, sample metadata, and gene metadata, created in the Set Up stage.</p>
<pre class="r"><code># load DGElist previously created in the set up
dge &lt;- readRDS(here::here(&quot;0_data/rds_object/dge.rds&quot;))</code></pre>
</div>
</div>
<div id="differential-gene-expression-analysis" class="section level1">
<h1>Differential Gene Expression Analysis</h1>
<p>Here multiple methods of identifying differential expression will be compared. These methods include</p>
<ol style="list-style-type: decimal">
<li>Generalised Linear Model (GLM) with Likelihood Ratio Test (LRT),</li>
<li>Generalised Linear Model (GLM) with Quasi-likelihood F-tests (QLF), and</li>
<li>Limma-voom</li>
</ol>
<p><strong>These methods have been internally assessed and only the GLM:QLF + TREAT method will be run</strong></p>
<div id="initial-parameterisation" class="section level2">
<h2>Initial Parameterisation</h2>
<p>The varying methods used to identify differential expression all rely on similar initial parameters. These include:</p>
<ol style="list-style-type: decimal">
<li>The Design Matrix,</li>
<li>Estimation of Dispersion, and</li>
<li>Contrast Matrix</li>
</ol>
<div id="design-matrix" class="section level3">
<h3>Design Matrix</h3>
<p>The experimental design can be parameterised in a one-way layout where one coefficient is assigned to each group. The design matrix formulated below contains the predictors of each sample</p>
<pre class="r"><code>#setup desgin matrix with sample_group
designMatrix &lt;- model.matrix(~ 0 + sample_group,
                             data = dge$samples)

#remove &quot;sample_group&quot; from each column names
colnames(designMatrix) &lt;- gsub(&quot;sample_group&quot;, &quot;&quot;, colnames(designMatrix))

#remove &quot;S48h_&quot; from row names
rownames(designMatrix) &lt;- gsub(&quot;S48h_&quot;, &quot;&quot;, rownames(designMatrix))

#display the design matrix
designMatrix %&gt;% as.data.frame()</code></pre>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["CKI"],"name":[1],"type":["dbl"],"align":["right"]},{"label":["Mac"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["Nme"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["Omt"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["Tri"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["UT"],"name":[6],"type":["dbl"],"align":["right"]}],"data":[{"1":"1","2":"0","3":"0","4":"0","5":"0","6":"0","_rn_":"CKI1"},{"1":"1","2":"0","3":"0","4":"0","5":"0","6":"0","_rn_":"CKI2"},{"1":"1","2":"0","3":"0","4":"0","5":"0","6":"0","_rn_":"CKI3"},{"1":"0","2":"1","3":"0","4":"0","5":"0","6":"0","_rn_":"Mac1"},{"1":"0","2":"1","3":"0","4":"0","5":"0","6":"0","_rn_":"Mac2"},{"1":"0","2":"1","3":"0","4":"0","5":"0","6":"0","_rn_":"Mac3"},{"1":"0","2":"0","3":"1","4":"0","5":"0","6":"0","_rn_":"Nme1"},{"1":"0","2":"0","3":"1","4":"0","5":"0","6":"0","_rn_":"Nme2"},{"1":"0","2":"0","3":"1","4":"0","5":"0","6":"0","_rn_":"Nme3"},{"1":"0","2":"0","3":"0","4":"1","5":"0","6":"0","_rn_":"OMT1"},{"1":"0","2":"0","3":"0","4":"1","5":"0","6":"0","_rn_":"OMT2"},{"1":"0","2":"0","3":"0","4":"1","5":"0","6":"0","_rn_":"OMT3"},{"1":"0","2":"0","3":"0","4":"0","5":"1","6":"0","_rn_":"Tri1"},{"1":"0","2":"0","3":"0","4":"0","5":"1","6":"0","_rn_":"Tri2"},{"1":"0","2":"0","3":"0","4":"0","5":"1","6":"0","_rn_":"Tri3"},{"1":"0","2":"0","3":"0","4":"0","5":"0","6":"1","_rn_":"UT1"},{"1":"0","2":"0","3":"0","4":"0","5":"0","6":"1","_rn_":"UT2"},{"1":"0","2":"0","3":"0","4":"0","5":"0","6":"1","_rn_":"UT3"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="contrast-matrix" class="section level3">
<h3>Contrast Matrix</h3>
<p>The contrast matrix is required to provide a coefficient to each comparison and later used to test for significant differential expression with each comparison group</p>
<pre class="r"><code># make specific contrasts
contrast &lt;- limma::makeContrasts(
  Mac_vs_CKI = Mac - CKI,
  Nme_vs_CKI = Nme - CKI,
  Omt_vs_CKI = Omt - CKI,
  Tri_vs_CKI = Tri - CKI,
  UT_vs_CKI = UT - CKI,
  levels = designMatrix)

#rename the five comparisons
colnames(contrast) &lt;- c(&quot;CKI-Mac vs CKI&quot;, &quot;CKI-Nme vs CKI&quot;, &quot;CKI-Omt vs CKI&quot;, &quot;CKI-Tri vs CKI&quot;, &quot;UT vs CKI&quot;)

contrast %&gt;% as.data.frame()</code></pre>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["CKI-Mac vs CKI"],"name":[1],"type":["dbl"],"align":["right"]},{"label":["CKI-Nme vs CKI"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["CKI-Omt vs CKI"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["CKI-Tri vs CKI"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["UT vs CKI"],"name":[5],"type":["dbl"],"align":["right"]}],"data":[{"1":"-1","2":"-1","3":"-1","4":"-1","5":"-1","_rn_":"CKI"},{"1":"1","2":"0","3":"0","4":"0","5":"0","_rn_":"Mac"},{"1":"0","2":"1","3":"0","4":"0","5":"0","_rn_":"Nme"},{"1":"0","2":"0","3":"1","4":"0","5":"0","_rn_":"Omt"},{"1":"0","2":"0","3":"0","4":"1","5":"0","_rn_":"Tri"},{"1":"0","2":"0","3":"0","4":"0","5":"1","_rn_":"UT"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<pre class="r"><code>#generate data frame of contrast colname for downstream analysis
comparison_group=colnames(contrast) %&gt;% as.data.frame()</code></pre>
</div>
<div id="estimating-dispersion" class="section level3">
<h3>Estimating Dispersion</h3>
<p>The negative binomial dispersion is calculated using the <code>edgeR</code> User guide recommended method of using the <code>estimateDisp</code> function. This method calculates the common dispersion, trended dispersion and tagwise dispersions in one run. This function also returns the formal <code>DGEList</code> object but with additional entries for the negative binomial dispersion for all genes.</p>
<p>The dispersion can then be visualised through the <code>plotBCV</code> function which shows the biological coefficient of variance (BCV) of each gene</p>
<pre class="r"><code># setting robust to T is highly recommneded by the edgeR user guide
NB_dispersion &lt;- estimateDisp(y = dge, design = designMatrix, robust = TRUE)

## Display the common dispersion parameter of NB model
# NB_dispersion$common.dispersion

#visualisation of dispersion estimate thorugh the BCV plot which is just the square root of the dispersion parameter under the NB model
edgeR::plotBCV(NB_dispersion)</code></pre>
<p><img src="figure/dge.Rmd/NBDispersion-1.png" width="1056" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#save
invisible(dev.print(svg, here::here(&quot;2_plots/2_dge/NB_dispersion.svg&quot;)))</code></pre>
</div>
</div>
<div id="generalised-linear-model-glm-with-quasi-likelihood-f-test-qlf" class="section level2">
<h2>Generalised Linear Model (GLM) with Quasi-likelihood F-Test (QLF)</h2>
<p>The negative binomial model can be extended with quasi-likelihood methods to account for gene specific variability from both biological and technical sources.</p>
<p><em>While the likelihood ratio test is a more obvious choice for inferences with GLMs, the QL F-test is preferred as it reflects the uncertainty in estimating the dispersion for each gene. It provides more robust and reliable error rate control when the number of replicates is small. The QL dispersion estimation and hypothesis testing can be done by using the functions <code>glmQLFit()</code> and <code>glmQLFTest()</code>.</em></p>
<div id="extending-the-nb-dispersion-with-the-ql-method" class="section level3">
<h3>Extending the NB dispersion with the QL method</h3>
<p><em>For the QL dispersions, estimation can be performed using the <code>glmQLFit</code> function. This returns a DGEGLM object containing the estimated values of the GLM coefficients for each gene, as well as the fitted mean-QL dispersion trend, the squeezed QL estimates and the prior degrees of freedom (df). These can be visualized with the <code>plotQLDisp</code> function.</em></p>
<pre class="r"><code>#extend the NB dispersion with QL method
qlf_dispersion &lt;- edgeR::glmQLFit(y = NB_dispersion, design = designMatrix, robust = TRUE)

## view gene coefficients
# head(qlf_dispersion$coefficients)

#visualise the disperion parameters after apply QL method to NB dispersion estimate 
plotQLDisp(glmfit = qlf_dispersion)</code></pre>
<p><img src="figure/dge.Rmd/qlf_dispersion-1.png" width="1056" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#save
invisible(dev.print(svg, here::here(&quot;2_plots/2_dge/qlf_dispersion.svg&quot;)))</code></pre>
</div>
<div id="apply-ql-f-test" class="section level3">
<h3>Apply QL F-test</h3>
<p>Following the application of the QL method to NB dispersion estimate, the actually testing for DE genes can be performed. This can be achieved through the implementation of QL F-test. Unlike the LRT test, the QLF enable a greater control of type 1 errors.</p>
<pre class="r"><code># Create list object that for loop will populate
qlf=list()
qlf_decideTest=list()
qlf_all=list()
qlf_sig=list()

for (i in 1:ncol(contrast)){
  #at each iteration, let x = name of each contrast group
  x=comparison_group[i,]

  #populate list with DGELRT object for every comparison
  qlf[[x]] &lt;-
    edgeR::glmQLFTest(glmfit = qlf_dispersion, contrast = contrast[,x])

  #populate significant list with decide test
  qlf_decideTest[[x]] &lt;-
    decideTests(qlf[[x]], p.value = 0.05, adjust.method = &quot;fdr&quot;) %&gt;% summary() %&gt;% as.data.frame()

  #populate all list with list of all DE genes
  qlf_all[[x]] &lt;-
    edgeR::topTags(object = qlf[[x]], n = Inf) %&gt;% as.data.frame()

  #populate sig list with list of significant DE genes
  qlf_sig[[x]] &lt;-
    edgeR::topTags(object = qlf[[x]], n = Inf, adjust.method = &quot;fdr&quot;, p.value = 0.05, sort.by = &quot;PValue&quot;) %&gt;% as.data.frame()
}</code></pre>
<div id="visualisation" class="section level4 tabset">
<h4 class="tabset">Visualisation</h4>
<p><strong>Visualisation of QL F-test has been disable, the code, however, remains.</strong></p>
<div id="p-value-histogram" class="section level5">
<h5>P Value histogram</h5>
<pre class="r"><code>#generate p value histogram for all comparison
lapply(1:length(qlf),
       function(x){
         hist(x = qlf[[x]]$table$PValue,
              breaks = 50,
              main = paste0(&quot;P-Values QL F-test &quot;, names(qlf[x])),
              xlab = &quot;P-Value&quot;,
              col = &quot;gray50&quot;)

#save
# invisible(dev.print(pdf, here::here(paste0(&quot;2_plots/2_dge/qlf_pValue_histogram_&quot;, names(qlf[x]), &quot;.svg&quot;))))
})</code></pre>
</div>
<div id="mean-difference-plot" class="section level5">
<h5>Mean-Difference plot</h5>
<p>The differential expression visualised through <code>plotMD</code> function, where the <code>logFC</code> is plotted against the relative abundance of the gene <code>logCPM</code>. Significant genes with FDR of 0.05 or less are highlighted.</p>
<pre class="r"><code>lapply(1:length(qlf_sig),
       function(x) {

         #create data.frame specific for the custome MA plot
         MAplotData_qlf &lt;- dplyr::select(as.data.frame(qlf_sig[[x]]), logCPM, logFC, FDR)
         colnames(MAplotData_qlf) &lt;- c(&quot;baseMeanLog2&quot;, &quot;log2FoldChange&quot;, &quot;padj&quot;)

         #create custom MA plot
         MAplot_qlf &lt;- ggpubr::ggmaplot(
           data = MAplotData_qlf,
           fdr = 0.05,
           fc = 1.5,
           genenames = as.vector(rownames(MAplotData_qlf)),
           size = 1.5,
           alpha = 0.8,
           label.rectangle = TRUE,
           palette = c(&quot;firebrick3&quot;, &quot;dodgerblue3&quot;, &quot;gray50&quot;),
           top = 5,
           select.top.method = c(&quot;padj&quot;, &quot;fc&quot;),
           main = &quot;MA Plot from GLM-QLF &quot;,
           legend = &quot;bottom&quot;,
           submain = names(qlf_sig[x]),
           xlab = expression(&quot;log&quot;[2]*&quot;CPM&quot;),
           ylab = expression(&quot;log&quot;[2]*&quot;FC&quot;),
           ylim = c(-8, 8),
           font.label = c(13, &quot;italic&quot;, &quot;gray30&quot;),
           ggtheme = theme_minimal())

         #display plot
         MAplot_qlf

         #save MA plot
         ggsave(paste0(&quot;MA_qlf_&quot;, names(qlf_sig[x]), &quot;.svg&quot;), plot = MAplot_qlf, path = here::here((&quot;2_plots/2_dge/&quot;))
         })</code></pre>
</div>
<div id="volcano-plot" class="section level5">
<h5>Volcano Plot</h5>
<p>Here the <code>ggplot</code> function can also be used to illustrate the DE genes significant <code>pValue</code> and <code>logFC</code>. In this instance, the significance is determined as genes with <code>PValue &lt; 0.05</code> <strong>and</strong> <code>abs(logFC)&gt;log(2)</code>.</p>
<pre class="r"><code>lapply(1:length(qlf_sig),
       function(x) {

         #add an extra column and determine whether the DE genes are significant
         qlf_sig[[x]] &lt;- qlf_sig[[x]] %&gt;% as.data.frame() %&gt;%
           dplyr::mutate(Expression = case_when
                         (FDR &lt;= 0.05 &amp; logFC &gt;= 1.5 ~ &quot;Up-regulated&quot;,
                          FDR &lt;= 0.05 &amp; logFC &lt;= -1.5 ~ &quot;Down-regulated&quot;,
                          TRUE ~ &quot;Insignificant&quot;))

         #adding labels to top genes
         top &lt;- 5
         top_genes_qlf &lt;- bind_rows(
           qlf_sig[[x]] %&gt;%
             filter(Expression == &#39;Up-regulated&#39;) %&gt;%
             arrange(FDR, desc(abs(logFC))) %&gt;%
             head(top),
           qlf_sig[[x]] %&gt;%
             filter(Expression == &#39;Down-regulated&#39;) %&gt;%
             arrange(FDR, desc(abs(logFC))) %&gt;%
             head(top)
           )
         invisible(top_genes_qlf %&gt;% as.data.frame())

         #generate volcano plot with the allDEgene data.frame
         volcano_qlf &lt;- qlf_sig[[x]] %&gt;%
           ggplot(aes(x = logFC,
                      y = -log(FDR, 10))) +
           geom_point(aes(colour = Expression),
                      size = 1.5,
                      alpha = 0.8,
                      show.legend = FALSE) +
           geom_label_repel(data = top_genes_qlf,
                            mapping = aes(logFC, -log(FDR,10), label = gene),
                            size = 3) +
           xlab(expression(&quot;log&quot;[2]*&quot;FC&quot;)) +
           ylab(expression(&quot;-log&quot;[10]*&quot;FDR&quot;)) +
           xlim(-8,8)+
           ylim(0,30)+
           scale_color_manual(values = c(&quot;dodgerblue3&quot;, &quot;gray50&quot;, &quot;firebrick3&quot;)) +
           guides(colour = guide_legend(override.aes = list(size=1.5))) +
           labs(title = &quot;Volcano Plot: GLM-QLF&quot;,
                subtitle = paste0(names(qlf_sig[x])),
                colour = &quot;Expression&quot;)

         #save to directory
         ggsave(paste0(&quot;volcano_qlf_&quot;, names(qlf_sig[x]), &quot;.svg&quot;), plot = volcano_qlf, path = here::here(&quot;2_plots/2_dge/&quot;))

         #display
         volcano_qlf
         })</code></pre>
</div>
</div>
</div>
<div id="apply-treat" class="section level3">
<h3>Apply TREAT</h3>
<p>Following the application of the QL method to NB dispersion estimate, the actual testing for DE genes can be performed. This can be achieved through the implementation of TREAT. Unlike the QL-F test, TREAT enables identificaiton of DE genes relative to a <code>logFC</code> threshold. The current <code>logFC</code> threshold is <code>|logFC&gt;=1.5|</code></p>
<pre class="r"><code># Create list object
treat=list()
treat_decideTest=list()
treat_all=list()
treat_sig=list()

for (i in 1:ncol(contrast)){
  #at each iteration, let x = name of each contrast group
  x=comparison_group[i,]

  #populate list with DGELRT object for every comparison
  treat[[x]] &lt;-
    edgeR::glmTreat(glmfit =  qlf_dispersion, contrast = contrast[,x], lfc = 1.5)

  #populate significant list with decide test
  treat_decideTest[[x]] &lt;- decideTests(treat[[x]], p.value = 0.05, adjust.method = &quot;fdr&quot;) %&gt;% summary() %&gt;% as.data.frame()


  #populate all list with list of all DE genes
  treat_all[[x]] &lt;-
    edgeR::topTags(object = treat[[x]], n = Inf) %&gt;% as.data.frame()

  #populate sig list with list of significant DE genes
  treat_sig[[x]] &lt;-
    edgeR::topTags(object = treat[[x]], n = Inf, adjust.method = &quot;fdr&quot;, p.value = 0.05, sort.by = &quot;PValue&quot;) %&gt;% as.data.frame()
}</code></pre>
<div id="visualisation-1" class="section level4 tabset">
<h4 class="tabset">Visualisation</h4>
<div id="p-value-histogram-1" class="section level5">
<h5>P Value histogram</h5>
<pre class="r"><code>lapply(1:length(treat),
       function(x){
         hist(x = treat[[x]]$table$PValue,
              breaks = 50,
              main = paste0(&quot;P-Values treat &quot;, names(treat[x])),
              xlab = &quot;P-Value&quot;,
              col = &quot;gray50&quot;)

invisible(dev.print(pdf, here::here(paste0(&quot;2_plots/2_dge/pValue_treat_&quot;, names(treat[x]), &quot;.svg&quot;))))
})</code></pre>
<p><img src="figure/dge.Rmd/pValueHistogram_treat-1.png" width="1056" style="display: block; margin: auto;" /><img src="figure/dge.Rmd/pValueHistogram_treat-2.png" width="1056" style="display: block; margin: auto;" /><img src="figure/dge.Rmd/pValueHistogram_treat-3.png" width="1056" style="display: block; margin: auto;" /><img src="figure/dge.Rmd/pValueHistogram_treat-4.png" width="1056" style="display: block; margin: auto;" /><img src="figure/dge.Rmd/pValueHistogram_treat-5.png" width="1056" style="display: block; margin: auto;" /></p>
<pre><code>[[1]]
png 
  2 

[[2]]
png 
  2 

[[3]]
png 
  2 

[[4]]
png 
  2 

[[5]]
png 
  2 </code></pre>
</div>
<div id="mean-difference-plot-1" class="section level5">
<h5>Mean-Difference plot</h5>
<p>The differential expression can initial visualised through <code>plotMD</code> function, where the <code>logFC</code> is plotted against the relative abundance of the gene <code>logCPM</code>. Significant genes with FDR of 0.05 or less are highlighted.</p>
<pre class="r"><code>MAplot=list()
lapply(1:length(treat_all),
       function(x) {

         #create data.frame specific for the custome MA plot
         MAplotData_treat &lt;- dplyr::select(as.data.frame(treat_all[[x]]), logCPM, logFC, FDR)
         colnames(MAplotData_treat) &lt;- c(&quot;baseMeanLog2&quot;, &quot;log2FoldChange&quot;, &quot;padj&quot;)

         #create custom MA plot
         MAplot_treat &lt;- ggpubr::ggmaplot(
           data = MAplotData_treat,
           fdr = 0.05,
           fc = 2.828,
           genenames = as.vector(rownames(MAplotData_treat)),
           size = 1.5,
           alpha = 0.8,
           label.rectangle = TRUE,
           palette = c(&quot;firebrick3&quot;, &quot;dodgerblue3&quot;, &quot;gray50&quot;),
           top = 5,
           ylim = c(-8, 8),
           select.top.method = c(&quot;padj&quot;, &quot;fc&quot;),
           main = &quot;MA Plot from GLM:QLF-TREAT&quot;,
           submain = names(treat_all[x]),
           legend = &quot;NA&quot;,
           legend.title = &quot;Expression&quot;,
           xlab = expression(&quot;log&quot;[2]*&quot;CPM&quot;),
           ylab = expression(&quot;log&quot;[2]*&quot;FC&quot;),
           font.label = c(13, &quot;italic&quot;, &quot;gray30&quot;), 
           ggtheme = theme_minimal())
         
         #save MA plot
         ggsave(filename = paste0(&quot;MA_treat_&quot;, names(treat_all[x]), &quot;.svg&quot;),
                plot = MAplot_treat,
                path = here::here(&quot;2_plots/2_dge&quot;))

         #display plot
         MAplot_treat
         })</code></pre>
<pre><code>[[1]]</code></pre>
<p><img src="figure/dge.Rmd/MAplotCustom_treat-1.png" width="1056" style="display: block; margin: auto;" /></p>
<pre><code>
[[2]]</code></pre>
<p><img src="figure/dge.Rmd/MAplotCustom_treat-2.png" width="1056" style="display: block; margin: auto;" /></p>
<pre><code>
[[3]]</code></pre>
<p><img src="figure/dge.Rmd/MAplotCustom_treat-3.png" width="1056" style="display: block; margin: auto;" /></p>
<pre><code>
[[4]]</code></pre>
<p><img src="figure/dge.Rmd/MAplotCustom_treat-4.png" width="1056" style="display: block; margin: auto;" /></p>
<pre><code>
[[5]]</code></pre>
<p><img src="figure/dge.Rmd/MAplotCustom_treat-5.png" width="1056" style="display: block; margin: auto;" /></p>
</div>
<div id="volcano-plot-1" class="section level5">
<h5>Volcano Plot</h5>
<p>Here the <code>ggplot</code> function can also be used to illustrate the DE genes significant <code>pValue</code> and <code>logFC</code>. In this instance, the significance is determined as genes with <code>PValue &lt; 0.05</code> <strong>and</strong> <code>abs(logFC)&gt;log(2)</code>.</p>
<pre class="r"><code>lapply(1:length(treat_all),
       function(x) {

         #add an extra column and determine whether the DE genes are significant
         treat_all[[x]] &lt;- treat_all[[x]] %&gt;% as.data.frame() %&gt;%
           dplyr::mutate(Expression = case_when
                         (FDR &lt;= 0.05 &amp; logFC &gt;= 1.5 ~ &quot;Up-regulated&quot;,
                          FDR &lt;= 0.05 &amp; logFC &lt;= -1.5 ~ &quot;Down-regulated&quot;,
                          TRUE ~ &quot;Insignificant&quot;))

         #adding labels to top genes
         top &lt;- 5
         top_genes_treat &lt;- bind_rows(
           treat_all[[x]] %&gt;%
             filter(Expression == &#39;Up-regulated&#39;) %&gt;%
             arrange(FDR, desc(abs(logFC))) %&gt;%
             head(top),
           treat_all[[x]] %&gt;%
             filter(Expression == &#39;Down-regulated&#39;) %&gt;%
             arrange(FDR, desc(abs(logFC))) %&gt;%
             head(top)
           )
         invisible(top_genes_treat %&gt;% as.data.frame())

         #generate volcano plot with the allDEgene data.frame
         volcano_treat &lt;- treat_all[[x]] %&gt;%
           ggplot(aes(x = logFC,
                      y = -log(FDR, 10))) +
           geom_point(aes(colour = Expression),
                      size = 1.5,
                      alpha = 0.8,
                      show.legend = FALSE) +
           geom_label_repel(data = top_genes_treat,
                            mapping = aes(logFC, -log(FDR,10), label = gene),
                            size = 3) +
           xlab(expression(&quot;log&quot;[2]*&quot;FC&quot;)) +
           ylab(expression(&quot;-log&quot;[10]*&quot;FDR&quot;)) +
           xlim(-8,8)+
           ylim(0,30)+
           scale_color_manual(values = c(&quot;dodgerblue3&quot;, &quot;gray50&quot;, &quot;firebrick3&quot;)) +
           guides(colour = guide_legend(override.aes = list(size=1.5))) +
  
           labs(title = &quot;Volcano Plot: GLM:QLF-TREAT&quot;,
                subtitle = paste0(names(treat_all[x])),
                colour = &quot;Expression&quot;)

         #save to directory
         ggsave(paste0(&quot;volcano_treat_&quot;, names(treat_all[x]), &quot;.svg&quot;),
                plot = volcano_treat,
                path = here::here(&quot;2_plots/2_dge/&quot;))

         #display
         volcano_treat
         })</code></pre>
<pre><code>[[1]]</code></pre>
<p><img src="figure/dge.Rmd/volcano_treat-1.png" width="1056" style="display: block; margin: auto;" /></p>
<pre><code>
[[2]]</code></pre>
<p><img src="figure/dge.Rmd/volcano_treat-2.png" width="1056" style="display: block; margin: auto;" /></p>
<pre><code>
[[3]]</code></pre>
<p><img src="figure/dge.Rmd/volcano_treat-3.png" width="1056" style="display: block; margin: auto;" /></p>
<pre><code>
[[4]]</code></pre>
<p><img src="figure/dge.Rmd/volcano_treat-4.png" width="1056" style="display: block; margin: auto;" /></p>
<pre><code>
[[5]]</code></pre>
<p><img src="figure/dge.Rmd/volcano_treat-5.png" width="1056" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="heatmaps" class="section level4 tabset">
<h4 class="tabset">Heatmaps</h4>
<div id="top-common" class="section level5">
<h5>Top Common</h5>
<p>This heatmap displays the expression of the most significant DE genes common among all five comparisons</p>
<pre class="r"><code>#create matrix with log cpm counts
logCPM &lt;- cpm(dge, prior.count=3, log=TRUE)
rownames(logCPM) &lt;- dge$genes$gene
colnames(logCPM) &lt;- paste(dge$samples$pretty_name, sep=&quot;-&quot;)

#join common significant DE genes into df
common &lt;- join_all(list(treat_sig$`UT vs CKI`,
                        treat_sig$`CKI-Mac vs CKI`,
                        treat_sig$`CKI-Nme vs CKI`,
                        treat_sig$`CKI-Omt vs CKI`,
                        treat_sig$`CKI-Tri vs CKI`), by = &#39;gene&#39;, type = &#39;inner&#39;)

kable(common, digits = Inf) %&gt;%
  kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  scroll_box(height = &quot;600px&quot;)</code></pre>
<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
gene
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
seqnames
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
start
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
end
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
width
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
strand
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
gene_id
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
gene_name
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
gene_biotype
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
seq_coord_system
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
description
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
gene_id_version
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
canonical_transcript
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
entrezid
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
logFC
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
unshrunk.logFC
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
logCPM
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
PValue
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
FDR
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
seqnames
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
start
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
end
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
width
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
strand
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
gene_id
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
gene_name
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
gene_biotype
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
seq_coord_system
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
description
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
gene_id_version
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
canonical_transcript
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
entrezid
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
logFC
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
unshrunk.logFC
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
logCPM
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
PValue
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
FDR
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
seqnames
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
start
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
end
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
width
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
strand
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
gene_id
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
gene_name
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
gene_biotype
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
seq_coord_system
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
description
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
gene_id_version
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
canonical_transcript
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
entrezid
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
logFC
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
unshrunk.logFC
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
logCPM
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
PValue
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
FDR
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
seqnames
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
start
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
end
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
width
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
strand
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
gene_id
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
gene_name
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
gene_biotype
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
seq_coord_system
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
description
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
gene_id_version
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
canonical_transcript
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
entrezid
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
logFC
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
unshrunk.logFC
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
logCPM
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
PValue
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
FDR
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
seqnames
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
start
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
end
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
width
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
strand
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
gene_id
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
gene_name
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
gene_biotype
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
seq_coord_system
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
description
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
gene_id_version
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
canonical_transcript
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
entrezid
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
logFC
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
unshrunk.logFC
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
logCPM
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
PValue
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
FDR
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
CYP1A1
</td>
<td style="text-align:left;">
15
</td>
<td style="text-align:right;">
74719542
</td>
<td style="text-align:right;">
74725536
</td>
<td style="text-align:right;">
5995
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000140465
</td>
<td style="text-align:left;">
CYP1A1
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
cytochrome P450 family 1 subfamily A member 1 [Source:HGNC Symbol;Acc:HGNC:2595]
</td>
<td style="text-align:left;">
ENSG00000140465.14
</td>
<td style="text-align:left;">
ENST00000379727
</td>
<td style="text-align:left;">
1543
</td>
<td style="text-align:right;">
-6.875442
</td>
<td style="text-align:right;">
-6.888598
</td>
<td style="text-align:right;">
4.1189921
</td>
<td style="text-align:right;">
2.972544e-30
</td>
<td style="text-align:right;">
3.941296e-26
</td>
<td style="text-align:left;">
15
</td>
<td style="text-align:right;">
74719542
</td>
<td style="text-align:right;">
74725536
</td>
<td style="text-align:right;">
5995
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000140465
</td>
<td style="text-align:left;">
CYP1A1
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
cytochrome P450 family 1 subfamily A member 1 [Source:HGNC Symbol;Acc:HGNC:2595]
</td>
<td style="text-align:left;">
ENSG00000140465.14
</td>
<td style="text-align:left;">
ENST00000379727
</td>
<td style="text-align:left;">
1543
</td>
<td style="text-align:right;">
-2.850475
</td>
<td style="text-align:right;">
-2.851174
</td>
<td style="text-align:right;">
4.1189921
</td>
<td style="text-align:right;">
2.914189e-18
</td>
<td style="text-align:right;">
2.033644e-15
</td>
<td style="text-align:left;">
15
</td>
<td style="text-align:right;">
74719542
</td>
<td style="text-align:right;">
74725536
</td>
<td style="text-align:right;">
5995
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000140465
</td>
<td style="text-align:left;">
CYP1A1
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
cytochrome P450 family 1 subfamily A member 1 [Source:HGNC Symbol;Acc:HGNC:2595]
</td>
<td style="text-align:left;">
ENSG00000140465.14
</td>
<td style="text-align:left;">
ENST00000379727
</td>
<td style="text-align:left;">
1543
</td>
<td style="text-align:right;">
-4.219593
</td>
<td style="text-align:right;">
-4.221574
</td>
<td style="text-align:right;">
4.1189921
</td>
<td style="text-align:right;">
1.149757e-25
</td>
<td style="text-align:right;">
1.524462e-21
</td>
<td style="text-align:left;">
15
</td>
<td style="text-align:right;">
74719542
</td>
<td style="text-align:right;">
74725536
</td>
<td style="text-align:right;">
5995
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000140465
</td>
<td style="text-align:left;">
CYP1A1
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
cytochrome P450 family 1 subfamily A member 1 [Source:HGNC Symbol;Acc:HGNC:2595]
</td>
<td style="text-align:left;">
ENSG00000140465.14
</td>
<td style="text-align:left;">
ENST00000379727
</td>
<td style="text-align:left;">
1543
</td>
<td style="text-align:right;">
-2.593100
</td>
<td style="text-align:right;">
-2.593666
</td>
<td style="text-align:right;">
4.1189921
</td>
<td style="text-align:right;">
7.491515e-16
</td>
<td style="text-align:right;">
1.141724e-13
</td>
<td style="text-align:left;">
15
</td>
<td style="text-align:right;">
74719542
</td>
<td style="text-align:right;">
74725536
</td>
<td style="text-align:right;">
5995
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000140465
</td>
<td style="text-align:left;">
CYP1A1
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
cytochrome P450 family 1 subfamily A member 1 [Source:HGNC Symbol;Acc:HGNC:2595]
</td>
<td style="text-align:left;">
ENSG00000140465.14
</td>
<td style="text-align:left;">
ENST00000379727
</td>
<td style="text-align:left;">
1543
</td>
<td style="text-align:right;">
-3.722120
</td>
<td style="text-align:right;">
-3.723497
</td>
<td style="text-align:right;">
4.1189921
</td>
<td style="text-align:right;">
1.629483e-23
</td>
<td style="text-align:right;">
7.201773e-20
</td>
</tr>
<tr>
<td style="text-align:left;">
ABCA1
</td>
<td style="text-align:left;">
9
</td>
<td style="text-align:right;">
104781006
</td>
<td style="text-align:right;">
104928155
</td>
<td style="text-align:right;">
147150
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000165029
</td>
<td style="text-align:left;">
ABCA1
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
ATP binding cassette subfamily A member 1 [Source:HGNC Symbol;Acc:HGNC:29]
</td>
<td style="text-align:left;">
ENSG00000165029.17
</td>
<td style="text-align:left;">
ENST00000374736
</td>
<td style="text-align:left;">
19
</td>
<td style="text-align:right;">
3.163827
</td>
<td style="text-align:right;">
3.164634
</td>
<td style="text-align:right;">
5.8109715
</td>
<td style="text-align:right;">
3.972089e-23
</td>
<td style="text-align:right;">
1.316648e-19
</td>
<td style="text-align:left;">
9
</td>
<td style="text-align:right;">
104781006
</td>
<td style="text-align:right;">
104928155
</td>
<td style="text-align:right;">
147150
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000165029
</td>
<td style="text-align:left;">
ABCA1
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
ATP binding cassette subfamily A member 1 [Source:HGNC Symbol;Acc:HGNC:29]
</td>
<td style="text-align:left;">
ENSG00000165029.17
</td>
<td style="text-align:left;">
ENST00000374736
</td>
<td style="text-align:left;">
19
</td>
<td style="text-align:right;">
2.769970
</td>
<td style="text-align:right;">
2.770745
</td>
<td style="text-align:right;">
5.8109715
</td>
<td style="text-align:right;">
3.737055e-19
</td>
<td style="text-align:right;">
3.096851e-16
</td>
<td style="text-align:left;">
9
</td>
<td style="text-align:right;">
104781006
</td>
<td style="text-align:right;">
104928155
</td>
<td style="text-align:right;">
147150
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000165029
</td>
<td style="text-align:left;">
ABCA1
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
ATP binding cassette subfamily A member 1 [Source:HGNC Symbol;Acc:HGNC:29]
</td>
<td style="text-align:left;">
ENSG00000165029.17
</td>
<td style="text-align:left;">
ENST00000374736
</td>
<td style="text-align:left;">
19
</td>
<td style="text-align:right;">
2.734197
</td>
<td style="text-align:right;">
2.734969
</td>
<td style="text-align:right;">
5.8109715
</td>
<td style="text-align:right;">
9.005762e-19
</td>
<td style="text-align:right;">
1.326749e-15
</td>
<td style="text-align:left;">
9
</td>
<td style="text-align:right;">
104781006
</td>
<td style="text-align:right;">
104928155
</td>
<td style="text-align:right;">
147150
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000165029
</td>
<td style="text-align:left;">
ABCA1
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
ATP binding cassette subfamily A member 1 [Source:HGNC Symbol;Acc:HGNC:29]
</td>
<td style="text-align:left;">
ENSG00000165029.17
</td>
<td style="text-align:left;">
ENST00000374736
</td>
<td style="text-align:left;">
19
</td>
<td style="text-align:right;">
2.899521
</td>
<td style="text-align:right;">
2.900308
</td>
<td style="text-align:right;">
5.8109715
</td>
<td style="text-align:right;">
1.580998e-20
</td>
<td style="text-align:right;">
7.486590e-18
</td>
<td style="text-align:left;">
9
</td>
<td style="text-align:right;">
104781006
</td>
<td style="text-align:right;">
104928155
</td>
<td style="text-align:right;">
147150
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000165029
</td>
<td style="text-align:left;">
ABCA1
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
ATP binding cassette subfamily A member 1 [Source:HGNC Symbol;Acc:HGNC:29]
</td>
<td style="text-align:left;">
ENSG00000165029.17
</td>
<td style="text-align:left;">
ENST00000374736
</td>
<td style="text-align:left;">
19
</td>
<td style="text-align:right;">
2.691747
</td>
<td style="text-align:right;">
2.692514
</td>
<td style="text-align:right;">
5.8109715
</td>
<td style="text-align:right;">
3.034331e-18
</td>
<td style="text-align:right;">
2.682146e-15
</td>
</tr>
<tr>
<td style="text-align:left;">
AKR1B10
</td>
<td style="text-align:left;">
7
</td>
<td style="text-align:right;">
134527567
</td>
<td style="text-align:right;">
134541412
</td>
<td style="text-align:right;">
13846
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000198074
</td>
<td style="text-align:left;">
AKR1B10
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
aldo-keto reductase family 1 member B10 [Source:HGNC Symbol;Acc:HGNC:382]
</td>
<td style="text-align:left;">
ENSG00000198074.10
</td>
<td style="text-align:left;">
ENST00000359579
</td>
<td style="text-align:left;">
57016
</td>
<td style="text-align:right;">
-5.445797
</td>
<td style="text-align:right;">
-5.468945
</td>
<td style="text-align:right;">
2.0773700
</td>
<td style="text-align:right;">
9.894575e-21
</td>
<td style="text-align:right;">
1.639902e-17
</td>
<td style="text-align:left;">
7
</td>
<td style="text-align:right;">
134527567
</td>
<td style="text-align:right;">
134541412
</td>
<td style="text-align:right;">
13846
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000198074
</td>
<td style="text-align:left;">
AKR1B10
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
aldo-keto reductase family 1 member B10 [Source:HGNC Symbol;Acc:HGNC:382]
</td>
<td style="text-align:left;">
ENSG00000198074.10
</td>
<td style="text-align:left;">
ENST00000359579
</td>
<td style="text-align:left;">
57016
</td>
<td style="text-align:right;">
-2.420471
</td>
<td style="text-align:right;">
-2.422820
</td>
<td style="text-align:right;">
2.0773700
</td>
<td style="text-align:right;">
6.943818e-09
</td>
<td style="text-align:right;">
7.365447e-07
</td>
<td style="text-align:left;">
7
</td>
<td style="text-align:right;">
134527567
</td>
<td style="text-align:right;">
134541412
</td>
<td style="text-align:right;">
13846
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000198074
</td>
<td style="text-align:left;">
AKR1B10
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
aldo-keto reductase family 1 member B10 [Source:HGNC Symbol;Acc:HGNC:382]
</td>
<td style="text-align:left;">
ENSG00000198074.10
</td>
<td style="text-align:left;">
ENST00000359579
</td>
<td style="text-align:left;">
57016
</td>
<td style="text-align:right;">
-2.921591
</td>
<td style="text-align:right;">
-2.925131
</td>
<td style="text-align:right;">
2.0773700
</td>
<td style="text-align:right;">
1.225271e-12
</td>
<td style="text-align:right;">
2.673242e-10
</td>
<td style="text-align:left;">
7
</td>
<td style="text-align:right;">
134527567
</td>
<td style="text-align:right;">
134541412
</td>
<td style="text-align:right;">
13846
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000198074
</td>
<td style="text-align:left;">
AKR1B10
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
aldo-keto reductase family 1 member B10 [Source:HGNC Symbol;Acc:HGNC:382]
</td>
<td style="text-align:left;">
ENSG00000198074.10
</td>
<td style="text-align:left;">
ENST00000359579
</td>
<td style="text-align:left;">
57016
</td>
<td style="text-align:right;">
-2.232062
</td>
<td style="text-align:right;">
-2.234057
</td>
<td style="text-align:right;">
2.0773700
</td>
<td style="text-align:right;">
3.414854e-07
</td>
<td style="text-align:right;">
1.254226e-05
</td>
<td style="text-align:left;">
7
</td>
<td style="text-align:right;">
134527567
</td>
<td style="text-align:right;">
134541412
</td>
<td style="text-align:right;">
13846
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000198074
</td>
<td style="text-align:left;">
AKR1B10
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
aldo-keto reductase family 1 member B10 [Source:HGNC Symbol;Acc:HGNC:382]
</td>
<td style="text-align:left;">
ENSG00000198074.10
</td>
<td style="text-align:left;">
ENST00000359579
</td>
<td style="text-align:left;">
57016
</td>
<td style="text-align:right;">
-3.273234
</td>
<td style="text-align:right;">
-3.277902
</td>
<td style="text-align:right;">
2.0773700
</td>
<td style="text-align:right;">
2.151157e-14
</td>
<td style="text-align:right;">
9.507396e-12
</td>
</tr>
<tr>
<td style="text-align:left;">
ASB2
</td>
<td style="text-align:left;">
14
</td>
<td style="text-align:right;">
93934166
</td>
<td style="text-align:right;">
93976739
</td>
<td style="text-align:right;">
42574
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000100628
</td>
<td style="text-align:left;">
ASB2
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
ankyrin repeat and SOCS box containing 2 [Source:HGNC Symbol;Acc:HGNC:16012]
</td>
<td style="text-align:left;">
ENSG00000100628.12
</td>
<td style="text-align:left;">
ENST00000555019
</td>
<td style="text-align:left;">
51676
</td>
<td style="text-align:right;">
-3.760831
</td>
<td style="text-align:right;">
-3.763713
</td>
<td style="text-align:right;">
3.2479347
</td>
<td style="text-align:right;">
1.678241e-19
</td>
<td style="text-align:right;">
2.225180e-16
</td>
<td style="text-align:left;">
14
</td>
<td style="text-align:right;">
93934166
</td>
<td style="text-align:right;">
93976739
</td>
<td style="text-align:right;">
42574
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000100628
</td>
<td style="text-align:left;">
ASB2
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
ankyrin repeat and SOCS box containing 2 [Source:HGNC Symbol;Acc:HGNC:16012]
</td>
<td style="text-align:left;">
ENSG00000100628.12
</td>
<td style="text-align:left;">
ENST00000555019
</td>
<td style="text-align:left;">
51676
</td>
<td style="text-align:right;">
-2.709970
</td>
<td style="text-align:right;">
-2.711242
</td>
<td style="text-align:right;">
3.2479347
</td>
<td style="text-align:right;">
2.660748e-13
</td>
<td style="text-align:right;">
6.784397e-11
</td>
<td style="text-align:left;">
14
</td>
<td style="text-align:right;">
93934166
</td>
<td style="text-align:right;">
93976739
</td>
<td style="text-align:right;">
42574
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000100628
</td>
<td style="text-align:left;">
ASB2
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
ankyrin repeat and SOCS box containing 2 [Source:HGNC Symbol;Acc:HGNC:16012]
</td>
<td style="text-align:left;">
ENSG00000100628.12
</td>
<td style="text-align:left;">
ENST00000555019
</td>
<td style="text-align:left;">
51676
</td>
<td style="text-align:right;">
-3.073330
</td>
<td style="text-align:right;">
-3.075028
</td>
<td style="text-align:right;">
3.2479347
</td>
<td style="text-align:right;">
5.563116e-16
</td>
<td style="text-align:right;">
2.950454e-13
</td>
<td style="text-align:left;">
14
</td>
<td style="text-align:right;">
93934166
</td>
<td style="text-align:right;">
93976739
</td>
<td style="text-align:right;">
42574
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000100628
</td>
<td style="text-align:left;">
ASB2
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
ankyrin repeat and SOCS box containing 2 [Source:HGNC Symbol;Acc:HGNC:16012]
</td>
<td style="text-align:left;">
ENSG00000100628.12
</td>
<td style="text-align:left;">
ENST00000555019
</td>
<td style="text-align:left;">
51676
</td>
<td style="text-align:right;">
-3.259348
</td>
<td style="text-align:right;">
-3.261319
</td>
<td style="text-align:right;">
3.2479347
</td>
<td style="text-align:right;">
7.743555e-17
</td>
<td style="text-align:right;">
1.368957e-14
</td>
<td style="text-align:left;">
14
</td>
<td style="text-align:right;">
93934166
</td>
<td style="text-align:right;">
93976739
</td>
<td style="text-align:right;">
42574
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000100628
</td>
<td style="text-align:left;">
ASB2
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
ankyrin repeat and SOCS box containing 2 [Source:HGNC Symbol;Acc:HGNC:16012]
</td>
<td style="text-align:left;">
ENSG00000100628.12
</td>
<td style="text-align:left;">
ENST00000555019
</td>
<td style="text-align:left;">
51676
</td>
<td style="text-align:right;">
-2.741608
</td>
<td style="text-align:right;">
-2.742919
</td>
<td style="text-align:right;">
3.2479347
</td>
<td style="text-align:right;">
1.758141e-13
</td>
<td style="text-align:right;">
6.300321e-11
</td>
</tr>
<tr>
<td style="text-align:left;">
SLCO2B1
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:right;">
75100563
</td>
<td style="text-align:right;">
75206549
</td>
<td style="text-align:right;">
105987
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000137491
</td>
<td style="text-align:left;">
SLCO2B1
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
solute carrier organic anion transporter family member 2B1 [Source:HGNC Symbol;Acc:HGNC:10962]
</td>
<td style="text-align:left;">
ENSG00000137491.15
</td>
<td style="text-align:left;">
ENST00000289575
</td>
<td style="text-align:left;">
11309
</td>
<td style="text-align:right;">
-6.272838
</td>
<td style="text-align:right;">
-6.328333
</td>
<td style="text-align:right;">
1.1269140
</td>
<td style="text-align:right;">
4.462780e-19
</td>
<td style="text-align:right;">
4.551692e-16
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:right;">
75100563
</td>
<td style="text-align:right;">
75206549
</td>
<td style="text-align:right;">
105987
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000137491
</td>
<td style="text-align:left;">
SLCO2B1
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
solute carrier organic anion transporter family member 2B1 [Source:HGNC Symbol;Acc:HGNC:10962]
</td>
<td style="text-align:left;">
ENSG00000137491.15
</td>
<td style="text-align:left;">
ENST00000289575
</td>
<td style="text-align:left;">
11309
</td>
<td style="text-align:right;">
-4.885934
</td>
<td style="text-align:right;">
-4.906461
</td>
<td style="text-align:right;">
1.1269140
</td>
<td style="text-align:right;">
5.134357e-17
</td>
<td style="text-align:right;">
2.836518e-14
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:right;">
75100563
</td>
<td style="text-align:right;">
75206549
</td>
<td style="text-align:right;">
105987
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000137491
</td>
<td style="text-align:left;">
SLCO2B1
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
solute carrier organic anion transporter family member 2B1 [Source:HGNC Symbol;Acc:HGNC:10962]
</td>
<td style="text-align:left;">
ENSG00000137491.15
</td>
<td style="text-align:left;">
ENST00000289575
</td>
<td style="text-align:left;">
11309
</td>
<td style="text-align:right;">
-5.129920
</td>
<td style="text-align:right;">
-5.154404
</td>
<td style="text-align:right;">
1.1269140
</td>
<td style="text-align:right;">
1.319661e-17
</td>
<td style="text-align:right;">
1.249813e-14
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:right;">
75100563
</td>
<td style="text-align:right;">
75206549
</td>
<td style="text-align:right;">
105987
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000137491
</td>
<td style="text-align:left;">
SLCO2B1
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
solute carrier organic anion transporter family member 2B1 [Source:HGNC Symbol;Acc:HGNC:10962]
</td>
<td style="text-align:left;">
ENSG00000137491.15
</td>
<td style="text-align:left;">
ENST00000289575
</td>
<td style="text-align:left;">
11309
</td>
<td style="text-align:right;">
-5.478410
</td>
<td style="text-align:right;">
-5.509832
</td>
<td style="text-align:right;">
1.1269140
</td>
<td style="text-align:right;">
8.381793e-18
</td>
<td style="text-align:right;">
1.852237e-15
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:right;">
75100563
</td>
<td style="text-align:right;">
75206549
</td>
<td style="text-align:right;">
105987
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000137491
</td>
<td style="text-align:left;">
SLCO2B1
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
solute carrier organic anion transporter family member 2B1 [Source:HGNC Symbol;Acc:HGNC:10962]
</td>
<td style="text-align:left;">
ENSG00000137491.15
</td>
<td style="text-align:left;">
ENST00000289575
</td>
<td style="text-align:left;">
11309
</td>
<td style="text-align:right;">
-6.020703
</td>
<td style="text-align:right;">
-6.067078
</td>
<td style="text-align:right;">
1.1269140
</td>
<td style="text-align:right;">
1.869676e-18
</td>
<td style="text-align:right;">
1.829289e-15
</td>
</tr>
<tr>
<td style="text-align:left;">
RSAD2
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:right;">
6865557
</td>
<td style="text-align:right;">
6898239
</td>
<td style="text-align:right;">
32683
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000134321
</td>
<td style="text-align:left;">
RSAD2
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
radical S-adenosyl methionine domain containing 2 [Source:HGNC Symbol;Acc:HGNC:30908]
</td>
<td style="text-align:left;">
ENSG00000134321.13
</td>
<td style="text-align:left;">
ENST00000382040
</td>
<td style="text-align:left;">
91543
</td>
<td style="text-align:right;">
3.316251
</td>
<td style="text-align:right;">
3.320304
</td>
<td style="text-align:right;">
3.6629646
</td>
<td style="text-align:right;">
7.550943e-18
</td>
<td style="text-align:right;">
7.151283e-15
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:right;">
6865557
</td>
<td style="text-align:right;">
6898239
</td>
<td style="text-align:right;">
32683
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000134321
</td>
<td style="text-align:left;">
RSAD2
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
radical S-adenosyl methionine domain containing 2 [Source:HGNC Symbol;Acc:HGNC:30908]
</td>
<td style="text-align:left;">
ENSG00000134321.13
</td>
<td style="text-align:left;">
ENST00000382040
</td>
<td style="text-align:left;">
91543
</td>
<td style="text-align:right;">
2.131068
</td>
<td style="text-align:right;">
2.134546
</td>
<td style="text-align:right;">
3.6629646
</td>
<td style="text-align:right;">
4.241780e-06
</td>
<td style="text-align:right;">
2.615896e-04
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:right;">
6865557
</td>
<td style="text-align:right;">
6898239
</td>
<td style="text-align:right;">
32683
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000134321
</td>
<td style="text-align:left;">
RSAD2
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
radical S-adenosyl methionine domain containing 2 [Source:HGNC Symbol;Acc:HGNC:30908]
</td>
<td style="text-align:left;">
ENSG00000134321.13
</td>
<td style="text-align:left;">
ENST00000382040
</td>
<td style="text-align:left;">
91543
</td>
<td style="text-align:right;">
2.380426
</td>
<td style="text-align:right;">
2.384066
</td>
<td style="text-align:right;">
3.6629646
</td>
<td style="text-align:right;">
8.643369e-09
</td>
<td style="text-align:right;">
8.365140e-07
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:right;">
6865557
</td>
<td style="text-align:right;">
6898239
</td>
<td style="text-align:right;">
32683
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000134321
</td>
<td style="text-align:left;">
RSAD2
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
radical S-adenosyl methionine domain containing 2 [Source:HGNC Symbol;Acc:HGNC:30908]
</td>
<td style="text-align:left;">
ENSG00000134321.13
</td>
<td style="text-align:left;">
ENST00000382040
</td>
<td style="text-align:left;">
91543
</td>
<td style="text-align:right;">
4.014211
</td>
<td style="text-align:right;">
4.018437
</td>
<td style="text-align:right;">
3.6629646
</td>
<td style="text-align:right;">
6.001416e-23
</td>
<td style="text-align:right;">
4.973298e-20
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:right;">
6865557
</td>
<td style="text-align:right;">
6898239
</td>
<td style="text-align:right;">
32683
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000134321
</td>
<td style="text-align:left;">
RSAD2
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
radical S-adenosyl methionine domain containing 2 [Source:HGNC Symbol;Acc:HGNC:30908]
</td>
<td style="text-align:left;">
ENSG00000134321.13
</td>
<td style="text-align:left;">
ENST00000382040
</td>
<td style="text-align:left;">
91543
</td>
<td style="text-align:right;">
2.299907
</td>
<td style="text-align:right;">
2.303498
</td>
<td style="text-align:right;">
3.6629646
</td>
<td style="text-align:right;">
7.041832e-08
</td>
<td style="text-align:right;">
6.023719e-06
</td>
</tr>
<tr>
<td style="text-align:left;">
FER1L4
</td>
<td style="text-align:left;">
20
</td>
<td style="text-align:right;">
35558737
</td>
<td style="text-align:right;">
35607562
</td>
<td style="text-align:right;">
48826
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000088340
</td>
<td style="text-align:left;">
FER1L4
</td>
<td style="text-align:left;">
transcribed_unitary_pseudogene
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
fer-1 like family member 4 (pseudogene) [Source:HGNC Symbol;Acc:HGNC:15801]
</td>
<td style="text-align:left;">
ENSG00000088340.17
</td>
<td style="text-align:left;">
ENST00000615531
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
-3.031072
</td>
<td style="text-align:right;">
-3.032139
</td>
<td style="text-align:right;">
4.1105526
</td>
<td style="text-align:right;">
2.918322e-17
</td>
<td style="text-align:right;">
2.418377e-14
</td>
<td style="text-align:left;">
20
</td>
<td style="text-align:right;">
35558737
</td>
<td style="text-align:right;">
35607562
</td>
<td style="text-align:right;">
48826
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000088340
</td>
<td style="text-align:left;">
FER1L4
</td>
<td style="text-align:left;">
transcribed_unitary_pseudogene
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
fer-1 like family member 4 (pseudogene) [Source:HGNC Symbol;Acc:HGNC:15801]
</td>
<td style="text-align:left;">
ENSG00000088340.17
</td>
<td style="text-align:left;">
ENST00000615531
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
-2.182715
</td>
<td style="text-align:right;">
-2.183242
</td>
<td style="text-align:right;">
4.1105526
</td>
<td style="text-align:right;">
2.096171e-09
</td>
<td style="text-align:right;">
2.416794e-07
</td>
<td style="text-align:left;">
20
</td>
<td style="text-align:right;">
35558737
</td>
<td style="text-align:right;">
35607562
</td>
<td style="text-align:right;">
48826
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000088340
</td>
<td style="text-align:left;">
FER1L4
</td>
<td style="text-align:left;">
transcribed_unitary_pseudogene
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
fer-1 like family member 4 (pseudogene) [Source:HGNC Symbol;Acc:HGNC:15801]
</td>
<td style="text-align:left;">
ENSG00000088340.17
</td>
<td style="text-align:left;">
ENST00000615531
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
-2.569512
</td>
<td style="text-align:right;">
-2.570247
</td>
<td style="text-align:right;">
4.1105526
</td>
<td style="text-align:right;">
1.310740e-13
</td>
<td style="text-align:right;">
3.862023e-11
</td>
<td style="text-align:left;">
20
</td>
<td style="text-align:right;">
35558737
</td>
<td style="text-align:right;">
35607562
</td>
<td style="text-align:right;">
48826
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000088340
</td>
<td style="text-align:left;">
FER1L4
</td>
<td style="text-align:left;">
transcribed_unitary_pseudogene
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
fer-1 like family member 4 (pseudogene) [Source:HGNC Symbol;Acc:HGNC:15801]
</td>
<td style="text-align:left;">
ENSG00000088340.17
</td>
<td style="text-align:left;">
ENST00000615531
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
-2.412069
</td>
<td style="text-align:right;">
-2.412710
</td>
<td style="text-align:right;">
4.1105526
</td>
<td style="text-align:right;">
6.229675e-12
</td>
<td style="text-align:right;">
4.916623e-10
</td>
<td style="text-align:left;">
20
</td>
<td style="text-align:right;">
35558737
</td>
<td style="text-align:right;">
35607562
</td>
<td style="text-align:right;">
48826
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000088340
</td>
<td style="text-align:left;">
FER1L4
</td>
<td style="text-align:left;">
transcribed_unitary_pseudogene
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
fer-1 like family member 4 (pseudogene) [Source:HGNC Symbol;Acc:HGNC:15801]
</td>
<td style="text-align:left;">
ENSG00000088340.17
</td>
<td style="text-align:left;">
ENST00000615531
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
-2.403608
</td>
<td style="text-align:right;">
-2.404246
</td>
<td style="text-align:right;">
4.1105526
</td>
<td style="text-align:right;">
7.417596e-12
</td>
<td style="text-align:right;">
1.725437e-09
</td>
</tr>
<tr>
<td style="text-align:left;">
KCNQ5
</td>
<td style="text-align:left;">
6
</td>
<td style="text-align:right;">
72621792
</td>
<td style="text-align:right;">
73198853
</td>
<td style="text-align:right;">
577062
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000185760
</td>
<td style="text-align:left;">
KCNQ5
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
potassium voltage-gated channel subfamily Q member 5 [Source:HGNC Symbol;Acc:HGNC:6299]
</td>
<td style="text-align:left;">
ENSG00000185760.17
</td>
<td style="text-align:left;">
ENST00000370398
</td>
<td style="text-align:left;">
56479
</td>
<td style="text-align:right;">
2.955595
</td>
<td style="text-align:right;">
2.957365
</td>
<td style="text-align:right;">
4.5890088
</td>
<td style="text-align:right;">
7.683113e-17
</td>
<td style="text-align:right;">
5.992376e-14
</td>
<td style="text-align:left;">
6
</td>
<td style="text-align:right;">
72621792
</td>
<td style="text-align:right;">
73198853
</td>
<td style="text-align:right;">
577062
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000185760
</td>
<td style="text-align:left;">
KCNQ5
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
potassium voltage-gated channel subfamily Q member 5 [Source:HGNC Symbol;Acc:HGNC:6299]
</td>
<td style="text-align:left;">
ENSG00000185760.17
</td>
<td style="text-align:left;">
ENST00000370398
</td>
<td style="text-align:left;">
56479
</td>
<td style="text-align:right;">
2.703420
</td>
<td style="text-align:right;">
2.705139
</td>
<td style="text-align:right;">
4.5890088
</td>
<td style="text-align:right;">
3.205488e-14
</td>
<td style="text-align:right;">
1.036623e-11
</td>
<td style="text-align:left;">
6
</td>
<td style="text-align:right;">
72621792
</td>
<td style="text-align:right;">
73198853
</td>
<td style="text-align:right;">
577062
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000185760
</td>
<td style="text-align:left;">
KCNQ5
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
potassium voltage-gated channel subfamily Q member 5 [Source:HGNC Symbol;Acc:HGNC:6299]
</td>
<td style="text-align:left;">
ENSG00000185760.17
</td>
<td style="text-align:left;">
ENST00000370398
</td>
<td style="text-align:left;">
56479
</td>
<td style="text-align:right;">
2.881946
</td>
<td style="text-align:right;">
2.883701
</td>
<td style="text-align:right;">
4.5890088
</td>
<td style="text-align:right;">
4.239327e-16
</td>
<td style="text-align:right;">
2.342051e-13
</td>
<td style="text-align:left;">
6
</td>
<td style="text-align:right;">
72621792
</td>
<td style="text-align:right;">
73198853
</td>
<td style="text-align:right;">
577062
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000185760
</td>
<td style="text-align:left;">
KCNQ5
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
potassium voltage-gated channel subfamily Q member 5 [Source:HGNC Symbol;Acc:HGNC:6299]
</td>
<td style="text-align:left;">
ENSG00000185760.17
</td>
<td style="text-align:left;">
ENST00000370398
</td>
<td style="text-align:left;">
56479
</td>
<td style="text-align:right;">
2.520910
</td>
<td style="text-align:right;">
2.522586
</td>
<td style="text-align:right;">
4.5890088
</td>
<td style="text-align:right;">
3.716616e-12
</td>
<td style="text-align:right;">
3.079913e-10
</td>
<td style="text-align:left;">
6
</td>
<td style="text-align:right;">
72621792
</td>
<td style="text-align:right;">
73198853
</td>
<td style="text-align:right;">
577062
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000185760
</td>
<td style="text-align:left;">
KCNQ5
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
potassium voltage-gated channel subfamily Q member 5 [Source:HGNC Symbol;Acc:HGNC:6299]
</td>
<td style="text-align:left;">
ENSG00000185760.17
</td>
<td style="text-align:left;">
ENST00000370398
</td>
<td style="text-align:left;">
56479
</td>
<td style="text-align:right;">
2.877144
</td>
<td style="text-align:right;">
2.878899
</td>
<td style="text-align:right;">
4.5890088
</td>
<td style="text-align:right;">
5.279250e-16
</td>
<td style="text-align:right;">
3.181708e-13
</td>
</tr>
<tr>
<td style="text-align:left;">
TM7SF2
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:right;">
65111845
</td>
<td style="text-align:right;">
65116384
</td>
<td style="text-align:right;">
4540
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000149809
</td>
<td style="text-align:left;">
TM7SF2
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
transmembrane 7 superfamily member 2 [Source:HGNC Symbol;Acc:HGNC:11863]
</td>
<td style="text-align:left;">
ENSG00000149809.17
</td>
<td style="text-align:left;">
ENST00000279263
</td>
<td style="text-align:left;">
7108
</td>
<td style="text-align:right;">
-2.390531
</td>
<td style="text-align:right;">
-2.390914
</td>
<td style="text-align:right;">
4.8228286
</td>
<td style="text-align:right;">
6.970511e-16
</td>
<td style="text-align:right;">
4.621100e-13
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:right;">
65111845
</td>
<td style="text-align:right;">
65116384
</td>
<td style="text-align:right;">
4540
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000149809
</td>
<td style="text-align:left;">
TM7SF2
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
transmembrane 7 superfamily member 2 [Source:HGNC Symbol;Acc:HGNC:11863]
</td>
<td style="text-align:left;">
ENSG00000149809.17
</td>
<td style="text-align:left;">
ENST00000279263
</td>
<td style="text-align:left;">
7108
</td>
<td style="text-align:right;">
-2.232172
</td>
<td style="text-align:right;">
-2.232506
</td>
<td style="text-align:right;">
4.8228286
</td>
<td style="text-align:right;">
1.376733e-13
</td>
<td style="text-align:right;">
3.802939e-11
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:right;">
65111845
</td>
<td style="text-align:right;">
65116384
</td>
<td style="text-align:right;">
4540
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000149809
</td>
<td style="text-align:left;">
TM7SF2
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
transmembrane 7 superfamily member 2 [Source:HGNC Symbol;Acc:HGNC:11863]
</td>
<td style="text-align:left;">
ENSG00000149809.17
</td>
<td style="text-align:left;">
ENST00000279263
</td>
<td style="text-align:left;">
7108
</td>
<td style="text-align:right;">
-2.625352
</td>
<td style="text-align:right;">
-2.625818
</td>
<td style="text-align:right;">
4.8228286
</td>
<td style="text-align:right;">
1.557373e-18
</td>
<td style="text-align:right;">
1.877200e-15
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:right;">
65111845
</td>
<td style="text-align:right;">
65116384
</td>
<td style="text-align:right;">
4540
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000149809
</td>
<td style="text-align:left;">
TM7SF2
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
transmembrane 7 superfamily member 2 [Source:HGNC Symbol;Acc:HGNC:11863]
</td>
<td style="text-align:left;">
ENSG00000149809.17
</td>
<td style="text-align:left;">
ENST00000279263
</td>
<td style="text-align:left;">
7108
</td>
<td style="text-align:right;">
-2.902769
</td>
<td style="text-align:right;">
-2.903352
</td>
<td style="text-align:right;">
4.8228286
</td>
<td style="text-align:right;">
6.372912e-21
</td>
<td style="text-align:right;">
3.249940e-18
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:right;">
65111845
</td>
<td style="text-align:right;">
65116384
</td>
<td style="text-align:right;">
4540
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000149809
</td>
<td style="text-align:left;">
TM7SF2
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
transmembrane 7 superfamily member 2 [Source:HGNC Symbol;Acc:HGNC:11863]
</td>
<td style="text-align:left;">
ENSG00000149809.17
</td>
<td style="text-align:left;">
ENST00000279263
</td>
<td style="text-align:left;">
7108
</td>
<td style="text-align:right;">
-2.513387
</td>
<td style="text-align:right;">
-2.513811
</td>
<td style="text-align:right;">
4.8228286
</td>
<td style="text-align:right;">
3.439685e-17
</td>
<td style="text-align:right;">
2.682752e-14
</td>
</tr>
<tr>
<td style="text-align:left;">
C6orf132
</td>
<td style="text-align:left;">
6
</td>
<td style="text-align:right;">
42101119
</td>
<td style="text-align:right;">
42142620
</td>
<td style="text-align:right;">
41502
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000188112
</td>
<td style="text-align:left;">
C6orf132
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
chromosome 6 open reading frame 132 [Source:HGNC Symbol;Acc:HGNC:21288]
</td>
<td style="text-align:left;">
ENSG00000188112.9
</td>
<td style="text-align:left;">
ENST00000341865
</td>
<td style="text-align:left;">
647024
</td>
<td style="text-align:right;">
2.857328
</td>
<td style="text-align:right;">
2.857709
</td>
<td style="text-align:right;">
6.4909086
</td>
<td style="text-align:right;">
2.297622e-15
</td>
<td style="text-align:right;">
1.384735e-12
</td>
<td style="text-align:left;">
6
</td>
<td style="text-align:right;">
42101119
</td>
<td style="text-align:right;">
42142620
</td>
<td style="text-align:right;">
41502
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000188112
</td>
<td style="text-align:left;">
C6orf132
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
chromosome 6 open reading frame 132 [Source:HGNC Symbol;Acc:HGNC:21288]
</td>
<td style="text-align:left;">
ENSG00000188112.9
</td>
<td style="text-align:left;">
ENST00000341865
</td>
<td style="text-align:left;">
647024
</td>
<td style="text-align:right;">
2.301604
</td>
<td style="text-align:right;">
2.301957
</td>
<td style="text-align:right;">
6.4909086
</td>
<td style="text-align:right;">
2.111561e-10
</td>
<td style="text-align:right;">
2.947072e-08
</td>
<td style="text-align:left;">
6
</td>
<td style="text-align:right;">
42101119
</td>
<td style="text-align:right;">
42142620
</td>
<td style="text-align:right;">
41502
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000188112
</td>
<td style="text-align:left;">
C6orf132
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
chromosome 6 open reading frame 132 [Source:HGNC Symbol;Acc:HGNC:21288]
</td>
<td style="text-align:left;">
ENSG00000188112.9
</td>
<td style="text-align:left;">
ENST00000341865
</td>
<td style="text-align:left;">
647024
</td>
<td style="text-align:right;">
2.310534
</td>
<td style="text-align:right;">
2.310887
</td>
<td style="text-align:right;">
6.4909086
</td>
<td style="text-align:right;">
1.642099e-10
</td>
<td style="text-align:right;">
2.113844e-08
</td>
<td style="text-align:left;">
6
</td>
<td style="text-align:right;">
42101119
</td>
<td style="text-align:right;">
42142620
</td>
<td style="text-align:right;">
41502
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000188112
</td>
<td style="text-align:left;">
C6orf132
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
chromosome 6 open reading frame 132 [Source:HGNC Symbol;Acc:HGNC:21288]
</td>
<td style="text-align:left;">
ENSG00000188112.9
</td>
<td style="text-align:left;">
ENST00000341865
</td>
<td style="text-align:left;">
647024
</td>
<td style="text-align:right;">
2.452535
</td>
<td style="text-align:right;">
2.452897
</td>
<td style="text-align:right;">
6.4909086
</td>
<td style="text-align:right;">
6.334594e-12
</td>
<td style="text-align:right;">
4.969845e-10
</td>
<td style="text-align:left;">
6
</td>
<td style="text-align:right;">
42101119
</td>
<td style="text-align:right;">
42142620
</td>
<td style="text-align:right;">
41502
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000188112
</td>
<td style="text-align:left;">
C6orf132
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
chromosome 6 open reading frame 132 [Source:HGNC Symbol;Acc:HGNC:21288]
</td>
<td style="text-align:left;">
ENSG00000188112.9
</td>
<td style="text-align:left;">
ENST00000341865
</td>
<td style="text-align:left;">
647024
</td>
<td style="text-align:right;">
2.488830
</td>
<td style="text-align:right;">
2.489194
</td>
<td style="text-align:right;">
6.4909086
</td>
<td style="text-align:right;">
2.845581e-12
</td>
<td style="text-align:right;">
7.468557e-10
</td>
</tr>
<tr>
<td style="text-align:left;">
TNC
</td>
<td style="text-align:left;">
9
</td>
<td style="text-align:right;">
115019575
</td>
<td style="text-align:right;">
115118257
</td>
<td style="text-align:right;">
98683
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000041982
</td>
<td style="text-align:left;">
TNC
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
tenascin C [Source:HGNC Symbol;Acc:HGNC:5318]
</td>
<td style="text-align:left;">
ENSG00000041982.16
</td>
<td style="text-align:left;">
ENST00000350763
</td>
<td style="text-align:left;">
3371
</td>
<td style="text-align:right;">
2.310323
</td>
<td style="text-align:right;">
2.310815
</td>
<td style="text-align:right;">
5.6044356
</td>
<td style="text-align:right;">
1.531323e-14
</td>
<td style="text-align:right;">
7.519928e-12
</td>
<td style="text-align:left;">
9
</td>
<td style="text-align:right;">
115019575
</td>
<td style="text-align:right;">
115118257
</td>
<td style="text-align:right;">
98683
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000041982
</td>
<td style="text-align:left;">
TNC
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
tenascin C [Source:HGNC Symbol;Acc:HGNC:5318]
</td>
<td style="text-align:left;">
ENSG00000041982.16
</td>
<td style="text-align:left;">
ENST00000350763
</td>
<td style="text-align:left;">
3371
</td>
<td style="text-align:right;">
2.007560
</td>
<td style="text-align:right;">
2.008024
</td>
<td style="text-align:right;">
5.6044356
</td>
<td style="text-align:right;">
2.299802e-09
</td>
<td style="text-align:right;">
2.628714e-07
</td>
<td style="text-align:left;">
9
</td>
<td style="text-align:right;">
115019575
</td>
<td style="text-align:right;">
115118257
</td>
<td style="text-align:right;">
98683
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000041982
</td>
<td style="text-align:left;">
TNC
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
tenascin C [Source:HGNC Symbol;Acc:HGNC:5318]
</td>
<td style="text-align:left;">
ENSG00000041982.16
</td>
<td style="text-align:left;">
ENST00000350763
</td>
<td style="text-align:left;">
3371
</td>
<td style="text-align:right;">
1.895754
</td>
<td style="text-align:right;">
1.896205
</td>
<td style="text-align:right;">
5.6044356
</td>
<td style="text-align:right;">
2.836624e-07
</td>
<td style="text-align:right;">
2.124904e-05
</td>
<td style="text-align:left;">
9
</td>
<td style="text-align:right;">
115019575
</td>
<td style="text-align:right;">
115118257
</td>
<td style="text-align:right;">
98683
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000041982
</td>
<td style="text-align:left;">
TNC
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
tenascin C [Source:HGNC Symbol;Acc:HGNC:5318]
</td>
<td style="text-align:left;">
ENSG00000041982.16
</td>
<td style="text-align:left;">
ENST00000350763
</td>
<td style="text-align:left;">
3371
</td>
<td style="text-align:right;">
2.099495
</td>
<td style="text-align:right;">
2.099968
</td>
<td style="text-align:right;">
5.6044356
</td>
<td style="text-align:right;">
5.180765e-11
</td>
<td style="text-align:right;">
3.417501e-09
</td>
<td style="text-align:left;">
9
</td>
<td style="text-align:right;">
115019575
</td>
<td style="text-align:right;">
115118257
</td>
<td style="text-align:right;">
98683
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000041982
</td>
<td style="text-align:left;">
TNC
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
tenascin C [Source:HGNC Symbol;Acc:HGNC:5318]
</td>
<td style="text-align:left;">
ENSG00000041982.16
</td>
<td style="text-align:left;">
ENST00000350763
</td>
<td style="text-align:left;">
3371
</td>
<td style="text-align:right;">
2.012420
</td>
<td style="text-align:right;">
2.012884
</td>
<td style="text-align:right;">
5.6044356
</td>
<td style="text-align:right;">
1.922296e-09
</td>
<td style="text-align:right;">
2.404255e-07
</td>
</tr>
<tr>
<td style="text-align:left;">
F2RL3
</td>
<td style="text-align:left;">
19
</td>
<td style="text-align:right;">
16888999
</td>
<td style="text-align:right;">
16892606
</td>
<td style="text-align:right;">
3608
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000127533
</td>
<td style="text-align:left;">
F2RL3
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
F2R like thrombin or trypsin receptor 3 [Source:HGNC Symbol;Acc:HGNC:3540]
</td>
<td style="text-align:left;">
ENSG00000127533.4
</td>
<td style="text-align:left;">
ENST00000248076
</td>
<td style="text-align:left;">
9002
</td>
<td style="text-align:right;">
-2.884927
</td>
<td style="text-align:right;">
-2.886878
</td>
<td style="text-align:right;">
2.5172859
</td>
<td style="text-align:right;">
3.856177e-14
</td>
<td style="text-align:right;">
1.763071e-11
</td>
<td style="text-align:left;">
19
</td>
<td style="text-align:right;">
16888999
</td>
<td style="text-align:right;">
16892606
</td>
<td style="text-align:right;">
3608
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000127533
</td>
<td style="text-align:left;">
F2RL3
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
F2R like thrombin or trypsin receptor 3 [Source:HGNC Symbol;Acc:HGNC:3540]
</td>
<td style="text-align:left;">
ENSG00000127533.4
</td>
<td style="text-align:left;">
ENST00000248076
</td>
<td style="text-align:left;">
9002
</td>
<td style="text-align:right;">
-5.208385
</td>
<td style="text-align:right;">
-5.219416
</td>
<td style="text-align:right;">
2.5172859
</td>
<td style="text-align:right;">
2.161588e-22
</td>
<td style="text-align:right;">
4.776750e-19
</td>
<td style="text-align:left;">
19
</td>
<td style="text-align:right;">
16888999
</td>
<td style="text-align:right;">
16892606
</td>
<td style="text-align:right;">
3608
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000127533
</td>
<td style="text-align:left;">
F2RL3
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
F2R like thrombin or trypsin receptor 3 [Source:HGNC Symbol;Acc:HGNC:3540]
</td>
<td style="text-align:left;">
ENSG00000127533.4
</td>
<td style="text-align:left;">
ENST00000248076
</td>
<td style="text-align:left;">
9002
</td>
<td style="text-align:right;">
-4.996314
</td>
<td style="text-align:right;">
-5.005787
</td>
<td style="text-align:right;">
2.5172859
</td>
<td style="text-align:right;">
3.493425e-22
</td>
<td style="text-align:right;">
9.263864e-19
</td>
<td style="text-align:left;">
19
</td>
<td style="text-align:right;">
16888999
</td>
<td style="text-align:right;">
16892606
</td>
<td style="text-align:right;">
3608
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000127533
</td>
<td style="text-align:left;">
F2RL3
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
F2R like thrombin or trypsin receptor 3 [Source:HGNC Symbol;Acc:HGNC:3540]
</td>
<td style="text-align:left;">
ENSG00000127533.4
</td>
<td style="text-align:left;">
ENST00000248076
</td>
<td style="text-align:left;">
9002
</td>
<td style="text-align:right;">
-4.555775
</td>
<td style="text-align:right;">
-4.562673
</td>
<td style="text-align:right;">
2.5172859
</td>
<td style="text-align:right;">
6.171375e-21
</td>
<td style="text-align:right;">
3.249940e-18
</td>
<td style="text-align:left;">
19
</td>
<td style="text-align:right;">
16888999
</td>
<td style="text-align:right;">
16892606
</td>
<td style="text-align:right;">
3608
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000127533
</td>
<td style="text-align:left;">
F2RL3
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
F2R like thrombin or trypsin receptor 3 [Source:HGNC Symbol;Acc:HGNC:3540]
</td>
<td style="text-align:left;">
ENSG00000127533.4
</td>
<td style="text-align:left;">
ENST00000248076
</td>
<td style="text-align:left;">
9002
</td>
<td style="text-align:right;">
-4.763857
</td>
<td style="text-align:right;">
-4.771884
</td>
<td style="text-align:right;">
2.5172859
</td>
<td style="text-align:right;">
1.893250e-21
</td>
<td style="text-align:right;">
6.275650e-18
</td>
</tr>
<tr>
<td style="text-align:left;">
DUSP13
</td>
<td style="text-align:left;">
10
</td>
<td style="text-align:right;">
75094432
</td>
<td style="text-align:right;">
75109221
</td>
<td style="text-align:right;">
14790
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000079393
</td>
<td style="text-align:left;">
DUSP13
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
dual specificity phosphatase 13 [Source:HGNC Symbol;Acc:HGNC:19681]
</td>
<td style="text-align:left;">
ENSG00000079393.21
</td>
<td style="text-align:left;">
ENST00000478873
</td>
<td style="text-align:left;">
51207
</td>
<td style="text-align:right;">
-4.486472
</td>
<td style="text-align:right;">
-4.520295
</td>
<td style="text-align:right;">
0.5431086
</td>
<td style="text-align:right;">
1.668553e-12
</td>
<td style="text-align:right;">
5.267464e-10
</td>
<td style="text-align:left;">
10
</td>
<td style="text-align:right;">
75094432
</td>
<td style="text-align:right;">
75109221
</td>
<td style="text-align:right;">
14790
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000079393
</td>
<td style="text-align:left;">
DUSP13
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
dual specificity phosphatase 13 [Source:HGNC Symbol;Acc:HGNC:19681]
</td>
<td style="text-align:left;">
ENSG00000079393.21
</td>
<td style="text-align:left;">
ENST00000478873
</td>
<td style="text-align:left;">
51207
</td>
<td style="text-align:right;">
-2.263955
</td>
<td style="text-align:right;">
-2.269905
</td>
<td style="text-align:right;">
0.5431086
</td>
<td style="text-align:right;">
1.298761e-04
</td>
<td style="text-align:right;">
6.128210e-03
</td>
<td style="text-align:left;">
10
</td>
<td style="text-align:right;">
75094432
</td>
<td style="text-align:right;">
75109221
</td>
<td style="text-align:right;">
14790
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000079393
</td>
<td style="text-align:left;">
DUSP13
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
dual specificity phosphatase 13 [Source:HGNC Symbol;Acc:HGNC:19681]
</td>
<td style="text-align:left;">
ENSG00000079393.21
</td>
<td style="text-align:left;">
ENST00000478873
</td>
<td style="text-align:left;">
51207
</td>
<td style="text-align:right;">
-3.820138
</td>
<td style="text-align:right;">
-3.840772
</td>
<td style="text-align:right;">
0.5431086
</td>
<td style="text-align:right;">
6.779080e-11
</td>
<td style="text-align:right;">
9.461455e-09
</td>
<td style="text-align:left;">
10
</td>
<td style="text-align:right;">
75094432
</td>
<td style="text-align:right;">
75109221
</td>
<td style="text-align:right;">
14790
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000079393
</td>
<td style="text-align:left;">
DUSP13
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
dual specificity phosphatase 13 [Source:HGNC Symbol;Acc:HGNC:19681]
</td>
<td style="text-align:left;">
ENSG00000079393.21
</td>
<td style="text-align:left;">
ENST00000478873
</td>
<td style="text-align:left;">
51207
</td>
<td style="text-align:right;">
-3.158914
</td>
<td style="text-align:right;">
-3.171336
</td>
<td style="text-align:right;">
0.5431086
</td>
<td style="text-align:right;">
1.448846e-08
</td>
<td style="text-align:right;">
6.468098e-07
</td>
<td style="text-align:left;">
10
</td>
<td style="text-align:right;">
75094432
</td>
<td style="text-align:right;">
75109221
</td>
<td style="text-align:right;">
14790
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000079393
</td>
<td style="text-align:left;">
DUSP13
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
dual specificity phosphatase 13 [Source:HGNC Symbol;Acc:HGNC:19681]
</td>
<td style="text-align:left;">
ENSG00000079393.21
</td>
<td style="text-align:left;">
ENST00000478873
</td>
<td style="text-align:left;">
51207
</td>
<td style="text-align:right;">
-2.629572
</td>
<td style="text-align:right;">
-2.637722
</td>
<td style="text-align:right;">
0.5431086
</td>
<td style="text-align:right;">
2.297146e-06
</td>
<td style="text-align:right;">
1.471394e-04
</td>
</tr>
<tr>
<td style="text-align:left;">
CXCL1
</td>
<td style="text-align:left;">
4
</td>
<td style="text-align:right;">
73869393
</td>
<td style="text-align:right;">
73871308
</td>
<td style="text-align:right;">
1916
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000163739
</td>
<td style="text-align:left;">
CXCL1
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
C-X-C motif chemokine ligand 1 [Source:HGNC Symbol;Acc:HGNC:4602]
</td>
<td style="text-align:left;">
ENSG00000163739.5
</td>
<td style="text-align:left;">
ENST00000395761
</td>
<td style="text-align:left;">
2919
</td>
<td style="text-align:right;">
2.770702
</td>
<td style="text-align:right;">
2.775106
</td>
<td style="text-align:right;">
4.2190122
</td>
<td style="text-align:right;">
3.520875e-11
</td>
<td style="text-align:right;">
9.153583e-09
</td>
<td style="text-align:left;">
4
</td>
<td style="text-align:right;">
73869393
</td>
<td style="text-align:right;">
73871308
</td>
<td style="text-align:right;">
1916
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000163739
</td>
<td style="text-align:left;">
CXCL1
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
C-X-C motif chemokine ligand 1 [Source:HGNC Symbol;Acc:HGNC:4602]
</td>
<td style="text-align:left;">
ENSG00000163739.5
</td>
<td style="text-align:left;">
ENST00000395761
</td>
<td style="text-align:left;">
2919
</td>
<td style="text-align:right;">
4.913343
</td>
<td style="text-align:right;">
4.918332
</td>
<td style="text-align:right;">
4.2190122
</td>
<td style="text-align:right;">
2.762991e-26
</td>
<td style="text-align:right;">
1.831725e-22
</td>
<td style="text-align:left;">
4
</td>
<td style="text-align:right;">
73869393
</td>
<td style="text-align:right;">
73871308
</td>
<td style="text-align:right;">
1916
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000163739
</td>
<td style="text-align:left;">
CXCL1
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
C-X-C motif chemokine ligand 1 [Source:HGNC Symbol;Acc:HGNC:4602]
</td>
<td style="text-align:left;">
ENSG00000163739.5
</td>
<td style="text-align:left;">
ENST00000395761
</td>
<td style="text-align:left;">
2919
</td>
<td style="text-align:right;">
3.355212
</td>
<td style="text-align:right;">
3.359868
</td>
<td style="text-align:right;">
4.2190122
</td>
<td style="text-align:right;">
3.265801e-16
</td>
<td style="text-align:right;">
1.882663e-13
</td>
<td style="text-align:left;">
4
</td>
<td style="text-align:right;">
73869393
</td>
<td style="text-align:right;">
73871308
</td>
<td style="text-align:right;">
1916
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000163739
</td>
<td style="text-align:left;">
CXCL1
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
C-X-C motif chemokine ligand 1 [Source:HGNC Symbol;Acc:HGNC:4602]
</td>
<td style="text-align:left;">
ENSG00000163739.5
</td>
<td style="text-align:left;">
ENST00000395761
</td>
<td style="text-align:left;">
2919
</td>
<td style="text-align:right;">
3.105230
</td>
<td style="text-align:right;">
3.109791
</td>
<td style="text-align:right;">
4.2190122
</td>
<td style="text-align:right;">
4.412935e-14
</td>
<td style="text-align:right;">
5.044060e-12
</td>
<td style="text-align:left;">
4
</td>
<td style="text-align:right;">
73869393
</td>
<td style="text-align:right;">
73871308
</td>
<td style="text-align:right;">
1916
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000163739
</td>
<td style="text-align:left;">
CXCL1
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
C-X-C motif chemokine ligand 1 [Source:HGNC Symbol;Acc:HGNC:4602]
</td>
<td style="text-align:left;">
ENSG00000163739.5
</td>
<td style="text-align:left;">
ENST00000395761
</td>
<td style="text-align:left;">
2919
</td>
<td style="text-align:right;">
3.773446
</td>
<td style="text-align:right;">
3.778228
</td>
<td style="text-align:right;">
4.2190122
</td>
<td style="text-align:right;">
2.565966e-19
</td>
<td style="text-align:right;">
3.402214e-16
</td>
</tr>
<tr>
<td style="text-align:left;">
COLEC12
</td>
<td style="text-align:left;">
18
</td>
<td style="text-align:right;">
316737
</td>
<td style="text-align:right;">
500722
</td>
<td style="text-align:right;">
183986
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000158270
</td>
<td style="text-align:left;">
COLEC12
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
collectin subfamily member 12 [Source:HGNC Symbol;Acc:HGNC:16016]
</td>
<td style="text-align:left;">
ENSG00000158270.12
</td>
<td style="text-align:left;">
ENST00000400256
</td>
<td style="text-align:left;">
81035
</td>
<td style="text-align:right;">
-3.169759
</td>
<td style="text-align:right;">
-3.177577
</td>
<td style="text-align:right;">
1.0653656
</td>
<td style="text-align:right;">
5.025309e-11
</td>
<td style="text-align:right;">
1.281357e-08
</td>
<td style="text-align:left;">
18
</td>
<td style="text-align:right;">
316737
</td>
<td style="text-align:right;">
500722
</td>
<td style="text-align:right;">
183986
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000158270
</td>
<td style="text-align:left;">
COLEC12
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
collectin subfamily member 12 [Source:HGNC Symbol;Acc:HGNC:16016]
</td>
<td style="text-align:left;">
ENSG00000158270.12
</td>
<td style="text-align:left;">
ENST00000400256
</td>
<td style="text-align:left;">
81035
</td>
<td style="text-align:right;">
-3.093399
</td>
<td style="text-align:right;">
-3.100759
</td>
<td style="text-align:right;">
1.0653656
</td>
<td style="text-align:right;">
1.791251e-10
</td>
<td style="text-align:right;">
2.668561e-08
</td>
<td style="text-align:left;">
18
</td>
<td style="text-align:right;">
316737
</td>
<td style="text-align:right;">
500722
</td>
<td style="text-align:right;">
183986
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000158270
</td>
<td style="text-align:left;">
COLEC12
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
collectin subfamily member 12 [Source:HGNC Symbol;Acc:HGNC:16016]
</td>
<td style="text-align:left;">
ENSG00000158270.12
</td>
<td style="text-align:left;">
ENST00000400256
</td>
<td style="text-align:left;">
81035
</td>
<td style="text-align:right;">
-3.692826
</td>
<td style="text-align:right;">
-3.704506
</td>
<td style="text-align:right;">
1.0653656
</td>
<td style="text-align:right;">
5.368158e-13
</td>
<td style="text-align:right;">
1.342951e-10
</td>
<td style="text-align:left;">
18
</td>
<td style="text-align:right;">
316737
</td>
<td style="text-align:right;">
500722
</td>
<td style="text-align:right;">
183986
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000158270
</td>
<td style="text-align:left;">
COLEC12
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
collectin subfamily member 12 [Source:HGNC Symbol;Acc:HGNC:16016]
</td>
<td style="text-align:left;">
ENSG00000158270.12
</td>
<td style="text-align:left;">
ENST00000400256
</td>
<td style="text-align:left;">
81035
</td>
<td style="text-align:right;">
-3.525824
</td>
<td style="text-align:right;">
-3.536109
</td>
<td style="text-align:right;">
1.0653656
</td>
<td style="text-align:right;">
3.640378e-12
</td>
<td style="text-align:right;">
3.035709e-10
</td>
<td style="text-align:left;">
18
</td>
<td style="text-align:right;">
316737
</td>
<td style="text-align:right;">
500722
</td>
<td style="text-align:right;">
183986
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000158270
</td>
<td style="text-align:left;">
COLEC12
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
collectin subfamily member 12 [Source:HGNC Symbol;Acc:HGNC:16016]
</td>
<td style="text-align:left;">
ENSG00000158270.12
</td>
<td style="text-align:left;">
ENST00000400256
</td>
<td style="text-align:left;">
81035
</td>
<td style="text-align:right;">
-4.151772
</td>
<td style="text-align:right;">
-4.168241
</td>
<td style="text-align:right;">
1.0653656
</td>
<td style="text-align:right;">
4.098610e-14
</td>
<td style="text-align:right;">
1.753015e-11
</td>
</tr>
<tr>
<td style="text-align:left;">
IL6
</td>
<td style="text-align:left;">
7
</td>
<td style="text-align:right;">
22725884
</td>
<td style="text-align:right;">
22732002
</td>
<td style="text-align:right;">
6119
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000136244
</td>
<td style="text-align:left;">
IL6
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
interleukin 6 [Source:HGNC Symbol;Acc:HGNC:6018]
</td>
<td style="text-align:left;">
ENSG00000136244.12
</td>
<td style="text-align:left;">
ENST00000258743
</td>
<td style="text-align:left;">
3569
</td>
<td style="text-align:right;">
2.181842
</td>
<td style="text-align:right;">
2.182874
</td>
<td style="text-align:right;">
6.1315177
</td>
<td style="text-align:right;">
7.647256e-11
</td>
<td style="text-align:right;">
1.877685e-08
</td>
<td style="text-align:left;">
7
</td>
<td style="text-align:right;">
22725884
</td>
<td style="text-align:right;">
22732002
</td>
<td style="text-align:right;">
6119
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000136244
</td>
<td style="text-align:left;">
IL6
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
interleukin 6 [Source:HGNC Symbol;Acc:HGNC:6018]
</td>
<td style="text-align:left;">
ENSG00000136244.12
</td>
<td style="text-align:left;">
ENST00000258743
</td>
<td style="text-align:left;">
3569
</td>
<td style="text-align:right;">
4.866167
</td>
<td style="text-align:right;">
4.867445
</td>
<td style="text-align:right;">
6.1315177
</td>
<td style="text-align:right;">
7.414143e-34
</td>
<td style="text-align:right;">
9.830412e-30
</td>
<td style="text-align:left;">
7
</td>
<td style="text-align:right;">
22725884
</td>
<td style="text-align:right;">
22732002
</td>
<td style="text-align:right;">
6119
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000136244
</td>
<td style="text-align:left;">
IL6
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
interleukin 6 [Source:HGNC Symbol;Acc:HGNC:6018]
</td>
<td style="text-align:left;">
ENSG00000136244.12
</td>
<td style="text-align:left;">
ENST00000258743
</td>
<td style="text-align:left;">
3569
</td>
<td style="text-align:right;">
2.611423
</td>
<td style="text-align:right;">
2.612530
</td>
<td style="text-align:right;">
6.1315177
</td>
<td style="text-align:right;">
7.672616e-17
</td>
<td style="text-align:right;">
6.358201e-14
</td>
<td style="text-align:left;">
7
</td>
<td style="text-align:right;">
22725884
</td>
<td style="text-align:right;">
22732002
</td>
<td style="text-align:right;">
6119
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000136244
</td>
<td style="text-align:left;">
IL6
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
interleukin 6 [Source:HGNC Symbol;Acc:HGNC:6018]
</td>
<td style="text-align:left;">
ENSG00000136244.12
</td>
<td style="text-align:left;">
ENST00000258743
</td>
<td style="text-align:left;">
3569
</td>
<td style="text-align:right;">
4.097265
</td>
<td style="text-align:right;">
4.098511
</td>
<td style="text-align:right;">
6.1315177
</td>
<td style="text-align:right;">
1.061108e-29
</td>
<td style="text-align:right;">
1.396151e-25
</td>
<td style="text-align:left;">
7
</td>
<td style="text-align:right;">
22725884
</td>
<td style="text-align:right;">
22732002
</td>
<td style="text-align:right;">
6119
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000136244
</td>
<td style="text-align:left;">
IL6
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
interleukin 6 [Source:HGNC Symbol;Acc:HGNC:6018]
</td>
<td style="text-align:left;">
ENSG00000136244.12
</td>
<td style="text-align:left;">
ENST00000258743
</td>
<td style="text-align:left;">
3569
</td>
<td style="text-align:right;">
3.377613
</td>
<td style="text-align:right;">
3.378809
</td>
<td style="text-align:right;">
6.1315177
</td>
<td style="text-align:right;">
1.508097e-24
</td>
<td style="text-align:right;">
1.999585e-20
</td>
</tr>
<tr>
<td style="text-align:left;">
ALDH3A1
</td>
<td style="text-align:left;">
17
</td>
<td style="text-align:right;">
19737984
</td>
<td style="text-align:right;">
19748943
</td>
<td style="text-align:right;">
10960
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000108602
</td>
<td style="text-align:left;">
ALDH3A1
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
aldehyde dehydrogenase 3 family member A1 [Source:HGNC Symbol;Acc:HGNC:405]
</td>
<td style="text-align:left;">
ENSG00000108602.18
</td>
<td style="text-align:left;">
ENST00000225740
</td>
<td style="text-align:left;">
218
</td>
<td style="text-align:right;">
-3.857799
</td>
<td style="text-align:right;">
-3.892943
</td>
<td style="text-align:right;">
-0.2219792
</td>
<td style="text-align:right;">
8.123435e-10
</td>
<td style="text-align:right;">
1.709661e-07
</td>
<td style="text-align:left;">
17
</td>
<td style="text-align:right;">
19737984
</td>
<td style="text-align:right;">
19748943
</td>
<td style="text-align:right;">
10960
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000108602
</td>
<td style="text-align:left;">
ALDH3A1
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
aldehyde dehydrogenase 3 family member A1 [Source:HGNC Symbol;Acc:HGNC:405]
</td>
<td style="text-align:left;">
ENSG00000108602.18
</td>
<td style="text-align:left;">
ENST00000225740
</td>
<td style="text-align:left;">
218
</td>
<td style="text-align:right;">
-3.761311
</td>
<td style="text-align:right;">
-3.793968
</td>
<td style="text-align:right;">
-0.2219792
</td>
<td style="text-align:right;">
2.352845e-09
</td>
<td style="text-align:right;">
2.666357e-07
</td>
<td style="text-align:left;">
17
</td>
<td style="text-align:right;">
19737984
</td>
<td style="text-align:right;">
19748943
</td>
<td style="text-align:right;">
10960
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000108602
</td>
<td style="text-align:left;">
ALDH3A1
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
aldehyde dehydrogenase 3 family member A1 [Source:HGNC Symbol;Acc:HGNC:405]
</td>
<td style="text-align:left;">
ENSG00000108602.18
</td>
<td style="text-align:left;">
ENST00000225740
</td>
<td style="text-align:left;">
218
</td>
<td style="text-align:right;">
-3.648080
</td>
<td style="text-align:right;">
-3.678036
</td>
<td style="text-align:right;">
-0.2219792
</td>
<td style="text-align:right;">
3.296711e-09
</td>
<td style="text-align:right;">
3.441819e-07
</td>
<td style="text-align:left;">
17
</td>
<td style="text-align:right;">
19737984
</td>
<td style="text-align:right;">
19748943
</td>
<td style="text-align:right;">
10960
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000108602
</td>
<td style="text-align:left;">
ALDH3A1
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
aldehyde dehydrogenase 3 family member A1 [Source:HGNC Symbol;Acc:HGNC:405]
</td>
<td style="text-align:left;">
ENSG00000108602.18
</td>
<td style="text-align:left;">
ENST00000225740
</td>
<td style="text-align:left;">
218
</td>
<td style="text-align:right;">
-2.438819
</td>
<td style="text-align:right;">
-2.450233
</td>
<td style="text-align:right;">
-0.2219792
</td>
<td style="text-align:right;">
7.685977e-05
</td>
<td style="text-align:right;">
1.948535e-03
</td>
<td style="text-align:left;">
17
</td>
<td style="text-align:right;">
19737984
</td>
<td style="text-align:right;">
19748943
</td>
<td style="text-align:right;">
10960
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000108602
</td>
<td style="text-align:left;">
ALDH3A1
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
aldehyde dehydrogenase 3 family member A1 [Source:HGNC Symbol;Acc:HGNC:405]
</td>
<td style="text-align:left;">
ENSG00000108602.18
</td>
<td style="text-align:left;">
ENST00000225740
</td>
<td style="text-align:left;">
218
</td>
<td style="text-align:right;">
-3.838474
</td>
<td style="text-align:right;">
-3.873034
</td>
<td style="text-align:right;">
-0.2219792
</td>
<td style="text-align:right;">
1.730282e-09
</td>
<td style="text-align:right;">
2.219712e-07
</td>
</tr>
<tr>
<td style="text-align:left;">
C6orf48
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
</td>
<td style="text-align:right;">
2.073542
</td>
<td style="text-align:right;">
2.073623
</td>
<td style="text-align:right;">
8.3509306
</td>
<td style="text-align:right;">
4.039192e-09
</td>
<td style="text-align:right;">
7.543049e-07
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
</td>
<td style="text-align:right;">
2.489797
</td>
<td style="text-align:right;">
2.489885
</td>
<td style="text-align:right;">
8.3509306
</td>
<td style="text-align:right;">
7.733699e-14
</td>
<td style="text-align:right;">
2.330480e-11
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
</td>
<td style="text-align:right;">
2.543025
</td>
<td style="text-align:right;">
2.543113
</td>
<td style="text-align:right;">
8.3509306
</td>
<td style="text-align:right;">
2.467322e-14
</td>
<td style="text-align:right;">
1.022319e-11
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
</td>
<td style="text-align:right;">
1.938183
</td>
<td style="text-align:right;">
1.938261
</td>
<td style="text-align:right;">
8.3509306
</td>
<td style="text-align:right;">
3.523764e-07
</td>
<td style="text-align:right;">
1.287096e-05
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
</td>
<td style="text-align:right;">
2.324490
</td>
<td style="text-align:right;">
2.324575
</td>
<td style="text-align:right;">
8.3509306
</td>
<td style="text-align:right;">
3.699089e-12
</td>
<td style="text-align:right;">
9.431965e-10
</td>
</tr>
<tr>
<td style="text-align:left;">
FILIP1L
</td>
<td style="text-align:left;">
3
</td>
<td style="text-align:right;">
99828811
</td>
<td style="text-align:right;">
100114513
</td>
<td style="text-align:right;">
285703
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000168386
</td>
<td style="text-align:left;">
FILIP1L
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
filamin A interacting protein 1 like [Source:HGNC Symbol;Acc:HGNC:24589]
</td>
<td style="text-align:left;">
ENSG00000168386.19
</td>
<td style="text-align:left;">
ENST00000477258
</td>
<td style="text-align:left;">
11259
</td>
<td style="text-align:right;">
2.234247
</td>
<td style="text-align:right;">
2.235621
</td>
<td style="text-align:right;">
4.2902029
</td>
<td style="text-align:right;">
1.215157e-08
</td>
<td style="text-align:right;">
1.989106e-06
</td>
<td style="text-align:left;">
3
</td>
<td style="text-align:right;">
99828811
</td>
<td style="text-align:right;">
100114513
</td>
<td style="text-align:right;">
285703
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000168386
</td>
<td style="text-align:left;">
FILIP1L
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
filamin A interacting protein 1 like [Source:HGNC Symbol;Acc:HGNC:24589]
</td>
<td style="text-align:left;">
ENSG00000168386.19
</td>
<td style="text-align:left;">
ENST00000477258
</td>
<td style="text-align:left;">
11259
</td>
<td style="text-align:right;">
2.226675
</td>
<td style="text-align:right;">
2.228047
</td>
<td style="text-align:right;">
4.2902029
</td>
<td style="text-align:right;">
1.652324e-08
</td>
<td style="text-align:right;">
1.672379e-06
</td>
<td style="text-align:left;">
3
</td>
<td style="text-align:right;">
99828811
</td>
<td style="text-align:right;">
100114513
</td>
<td style="text-align:right;">
285703
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000168386
</td>
<td style="text-align:left;">
FILIP1L
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
filamin A interacting protein 1 like [Source:HGNC Symbol;Acc:HGNC:24589]
</td>
<td style="text-align:left;">
ENSG00000168386.19
</td>
<td style="text-align:left;">
ENST00000477258
</td>
<td style="text-align:left;">
11259
</td>
<td style="text-align:right;">
1.953605
</td>
<td style="text-align:right;">
1.954900
</td>
<td style="text-align:right;">
4.2902029
</td>
<td style="text-align:right;">
3.431161e-05
</td>
<td style="text-align:right;">
1.574179e-03
</td>
<td style="text-align:left;">
3
</td>
<td style="text-align:right;">
99828811
</td>
<td style="text-align:right;">
100114513
</td>
<td style="text-align:right;">
285703
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000168386
</td>
<td style="text-align:left;">
FILIP1L
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
filamin A interacting protein 1 like [Source:HGNC Symbol;Acc:HGNC:24589]
</td>
<td style="text-align:left;">
ENSG00000168386.19
</td>
<td style="text-align:left;">
ENST00000477258
</td>
<td style="text-align:left;">
11259
</td>
<td style="text-align:right;">
2.555404
</td>
<td style="text-align:right;">
2.556852
</td>
<td style="text-align:right;">
4.2902029
</td>
<td style="text-align:right;">
2.188139e-12
</td>
<td style="text-align:right;">
1.921360e-10
</td>
<td style="text-align:left;">
3
</td>
<td style="text-align:right;">
99828811
</td>
<td style="text-align:right;">
100114513
</td>
<td style="text-align:right;">
285703
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000168386
</td>
<td style="text-align:left;">
FILIP1L
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
filamin A interacting protein 1 like [Source:HGNC Symbol;Acc:HGNC:24589]
</td>
<td style="text-align:left;">
ENSG00000168386.19
</td>
<td style="text-align:left;">
ENST00000477258
</td>
<td style="text-align:left;">
11259
</td>
<td style="text-align:right;">
2.269559
</td>
<td style="text-align:right;">
2.270941
</td>
<td style="text-align:right;">
4.2902029
</td>
<td style="text-align:right;">
5.088105e-09
</td>
<td style="text-align:right;">
5.440579e-07
</td>
</tr>
<tr>
<td style="text-align:left;">
CA9
</td>
<td style="text-align:left;">
9
</td>
<td style="text-align:right;">
35673918
</td>
<td style="text-align:right;">
35681159
</td>
<td style="text-align:right;">
7242
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000107159
</td>
<td style="text-align:left;">
CA9
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
carbonic anhydrase 9 [Source:HGNC Symbol;Acc:HGNC:1383]
</td>
<td style="text-align:left;">
ENSG00000107159.13
</td>
<td style="text-align:left;">
ENST00000378357
</td>
<td style="text-align:left;">
768
</td>
<td style="text-align:right;">
-2.122819
</td>
<td style="text-align:right;">
-2.123749
</td>
<td style="text-align:right;">
3.3039973
</td>
<td style="text-align:right;">
1.234601e-08
</td>
<td style="text-align:right;">
1.996290e-06
</td>
<td style="text-align:left;">
9
</td>
<td style="text-align:right;">
35673918
</td>
<td style="text-align:right;">
35681159
</td>
<td style="text-align:right;">
7242
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000107159
</td>
<td style="text-align:left;">
CA9
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
carbonic anhydrase 9 [Source:HGNC Symbol;Acc:HGNC:1383]
</td>
<td style="text-align:left;">
ENSG00000107159.13
</td>
<td style="text-align:left;">
ENST00000378357
</td>
<td style="text-align:left;">
768
</td>
<td style="text-align:right;">
-2.067587
</td>
<td style="text-align:right;">
-2.068472
</td>
<td style="text-align:right;">
3.3039973
</td>
<td style="text-align:right;">
8.239259e-08
</td>
<td style="text-align:right;">
7.381374e-06
</td>
<td style="text-align:left;">
9
</td>
<td style="text-align:right;">
35673918
</td>
<td style="text-align:right;">
35681159
</td>
<td style="text-align:right;">
7242
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000107159
</td>
<td style="text-align:left;">
CA9
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
carbonic anhydrase 9 [Source:HGNC Symbol;Acc:HGNC:1383]
</td>
<td style="text-align:left;">
ENSG00000107159.13
</td>
<td style="text-align:left;">
ENST00000378357
</td>
<td style="text-align:left;">
768
</td>
<td style="text-align:right;">
-2.349424
</td>
<td style="text-align:right;">
-2.350559
</td>
<td style="text-align:right;">
3.3039973
</td>
<td style="text-align:right;">
2.842691e-11
</td>
<td style="text-align:right;">
4.234971e-09
</td>
<td style="text-align:left;">
9
</td>
<td style="text-align:right;">
35673918
</td>
<td style="text-align:right;">
35681159
</td>
<td style="text-align:right;">
7242
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000107159
</td>
<td style="text-align:left;">
CA9
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
carbonic anhydrase 9 [Source:HGNC Symbol;Acc:HGNC:1383]
</td>
<td style="text-align:left;">
ENSG00000107159.13
</td>
<td style="text-align:left;">
ENST00000378357
</td>
<td style="text-align:left;">
768
</td>
<td style="text-align:right;">
-3.000052
</td>
<td style="text-align:right;">
-3.001994
</td>
<td style="text-align:right;">
3.3039973
</td>
<td style="text-align:right;">
1.134231e-16
</td>
<td style="text-align:right;">
1.953087e-14
</td>
<td style="text-align:left;">
9
</td>
<td style="text-align:right;">
35673918
</td>
<td style="text-align:right;">
35681159
</td>
<td style="text-align:right;">
7242
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000107159
</td>
<td style="text-align:left;">
CA9
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
carbonic anhydrase 9 [Source:HGNC Symbol;Acc:HGNC:1383]
</td>
<td style="text-align:left;">
ENSG00000107159.13
</td>
<td style="text-align:left;">
ENST00000378357
</td>
<td style="text-align:left;">
768
</td>
<td style="text-align:right;">
-2.232552
</td>
<td style="text-align:right;">
-2.233578
</td>
<td style="text-align:right;">
3.3039973
</td>
<td style="text-align:right;">
8.265725e-10
</td>
<td style="text-align:right;">
1.129848e-07
</td>
</tr>
<tr>
<td style="text-align:left;">
LCN2
</td>
<td style="text-align:left;">
9
</td>
<td style="text-align:right;">
128149071
</td>
<td style="text-align:right;">
128153453
</td>
<td style="text-align:right;">
4383
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000148346
</td>
<td style="text-align:left;">
LCN2
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
lipocalin 2 [Source:HGNC Symbol;Acc:HGNC:6526]
</td>
<td style="text-align:left;">
ENSG00000148346.12
</td>
<td style="text-align:left;">
ENST00000277480
</td>
<td style="text-align:left;">
3934
</td>
<td style="text-align:right;">
2.750052
</td>
<td style="text-align:right;">
2.753132
</td>
<td style="text-align:right;">
4.7728733
</td>
<td style="text-align:right;">
5.230336e-08
</td>
<td style="text-align:right;">
7.537937e-06
</td>
<td style="text-align:left;">
9
</td>
<td style="text-align:right;">
128149071
</td>
<td style="text-align:right;">
128153453
</td>
<td style="text-align:right;">
4383
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000148346
</td>
<td style="text-align:left;">
LCN2
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
lipocalin 2 [Source:HGNC Symbol;Acc:HGNC:6526]
</td>
<td style="text-align:left;">
ENSG00000148346.12
</td>
<td style="text-align:left;">
ENST00000277480
</td>
<td style="text-align:left;">
3934
</td>
<td style="text-align:right;">
4.968006
</td>
<td style="text-align:right;">
4.971508
</td>
<td style="text-align:right;">
4.7728733
</td>
<td style="text-align:right;">
4.579151e-18
</td>
<td style="text-align:right;">
2.891189e-15
</td>
<td style="text-align:left;">
9
</td>
<td style="text-align:right;">
128149071
</td>
<td style="text-align:right;">
128153453
</td>
<td style="text-align:right;">
4383
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000148346
</td>
<td style="text-align:left;">
LCN2
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
lipocalin 2 [Source:HGNC Symbol;Acc:HGNC:6526]
</td>
<td style="text-align:left;">
ENSG00000148346.12
</td>
<td style="text-align:left;">
ENST00000277480
</td>
<td style="text-align:left;">
3934
</td>
<td style="text-align:right;">
3.815031
</td>
<td style="text-align:right;">
3.818391
</td>
<td style="text-align:right;">
4.7728733
</td>
<td style="text-align:right;">
1.091915e-13
</td>
<td style="text-align:right;">
3.531147e-11
</td>
<td style="text-align:left;">
9
</td>
<td style="text-align:right;">
128149071
</td>
<td style="text-align:right;">
128153453
</td>
<td style="text-align:right;">
4383
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000148346
</td>
<td style="text-align:left;">
LCN2
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
lipocalin 2 [Source:HGNC Symbol;Acc:HGNC:6526]
</td>
<td style="text-align:left;">
ENSG00000148346.12
</td>
<td style="text-align:left;">
ENST00000277480
</td>
<td style="text-align:left;">
3934
</td>
<td style="text-align:right;">
2.669926
</td>
<td style="text-align:right;">
2.672975
</td>
<td style="text-align:right;">
4.7728733
</td>
<td style="text-align:right;">
1.838869e-07
</td>
<td style="text-align:right;">
7.046696e-06
</td>
<td style="text-align:left;">
9
</td>
<td style="text-align:right;">
128149071
</td>
<td style="text-align:right;">
128153453
</td>
<td style="text-align:right;">
4383
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000148346
</td>
<td style="text-align:left;">
LCN2
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
lipocalin 2 [Source:HGNC Symbol;Acc:HGNC:6526]
</td>
<td style="text-align:left;">
ENSG00000148346.12
</td>
<td style="text-align:left;">
ENST00000277480
</td>
<td style="text-align:left;">
3934
</td>
<td style="text-align:right;">
3.704796
</td>
<td style="text-align:right;">
3.708136
</td>
<td style="text-align:right;">
4.7728733
</td>
<td style="text-align:right;">
3.758690e-13
</td>
<td style="text-align:right;">
1.245912e-10
</td>
</tr>
<tr>
<td style="text-align:left;">
RPL22L1
</td>
<td style="text-align:left;">
3
</td>
<td style="text-align:right;">
170864875
</td>
<td style="text-align:right;">
170870208
</td>
<td style="text-align:right;">
5334
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000163584
</td>
<td style="text-align:left;">
RPL22L1
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
ribosomal protein L22 like 1 [Source:HGNC Symbol;Acc:HGNC:27610]
</td>
<td style="text-align:left;">
ENSG00000163584.18
</td>
<td style="text-align:left;">
ENST00000295830
</td>
<td style="text-align:left;">
200916
</td>
<td style="text-align:right;">
1.926659
</td>
<td style="text-align:right;">
1.927162
</td>
<td style="text-align:right;">
5.8187461
</td>
<td style="text-align:right;">
7.981174e-08
</td>
<td style="text-align:right;">
1.102317e-05
</td>
<td style="text-align:left;">
3
</td>
<td style="text-align:right;">
170864875
</td>
<td style="text-align:right;">
170870208
</td>
<td style="text-align:right;">
5334
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000163584
</td>
<td style="text-align:left;">
RPL22L1
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
ribosomal protein L22 like 1 [Source:HGNC Symbol;Acc:HGNC:27610]
</td>
<td style="text-align:left;">
ENSG00000163584.18
</td>
<td style="text-align:left;">
ENST00000295830
</td>
<td style="text-align:left;">
200916
</td>
<td style="text-align:right;">
2.443216
</td>
<td style="text-align:right;">
2.443773
</td>
<td style="text-align:right;">
5.8187461
</td>
<td style="text-align:right;">
2.154689e-16
</td>
<td style="text-align:right;">
1.020322e-13
</td>
<td style="text-align:left;">
3
</td>
<td style="text-align:right;">
170864875
</td>
<td style="text-align:right;">
170870208
</td>
<td style="text-align:right;">
5334
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000163584
</td>
<td style="text-align:left;">
RPL22L1
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
ribosomal protein L22 like 1 [Source:HGNC Symbol;Acc:HGNC:27610]
</td>
<td style="text-align:left;">
ENSG00000163584.18
</td>
<td style="text-align:left;">
ENST00000295830
</td>
<td style="text-align:left;">
200916
</td>
<td style="text-align:right;">
2.458410
</td>
<td style="text-align:right;">
2.458967
</td>
<td style="text-align:right;">
5.8187461
</td>
<td style="text-align:right;">
1.257912e-16
</td>
<td style="text-align:right;">
9.810970e-14
</td>
<td style="text-align:left;">
3
</td>
<td style="text-align:right;">
170864875
</td>
<td style="text-align:right;">
170870208
</td>
<td style="text-align:right;">
5334
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000163584
</td>
<td style="text-align:left;">
RPL22L1
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
ribosomal protein L22 like 1 [Source:HGNC Symbol;Acc:HGNC:27610]
</td>
<td style="text-align:left;">
ENSG00000163584.18
</td>
<td style="text-align:left;">
ENST00000295830
</td>
<td style="text-align:left;">
200916
</td>
<td style="text-align:right;">
2.576553
</td>
<td style="text-align:right;">
2.577120
</td>
<td style="text-align:right;">
5.8187461
</td>
<td style="text-align:right;">
3.775399e-18
</td>
<td style="text-align:right;">
9.270003e-16
</td>
<td style="text-align:left;">
3
</td>
<td style="text-align:right;">
170864875
</td>
<td style="text-align:right;">
170870208
</td>
<td style="text-align:right;">
5334
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000163584
</td>
<td style="text-align:left;">
RPL22L1
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
ribosomal protein L22 like 1 [Source:HGNC Symbol;Acc:HGNC:27610]
</td>
<td style="text-align:left;">
ENSG00000163584.18
</td>
<td style="text-align:left;">
ENST00000295830
</td>
<td style="text-align:left;">
200916
</td>
<td style="text-align:right;">
2.710758
</td>
<td style="text-align:right;">
2.711335
</td>
<td style="text-align:right;">
5.8187461
</td>
<td style="text-align:right;">
8.668150e-20
</td>
<td style="text-align:right;">
1.915517e-16
</td>
</tr>
<tr>
<td style="text-align:left;">
KRT17
</td>
<td style="text-align:left;">
17
</td>
<td style="text-align:right;">
41619442
</td>
<td style="text-align:right;">
41624842
</td>
<td style="text-align:right;">
5401
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000128422
</td>
<td style="text-align:left;">
KRT17
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
keratin 17 [Source:HGNC Symbol;Acc:HGNC:6427]
</td>
<td style="text-align:left;">
ENSG00000128422.17
</td>
<td style="text-align:left;">
ENST00000311208
</td>
<td style="text-align:left;">
3872
</td>
<td style="text-align:right;">
3.441144
</td>
<td style="text-align:right;">
3.469302
</td>
<td style="text-align:right;">
1.3570823
</td>
<td style="text-align:right;">
1.316226e-07
</td>
<td style="text-align:right;">
1.780801e-05
</td>
<td style="text-align:left;">
17
</td>
<td style="text-align:right;">
41619442
</td>
<td style="text-align:right;">
41624842
</td>
<td style="text-align:right;">
5401
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000128422
</td>
<td style="text-align:left;">
KRT17
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
keratin 17 [Source:HGNC Symbol;Acc:HGNC:6427]
</td>
<td style="text-align:left;">
ENSG00000128422.17
</td>
<td style="text-align:left;">
ENST00000311208
</td>
<td style="text-align:left;">
3872
</td>
<td style="text-align:right;">
4.122394
</td>
<td style="text-align:right;">
4.151615
</td>
<td style="text-align:right;">
1.3570823
</td>
<td style="text-align:right;">
5.192255e-11
</td>
<td style="text-align:right;">
8.605514e-09
</td>
<td style="text-align:left;">
17
</td>
<td style="text-align:right;">
41619442
</td>
<td style="text-align:right;">
41624842
</td>
<td style="text-align:right;">
5401
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000128422
</td>
<td style="text-align:left;">
KRT17
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
keratin 17 [Source:HGNC Symbol;Acc:HGNC:6427]
</td>
<td style="text-align:left;">
ENSG00000128422.17
</td>
<td style="text-align:left;">
ENST00000311208
</td>
<td style="text-align:left;">
3872
</td>
<td style="text-align:right;">
3.292126
</td>
<td style="text-align:right;">
3.319982
</td>
<td style="text-align:right;">
1.3570823
</td>
<td style="text-align:right;">
7.100443e-07
</td>
<td style="text-align:right;">
4.827937e-05
</td>
<td style="text-align:left;">
17
</td>
<td style="text-align:right;">
41619442
</td>
<td style="text-align:right;">
41624842
</td>
<td style="text-align:right;">
5401
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000128422
</td>
<td style="text-align:left;">
KRT17
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
keratin 17 [Source:HGNC Symbol;Acc:HGNC:6427]
</td>
<td style="text-align:left;">
ENSG00000128422.17
</td>
<td style="text-align:left;">
ENST00000311208
</td>
<td style="text-align:left;">
3872
</td>
<td style="text-align:right;">
2.930129
</td>
<td style="text-align:right;">
2.957084
</td>
<td style="text-align:right;">
1.3570823
</td>
<td style="text-align:right;">
3.322043e-05
</td>
<td style="text-align:right;">
8.866652e-04
</td>
<td style="text-align:left;">
17
</td>
<td style="text-align:right;">
41619442
</td>
<td style="text-align:right;">
41624842
</td>
<td style="text-align:right;">
5401
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000128422
</td>
<td style="text-align:left;">
KRT17
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
keratin 17 [Source:HGNC Symbol;Acc:HGNC:6427]
</td>
<td style="text-align:left;">
ENSG00000128422.17
</td>
<td style="text-align:left;">
ENST00000311208
</td>
<td style="text-align:left;">
3872
</td>
<td style="text-align:right;">
3.144169
</td>
<td style="text-align:right;">
3.171687
</td>
<td style="text-align:right;">
1.3570823
</td>
<td style="text-align:right;">
3.750551e-06
</td>
<td style="text-align:right;">
2.323138e-04
</td>
</tr>
<tr>
<td style="text-align:left;">
TMEM154
</td>
<td style="text-align:left;">
4
</td>
<td style="text-align:right;">
152618628
</td>
<td style="text-align:right;">
152680012
</td>
<td style="text-align:right;">
61385
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000170006
</td>
<td style="text-align:left;">
TMEM154
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
transmembrane protein 154 [Source:HGNC Symbol;Acc:HGNC:26489]
</td>
<td style="text-align:left;">
ENSG00000170006.12
</td>
<td style="text-align:left;">
ENST00000304385
</td>
<td style="text-align:left;">
201799
</td>
<td style="text-align:right;">
2.081623
</td>
<td style="text-align:right;">
2.082208
</td>
<td style="text-align:right;">
5.6405248
</td>
<td style="text-align:right;">
8.425950e-07
</td>
<td style="text-align:right;">
1.034441e-04
</td>
<td style="text-align:left;">
4
</td>
<td style="text-align:right;">
152618628
</td>
<td style="text-align:right;">
152680012
</td>
<td style="text-align:right;">
61385
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000170006
</td>
<td style="text-align:left;">
TMEM154
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
transmembrane protein 154 [Source:HGNC Symbol;Acc:HGNC:26489]
</td>
<td style="text-align:left;">
ENSG00000170006.12
</td>
<td style="text-align:left;">
ENST00000304385
</td>
<td style="text-align:left;">
201799
</td>
<td style="text-align:right;">
2.355885
</td>
<td style="text-align:right;">
2.356501
</td>
<td style="text-align:right;">
5.6405248
</td>
<td style="text-align:right;">
1.179871e-09
</td>
<td style="text-align:right;">
1.422173e-07
</td>
<td style="text-align:left;">
4
</td>
<td style="text-align:right;">
152618628
</td>
<td style="text-align:right;">
152680012
</td>
<td style="text-align:right;">
61385
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000170006
</td>
<td style="text-align:left;">
TMEM154
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
transmembrane protein 154 [Source:HGNC Symbol;Acc:HGNC:26489]
</td>
<td style="text-align:left;">
ENSG00000170006.12
</td>
<td style="text-align:left;">
ENST00000304385
</td>
<td style="text-align:left;">
201799
</td>
<td style="text-align:right;">
2.569568
</td>
<td style="text-align:right;">
2.570204
</td>
<td style="text-align:right;">
5.6405248
</td>
<td style="text-align:right;">
1.209089e-11
</td>
<td style="text-align:right;">
2.003914e-09
</td>
<td style="text-align:left;">
4
</td>
<td style="text-align:right;">
152618628
</td>
<td style="text-align:right;">
152680012
</td>
<td style="text-align:right;">
61385
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000170006
</td>
<td style="text-align:left;">
TMEM154
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
transmembrane protein 154 [Source:HGNC Symbol;Acc:HGNC:26489]
</td>
<td style="text-align:left;">
ENSG00000170006.12
</td>
<td style="text-align:left;">
ENST00000304385
</td>
<td style="text-align:left;">
201799
</td>
<td style="text-align:right;">
2.577730
</td>
<td style="text-align:right;">
2.578367
</td>
<td style="text-align:right;">
5.6405248
</td>
<td style="text-align:right;">
1.098922e-11
</td>
<td style="text-align:right;">
8.326063e-10
</td>
<td style="text-align:left;">
4
</td>
<td style="text-align:right;">
152618628
</td>
<td style="text-align:right;">
152680012
</td>
<td style="text-align:right;">
61385
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000170006
</td>
<td style="text-align:left;">
TMEM154
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
transmembrane protein 154 [Source:HGNC Symbol;Acc:HGNC:26489]
</td>
<td style="text-align:left;">
ENSG00000170006.12
</td>
<td style="text-align:left;">
ENST00000304385
</td>
<td style="text-align:left;">
201799
</td>
<td style="text-align:right;">
2.526503
</td>
<td style="text-align:right;">
2.527136
</td>
<td style="text-align:right;">
5.6405248
</td>
<td style="text-align:right;">
3.069812e-11
</td>
<td style="text-align:right;">
5.985682e-09
</td>
</tr>
<tr>
<td style="text-align:left;">
ZFAS1
</td>
<td style="text-align:left;">
20
</td>
<td style="text-align:right;">
49278178
</td>
<td style="text-align:right;">
49299600
</td>
<td style="text-align:right;">
21423
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000177410
</td>
<td style="text-align:left;">
ZFAS1
</td>
<td style="text-align:left;">
lncRNA
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
ZNFX1 antisense RNA 1 [Source:HGNC Symbol;Acc:HGNC:33101]
</td>
<td style="text-align:left;">
ENSG00000177410.13
</td>
<td style="text-align:left;">
ENST00000620594
</td>
<td style="text-align:left;">
441951
</td>
<td style="text-align:right;">
2.058415
</td>
<td style="text-align:right;">
2.058597
</td>
<td style="text-align:right;">
8.1056151
</td>
<td style="text-align:right;">
1.039498e-05
</td>
<td style="text-align:right;">
1.085252e-03
</td>
<td style="text-align:left;">
20
</td>
<td style="text-align:right;">
49278178
</td>
<td style="text-align:right;">
49299600
</td>
<td style="text-align:right;">
21423
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000177410
</td>
<td style="text-align:left;">
ZFAS1
</td>
<td style="text-align:left;">
lncRNA
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
ZNFX1 antisense RNA 1 [Source:HGNC Symbol;Acc:HGNC:33101]
</td>
<td style="text-align:left;">
ENSG00000177410.13
</td>
<td style="text-align:left;">
ENST00000620594
</td>
<td style="text-align:left;">
441951
</td>
<td style="text-align:right;">
3.286977
</td>
<td style="text-align:right;">
3.287192
</td>
<td style="text-align:right;">
8.1056151
</td>
<td style="text-align:right;">
1.603719e-12
</td>
<td style="text-align:right;">
3.604018e-10
</td>
<td style="text-align:left;">
20
</td>
<td style="text-align:right;">
49278178
</td>
<td style="text-align:right;">
49299600
</td>
<td style="text-align:right;">
21423
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000177410
</td>
<td style="text-align:left;">
ZFAS1
</td>
<td style="text-align:left;">
lncRNA
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
ZNFX1 antisense RNA 1 [Source:HGNC Symbol;Acc:HGNC:33101]
</td>
<td style="text-align:left;">
ENSG00000177410.13
</td>
<td style="text-align:left;">
ENST00000620594
</td>
<td style="text-align:left;">
441951
</td>
<td style="text-align:right;">
3.407290
</td>
<td style="text-align:right;">
3.407508
</td>
<td style="text-align:right;">
8.1056151
</td>
<td style="text-align:right;">
6.388080e-13
</td>
<td style="text-align:right;">
1.568510e-10
</td>
<td style="text-align:left;">
20
</td>
<td style="text-align:right;">
49278178
</td>
<td style="text-align:right;">
49299600
</td>
<td style="text-align:right;">
21423
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000177410
</td>
<td style="text-align:left;">
ZFAS1
</td>
<td style="text-align:left;">
lncRNA
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
ZNFX1 antisense RNA 1 [Source:HGNC Symbol;Acc:HGNC:33101]
</td>
<td style="text-align:left;">
ENSG00000177410.13
</td>
<td style="text-align:left;">
ENST00000620594
</td>
<td style="text-align:left;">
441951
</td>
<td style="text-align:right;">
3.703495
</td>
<td style="text-align:right;">
3.703717
</td>
<td style="text-align:right;">
8.1056151
</td>
<td style="text-align:right;">
1.085944e-13
</td>
<td style="text-align:right;">
1.124886e-11
</td>
<td style="text-align:left;">
20
</td>
<td style="text-align:right;">
49278178
</td>
<td style="text-align:right;">
49299600
</td>
<td style="text-align:right;">
21423
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000177410
</td>
<td style="text-align:left;">
ZFAS1
</td>
<td style="text-align:left;">
lncRNA
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
ZNFX1 antisense RNA 1 [Source:HGNC Symbol;Acc:HGNC:33101]
</td>
<td style="text-align:left;">
ENSG00000177410.13
</td>
<td style="text-align:left;">
ENST00000620594
</td>
<td style="text-align:left;">
441951
</td>
<td style="text-align:right;">
3.344575
</td>
<td style="text-align:right;">
3.344792
</td>
<td style="text-align:right;">
8.1056151
</td>
<td style="text-align:right;">
1.028369e-12
</td>
<td style="text-align:right;">
2.964162e-10
</td>
</tr>
<tr>
<td style="text-align:left;">
SOST
</td>
<td style="text-align:left;">
17
</td>
<td style="text-align:right;">
43753738
</td>
<td style="text-align:right;">
43758791
</td>
<td style="text-align:right;">
5054
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000167941
</td>
<td style="text-align:left;">
SOST
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
sclerostin [Source:HGNC Symbol;Acc:HGNC:13771]
</td>
<td style="text-align:left;">
ENSG00000167941.3
</td>
<td style="text-align:left;">
ENST00000301691
</td>
<td style="text-align:left;">
50964
</td>
<td style="text-align:right;">
-2.192300
</td>
<td style="text-align:right;">
-2.194680
</td>
<td style="text-align:right;">
1.7670916
</td>
<td style="text-align:right;">
1.998125e-05
</td>
<td style="text-align:right;">
1.878946e-03
</td>
<td style="text-align:left;">
17
</td>
<td style="text-align:right;">
43753738
</td>
<td style="text-align:right;">
43758791
</td>
<td style="text-align:right;">
5054
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000167941
</td>
<td style="text-align:left;">
SOST
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
sclerostin [Source:HGNC Symbol;Acc:HGNC:13771]
</td>
<td style="text-align:left;">
ENSG00000167941.3
</td>
<td style="text-align:left;">
ENST00000301691
</td>
<td style="text-align:left;">
50964
</td>
<td style="text-align:right;">
-2.615399
</td>
<td style="text-align:right;">
-2.618812
</td>
<td style="text-align:right;">
1.7670916
</td>
<td style="text-align:right;">
3.941092e-08
</td>
<td style="text-align:right;">
3.759348e-06
</td>
<td style="text-align:left;">
17
</td>
<td style="text-align:right;">
43753738
</td>
<td style="text-align:right;">
43758791
</td>
<td style="text-align:right;">
5054
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000167941
</td>
<td style="text-align:left;">
SOST
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
sclerostin [Source:HGNC Symbol;Acc:HGNC:13771]
</td>
<td style="text-align:left;">
ENSG00000167941.3
</td>
<td style="text-align:left;">
ENST00000301691
</td>
<td style="text-align:left;">
50964
</td>
<td style="text-align:right;">
-4.180219
</td>
<td style="text-align:right;">
-4.191661
</td>
<td style="text-align:right;">
1.7670916
</td>
<td style="text-align:right;">
9.240815e-15
</td>
<td style="text-align:right;">
4.224964e-12
</td>
<td style="text-align:left;">
17
</td>
<td style="text-align:right;">
43753738
</td>
<td style="text-align:right;">
43758791
</td>
<td style="text-align:right;">
5054
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000167941
</td>
<td style="text-align:left;">
SOST
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
sclerostin [Source:HGNC Symbol;Acc:HGNC:13771]
</td>
<td style="text-align:left;">
ENSG00000167941.3
</td>
<td style="text-align:left;">
ENST00000301691
</td>
<td style="text-align:left;">
50964
</td>
<td style="text-align:right;">
-3.263262
</td>
<td style="text-align:right;">
-3.269002
</td>
<td style="text-align:right;">
1.7670916
</td>
<td style="text-align:right;">
2.078592e-11
</td>
<td style="text-align:right;">
1.497829e-09
</td>
<td style="text-align:left;">
17
</td>
<td style="text-align:right;">
43753738
</td>
<td style="text-align:right;">
43758791
</td>
<td style="text-align:right;">
5054
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000167941
</td>
<td style="text-align:left;">
SOST
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
sclerostin [Source:HGNC Symbol;Acc:HGNC:13771]
</td>
<td style="text-align:left;">
ENSG00000167941.3
</td>
<td style="text-align:left;">
ENST00000301691
</td>
<td style="text-align:left;">
50964
</td>
<td style="text-align:right;">
-3.531066
</td>
<td style="text-align:right;">
-3.538122
</td>
<td style="text-align:right;">
1.7670916
</td>
<td style="text-align:right;">
1.648556e-12
</td>
<td style="text-align:right;">
4.553791e-10
</td>
</tr>
<tr>
<td style="text-align:left;">
KRT34
</td>
<td style="text-align:left;">
17
</td>
<td style="text-align:right;">
41377669
</td>
<td style="text-align:right;">
41382306
</td>
<td style="text-align:right;">
4638
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000131737
</td>
<td style="text-align:left;">
KRT34
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
keratin 34 [Source:HGNC Symbol;Acc:HGNC:6452]
</td>
<td style="text-align:left;">
ENSG00000131737.7
</td>
<td style="text-align:left;">
ENST00000394001
</td>
<td style="text-align:left;">
3885
</td>
<td style="text-align:right;">
2.536622
</td>
<td style="text-align:right;">
2.540735
</td>
<td style="text-align:right;">
4.6437104
</td>
<td style="text-align:right;">
9.978814e-05
</td>
<td style="text-align:right;">
7.678126e-03
</td>
<td style="text-align:left;">
17
</td>
<td style="text-align:right;">
41377669
</td>
<td style="text-align:right;">
41382306
</td>
<td style="text-align:right;">
4638
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000131737
</td>
<td style="text-align:left;">
KRT34
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
keratin 34 [Source:HGNC Symbol;Acc:HGNC:6452]
</td>
<td style="text-align:left;">
ENSG00000131737.7
</td>
<td style="text-align:left;">
ENST00000394001
</td>
<td style="text-align:left;">
3885
</td>
<td style="text-align:right;">
4.310192
</td>
<td style="text-align:right;">
4.314909
</td>
<td style="text-align:right;">
4.6437104
</td>
<td style="text-align:right;">
2.665785e-11
</td>
<td style="text-align:right;">
4.650742e-09
</td>
<td style="text-align:left;">
17
</td>
<td style="text-align:right;">
41377669
</td>
<td style="text-align:right;">
41382306
</td>
<td style="text-align:right;">
4638
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000131737
</td>
<td style="text-align:left;">
KRT34
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
keratin 34 [Source:HGNC Symbol;Acc:HGNC:6452]
</td>
<td style="text-align:left;">
ENSG00000131737.7
</td>
<td style="text-align:left;">
ENST00000394001
</td>
<td style="text-align:left;">
3885
</td>
<td style="text-align:right;">
4.256411
</td>
<td style="text-align:right;">
4.261118
</td>
<td style="text-align:right;">
4.6437104
</td>
<td style="text-align:right;">
3.743492e-11
</td>
<td style="text-align:right;">
5.392161e-09
</td>
<td style="text-align:left;">
17
</td>
<td style="text-align:right;">
41377669
</td>
<td style="text-align:right;">
41382306
</td>
<td style="text-align:right;">
4638
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000131737
</td>
<td style="text-align:left;">
KRT34
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
keratin 34 [Source:HGNC Symbol;Acc:HGNC:6452]
</td>
<td style="text-align:left;">
ENSG00000131737.7
</td>
<td style="text-align:left;">
ENST00000394001
</td>
<td style="text-align:left;">
3885
</td>
<td style="text-align:right;">
4.570593
</td>
<td style="text-align:right;">
4.575351
</td>
<td style="text-align:right;">
4.6437104
</td>
<td style="text-align:right;">
5.077930e-12
</td>
<td style="text-align:right;">
4.033361e-10
</td>
<td style="text-align:left;">
17
</td>
<td style="text-align:right;">
41377669
</td>
<td style="text-align:right;">
41382306
</td>
<td style="text-align:right;">
4638
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000131737
</td>
<td style="text-align:left;">
KRT34
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
keratin 34 [Source:HGNC Symbol;Acc:HGNC:6452]
</td>
<td style="text-align:left;">
ENSG00000131737.7
</td>
<td style="text-align:left;">
ENST00000394001
</td>
<td style="text-align:left;">
3885
</td>
<td style="text-align:right;">
4.430011
</td>
<td style="text-align:right;">
4.434748
</td>
<td style="text-align:right;">
4.6437104
</td>
<td style="text-align:right;">
1.225827e-11
</td>
<td style="text-align:right;">
2.754786e-09
</td>
</tr>
<tr>
<td style="text-align:left;">
CXCL8
</td>
<td style="text-align:left;">
4
</td>
<td style="text-align:right;">
73740541
</td>
<td style="text-align:right;">
73743716
</td>
<td style="text-align:right;">
3176
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000169429
</td>
<td style="text-align:left;">
CXCL8
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
C-X-C motif chemokine ligand 8 [Source:HGNC Symbol;Acc:HGNC:6025]
</td>
<td style="text-align:left;">
ENSG00000169429.11
</td>
<td style="text-align:left;">
ENST00000307407
</td>
<td style="text-align:left;">
3576
</td>
<td style="text-align:right;">
2.083230
</td>
<td style="text-align:right;">
2.083522
</td>
<td style="text-align:right;">
9.7767522
</td>
<td style="text-align:right;">
1.284342e-04
</td>
<td style="text-align:right;">
9.675618e-03
</td>
<td style="text-align:left;">
4
</td>
<td style="text-align:right;">
73740541
</td>
<td style="text-align:right;">
73743716
</td>
<td style="text-align:right;">
3176
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000169429
</td>
<td style="text-align:left;">
CXCL8
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
C-X-C motif chemokine ligand 8 [Source:HGNC Symbol;Acc:HGNC:6025]
</td>
<td style="text-align:left;">
ENSG00000169429.11
</td>
<td style="text-align:left;">
ENST00000307407
</td>
<td style="text-align:left;">
3576
</td>
<td style="text-align:right;">
6.758561
</td>
<td style="text-align:right;">
6.758940
</td>
<td style="text-align:right;">
9.7767522
</td>
<td style="text-align:right;">
1.497980e-15
</td>
<td style="text-align:right;">
5.942201e-13
</td>
<td style="text-align:left;">
4
</td>
<td style="text-align:right;">
73740541
</td>
<td style="text-align:right;">
73743716
</td>
<td style="text-align:right;">
3176
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000169429
</td>
<td style="text-align:left;">
CXCL8
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
C-X-C motif chemokine ligand 8 [Source:HGNC Symbol;Acc:HGNC:6025]
</td>
<td style="text-align:left;">
ENSG00000169429.11
</td>
<td style="text-align:left;">
ENST00000307407
</td>
<td style="text-align:left;">
3576
</td>
<td style="text-align:right;">
5.306451
</td>
<td style="text-align:right;">
5.306823
</td>
<td style="text-align:right;">
9.7767522
</td>
<td style="text-align:right;">
5.374733e-14
</td>
<td style="text-align:right;">
1.827271e-11
</td>
<td style="text-align:left;">
4
</td>
<td style="text-align:right;">
73740541
</td>
<td style="text-align:right;">
73743716
</td>
<td style="text-align:right;">
3176
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000169429
</td>
<td style="text-align:left;">
CXCL8
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
C-X-C motif chemokine ligand 8 [Source:HGNC Symbol;Acc:HGNC:6025]
</td>
<td style="text-align:left;">
ENSG00000169429.11
</td>
<td style="text-align:left;">
ENST00000307407
</td>
<td style="text-align:left;">
3576
</td>
<td style="text-align:right;">
5.070467
</td>
<td style="text-align:right;">
5.070837
</td>
<td style="text-align:right;">
9.7767522
</td>
<td style="text-align:right;">
1.142013e-13
</td>
<td style="text-align:right;">
1.173795e-11
</td>
<td style="text-align:left;">
4
</td>
<td style="text-align:right;">
73740541
</td>
<td style="text-align:right;">
73743716
</td>
<td style="text-align:right;">
3176
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000169429
</td>
<td style="text-align:left;">
CXCL8
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
C-X-C motif chemokine ligand 8 [Source:HGNC Symbol;Acc:HGNC:6025]
</td>
<td style="text-align:left;">
ENSG00000169429.11
</td>
<td style="text-align:left;">
ENST00000307407
</td>
<td style="text-align:left;">
3576
</td>
<td style="text-align:right;">
5.904394
</td>
<td style="text-align:right;">
5.904769
</td>
<td style="text-align:right;">
9.7767522
</td>
<td style="text-align:right;">
1.021393e-14
</td>
<td style="text-align:right;">
4.836661e-12
</td>
</tr>
<tr>
<td style="text-align:left;">
CXCL2
</td>
<td style="text-align:left;">
4
</td>
<td style="text-align:right;">
74097040
</td>
<td style="text-align:right;">
74099196
</td>
<td style="text-align:right;">
2157
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000081041
</td>
<td style="text-align:left;">
CXCL2
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
C-X-C motif chemokine ligand 2 [Source:HGNC Symbol;Acc:HGNC:4603]
</td>
<td style="text-align:left;">
ENSG00000081041.9
</td>
<td style="text-align:left;">
ENST00000508487
</td>
<td style="text-align:left;">
2920
</td>
<td style="text-align:right;">
2.180750
</td>
<td style="text-align:right;">
2.182429
</td>
<td style="text-align:right;">
5.8548683
</td>
<td style="text-align:right;">
1.354685e-04
</td>
<td style="text-align:right;">
1.009088e-02
</td>
<td style="text-align:left;">
4
</td>
<td style="text-align:right;">
74097040
</td>
<td style="text-align:right;">
74099196
</td>
<td style="text-align:right;">
2157
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000081041
</td>
<td style="text-align:left;">
CXCL2
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
C-X-C motif chemokine ligand 2 [Source:HGNC Symbol;Acc:HGNC:4603]
</td>
<td style="text-align:left;">
ENSG00000081041.9
</td>
<td style="text-align:left;">
ENST00000508487
</td>
<td style="text-align:left;">
2920
</td>
<td style="text-align:right;">
5.270257
</td>
<td style="text-align:right;">
5.272355
</td>
<td style="text-align:right;">
5.8548683
</td>
<td style="text-align:right;">
1.880269e-16
</td>
<td style="text-align:right;">
9.233512e-14
</td>
<td style="text-align:left;">
4
</td>
<td style="text-align:right;">
74097040
</td>
<td style="text-align:right;">
74099196
</td>
<td style="text-align:right;">
2157
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000081041
</td>
<td style="text-align:left;">
CXCL2
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
C-X-C motif chemokine ligand 2 [Source:HGNC Symbol;Acc:HGNC:4603]
</td>
<td style="text-align:left;">
ENSG00000081041.9
</td>
<td style="text-align:left;">
ENST00000508487
</td>
<td style="text-align:left;">
2920
</td>
<td style="text-align:right;">
3.963242
</td>
<td style="text-align:right;">
3.965258
</td>
<td style="text-align:right;">
5.8548683
</td>
<td style="text-align:right;">
4.179709e-13
</td>
<td style="text-align:right;">
1.086642e-10
</td>
<td style="text-align:left;">
4
</td>
<td style="text-align:right;">
74097040
</td>
<td style="text-align:right;">
74099196
</td>
<td style="text-align:right;">
2157
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000081041
</td>
<td style="text-align:left;">
CXCL2
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
C-X-C motif chemokine ligand 2 [Source:HGNC Symbol;Acc:HGNC:4603]
</td>
<td style="text-align:left;">
ENSG00000081041.9
</td>
<td style="text-align:left;">
ENST00000508487
</td>
<td style="text-align:left;">
2920
</td>
<td style="text-align:right;">
3.243235
</td>
<td style="text-align:right;">
3.245161
</td>
<td style="text-align:right;">
5.8548683
</td>
<td style="text-align:right;">
2.849323e-10
</td>
<td style="text-align:right;">
1.609801e-08
</td>
<td style="text-align:left;">
4
</td>
<td style="text-align:right;">
74097040
</td>
<td style="text-align:right;">
74099196
</td>
<td style="text-align:right;">
2157
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000081041
</td>
<td style="text-align:left;">
CXCL2
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
C-X-C motif chemokine ligand 2 [Source:HGNC Symbol;Acc:HGNC:4603]
</td>
<td style="text-align:left;">
ENSG00000081041.9
</td>
<td style="text-align:left;">
ENST00000508487
</td>
<td style="text-align:left;">
2920
</td>
<td style="text-align:right;">
4.491903
</td>
<td style="text-align:right;">
4.493961
</td>
<td style="text-align:right;">
5.8548683
</td>
<td style="text-align:right;">
1.234423e-14
</td>
<td style="text-align:right;">
5.643867e-12
</td>
</tr>
<tr>
<td style="text-align:left;">
RELN
</td>
<td style="text-align:left;">
7
</td>
<td style="text-align:right;">
103471381
</td>
<td style="text-align:right;">
103989658
</td>
<td style="text-align:right;">
518278
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000189056
</td>
<td style="text-align:left;">
RELN
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
reelin [Source:HGNC Symbol;Acc:HGNC:9957]
</td>
<td style="text-align:left;">
ENSG00000189056.15
</td>
<td style="text-align:left;">
ENST00000428762
</td>
<td style="text-align:left;">
5649
</td>
<td style="text-align:right;">
2.591063
</td>
<td style="text-align:right;">
2.613101
</td>
<td style="text-align:right;">
1.5244277
</td>
<td style="text-align:right;">
3.464263e-04
</td>
<td style="text-align:right;">
2.319832e-02
</td>
<td style="text-align:left;">
7
</td>
<td style="text-align:right;">
103471381
</td>
<td style="text-align:right;">
103989658
</td>
<td style="text-align:right;">
518278
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000189056
</td>
<td style="text-align:left;">
RELN
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
reelin [Source:HGNC Symbol;Acc:HGNC:9957]
</td>
<td style="text-align:left;">
ENSG00000189056.15
</td>
<td style="text-align:left;">
ENST00000428762
</td>
<td style="text-align:left;">
5649
</td>
<td style="text-align:right;">
3.176965
</td>
<td style="text-align:right;">
3.200450
</td>
<td style="text-align:right;">
1.5244277
</td>
<td style="text-align:right;">
8.334882e-07
</td>
<td style="text-align:right;">
6.208551e-05
</td>
<td style="text-align:left;">
7
</td>
<td style="text-align:right;">
103471381
</td>
<td style="text-align:right;">
103989658
</td>
<td style="text-align:right;">
518278
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000189056
</td>
<td style="text-align:left;">
RELN
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
reelin [Source:HGNC Symbol;Acc:HGNC:9957]
</td>
<td style="text-align:left;">
ENSG00000189056.15
</td>
<td style="text-align:left;">
ENST00000428762
</td>
<td style="text-align:left;">
5649
</td>
<td style="text-align:right;">
3.070156
</td>
<td style="text-align:right;">
3.093413
</td>
<td style="text-align:right;">
1.5244277
</td>
<td style="text-align:right;">
2.646592e-06
</td>
<td style="text-align:right;">
1.580683e-04
</td>
<td style="text-align:left;">
7
</td>
<td style="text-align:right;">
103471381
</td>
<td style="text-align:right;">
103989658
</td>
<td style="text-align:right;">
518278
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000189056
</td>
<td style="text-align:left;">
RELN
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
reelin [Source:HGNC Symbol;Acc:HGNC:9957]
</td>
<td style="text-align:left;">
ENSG00000189056.15
</td>
<td style="text-align:left;">
ENST00000428762
</td>
<td style="text-align:left;">
5649
</td>
<td style="text-align:right;">
4.384724
</td>
<td style="text-align:right;">
4.409848
</td>
<td style="text-align:right;">
1.5244277
</td>
<td style="text-align:right;">
3.996308e-13
</td>
<td style="text-align:right;">
3.924966e-11
</td>
<td style="text-align:left;">
7
</td>
<td style="text-align:right;">
103471381
</td>
<td style="text-align:right;">
103989658
</td>
<td style="text-align:right;">
518278
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000189056
</td>
<td style="text-align:left;">
RELN
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
reelin [Source:HGNC Symbol;Acc:HGNC:9957]
</td>
<td style="text-align:left;">
ENSG00000189056.15
</td>
<td style="text-align:left;">
ENST00000428762
</td>
<td style="text-align:left;">
5649
</td>
<td style="text-align:right;">
3.122716
</td>
<td style="text-align:right;">
3.146087
</td>
<td style="text-align:right;">
1.5244277
</td>
<td style="text-align:right;">
1.587076e-06
</td>
<td style="text-align:right;">
1.046918e-04
</td>
</tr>
<tr>
<td style="text-align:left;">
RCAN1
</td>
<td style="text-align:left;">
21
</td>
<td style="text-align:right;">
34513142
</td>
<td style="text-align:right;">
34615113
</td>
<td style="text-align:right;">
101972
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000159200
</td>
<td style="text-align:left;">
RCAN1
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
regulator of calcineurin 1 [Source:HGNC Symbol;Acc:HGNC:3040]
</td>
<td style="text-align:left;">
ENSG00000159200.18
</td>
<td style="text-align:left;">
ENST00000313806
</td>
<td style="text-align:left;">
1827
</td>
<td style="text-align:right;">
1.982334
</td>
<td style="text-align:right;">
1.982492
</td>
<td style="text-align:right;">
7.8075066
</td>
<td style="text-align:right;">
3.788909e-04
</td>
<td style="text-align:right;">
2.511857e-02
</td>
<td style="text-align:left;">
21
</td>
<td style="text-align:right;">
34513142
</td>
<td style="text-align:right;">
34615113
</td>
<td style="text-align:right;">
101972
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000159200
</td>
<td style="text-align:left;">
RCAN1
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
regulator of calcineurin 1 [Source:HGNC Symbol;Acc:HGNC:3040]
</td>
<td style="text-align:left;">
ENSG00000159200.18
</td>
<td style="text-align:left;">
ENST00000313806
</td>
<td style="text-align:left;">
1827
</td>
<td style="text-align:right;">
3.216600
</td>
<td style="text-align:right;">
3.216789
</td>
<td style="text-align:right;">
7.8075066
</td>
<td style="text-align:right;">
7.271345e-10
</td>
<td style="text-align:right;">
9.181977e-08
</td>
<td style="text-align:left;">
21
</td>
<td style="text-align:right;">
34513142
</td>
<td style="text-align:right;">
34615113
</td>
<td style="text-align:right;">
101972
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000159200
</td>
<td style="text-align:left;">
RCAN1
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
regulator of calcineurin 1 [Source:HGNC Symbol;Acc:HGNC:3040]
</td>
<td style="text-align:left;">
ENSG00000159200.18
</td>
<td style="text-align:left;">
ENST00000313806
</td>
<td style="text-align:left;">
1827
</td>
<td style="text-align:right;">
2.976713
</td>
<td style="text-align:right;">
2.976897
</td>
<td style="text-align:right;">
7.8075066
</td>
<td style="text-align:right;">
4.125280e-09
</td>
<td style="text-align:right;">
4.240085e-07
</td>
<td style="text-align:left;">
21
</td>
<td style="text-align:right;">
34513142
</td>
<td style="text-align:right;">
34615113
</td>
<td style="text-align:right;">
101972
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000159200
</td>
<td style="text-align:left;">
RCAN1
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
regulator of calcineurin 1 [Source:HGNC Symbol;Acc:HGNC:3040]
</td>
<td style="text-align:left;">
ENSG00000159200.18
</td>
<td style="text-align:left;">
ENST00000313806
</td>
<td style="text-align:left;">
1827
</td>
<td style="text-align:right;">
2.072702
</td>
<td style="text-align:right;">
2.072863
</td>
<td style="text-align:right;">
7.8075066
</td>
<td style="text-align:right;">
1.001296e-04
</td>
<td style="text-align:right;">
2.481529e-03
</td>
<td style="text-align:left;">
21
</td>
<td style="text-align:right;">
34513142
</td>
<td style="text-align:right;">
34615113
</td>
<td style="text-align:right;">
101972
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000159200
</td>
<td style="text-align:left;">
RCAN1
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
regulator of calcineurin 1 [Source:HGNC Symbol;Acc:HGNC:3040]
</td>
<td style="text-align:left;">
ENSG00000159200.18
</td>
<td style="text-align:left;">
ENST00000313806
</td>
<td style="text-align:left;">
1827
</td>
<td style="text-align:right;">
3.090642
</td>
<td style="text-align:right;">
3.090829
</td>
<td style="text-align:right;">
7.8075066
</td>
<td style="text-align:right;">
1.741082e-09
</td>
<td style="text-align:right;">
2.219712e-07
</td>
</tr>
<tr>
<td style="text-align:left;">
S100P
</td>
<td style="text-align:left;">
4
</td>
<td style="text-align:right;">
6693878
</td>
<td style="text-align:right;">
6697170
</td>
<td style="text-align:right;">
3293
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000163993
</td>
<td style="text-align:left;">
S100P
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
S100 calcium binding protein P [Source:HGNC Symbol;Acc:HGNC:10504]
</td>
<td style="text-align:left;">
ENSG00000163993.7
</td>
<td style="text-align:left;">
ENST00000296370
</td>
<td style="text-align:left;">
6286
</td>
<td style="text-align:right;">
2.564134
</td>
<td style="text-align:right;">
2.595384
</td>
<td style="text-align:right;">
2.2311440
</td>
<td style="text-align:right;">
7.888512e-04
</td>
<td style="text-align:right;">
4.864827e-02
</td>
<td style="text-align:left;">
4
</td>
<td style="text-align:right;">
6693878
</td>
<td style="text-align:right;">
6697170
</td>
<td style="text-align:right;">
3293
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000163993
</td>
<td style="text-align:left;">
S100P
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
S100 calcium binding protein P [Source:HGNC Symbol;Acc:HGNC:10504]
</td>
<td style="text-align:left;">
ENSG00000163993.7
</td>
<td style="text-align:left;">
ENST00000296370
</td>
<td style="text-align:left;">
6286
</td>
<td style="text-align:right;">
3.824826
</td>
<td style="text-align:right;">
3.859741
</td>
<td style="text-align:right;">
2.2311440
</td>
<td style="text-align:right;">
1.330030e-09
</td>
<td style="text-align:right;">
1.574542e-07
</td>
<td style="text-align:left;">
4
</td>
<td style="text-align:right;">
6693878
</td>
<td style="text-align:right;">
6697170
</td>
<td style="text-align:right;">
3293
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000163993
</td>
<td style="text-align:left;">
S100P
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
S100 calcium binding protein P [Source:HGNC Symbol;Acc:HGNC:10504]
</td>
<td style="text-align:left;">
ENSG00000163993.7
</td>
<td style="text-align:left;">
ENST00000296370
</td>
<td style="text-align:left;">
6286
</td>
<td style="text-align:right;">
3.385775
</td>
<td style="text-align:right;">
3.419754
</td>
<td style="text-align:right;">
2.2311440
</td>
<td style="text-align:right;">
2.191055e-07
</td>
<td style="text-align:right;">
1.669609e-05
</td>
<td style="text-align:left;">
4
</td>
<td style="text-align:right;">
6693878
</td>
<td style="text-align:right;">
6697170
</td>
<td style="text-align:right;">
3293
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000163993
</td>
<td style="text-align:left;">
S100P
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
S100 calcium binding protein P [Source:HGNC Symbol;Acc:HGNC:10504]
</td>
<td style="text-align:left;">
ENSG00000163993.7
</td>
<td style="text-align:left;">
ENST00000296370
</td>
<td style="text-align:left;">
6286
</td>
<td style="text-align:right;">
6.407633
</td>
<td style="text-align:right;">
6.444730
</td>
<td style="text-align:right;">
2.2311440
</td>
<td style="text-align:right;">
1.996800e-22
</td>
<td style="text-align:right;">
1.470865e-19
</td>
<td style="text-align:left;">
4
</td>
<td style="text-align:right;">
6693878
</td>
<td style="text-align:right;">
6697170
</td>
<td style="text-align:right;">
3293
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
ENSG00000163993
</td>
<td style="text-align:left;">
S100P
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
S100 calcium binding protein P [Source:HGNC Symbol;Acc:HGNC:10504]
</td>
<td style="text-align:left;">
ENSG00000163993.7
</td>
<td style="text-align:left;">
ENST00000296370
</td>
<td style="text-align:left;">
6286
</td>
<td style="text-align:right;">
3.550059
</td>
<td style="text-align:right;">
3.584425
</td>
<td style="text-align:right;">
2.2311440
</td>
<td style="text-align:right;">
3.602129e-08
</td>
<td style="text-align:right;">
3.249023e-06
</td>
</tr>
</tbody>
</table>
</div></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul>
<pre class="r"><code>#merge the log cpm counts with the top 30 common de genes
logCPM &lt;- logCPM[common$gene[1:30],]

#set colour palette
my_palette &lt;- colorRampPalette(c(
  rgb(32,121,226, maxColorValue = 255),
  # rgb(144,203,180, maxColorValue = 255), 
  rgb(254,238,229, maxColorValue = 255), 
  # rgb(251,192,52, maxColorValue = 255), 
  rgb(226,46,45, maxColorValue = 255)))(n = 201)

#df for heatmap annotation of sample type
anno &lt;- as.factor(dge$samples$sample_type) %&gt;% as.data.frame 
colnames(anno) &lt;- &quot;Sample Type&quot;
anno$`Sample Type` &lt;-  gsub(&quot;single_deletion&quot;, &quot;Single Deletion&quot;, anno$`Sample Type`)
anno$`Sample Type` &lt;-  gsub(&quot;UT&quot;, &quot;Untreated&quot;, anno$`Sample Type`)
rownames(anno) &lt;- colnames(logCPM) 

#setting colour of sample type annotation
anno_colour &lt;- RColorBrewer::brewer.pal(3, &quot;Set3&quot;)
names(anno_colour) &lt;- c(&quot;CKI&quot;, &quot;Single Deletion&quot;, &quot;Untreated&quot;)
anno_colours &lt;- list(&quot;Sample Type&quot; = anno_colour) 


pheatmap(mat = logCPM,
         scale = &quot;row&quot;,
         clustering_distance_rows = &quot;euclidean&quot;,
         treeheight_row = 60,
         treeheight_col = 30,
         cutree_rows = 4,
         cutree_cols = 5,
         main = &quot;Heatmap of top 30 common DE genes&quot;,
         legend_breaks = c(seq(-2, 10, by = 2), max(logCPM)),
         legend_labels = c(seq(-2, 10, by = 2), &quot;logCPM\n&quot;),
         angle_col =90,
         
         annotation_legend = T,
         annotation_col = anno,
         annotation_colors = list(&quot;Sample Type&quot; = anno_colour),
         annotation_names_col = FALSE,
         annotation = TRUE,
         
         # border_color = F,
         
         border_color = &quot;gray70&quot;,
         color = my_palette
         )</code></pre>
<p><img src="figure/dge.Rmd/heatmap_common-1.png" width="768" style="display: block; margin: auto;" /></p>
<pre class="r"><code>invisible(dev.print(device = svg, here::here(&quot;2_plots/2_dge/heatmap_logCPM_common.svg&quot;)))</code></pre>
</div>
<div id="top" class="section level5">
<h5>Top</h5>
<p>This heatmap displays the expression of the most significant DE genes among all five comparisons. top 10 most significant DE genes from each comparison were combined into a single data frame. NA values were assigned to genes that were not commonly in the top 10. Since only the gene column is important for filtering the logCPM dataframe, the values within this table are the <code>FDR</code> values</p>
<pre class="r"><code>#create matrix with log cpm counts
logCPM &lt;- cpm(dge, prior.count=3, log=TRUE)
rownames(logCPM) &lt;- dge$genes$gene
colnames(logCPM) &lt;- paste(dge$samples$pretty_name, sep=&quot;-&quot;)

top_unique=list()
for (i in 1:ncol(contrast)){
  #at each iteration, let x = name of each contrast group
  x=comparison_group[i,]
  
top_unique[[x]] &lt;- treat_sig[[x]][1:10,]
}

unique &lt;- as.data.frame(do.call(rbind, lapply(top_unique, as.data.frame))) %&gt;% rownames_to_column(&quot;group&quot;) %&gt;% dplyr::select(&quot;gene&quot;, &quot;group&quot;, &quot;FDR&quot;) 
unique$group &lt;- gsub(&quot;\\..*&quot;, &quot;&quot;, unique$group)
unique &lt;- unique %&gt;% pivot_wider(names_from = &quot;group&quot;, values_from = &quot;FDR&quot;) %&gt;% column_to_rownames(&quot;gene&quot;)
kable(unique, digits = Inf) %&gt;%
  kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  scroll_box(height = &quot;600px&quot;)</code></pre>
<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
CKI-Mac vs CKI
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
CKI-Nme vs CKI
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
CKI-Omt vs CKI
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
CKI-Tri vs CKI
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
UT vs CKI
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
IL6
</td>
<td style="text-align:right;">
9.830412e-30
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
1.396151e-25
</td>
<td style="text-align:right;">
1.999585e-20
</td>
<td style="text-align:right;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
CXCL1
</td>
<td style="text-align:right;">
1.831725e-22
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
3.402214e-16
</td>
<td style="text-align:right;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
IL1B
</td>
<td style="text-align:right;">
3.272592e-22
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
2.679042e-25
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
PTGS2
</td>
<td style="text-align:right;">
2.459666e-21
</td>
<td style="text-align:right;">
3.895596e-20
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
2.084981e-20
</td>
<td style="text-align:right;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
VSTM2L
</td>
<td style="text-align:right;">
4.702600e-21
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
4.877022e-22
</td>
<td style="text-align:right;">
3.402214e-16
</td>
<td style="text-align:right;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
F2RL3
</td>
<td style="text-align:right;">
4.776750e-19
</td>
<td style="text-align:right;">
9.263864e-19
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
6.275650e-18
</td>
<td style="text-align:right;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
CLDN1
</td>
<td style="text-align:right;">
5.362561e-19
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
GPC1
</td>
<td style="text-align:right;">
7.053140e-19
</td>
<td style="text-align:right;">
3.174221e-19
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
3.322323e-16
</td>
<td style="text-align:right;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
CXCL3
</td>
<td style="text-align:right;">
1.025833e-18
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
3.402214e-16
</td>
<td style="text-align:right;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
SOD2
</td>
<td style="text-align:right;">
1.556643e-18
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
CYP1A1
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
1.524462e-21
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
7.201773e-20
</td>
<td style="text-align:right;">
3.941296e-26
</td>
</tr>
<tr>
<td style="text-align:left;">
RARRES2
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
3.174221e-19
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
1.865034e-17
</td>
<td style="text-align:right;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
TACSTD2
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
1.277234e-17
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
HSP90B1
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
4.091986e-17
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
LRP5
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
2.123996e-16
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
ABCA1
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
1.326749e-15
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
1.316648e-19
</td>
</tr>
<tr>
<td style="text-align:left;">
IGFBP7
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
1.555549e-15
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
MKI67
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
1.396151e-25
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
RRM2
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
7.160231e-25
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
AKR1C1
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
8.000538e-25
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
TCF19
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
8.040098e-24
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
UCA1
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
8.279726e-24
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
KIF18B
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
1.075558e-21
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
SNHG3
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
2.537396e-21
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
RPL22L1
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
1.915517e-16
</td>
<td style="text-align:right;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
CEMIP
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
2.301546e-25
</td>
</tr>
<tr>
<td style="text-align:left;">
CYP1B1
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
4.622571e-22
</td>
</tr>
<tr>
<td style="text-align:left;">
AKR1C3
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
6.581147e-19
</td>
</tr>
<tr>
<td style="text-align:left;">
ANGPTL4
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
9.392354e-19
</td>
</tr>
<tr>
<td style="text-align:left;">
IFIT2
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
1.155837e-17
</td>
</tr>
<tr>
<td style="text-align:left;">
AKR1B10
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
1.639902e-17
</td>
</tr>
<tr>
<td style="text-align:left;">
IFITM10
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
2.022709e-16
</td>
</tr>
<tr>
<td style="text-align:left;">
ASB2
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
2.225180e-16
</td>
</tr>
</tbody>
</table>
</div>
<pre class="r"><code>#merge the log cpm counts with the top 30 common de genes
logCPM &lt;- logCPM[rownames(unique),]</code></pre>
<p>The gene list (rowname) was then used to filter the logCPM transformed dataframe. This heatmap illustrates the expression of the those 33 genes.</p>
<pre class="r"><code>#set colour palette
my_palette &lt;- colorRampPalette(c(
  rgb(32,121,226, maxColorValue = 255),
  # rgb(144,203,180, maxColorValue = 255), 
  rgb(254,238,229, maxColorValue = 255), 
  # rgb(251,192,52, maxColorValue = 255), 
  rgb(226,46,45, maxColorValue = 255)))(n = 201)

#df for heatmap annotation of sample type
anno &lt;- as.factor(dge$samples$sample_type) %&gt;% as.data.frame 
colnames(anno) &lt;- &quot;Sample Type&quot;
anno$`Sample Type` &lt;-  gsub(&quot;single_deletion&quot;, &quot;Single Deletion&quot;, anno$`Sample Type`)
anno$`Sample Type` &lt;-  gsub(&quot;UT&quot;, &quot;Untreated&quot;, anno$`Sample Type`)
rownames(anno) &lt;- colnames(logCPM) 

#setting colour of sample type annotation
anno_colour &lt;- RColorBrewer::brewer.pal(3, &quot;Set3&quot;)
names(anno_colour) &lt;- c(&quot;CKI&quot;, &quot;Single Deletion&quot;, &quot;Untreated&quot;)
anno_colours &lt;- list(&quot;Sample Type&quot; = anno_colour) 

pheatmap(mat = logCPM,
         # scale = &quot;row&quot;,
         clustering_distance_rows = &quot;euclidean&quot;,
         treeheight_row = 60,
         treeheight_col = 30,
         cutree_rows = 4,
         cutree_cols = 4,
         main = &quot;Heatmap of top 10 DE genes&quot;,
         legend_breaks = c(seq(-2, 10, by = 2), max(logCPM)),
         legend_labels = c(seq(-2, 10, by = 2), &quot;logCPM\n&quot;),
         angle_col =90,
         
         annotation_legend = T,
         annotation_col = anno,
         annotation_colors = list(&quot;Sample Type&quot; = anno_colour),
         annotation_names_col = FALSE,
         annotation = TRUE,
         
         # border_color = F,
         
         border_color = &quot;gray70&quot;,
         color = my_palette
         )</code></pre>
<p><img src="figure/dge.Rmd/heatmap_unique-1.png" width="768" style="display: block; margin: auto;" /></p>
<pre class="r"><code>invisible(dev.print(device = svg, here::here(&quot;2_plots/2_dge/heatmap_logCPM_unique.svg&quot;)))</code></pre>
</div>
</div>
</div>
</div>
<div id="export-data" class="section level2">
<h2>Export Data</h2>
<pre class="r"><code>designMatrix %&gt;% saveRDS(here::here(&quot;0_data/rds_object/designMatrix.rds&quot;))
contrast %&gt;% saveRDS(here::here(&quot;0_data/rds_object/contrastMatrix.rds&quot;))

# save rds object for use in downstream GO and KEGG analysis
qlf %&gt;% saveRDS(file = here::here(&quot;0_data/rds_object/qlf.rds&quot;))
qlf_all %&gt;% saveRDS(file = here::here(&quot;0_data/rds_object/qlf_all.rds&quot;))
qlf_sig %&gt;% saveRDS(file = here::here(&quot;0_data/rds_object/qlf_sig.rds&quot;))
treat %&gt;% saveRDS(file = here::here(&quot;0_data/rds_object/treat.rds&quot;))
treat_all %&gt;% saveRDS(file = here::here(&quot;0_data/rds_object/treat_all.rds&quot;))
treat_sig %&gt;% saveRDS(file = here::here(&quot;0_data/rds_object/treat_sig.rds&quot;))

#save each all comparison group in the output directory
writexl::write_xlsx(x = qlf_decideTest, here::here(&quot;3_output/qlf_decideTest.xlsx&quot;))
writexl::write_xlsx(x = qlf_sig, here::here(&quot;3_output/qlf_sig.xlsx&quot;))
writexl::write_xlsx(x = treat_decideTest, here::here(&quot;3_output/treat_decideTest.xlsx&quot;))
writexl::write_xlsx(x = treat_sig, here::here(&quot;3_output/treat_sig.xlsx&quot;))</code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.1.0 (2021-05-18)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19043)

Matrix products: default

locale:
[1] LC_COLLATE=English_Australia.1252  LC_CTYPE=English_Australia.1252   
[3] LC_MONETARY=English_Australia.1252 LC_NUMERIC=C                      
[5] LC_TIME=English_Australia.1252    

attached base packages:
[1] grid      stats     graphics  grDevices utils     datasets  methods  
[8] base     

other attached packages:
 [1] Glimma_2.2.0       edgeR_3.34.0       limma_3.48.1       ggrepel_0.9.1     
 [5] ggbiplot_0.55      scales_1.1.1       plyr_1.8.6         ggpubr_0.4.0      
 [9] RColorBrewer_1.1-2 pheatmap_1.0.12    cowplot_1.1.1      pander_0.6.4      
[13] kableExtra_1.3.4   forcats_0.5.1      stringr_1.4.0      purrr_0.3.4       
[17] tidyr_1.1.3        ggplot2_3.3.5      tidyverse_1.3.1    reshape2_1.4.4    
[21] tibble_3.1.2       readr_2.0.0        magrittr_2.0.1     dplyr_1.0.7       

loaded via a namespace (and not attached):
  [1] readxl_1.3.1                backports_1.2.1            
  [3] workflowr_1.6.2             systemfonts_1.0.2          
  [5] splines_4.1.0               BiocParallel_1.26.1        
  [7] GenomeInfoDb_1.28.4         digest_0.6.27              
  [9] htmltools_0.5.2             fansi_0.5.0                
 [11] memoise_2.0.0               tzdb_0.1.2                 
 [13] openxlsx_4.2.4              Biostrings_2.60.1          
 [15] annotate_1.70.0             modelr_0.1.8               
 [17] matrixStats_0.60.1          svglite_2.0.0              
 [19] colorspace_2.0-2            blob_1.2.2                 
 [21] rvest_1.0.2                 haven_2.4.3                
 [23] xfun_0.24                   crayon_1.4.1               
 [25] RCurl_1.98-1.3              jsonlite_1.7.2             
 [27] genefilter_1.74.0           survival_3.2-11            
 [29] glue_1.4.2                  gtable_0.3.0               
 [31] zlibbioc_1.38.0             XVector_0.32.0             
 [33] webshot_0.5.2               DelayedArray_0.18.0        
 [35] car_3.0-11                  BiocGenerics_0.38.0        
 [37] abind_1.4-5                 DBI_1.1.1                  
 [39] rstatix_0.7.0               Rcpp_1.0.6                 
 [41] viridisLite_0.4.0           xtable_1.8-4               
 [43] foreign_0.8-81              bit_4.0.4                  
 [45] stats4_4.1.0                htmlwidgets_1.5.4          
 [47] httr_1.4.2                  ellipsis_0.3.2             
 [49] farver_2.1.0                pkgconfig_2.0.3            
 [51] XML_3.99-0.6                sass_0.4.0                 
 [53] dbplyr_2.1.1                here_1.0.1                 
 [55] locfit_1.5-9.4              utf8_1.2.1                 
 [57] labeling_0.4.2              tidyselect_1.1.1           
 [59] rlang_0.4.11                later_1.2.0                
 [61] AnnotationDbi_1.54.1        munsell_0.5.0              
 [63] cellranger_1.1.0            tools_4.1.0                
 [65] cachem_1.0.5                cli_3.0.1                  
 [67] generics_0.1.0              RSQLite_2.2.7              
 [69] broom_0.7.9                 evaluate_0.14              
 [71] fastmap_1.1.0               yaml_2.2.1                 
 [73] knitr_1.36                  bit64_4.0.5                
 [75] fs_1.5.0                    zip_2.2.0                  
 [77] KEGGREST_1.32.0             whisker_0.4                
 [79] xml2_1.3.2                  compiler_4.1.0             
 [81] rstudioapi_0.13             curl_4.3.2                 
 [83] png_0.1-7                   ggsignif_0.6.3             
 [85] reprex_2.0.1                statmod_1.4.36             
 [87] geneplotter_1.70.0          bslib_0.3.1                
 [89] stringi_1.6.2               highr_0.9                  
 [91] lattice_0.20-44             Matrix_1.3-3               
 [93] vctrs_0.3.8                 pillar_1.6.4               
 [95] lifecycle_1.0.1             jquerylib_0.1.4            
 [97] data.table_1.14.0           bitops_1.0-7               
 [99] httpuv_1.6.1                GenomicRanges_1.44.0       
[101] R6_2.5.1                    promises_1.2.0.1           
[103] rio_0.5.27                  writexl_1.4.0              
[105] IRanges_2.26.0              assertthat_0.2.1           
[107] SummarizedExperiment_1.22.0 DESeq2_1.32.0              
[109] rprojroot_2.0.2             withr_2.4.2                
[111] S4Vectors_0.30.0            GenomeInfoDbData_1.2.6     
[113] parallel_4.1.0              hms_1.1.1                  
[115] rmarkdown_2.11              MatrixGenerics_1.4.3       
[117] carData_3.0-4               git2r_0.28.0               
[119] Biobase_2.52.0              lubridate_1.7.10           </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
