<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Ha Tran" />

<meta name="date" content="2021-08-09" />

<title>GO Analysis</title>

<script src="site_libs/header-attrs-2.11/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/lightable-0.0.1/lightable.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">CKI_DGE</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Setting Up
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="obtainData.html">Obtaining Data</a>
    </li>
    <li>
      <a href="setUp.html">QC and Pre-processing</a>
    </li>
  </ul>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Analysis
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="dge.html">DE Analysis</a>
    </li>
    <li>
      <a href="go.html">GO Term Enrichment Analysis</a>
    </li>
    <li>
      <a href="kegg.html">KEGG Pathway Enrichment Analysis</a>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">GO Analysis</h1>
<h4 class="author">Ha Tran</h4>
<h4 class="date">08/09/2021</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2021-11-13
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 5 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 2
</p>
<p>
<strong>Knit directory:</strong> <code>01_CKI_DGE/1_analysis/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges" class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown file has unstaged changes. To know which version of the R Markdown file created these results, you’ll want to first commit it to the Git repo. If you’re still working on the analysis, you can ignore this warning. When you’re finished, you can run <code>wflow_publish</code> to commit the R Markdown file and build the HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongobjectspresent"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>Environment:</strong> objects present </a>
</p>
</div>
<div id="strongEnvironmentstrongobjectspresent" class="panel-collapse collapse">
<div class="panel-body">
<p>
The global environment had objects present when the code in the R Markdown file was run. These objects can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment. Use <code>wflow_publish</code> or <code>wflow_build</code> to ensure that the code is always run in an empty environment.
</p>
<p>
The following objects were defined in the global environment when these results were created:
</p>
<table class="table table-condensed table-hover">
<thead>
<tr>
<th style="text-align:left;">
Name
</th>
<th style="text-align:left;">
Class
</th>
<th style="text-align:left;">
Size
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
anno
</td>
<td style="text-align:left;">
data.frame
</td>
<td style="text-align:left;">
2.3 Kb
</td>
</tr>
<tr>
<td style="text-align:left;">
anno_colour
</td>
<td style="text-align:left;">
character
</td>
<td style="text-align:left;">
624 bytes
</td>
</tr>
<tr>
<td style="text-align:left;">
anno_colours
</td>
<td style="text-align:left;">
list
</td>
<td style="text-align:left;">
912 bytes
</td>
</tr>
<tr>
<td style="text-align:left;">
common
</td>
<td style="text-align:left;">
data.frame
</td>
<td style="text-align:left;">
310 Kb
</td>
</tr>
<tr>
<td style="text-align:left;">
comparison_group
</td>
<td style="text-align:left;">
data.frame
</td>
<td style="text-align:left;">
1.1 Kb
</td>
</tr>
<tr>
<td style="text-align:left;">
contrast
</td>
<td style="text-align:left;">
matrix;array
</td>
<td style="text-align:left;">
1.7 Kb
</td>
</tr>
<tr>
<td style="text-align:left;">
designMatrix
</td>
<td style="text-align:left;">
matrix;array
</td>
<td style="text-align:left;">
3.5 Kb
</td>
</tr>
<tr>
<td style="text-align:left;">
dge
</td>
<td style="text-align:left;">
DGEList
</td>
<td style="text-align:left;">
12 Mb
</td>
</tr>
<tr>
<td style="text-align:left;">
i
</td>
<td style="text-align:left;">
integer
</td>
<td style="text-align:left;">
56 bytes
</td>
</tr>
<tr>
<td style="text-align:left;">
logCPM
</td>
<td style="text-align:left;">
matrix;array
</td>
<td style="text-align:left;">
8.4 Kb
</td>
</tr>
<tr>
<td style="text-align:left;">
MAplot
</td>
<td style="text-align:left;">
list
</td>
<td style="text-align:left;">
48 bytes
</td>
</tr>
<tr>
<td style="text-align:left;">
my_palette
</td>
<td style="text-align:left;">
character
</td>
<td style="text-align:left;">
12.6 Kb
</td>
</tr>
<tr>
<td style="text-align:left;">
NB_dispersion
</td>
<td style="text-align:left;">
DGEList
</td>
<td style="text-align:left;">
12.5 Mb
</td>
</tr>
<tr>
<td style="text-align:left;">
qlf
</td>
<td style="text-align:left;">
list
</td>
<td style="text-align:left;">
101 Mb
</td>
</tr>
<tr>
<td style="text-align:left;">
qlf_all
</td>
<td style="text-align:left;">
list
</td>
<td style="text-align:left;">
49.2 Mb
</td>
</tr>
<tr>
<td style="text-align:left;">
qlf_decideTest
</td>
<td style="text-align:left;">
list
</td>
<td style="text-align:left;">
10.8 Kb
</td>
</tr>
<tr>
<td style="text-align:left;">
qlf_dispersion
</td>
<td style="text-align:left;">
DGEGLM
</td>
<td style="text-align:left;">
20.6 Mb
</td>
</tr>
<tr>
<td style="text-align:left;">
qlf_sig
</td>
<td style="text-align:left;">
list
</td>
<td style="text-align:left;">
37.5 Mb
</td>
</tr>
<tr>
<td style="text-align:left;">
top_unique
</td>
<td style="text-align:left;">
list
</td>
<td style="text-align:left;">
256.4 Kb
</td>
</tr>
<tr>
<td style="text-align:left;">
treat
</td>
<td style="text-align:left;">
list
</td>
<td style="text-align:left;">
96.1 Mb
</td>
</tr>
<tr>
<td style="text-align:left;">
treat_all
</td>
<td style="text-align:left;">
list
</td>
<td style="text-align:left;">
49.2 Mb
</td>
</tr>
<tr>
<td style="text-align:left;">
treat_decideTest
</td>
<td style="text-align:left;">
list
</td>
<td style="text-align:left;">
10.8 Kb
</td>
</tr>
<tr>
<td style="text-align:left;">
treat_sig
</td>
<td style="text-align:left;">
list
</td>
<td style="text-align:left;">
1.7 Mb
</td>
</tr>
<tr>
<td style="text-align:left;">
unique
</td>
<td style="text-align:left;">
data.frame
</td>
<td style="text-align:left;">
4.6 Kb
</td>
</tr>
<tr>
<td style="text-align:left;">
x
</td>
<td style="text-align:left;">
character
</td>
<td style="text-align:left;">
120 bytes
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed12345code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(12345)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed12345code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(12345)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomtranmanhha13501CKIDGEtree848e971e9ccbf102898eeb089076953d68a0817ftargetblank848e971a"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/tranmanhha135/01_CKI_DGE/tree/848e971e9ccbf102898eeb089076953d68a0817f" target="_blank">848e971</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomtranmanhha13501CKIDGEtree848e971e9ccbf102898eeb089076953d68a0817ftargetblank848e971a" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/tranmanhha135/01_CKI_DGE/tree/848e971e9ccbf102898eeb089076953d68a0817f" target="_blank">848e971</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/

Untracked files:
    Untracked:  0_data/rds_object/qlf_all.rds
    Untracked:  0_data/rds_object/qlf_sig.rds
    Untracked:  0_data/rds_object/treat_all.rds
    Untracked:  0_data/rds_object/treat_sig.rds
    Untracked:  1_analysis/about.html
    Untracked:  1_analysis/dge.html
    Untracked:  1_analysis/figure/dge.Rmd/NBDispersion-1.png
    Untracked:  1_analysis/figure/dge.Rmd/coolmap-1.png
    Untracked:  1_analysis/figure/dge.Rmd/heatmap_common-1.png
    Untracked:  1_analysis/figure/dge.Rmd/heatmap_unique-1.png
    Untracked:  1_analysis/figure/dge.Rmd/qlf_dispersion-1.png
    Untracked:  1_analysis/figure/dge.Rmd/volcano_treat-1.png
    Untracked:  1_analysis/figure/dge.Rmd/volcano_treat-2.png
    Untracked:  1_analysis/figure/dge.Rmd/volcano_treat-3.png
    Untracked:  1_analysis/figure/dge.Rmd/volcano_treat-4.png
    Untracked:  1_analysis/figure/dge.Rmd/volcano_treat-5.png
    Untracked:  1_analysis/figure/go.Rmd/
    Untracked:  1_analysis/figure/kegg.Rmd/completePathway-1.png
    Untracked:  1_analysis/figure/kegg.Rmd/dotPlot-1.png
    Untracked:  1_analysis/figure/kegg.Rmd/partialPathway, -1.png
    Untracked:  1_analysis/figure/kegg.Rmd/partialPathway-1.png
    Untracked:  1_analysis/figure/setUp.Rmd/afterFiltering-2.png
    Untracked:  1_analysis/site_libs/
    Untracked:  2_plots/1_setup/after_normalisation.svg
    Untracked:  2_plots/2_dge/MA_treat_CKI-Mac vs CKI.svg
    Untracked:  2_plots/2_dge/MA_treat_CKI-Nme vs CKI.svg
    Untracked:  2_plots/2_dge/MA_treat_CKI-Omt vs CKI.svg
    Untracked:  2_plots/2_dge/MA_treat_CKI-Tri vs CKI.svg
    Untracked:  2_plots/2_dge/MA_treat_UT vs CKI.svg
    Untracked:  2_plots/2_dge/NB_dispersion.svg
    Untracked:  2_plots/2_dge/heatmap_logCPM_common.svg
    Untracked:  2_plots/2_dge/heatmap_logCPM_unique.svg
    Untracked:  2_plots/2_dge/pValue_treat_CKI-Mac vs CKI.svg
    Untracked:  2_plots/2_dge/pValue_treat_CKI-Nme vs CKI.svg
    Untracked:  2_plots/2_dge/pValue_treat_CKI-Omt vs CKI.svg
    Untracked:  2_plots/2_dge/pValue_treat_CKI-Tri vs CKI.svg
    Untracked:  2_plots/2_dge/pValue_treat_UT vs CKI.svg
    Untracked:  2_plots/2_dge/volcano_treat_CKI-Mac vs CKI.svg
    Untracked:  2_plots/2_dge/volcano_treat_CKI-Nme vs CKI.svg
    Untracked:  2_plots/2_dge/volcano_treat_CKI-Omt vs CKI.svg
    Untracked:  2_plots/2_dge/volcano_treat_CKI-Tri vs CKI.svg
    Untracked:  2_plots/2_dge/volcano_treat_UT vs CKI.svg
    Untracked:  2_plots/3_go/
    Untracked:  2_plots/4_kegg/
    Untracked:  3_output/enrichKEGG_treat_sig.xlsx
    Untracked:  3_output/goana_treat.xlsx
    Untracked:  3_output/kegga_treat_sig.xlsx
    Untracked:  3_output/qlf_decideTest.xlsx
    Untracked:  3_output/qlf_sig.xlsx
    Untracked:  3_output/treat_decideTest.xlsx
    Untracked:  3_output/treat_sig.xlsx
    Untracked:  Meeting Notes.pdf

Unstaged changes:
    Modified:   0_data/rds_object/dge.rds
    Deleted:    0_data/rds_object/dge_treat_unfiltered.rds
    Deleted:    0_data/rds_object/enrichGO_treat.rds
    Deleted:    0_data/rds_object/enrichKEGG_treat_filtered.rds
    Modified:   0_data/rds_object/qlf.rds
    Deleted:    0_data/rds_object/qlf_filtered.rds
    Modified:   0_data/rds_object/treat.rds
    Deleted:    0_data/rds_object/treat_filtered.rds
    Deleted:    0_data/rds_object/treat_unfiltered.rds
    Modified:   1_analysis/dge.Rmd
    Modified:   1_analysis/figure/dge.Rmd/MAplotCustom_treat-1.png
    Modified:   1_analysis/figure/dge.Rmd/MAplotCustom_treat-2.png
    Modified:   1_analysis/figure/dge.Rmd/MAplotCustom_treat-3.png
    Modified:   1_analysis/figure/dge.Rmd/MAplotCustom_treat-4.png
    Modified:   1_analysis/figure/dge.Rmd/MAplotCustom_treat-5.png
    Modified:   1_analysis/figure/dge.Rmd/pValueHistogram_treat-1.png
    Modified:   1_analysis/figure/dge.Rmd/pValueHistogram_treat-2.png
    Modified:   1_analysis/figure/dge.Rmd/pValueHistogram_treat-3.png
    Modified:   1_analysis/figure/dge.Rmd/pValueHistogram_treat-4.png
    Modified:   1_analysis/figure/dge.Rmd/pValueHistogram_treat-5.png
    Modified:   1_analysis/figure/setUp.Rmd/MDSplot-1.png
    Modified:   1_analysis/figure/setUp.Rmd/afterFiltering-1.png
    Modified:   1_analysis/figure/setUp.Rmd/libraryChecks-1.png
    Modified:   1_analysis/figure/setUp.Rmd/pcaSampleGroup-1.png
    Modified:   1_analysis/figure/setUp.Rmd/pcaSampleType-1.png
    Modified:   1_analysis/figure/setUp.Rmd/visualisingNormalisation-1.png
    Modified:   1_analysis/go.Rmd
    Modified:   1_analysis/kegg.Rmd
    Modified:   1_analysis/setUp.Rmd
    Modified:   2_plots/1_setup/counts_before_and_after_filtering.svg
    Modified:   2_plots/1_setup/library_size.svg
    Modified:   2_plots/1_setup/sample_group_PCA_plot.svg
    Modified:   2_plots/1_setup/sample_type_PCA_plot.svg
    Deleted:    2_plots/2_dge/MA_plot_treat_CKI-Mac vs CKI.svg
    Deleted:    2_plots/2_dge/MA_plot_treat_CKI-Nme vs CKI.svg
    Deleted:    2_plots/2_dge/MA_plot_treat_CKI-Omt vs CKI.svg
    Deleted:    2_plots/2_dge/MA_plot_treat_CKI-Tri vs CKI.svg
    Deleted:    2_plots/2_dge/MA_plot_treat_UT vs CKI.svg
    Deleted:    2_plots/2_dge/estimate_dispersion.svg
    Modified:   2_plots/2_dge/qlf_dispersion.svg
    Deleted:    2_plots/2_dge/qlf_pValue_histogram_CKI-Mac vs CKI.svg
    Deleted:    2_plots/2_dge/qlf_pValue_histogram_CKI-Nme vs CKI.svg
    Deleted:    2_plots/2_dge/qlf_pValue_histogram_CKI-Omt vs CKI.svg
    Deleted:    2_plots/2_dge/qlf_pValue_histogram_CKI-Tri vs CKI.svg
    Deleted:    2_plots/2_dge/qlf_pValue_histogram_UT vs CKI.svg
    Deleted:    2_plots/2_dge/treat_pValue_histogram_CKI-Mac vs CKI.svg
    Deleted:    2_plots/2_dge/treat_pValue_histogram_CKI-Nme vs CKI.svg
    Deleted:    2_plots/2_dge/treat_pValue_histogram_CKI-Omt vs CKI.svg
    Deleted:    2_plots/2_dge/treat_pValue_histogram_CKI-Tri vs CKI.svg
    Deleted:    2_plots/2_dge/treat_pValue_histogram_UT vs CKI.svg
    Deleted:    2_plots/2_dge/volcano_plot_treat_CKI-Mac vs CKI.svg
    Deleted:    2_plots/2_dge/volcano_plot_treat_CKI-Nme vs CKI.svg
    Deleted:    2_plots/2_dge/volcano_plot_treat_CKI-Omt vs CKI.svg
    Deleted:    2_plots/2_dge/volcano_plot_treat_CKI-Tri vs CKI.svg
    Deleted:    2_plots/2_dge/volcano_plot_treat_UT vs CKI.svg
    Deleted:    3_output/enrichGO_treat.xlsx
    Deleted:    3_output/enrichKEGG_treat_unfiltered.xlsx
    Deleted:    3_output/qlf_filtered.xlsx
    Deleted:    3_output/treat_filtered.xlsx

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>1_analysis/go.Rmd</code>) and HTML (<code>docs/go.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/01_CKI_DGE/blob/848e971e9ccbf102898eeb089076953d68a0817f/1_analysis/go.Rmd" target="_blank">848e971</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-11-08
</td>
<td>
adjust plot size, added dot plot for GO and KEGG, added heatmap for DE genes, minor cosmetic changes
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/01_CKI_DGE/848e971e9ccbf102898eeb089076953d68a0817f/docs/go.html" target="_blank">848e971</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-11-08
</td>
<td>
adjust plot size, added dot plot for GO and KEGG, added heatmap for DE genes, minor cosmetic changes
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/01_CKI_DGE/blob/70049b22b9a79841a2368454293f8825f08980dd/1_analysis/go.Rmd" target="_blank">70049b2</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-31
</td>
<td>
Remove old GO and KEGG code, adjusted directory paths, updated KEGG, minor cosmetic changes
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/01_CKI_DGE/70049b22b9a79841a2368454293f8825f08980dd/docs/go.html" target="_blank">70049b2</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-31
</td>
<td>
Remove old GO and KEGG code, adjusted directory paths, updated KEGG, minor cosmetic changes
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/01_CKI_DGE/6823b3c881eddd07176bce86153e0543293d7f02/docs/go.html" target="_blank">6823b3c</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-30
</td>
<td>
Add minor cosmetic changes
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/01_CKI_DGE/blob/7a6571ba908cf21e984811a75b8123b1e28a4d69/1_analysis/go.Rmd" target="_blank">7a6571b</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-30
</td>
<td>
minor changes
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/01_CKI_DGE/7a6571ba908cf21e984811a75b8123b1e28a4d69/docs/go.html" target="_blank">7a6571b</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-30
</td>
<td>
minor changes
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/01_CKI_DGE/f5f4f74bfd77b68627ec8414b040c66a60ed7344/docs/go.html" target="_blank">f5f4f74</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-30
</td>
<td>
please please please
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/01_CKI_DGE/blob/a6d8f8a05c71e8da432f180ed252874e35ca0ae1/1_analysis/go.Rmd" target="_blank">a6d8f8a</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-30
</td>
<td>
what is happening
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/01_CKI_DGE/a6d8f8a05c71e8da432f180ed252874e35ca0ae1/docs/go.html" target="_blank">a6d8f8a</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-30
</td>
<td>
what is happening
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/01_CKI_DGE/blob/d769a7cbf44fe4841bdf220ed214b5f73c00b47a/1_analysis/go.Rmd" target="_blank">d769a7c</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-30
</td>
<td>
Upload GO and KEGG analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/01_CKI_DGE/d769a7cbf44fe4841bdf220ed214b5f73c00b47a/docs/go.html" target="_blank">d769a7c</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-30
</td>
<td>
Upload GO and KEGG analysis
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="data-setup" class="section level1">
<h1>Data Setup</h1>
<div id="load-library" class="section level3">
<h3>Load Library</h3>
<pre class="r"><code># working with data
library(dplyr)
library(magrittr)
library(readr)
library(tibble)
library(reshape2)
library(tidyverse)

# Visualisation:
library(kableExtra)
library(ggplot2)
library(grid)
library(pander)
library(cowplot)

# Custom ggplot
library(ggbiplot)
library(ggrepel)

# Bioconductor packages:
library(edgeR)
library(limma)
library(Glimma)
library(clusterProfiler)
library(org.Hs.eg.db)
# library(enrichplot)
library(biomaRt)

theme_set(theme_minimal())</code></pre>
</div>
<div id="import-rds-data" class="section level3">
<h3>Import RDS Data</h3>
<p>DGElist object containing the raw feature count, sample metadata, and gene metadata, created in the Set Up stage.</p>
<pre class="r"><code># load DGElist previously created in the set up
dge &lt;- readRDS(here::here(&quot;0_data/rds_object/dge.rds&quot;))
designMatrix &lt;- readRDS(here::here(&quot;0_data/rds_object/designMatrix.rds&quot;))
contrast &lt;- readRDS(here::here(&quot;0_data/rds_object/contrastMatrix.rds&quot;))
treat &lt;- readRDS(here::here(&quot;0_data/rds_object/treat.rds&quot;))
treat_sig &lt;- readRDS(here::here(&quot;0_data/rds_object/treat_sig.rds&quot;))</code></pre>
</div>
</div>
<div id="gene-ontology-go-enrichment-analysis" class="section level1">
<h1>Gene Ontology (GO) Enrichment Analysis</h1>
<p>Gene ontology can be perfomred with many packages. <code>edgeR</code> can be used to perform GO enrihcment analysis</p>
<div id="using-the-edgergoanna-package" class="section level3 tabset">
<h3 class="tabset">Using the <code>edgeR::goanna</code> package</h3>
<pre class="r"><code>comparison_group &lt;- colnames(contrast) %&gt;% as.data.frame()
goSummaries &lt;- url(&quot;https://uofabioinformaticshub.github.io/summaries2GO/data/goSummaries.RDS&quot;) %&gt;%
  readRDS()
minPath &lt;- 3

# Create list object
goana_treat &lt;- list()
goana_treat_sig &lt;- list()

for (i in 1:ncol(contrast)) {
  # at each iteration, let x = name of each contrast group
  x &lt;- comparison_group[i, ]

  # populate the goanna list with all GO terms
  goana_treat[[x]] &lt;- edgeR::goana.DGELRT(
    de = treat[[x]],
    geneid = treat[[x]]$genes$entrezid,
    FDR = 0.05,
    species = &quot;Hs&quot;,
    trend = T,
    ont = &quot;ALL&quot;
  )

  # extract the top 300 enriched results
  goana_treat_sig[[x]] &lt;- topGO(results = goana_treat[[x]], ontology = c(&quot;MF&quot;, &quot;BP&quot;, &quot;CC&quot;), number = 300)

  # bind with goSummary to filter out ontologies above level 3
  goana_treat_sig[[x]] &lt;- goana_treat_sig[[x]] %&gt;%
    rownames_to_column(&quot;id&quot;) %&gt;%
    left_join(goSummaries) %&gt;%
    dplyr::filter(shortest_path &gt;= minPath) %&gt;%
    column_to_rownames(&quot;id&quot;)
}</code></pre>
<!-- ### Using the `clusterProfiler::enrichGO` package {.tabset} -->
<!-- GO enrichment analysis is performed using the DGE obtained from GLM:QLF + TREAT. -->
<!-- The top 30 most significant GO terms are displayed in relation to their respective comparison -->
<pre class="r"><code>comparison_group &lt;- colnames(contrast) %&gt;% as.data.frame()
goSummaries &lt;- url(&quot;https://uofabioinformaticshub.github.io/summaries2GO/data/goSummaries.RDS&quot;) %&gt;%
  readRDS()
minPath &lt;- 3

# Create list object
enrichGO_treat &lt;- list()
for (i in 1:ncol(contrast)) {
  # at each iteration, let x = name of each contrast group
  x &lt;- comparison_group[i, ]

  # populate the enrichGo list with all GO terms
  GOresults &lt;- enrichGO(
    gene = treat_filtered[[x]]$entrezid,
    keyType = &quot;ENTREZID&quot;,
    ont = &quot;ALL&quot;,
    OrgDb = org.Hs.eg.db,
    pAdjustMethod = &quot;fdr&quot;
  )

  enrichGO_treat[[x]] &lt;- GOresults@result

  enrichGO_treat[[x]] &lt;- enrichGO_treat[[x]] %&gt;%
    rownames_to_column(&quot;id&quot;) %&gt;%
    left_join(goSummaries) %&gt;%
    dplyr::filter(shortest_path &gt;= minPath) %&gt;%
    column_to_rownames(&quot;id&quot;)
}</code></pre>
<div id="untreated-vs-cki" class="section level4">
<h4>Untreated vs CKI</h4>
<pre class="r"><code>goana_treat_sig[[&quot;UT vs CKI&quot;]][1:30, ] %&gt;%
  as.data.frame() %&gt;%
  dplyr::select(-c(&quot;shortest_path&quot;, &quot;longest_path&quot;, &quot;terminal_node&quot;, &quot;ontology&quot;)) %&gt;%
  kable(digits = Inf) %&gt;%
  kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  scroll_box(height = &quot;600px&quot;)</code></pre>
<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Term
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Ont
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
N
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Up
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Down
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
P.Up
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
P.Down
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
<a href="GO:0005102" class="uri">GO:0005102</a>
</td>
<td style="text-align:left;">
signaling receptor binding
</td>
<td style="text-align:left;">
MF
</td>
<td style="text-align:right;">
861
</td>
<td style="text-align:right;">
26
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
5.713193e-10
</td>
<td style="text-align:right;">
8.416245e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0010033" class="uri">GO:0010033</a>
</td>
<td style="text-align:left;">
response to organic substance
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
2155
</td>
<td style="text-align:right;">
39
</td>
<td style="text-align:right;">
28
</td>
<td style="text-align:right;">
1.483479e-09
</td>
<td style="text-align:right;">
1.462338e-03
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0034754" class="uri">GO:0034754</a>
</td>
<td style="text-align:left;">
cellular hormone metabolic process
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
62
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
1.000000e+00
</td>
<td style="text-align:right;">
7.272928e-09
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0007166" class="uri">GO:0007166</a>
</td>
<td style="text-align:left;">
cell surface receptor signaling pathway
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
1896
</td>
<td style="text-align:right;">
36
</td>
<td style="text-align:right;">
21
</td>
<td style="text-align:right;">
1.399619e-08
</td>
<td style="text-align:right;">
6.156658e-02
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0030335" class="uri">GO:0030335</a>
</td>
<td style="text-align:left;">
positive regulation of cell migration
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
356
</td>
<td style="text-align:right;">
15
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
1.993030e-08
</td>
<td style="text-align:right;">
2.768071e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0043207" class="uri">GO:0043207</a>
</td>
<td style="text-align:left;">
response to external biotic stimulus
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
857
</td>
<td style="text-align:right;">
23
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
2.627677e-08
</td>
<td style="text-align:right;">
8.049658e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:2000147" class="uri">GO:2000147</a>
</td>
<td style="text-align:left;">
positive regulation of cell motility
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
364
</td>
<td style="text-align:right;">
15
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
3.084872e-08
</td>
<td style="text-align:right;">
2.968605e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0051272" class="uri">GO:0051272</a>
</td>
<td style="text-align:left;">
positive regulation of cellular component movement
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
372
</td>
<td style="text-align:right;">
15
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
4.255778e-08
</td>
<td style="text-align:right;">
3.122136e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0071310" class="uri">GO:0071310</a>
</td>
<td style="text-align:left;">
cellular response to organic substance
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
1761
</td>
<td style="text-align:right;">
32
</td>
<td style="text-align:right;">
22
</td>
<td style="text-align:right;">
6.218861e-08
</td>
<td style="text-align:right;">
7.357403e-03
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0034097" class="uri">GO:0034097</a>
</td>
<td style="text-align:left;">
response to cytokine
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
796
</td>
<td style="text-align:right;">
21
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
7.792062e-08
</td>
<td style="text-align:right;">
3.824849e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0019221" class="uri">GO:0019221</a>
</td>
<td style="text-align:left;">
cytokine-mediated signaling pathway
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
516
</td>
<td style="text-align:right;">
17
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
9.160960e-08
</td>
<td style="text-align:right;">
3.531970e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0042448" class="uri">GO:0042448</a>
</td>
<td style="text-align:left;">
progesterone metabolic process
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
1.000000e+00
</td>
<td style="text-align:right;">
1.075049e-07
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0030647" class="uri">GO:0030647</a>
</td>
<td style="text-align:left;">
aminoglycoside antibiotic metabolic process
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
1.000000e+00
</td>
<td style="text-align:right;">
2.097285e-07
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0044597" class="uri">GO:0044597</a>
</td>
<td style="text-align:left;">
daunorubicin metabolic process
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
1.000000e+00
</td>
<td style="text-align:right;">
2.097285e-07
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0044598" class="uri">GO:0044598</a>
</td>
<td style="text-align:left;">
doxorubicin metabolic process
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
1.000000e+00
</td>
<td style="text-align:right;">
2.097285e-07
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0030638" class="uri">GO:0030638</a>
</td>
<td style="text-align:left;">
polyketide metabolic process
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
1.000000e+00
</td>
<td style="text-align:right;">
2.097285e-07
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0008106" class="uri">GO:0008106</a>
</td>
<td style="text-align:left;">
alcohol dehydrogenase (NADP+) activity
</td>
<td style="text-align:left;">
MF
</td>
<td style="text-align:right;">
15
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
1.000000e+00
</td>
<td style="text-align:right;">
2.727044e-07
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0034308" class="uri">GO:0034308</a>
</td>
<td style="text-align:left;">
primary alcohol metabolic process
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
48
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
1.000000e+00
</td>
<td style="text-align:right;">
2.865939e-07
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0047023" class="uri">GO:0047023</a>
</td>
<td style="text-align:left;">
androsterone dehydrogenase activity
</td>
<td style="text-align:left;">
MF
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1.000000e+00
</td>
<td style="text-align:right;">
3.922194e-07
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0071395" class="uri">GO:0071395</a>
</td>
<td style="text-align:left;">
cellular response to jasmonic acid stimulus
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1.000000e+00
</td>
<td style="text-align:right;">
3.922194e-07
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0047086" class="uri">GO:0047086</a>
</td>
<td style="text-align:left;">
ketosteroid monooxygenase activity
</td>
<td style="text-align:left;">
MF
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1.000000e+00
</td>
<td style="text-align:right;">
3.922194e-07
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0018636" class="uri">GO:0018636</a>
</td>
<td style="text-align:left;">
phenanthrene 9,10-monooxygenase activity
</td>
<td style="text-align:left;">
MF
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1.000000e+00
</td>
<td style="text-align:right;">
3.922194e-07
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0009753" class="uri">GO:0009753</a>
</td>
<td style="text-align:left;">
response to jasmonic acid
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1.000000e+00
</td>
<td style="text-align:right;">
3.922194e-07
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0071357" class="uri">GO:0071357</a>
</td>
<td style="text-align:left;">
cellular response to type I interferon
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
73
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
3.952442e-07
</td>
<td style="text-align:right;">
3.976688e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0060337" class="uri">GO:0060337</a>
</td>
<td style="text-align:left;">
type I interferon signaling pathway
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
73
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
3.952442e-07
</td>
<td style="text-align:right;">
3.976688e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0016477" class="uri">GO:0016477</a>
</td>
<td style="text-align:left;">
cell migration
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
923
</td>
<td style="text-align:right;">
22
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
4.109866e-07
</td>
<td style="text-align:right;">
2.859990e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0004033" class="uri">GO:0004033</a>
</td>
<td style="text-align:left;">
aldo-keto reductase (NADP) activity
</td>
<td style="text-align:left;">
MF
</td>
<td style="text-align:right;">
17
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
1.000000e+00
</td>
<td style="text-align:right;">
4.240638e-07
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0004032" class="uri">GO:0004032</a>
</td>
<td style="text-align:left;">
alditol:NADP+ 1-oxidoreductase activity
</td>
<td style="text-align:left;">
MF
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
1.000000e+00
</td>
<td style="text-align:right;">
4.350426e-07
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0071345" class="uri">GO:0071345</a>
</td>
<td style="text-align:left;">
cellular response to cytokine stimulus
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
731
</td>
<td style="text-align:right;">
19
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
4.404555e-07
</td>
<td style="text-align:right;">
3.012425e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0045595" class="uri">GO:0045595</a>
</td>
<td style="text-align:left;">
regulation of cell differentiation
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
1003
</td>
<td style="text-align:right;">
23
</td>
<td style="text-align:right;">
13
</td>
<td style="text-align:right;">
4.896438e-07
</td>
<td style="text-align:right;">
4.858142e-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="cki-mac-vs-cki" class="section level4">
<h4>CKI-Mac vs CKI</h4>
<pre class="r"><code>goana_treat_sig[[&quot;CKI-Mac vs CKI&quot;]][1:30, ] %&gt;%
  as.data.frame() %&gt;%
  dplyr::select(-c(&quot;shortest_path&quot;, &quot;longest_path&quot;, &quot;terminal_node&quot;, &quot;ontology&quot;)) %&gt;%
  kable(digits = Inf) %&gt;%
  kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  scroll_box(height = &quot;600px&quot;)</code></pre>
<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Term
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Ont
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
N
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Up
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Down
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
P.Up
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
P.Down
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
<a href="GO:0071216" class="uri">GO:0071216</a>
</td>
<td style="text-align:left;">
cellular response to biotic stimulus
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
150
</td>
<td style="text-align:right;">
20
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
7.984907e-15
</td>
<td style="text-align:right;">
8.517907e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0071222" class="uri">GO:0071222</a>
</td>
<td style="text-align:left;">
cellular response to lipopolysaccharide
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
124
</td>
<td style="text-align:right;">
18
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
4.324524e-14
</td>
<td style="text-align:right;">
7.941049e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0071219" class="uri">GO:0071219</a>
</td>
<td style="text-align:left;">
cellular response to molecule of bacterial origin
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
129
</td>
<td style="text-align:right;">
18
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.065904e-13
</td>
<td style="text-align:right;">
8.106977e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0071396" class="uri">GO:0071396</a>
</td>
<td style="text-align:left;">
cellular response to lipid
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
362
</td>
<td style="text-align:right;">
26
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
9.487182e-13
</td>
<td style="text-align:right;">
7.672803e-02
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0005125" class="uri">GO:0005125</a>
</td>
<td style="text-align:left;">
cytokine activity
</td>
<td style="text-align:left;">
MF
</td>
<td style="text-align:right;">
78
</td>
<td style="text-align:right;">
15
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
2.090876e-12
</td>
<td style="text-align:right;">
7.753699e-03
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0032496" class="uri">GO:0032496</a>
</td>
<td style="text-align:left;">
response to lipopolysaccharide
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
197
</td>
<td style="text-align:right;">
20
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
2.560890e-12
</td>
<td style="text-align:right;">
2.529714e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0002237" class="uri">GO:0002237</a>
</td>
<td style="text-align:left;">
response to molecule of bacterial origin
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
207
</td>
<td style="text-align:right;">
20
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
8.692843e-12
</td>
<td style="text-align:right;">
1.390418e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:1901701" class="uri">GO:1901701</a>
</td>
<td style="text-align:left;">
cellular response to oxygen-containing compound
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
780
</td>
<td style="text-align:right;">
36
</td>
<td style="text-align:right;">
23
</td>
<td style="text-align:right;">
1.338162e-11
</td>
<td style="text-align:right;">
7.452955e-05
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0009617" class="uri">GO:0009617</a>
</td>
<td style="text-align:left;">
response to bacterium
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
315
</td>
<td style="text-align:right;">
24
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
1.412362e-11
</td>
<td style="text-align:right;">
2.598358e-02
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0005126" class="uri">GO:0005126</a>
</td>
<td style="text-align:left;">
cytokine receptor binding
</td>
<td style="text-align:left;">
MF
</td>
<td style="text-align:right;">
136
</td>
<td style="text-align:right;">
16
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
4.289058e-11
</td>
<td style="text-align:right;">
8.302874e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0062023" class="uri">GO:0062023</a>
</td>
<td style="text-align:left;">
collagen-containing extracellular matrix
</td>
<td style="text-align:left;">
CC
</td>
<td style="text-align:right;">
201
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
18
</td>
<td style="text-align:right;">
8.048545e-02
</td>
<td style="text-align:right;">
3.946609e-10
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0033993" class="uri">GO:0033993</a>
</td>
<td style="text-align:left;">
response to lipid
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
542
</td>
<td style="text-align:right;">
28
</td>
<td style="text-align:right;">
16
</td>
<td style="text-align:right;">
6.993144e-10
</td>
<td style="text-align:right;">
1.375302e-03
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0071310" class="uri">GO:0071310</a>
</td>
<td style="text-align:left;">
cellular response to organic substance
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
1761
</td>
<td style="text-align:right;">
53
</td>
<td style="text-align:right;">
44
</td>
<td style="text-align:right;">
1.124470e-09
</td>
<td style="text-align:right;">
1.537001e-06
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0048018" class="uri">GO:0048018</a>
</td>
<td style="text-align:left;">
receptor ligand activity
</td>
<td style="text-align:left;">
MF
</td>
<td style="text-align:right;">
162
</td>
<td style="text-align:right;">
17
</td>
<td style="text-align:right;">
10
</td>
<td style="text-align:right;">
1.192981e-09
</td>
<td style="text-align:right;">
2.102893e-04
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0030546" class="uri">GO:0030546</a>
</td>
<td style="text-align:left;">
signaling receptor activator activity
</td>
<td style="text-align:left;">
MF
</td>
<td style="text-align:right;">
168
</td>
<td style="text-align:right;">
17
</td>
<td style="text-align:right;">
10
</td>
<td style="text-align:right;">
1.943111e-09
</td>
<td style="text-align:right;">
2.709752e-04
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0019221" class="uri">GO:0019221</a>
</td>
<td style="text-align:left;">
cytokine-mediated signaling pathway
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
516
</td>
<td style="text-align:right;">
26
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
3.945078e-09
</td>
<td style="text-align:right;">
7.754062e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0006954" class="uri">GO:0006954</a>
</td>
<td style="text-align:left;">
inflammatory response
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
387
</td>
<td style="text-align:right;">
23
</td>
<td style="text-align:right;">
11
</td>
<td style="text-align:right;">
6.552197e-09
</td>
<td style="text-align:right;">
1.561263e-02
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0006952" class="uri">GO:0006952</a>
</td>
<td style="text-align:left;">
defense response
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
925
</td>
<td style="text-align:right;">
36
</td>
<td style="text-align:right;">
17
</td>
<td style="text-align:right;">
8.486203e-09
</td>
<td style="text-align:right;">
8.929437e-02
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:1901700" class="uri">GO:1901700</a>
</td>
<td style="text-align:left;">
response to oxygen-containing compound
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
1067
</td>
<td style="text-align:right;">
38
</td>
<td style="text-align:right;">
34
</td>
<td style="text-align:right;">
1.044894e-08
</td>
<td style="text-align:right;">
2.318140e-07
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0010033" class="uri">GO:0010033</a>
</td>
<td style="text-align:left;">
response to organic substance
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
2155
</td>
<td style="text-align:right;">
58
</td>
<td style="text-align:right;">
54
</td>
<td style="text-align:right;">
1.235957e-08
</td>
<td style="text-align:right;">
6.148077e-08
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0071345" class="uri">GO:0071345</a>
</td>
<td style="text-align:left;">
cellular response to cytokine stimulus
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
731
</td>
<td style="text-align:right;">
30
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
2.003667e-08
</td>
<td style="text-align:right;">
5.497199e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0070887" class="uri">GO:0070887</a>
</td>
<td style="text-align:left;">
cellular response to chemical stimulus
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
2128
</td>
<td style="text-align:right;">
57
</td>
<td style="text-align:right;">
51
</td>
<td style="text-align:right;">
2.292659e-08
</td>
<td style="text-align:right;">
7.541645e-07
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0019730" class="uri">GO:0019730</a>
</td>
<td style="text-align:left;">
antimicrobial humoral response
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
34
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
2.810355e-08
</td>
<td style="text-align:right;">
8.283040e-02
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0006959" class="uri">GO:0006959</a>
</td>
<td style="text-align:left;">
humoral immune response
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
82
</td>
<td style="text-align:right;">
11
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
6.604572e-08
</td>
<td style="text-align:right;">
1.260954e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0002526" class="uri">GO:0002526</a>
</td>
<td style="text-align:left;">
acute inflammatory response
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
53
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
7.372860e-08
</td>
<td style="text-align:right;">
5.278550e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0043207" class="uri">GO:0043207</a>
</td>
<td style="text-align:left;">
response to external biotic stimulus
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
857
</td>
<td style="text-align:right;">
32
</td>
<td style="text-align:right;">
14
</td>
<td style="text-align:right;">
9.805477e-08
</td>
<td style="text-align:right;">
1.924972e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0036499" class="uri">GO:0036499</a>
</td>
<td style="text-align:left;">
PERK-mediated unfolded protein response
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
22
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
1.235991e-07
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0034097" class="uri">GO:0034097</a>
</td>
<td style="text-align:left;">
response to cytokine
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
796
</td>
<td style="text-align:right;">
30
</td>
<td style="text-align:right;">
11
</td>
<td style="text-align:right;">
1.380023e-07
</td>
<td style="text-align:right;">
3.941103e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0005102" class="uri">GO:0005102</a>
</td>
<td style="text-align:left;">
signaling receptor binding
</td>
<td style="text-align:left;">
MF
</td>
<td style="text-align:right;">
861
</td>
<td style="text-align:right;">
32
</td>
<td style="text-align:right;">
28
</td>
<td style="text-align:right;">
2.084310e-07
</td>
<td style="text-align:right;">
5.373494e-06
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0071495" class="uri">GO:0071495</a>
</td>
<td style="text-align:left;">
cellular response to endogenous stimulus
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
896
</td>
<td style="text-align:right;">
24
</td>
<td style="text-align:right;">
30
</td>
<td style="text-align:right;">
4.316526e-04
</td>
<td style="text-align:right;">
2.574194e-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="cki-nme-vs-cki" class="section level4">
<h4>CKI-Nme vs CKI</h4>
<pre class="r"><code>goana_treat_sig[[&quot;CKI-Nme vs CKI&quot;]][1:30, ] %&gt;%
  as.data.frame() %&gt;%
  dplyr::select(-c(&quot;shortest_path&quot;, &quot;longest_path&quot;, &quot;terminal_node&quot;, &quot;ontology&quot;)) %&gt;%
  kable(digits = Inf) %&gt;%
  kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  scroll_box(height = &quot;600px&quot;)</code></pre>
<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Term
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Ont
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
N
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Up
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Down
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
P.Up
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
P.Down
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
<a href="GO:0016021" class="uri">GO:0016021</a>
</td>
<td style="text-align:left;">
integral component of membrane
</td>
<td style="text-align:left;">
CC
</td>
<td style="text-align:right;">
2641
</td>
<td style="text-align:right;">
26
</td>
<td style="text-align:right;">
102
</td>
<td style="text-align:right;">
7.408738e-01
</td>
<td style="text-align:right;">
1.660480e-14
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0062023" class="uri">GO:0062023</a>
</td>
<td style="text-align:left;">
collagen-containing extracellular matrix
</td>
<td style="text-align:left;">
CC
</td>
<td style="text-align:right;">
201
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
23
</td>
<td style="text-align:right;">
4.110607e-01
</td>
<td style="text-align:right;">
1.615989e-11
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0005886" class="uri">GO:0005886</a>
</td>
<td style="text-align:left;">
plasma membrane
</td>
<td style="text-align:left;">
CC
</td>
<td style="text-align:right;">
2806
</td>
<td style="text-align:right;">
34
</td>
<td style="text-align:right;">
98
</td>
<td style="text-align:right;">
2.460999e-01
</td>
<td style="text-align:right;">
3.076542e-11
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0005125" class="uri">GO:0005125</a>
</td>
<td style="text-align:left;">
cytokine activity
</td>
<td style="text-align:left;">
MF
</td>
<td style="text-align:right;">
78
</td>
<td style="text-align:right;">
12
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
2.739672e-10
</td>
<td style="text-align:right;">
3.283275e-02
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0004888" class="uri">GO:0004888</a>
</td>
<td style="text-align:left;">
transmembrane signaling receptor activity
</td>
<td style="text-align:left;">
MF
</td>
<td style="text-align:right;">
316
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
28
</td>
<td style="text-align:right;">
5.669704e-01
</td>
<td style="text-align:right;">
3.036367e-10
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0009888" class="uri">GO:0009888</a>
</td>
<td style="text-align:left;">
tissue development
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
1184
</td>
<td style="text-align:right;">
20
</td>
<td style="text-align:right;">
53
</td>
<td style="text-align:right;">
2.320772e-02
</td>
<td style="text-align:right;">
7.289877e-10
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0005794" class="uri">GO:0005794</a>
</td>
<td style="text-align:left;">
Golgi apparatus
</td>
<td style="text-align:left;">
CC
</td>
<td style="text-align:right;">
1199
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
51
</td>
<td style="text-align:right;">
9.930718e-01
</td>
<td style="text-align:right;">
7.308000e-10
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0048731" class="uri">GO:0048731</a>
</td>
<td style="text-align:left;">
system development
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
2913
</td>
<td style="text-align:right;">
43
</td>
<td style="text-align:right;">
94
</td>
<td style="text-align:right;">
7.402066e-03
</td>
<td style="text-align:right;">
1.358033e-09
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0048018" class="uri">GO:0048018</a>
</td>
<td style="text-align:left;">
receptor ligand activity
</td>
<td style="text-align:left;">
MF
</td>
<td style="text-align:right;">
162
</td>
<td style="text-align:right;">
15
</td>
<td style="text-align:right;">
10
</td>
<td style="text-align:right;">
1.740726e-09
</td>
<td style="text-align:right;">
3.497694e-03
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0031226" class="uri">GO:0031226</a>
</td>
<td style="text-align:left;">
intrinsic component of plasma membrane
</td>
<td style="text-align:left;">
CC
</td>
<td style="text-align:right;">
719
</td>
<td style="text-align:right;">
10
</td>
<td style="text-align:right;">
41
</td>
<td style="text-align:right;">
3.505146e-01
</td>
<td style="text-align:right;">
2.569726e-09
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0030546" class="uri">GO:0030546</a>
</td>
<td style="text-align:left;">
signaling receptor activator activity
</td>
<td style="text-align:left;">
MF
</td>
<td style="text-align:right;">
168
</td>
<td style="text-align:right;">
15
</td>
<td style="text-align:right;">
10
</td>
<td style="text-align:right;">
2.627992e-09
</td>
<td style="text-align:right;">
4.312956e-03
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0071216" class="uri">GO:0071216</a>
</td>
<td style="text-align:left;">
cellular response to biotic stimulus
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
150
</td>
<td style="text-align:right;">
13
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
4.019343e-09
</td>
<td style="text-align:right;">
7.583237e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0005783" class="uri">GO:0005783</a>
</td>
<td style="text-align:left;">
endoplasmic reticulum
</td>
<td style="text-align:left;">
CC
</td>
<td style="text-align:right;">
1384
</td>
<td style="text-align:right;">
15
</td>
<td style="text-align:right;">
54
</td>
<td style="text-align:right;">
3.906175e-01
</td>
<td style="text-align:right;">
5.962699e-09
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0010033" class="uri">GO:0010033</a>
</td>
<td style="text-align:left;">
response to organic substance
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
2155
</td>
<td style="text-align:right;">
48
</td>
<td style="text-align:right;">
70
</td>
<td style="text-align:right;">
1.410919e-08
</td>
<td style="text-align:right;">
5.436548e-08
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0010466" class="uri">GO:0010466</a>
</td>
<td style="text-align:left;">
negative regulation of peptidase activity
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
132
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
15
</td>
<td style="text-align:right;">
4.872328e-02
</td>
<td style="text-align:right;">
1.913706e-08
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0071310" class="uri">GO:0071310</a>
</td>
<td style="text-align:left;">
cellular response to organic substance
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
1761
</td>
<td style="text-align:right;">
42
</td>
<td style="text-align:right;">
58
</td>
<td style="text-align:right;">
2.082859e-08
</td>
<td style="text-align:right;">
7.836492e-07
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0043230" class="uri">GO:0043230</a>
</td>
<td style="text-align:left;">
extracellular organelle
</td>
<td style="text-align:left;">
CC
</td>
<td style="text-align:right;">
1455
</td>
<td style="text-align:right;">
17
</td>
<td style="text-align:right;">
54
</td>
<td style="text-align:right;">
2.561504e-01
</td>
<td style="text-align:right;">
2.530134e-08
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:1903561" class="uri">GO:1903561</a>
</td>
<td style="text-align:left;">
extracellular vesicle
</td>
<td style="text-align:left;">
CC
</td>
<td style="text-align:right;">
1455
</td>
<td style="text-align:right;">
17
</td>
<td style="text-align:right;">
54
</td>
<td style="text-align:right;">
2.561504e-01
</td>
<td style="text-align:right;">
2.530134e-08
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:1901700" class="uri">GO:1901700</a>
</td>
<td style="text-align:left;">
response to oxygen-containing compound
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
1067
</td>
<td style="text-align:right;">
26
</td>
<td style="text-align:right;">
45
</td>
<td style="text-align:right;">
1.883099e-05
</td>
<td style="text-align:right;">
2.595485e-08
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0036499" class="uri">GO:0036499</a>
</td>
<td style="text-align:left;">
PERK-mediated unfolded protein response
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
22
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
2.713962e-08
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0030198" class="uri">GO:0030198</a>
</td>
<td style="text-align:left;">
extracellular matrix organization
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
228
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
20
</td>
<td style="text-align:right;">
1.192361e-03
</td>
<td style="text-align:right;">
3.438846e-08
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0043062" class="uri">GO:0043062</a>
</td>
<td style="text-align:left;">
extracellular structure organization
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
229
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
20
</td>
<td style="text-align:right;">
1.255275e-03
</td>
<td style="text-align:right;">
3.886146e-08
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0045229" class="uri">GO:0045229</a>
</td>
<td style="text-align:left;">
external encapsulating structure organization
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
229
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
20
</td>
<td style="text-align:right;">
1.256603e-03
</td>
<td style="text-align:right;">
3.895788e-08
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0070062" class="uri">GO:0070062</a>
</td>
<td style="text-align:left;">
extracellular exosome
</td>
<td style="text-align:left;">
CC
</td>
<td style="text-align:right;">
1443
</td>
<td style="text-align:right;">
17
</td>
<td style="text-align:right;">
53
</td>
<td style="text-align:right;">
2.401018e-01
</td>
<td style="text-align:right;">
4.309503e-08
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0071222" class="uri">GO:0071222</a>
</td>
<td style="text-align:left;">
cellular response to lipopolysaccharide
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
124
</td>
<td style="text-align:right;">
11
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
5.368661e-08
</td>
<td style="text-align:right;">
6.611622e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0016477" class="uri">GO:0016477</a>
</td>
<td style="text-align:left;">
cell migration
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
923
</td>
<td style="text-align:right;">
25
</td>
<td style="text-align:right;">
41
</td>
<td style="text-align:right;">
6.350122e-06
</td>
<td style="text-align:right;">
6.484022e-08
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0010951" class="uri">GO:0010951</a>
</td>
<td style="text-align:left;">
negative regulation of endopeptidase activity
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
127
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
14
</td>
<td style="text-align:right;">
4.348211e-02
</td>
<td style="text-align:right;">
8.615818e-08
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0005788" class="uri">GO:0005788</a>
</td>
<td style="text-align:left;">
endoplasmic reticulum lumen
</td>
<td style="text-align:left;">
CC
</td>
<td style="text-align:right;">
196
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
17
</td>
<td style="text-align:right;">
8.561037e-04
</td>
<td style="text-align:right;">
8.660375e-08
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0071219" class="uri">GO:0071219</a>
</td>
<td style="text-align:left;">
cellular response to molecule of bacterial origin
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
129
</td>
<td style="text-align:right;">
11
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
9.053579e-08
</td>
<td style="text-align:right;">
6.882773e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0019730" class="uri">GO:0019730</a>
</td>
<td style="text-align:left;">
antimicrobial humoral response
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
34
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
9.537951e-08
</td>
<td style="text-align:right;">
3.151554e-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="cki-omt-vs-cki" class="section level4">
<h4>CKI-Omt vs CKI</h4>
<pre class="r"><code>goana_treat_sig[[&quot;CKI-Omt vs CKI&quot;]][1:30, ] %&gt;%
  as.data.frame() %&gt;%
  dplyr::select(-c(&quot;shortest_path&quot;, &quot;longest_path&quot;, &quot;terminal_node&quot;, &quot;ontology&quot;)) %&gt;%
  kable(digits = Inf) %&gt;%
  kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  scroll_box(height = &quot;600px&quot;)</code></pre>
<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Term
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Ont
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
N
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Up
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Down
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
P.Up
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
P.Down
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
<a href="GO:0000278" class="uri">GO:0000278</a>
</td>
<td style="text-align:left;">
mitotic cell cycle
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
883
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
113
</td>
<td style="text-align:right;">
0.97944266
</td>
<td style="text-align:right;">
3.409209e-44
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:1903047" class="uri">GO:1903047</a>
</td>
<td style="text-align:left;">
mitotic cell cycle process
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
772
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
102
</td>
<td style="text-align:right;">
0.93902066
</td>
<td style="text-align:right;">
4.179072e-41
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0000280" class="uri">GO:0000280</a>
</td>
<td style="text-align:left;">
nuclear division
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
342
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
71
</td>
<td style="text-align:right;">
0.38973819
</td>
<td style="text-align:right;">
1.936741e-39
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0140014" class="uri">GO:0140014</a>
</td>
<td style="text-align:left;">
mitotic nuclear division
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
269
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
61
</td>
<td style="text-align:right;">
0.64460060
</td>
<td style="text-align:right;">
8.549333e-38
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0048285" class="uri">GO:0048285</a>
</td>
<td style="text-align:left;">
organelle fission
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
383
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
73
</td>
<td style="text-align:right;">
0.51156505
</td>
<td style="text-align:right;">
9.224554e-38
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0098813" class="uri">GO:0098813</a>
</td>
<td style="text-align:left;">
nuclear chromosome segregation
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
218
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
56
</td>
<td style="text-align:right;">
0.89135065
</td>
<td style="text-align:right;">
8.845251e-37
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0000819" class="uri">GO:0000819</a>
</td>
<td style="text-align:left;">
sister chromatid segregation
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
177
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
50
</td>
<td style="text-align:right;">
0.94837163
</td>
<td style="text-align:right;">
7.528457e-36
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0000070" class="uri">GO:0000070</a>
</td>
<td style="text-align:left;">
mitotic sister chromatid segregation
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
153
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
46
</td>
<td style="text-align:right;">
0.92289952
</td>
<td style="text-align:right;">
2.321270e-34
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0000775" class="uri">GO:0000775</a>
</td>
<td style="text-align:left;">
chromosome, centromeric region
</td>
<td style="text-align:left;">
CC
</td>
<td style="text-align:right;">
180
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
46
</td>
<td style="text-align:right;">
1.00000000
</td>
<td style="text-align:right;">
2.168198e-30
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0051276" class="uri">GO:0051276</a>
</td>
<td style="text-align:left;">
chromosome organization
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
978
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
96
</td>
<td style="text-align:right;">
0.99891989
</td>
<td style="text-align:right;">
4.543130e-28
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0000793" class="uri">GO:0000793</a>
</td>
<td style="text-align:left;">
condensed chromosome
</td>
<td style="text-align:left;">
CC
</td>
<td style="text-align:right;">
169
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
42
</td>
<td style="text-align:right;">
0.95188695
</td>
<td style="text-align:right;">
1.331655e-26
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0000776" class="uri">GO:0000776</a>
</td>
<td style="text-align:left;">
kinetochore
</td>
<td style="text-align:left;">
CC
</td>
<td style="text-align:right;">
130
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
36
</td>
<td style="text-align:right;">
1.00000000
</td>
<td style="text-align:right;">
3.543949e-25
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0000779" class="uri">GO:0000779</a>
</td>
<td style="text-align:left;">
condensed chromosome, centromeric region
</td>
<td style="text-align:left;">
CC
</td>
<td style="text-align:right;">
110
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
34
</td>
<td style="text-align:right;">
1.00000000
</td>
<td style="text-align:right;">
3.937033e-25
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0010564" class="uri">GO:0010564</a>
</td>
<td style="text-align:left;">
regulation of cell cycle process
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
638
</td>
<td style="text-align:right;">
10
</td>
<td style="text-align:right;">
72
</td>
<td style="text-align:right;">
0.59582299
</td>
<td style="text-align:right;">
2.167617e-24
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0005694" class="uri">GO:0005694</a>
</td>
<td style="text-align:left;">
chromosome
</td>
<td style="text-align:left;">
CC
</td>
<td style="text-align:right;">
1206
</td>
<td style="text-align:right;">
14
</td>
<td style="text-align:right;">
106
</td>
<td style="text-align:right;">
0.96813930
</td>
<td style="text-align:right;">
2.465882e-24
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0006260" class="uri">GO:0006260</a>
</td>
<td style="text-align:left;">
DNA replication
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
257
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
48
</td>
<td style="text-align:right;">
1.00000000
</td>
<td style="text-align:right;">
9.040283e-24
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0000777" class="uri">GO:0000777</a>
</td>
<td style="text-align:left;">
condensed chromosome kinetochore
</td>
<td style="text-align:left;">
CC
</td>
<td style="text-align:right;">
99
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
31
</td>
<td style="text-align:right;">
1.00000000
</td>
<td style="text-align:right;">
5.158474e-23
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0051726" class="uri">GO:0051726</a>
</td>
<td style="text-align:left;">
regulation of cell cycle
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
987
</td>
<td style="text-align:right;">
16
</td>
<td style="text-align:right;">
87
</td>
<td style="text-align:right;">
0.56988349
</td>
<td style="text-align:right;">
1.278303e-21
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0044770" class="uri">GO:0044770</a>
</td>
<td style="text-align:left;">
cell cycle phase transition
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
544
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
62
</td>
<td style="text-align:right;">
0.94378142
</td>
<td style="text-align:right;">
2.444009e-21
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0044772" class="uri">GO:0044772</a>
</td>
<td style="text-align:left;">
mitotic cell cycle phase transition
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
515
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
60
</td>
<td style="text-align:right;">
0.92245467
</td>
<td style="text-align:right;">
2.956407e-21
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0071103" class="uri">GO:0071103</a>
</td>
<td style="text-align:left;">
DNA conformation change
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
241
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
41
</td>
<td style="text-align:right;">
1.00000000
</td>
<td style="text-align:right;">
3.787453e-20
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0007088" class="uri">GO:0007088</a>
</td>
<td style="text-align:left;">
regulation of mitotic nuclear division
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
98
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
28
</td>
<td style="text-align:right;">
0.09038307
</td>
<td style="text-align:right;">
4.188943e-20
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0051783" class="uri">GO:0051783</a>
</td>
<td style="text-align:left;">
regulation of nuclear division
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
109
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
29
</td>
<td style="text-align:right;">
0.01189635
</td>
<td style="text-align:right;">
8.793220e-20
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0051983" class="uri">GO:0051983</a>
</td>
<td style="text-align:left;">
regulation of chromosome segregation
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
80
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
24
</td>
<td style="text-align:right;">
0.73201661
</td>
<td style="text-align:right;">
1.485132e-18
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0051306" class="uri">GO:0051306</a>
</td>
<td style="text-align:left;">
mitotic sister chromatid separation
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
63
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
22
</td>
<td style="text-align:right;">
0.65983054
</td>
<td style="text-align:right;">
2.240639e-18
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0007346" class="uri">GO:0007346</a>
</td>
<td style="text-align:left;">
regulation of mitotic cell cycle
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
508
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
55
</td>
<td style="text-align:right;">
0.58671664
</td>
<td style="text-align:right;">
5.912455e-18
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0033045" class="uri">GO:0033045</a>
</td>
<td style="text-align:left;">
regulation of sister chromatid segregation
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
67
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
22
</td>
<td style="text-align:right;">
0.68047527
</td>
<td style="text-align:right;">
9.235988e-18
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0007091" class="uri">GO:0007091</a>
</td>
<td style="text-align:left;">
metaphase/anaphase transition of mitotic cell cycle
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
60
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
21
</td>
<td style="text-align:right;">
0.63812431
</td>
<td style="text-align:right;">
1.042396e-17
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0005819" class="uri">GO:0005819</a>
</td>
<td style="text-align:left;">
spindle
</td>
<td style="text-align:left;">
CC
</td>
<td style="text-align:right;">
331
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
44
</td>
<td style="text-align:right;">
0.99618666
</td>
<td style="text-align:right;">
1.256185e-17
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0010965" class="uri">GO:0010965</a>
</td>
<td style="text-align:left;">
regulation of mitotic sister chromatid separation
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
61
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
21
</td>
<td style="text-align:right;">
0.64281813
</td>
<td style="text-align:right;">
1.434542e-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="cki-tri-vs-cki" class="section level4">
<h4>CKI-Tri vs CKI</h4>
<pre class="r"><code>goana_treat_sig[[&quot;CKI-Tri vs CKI&quot;]][1:30, ] %&gt;%
  as.data.frame() %&gt;%
  dplyr::select(-c(&quot;shortest_path&quot;, &quot;longest_path&quot;, &quot;terminal_node&quot;, &quot;ontology&quot;)) %&gt;%
  kable(digits = Inf) %&gt;%
  kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  scroll_box(height = &quot;600px&quot;)</code></pre>
<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Term
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Ont
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
N
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Up
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Down
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
P.Up
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
P.Down
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
<a href="GO:1901701" class="uri">GO:1901701</a>
</td>
<td style="text-align:left;">
cellular response to oxygen-containing compound
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
780
</td>
<td style="text-align:right;">
34
</td>
<td style="text-align:right;">
25
</td>
<td style="text-align:right;">
1.366142e-12
</td>
<td style="text-align:right;">
2.250490e-04
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0005125" class="uri">GO:0005125</a>
</td>
<td style="text-align:left;">
cytokine activity
</td>
<td style="text-align:left;">
MF
</td>
<td style="text-align:right;">
78
</td>
<td style="text-align:right;">
14
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
2.378000e-12
</td>
<td style="text-align:right;">
1.621208e-02
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0071310" class="uri">GO:0071310</a>
</td>
<td style="text-align:left;">
cellular response to organic substance
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
1761
</td>
<td style="text-align:right;">
51
</td>
<td style="text-align:right;">
45
</td>
<td style="text-align:right;">
5.806925e-12
</td>
<td style="text-align:right;">
1.311982e-04
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0071216" class="uri">GO:0071216</a>
</td>
<td style="text-align:left;">
cellular response to biotic stimulus
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
150
</td>
<td style="text-align:right;">
16
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
8.553362e-12
</td>
<td style="text-align:right;">
9.014562e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0010033" class="uri">GO:0010033</a>
</td>
<td style="text-align:left;">
response to organic substance
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
2155
</td>
<td style="text-align:right;">
57
</td>
<td style="text-align:right;">
54
</td>
<td style="text-align:right;">
9.621688e-12
</td>
<td style="text-align:right;">
4.483444e-05
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0005126" class="uri">GO:0005126</a>
</td>
<td style="text-align:left;">
cytokine receptor binding
</td>
<td style="text-align:left;">
MF
</td>
<td style="text-align:right;">
136
</td>
<td style="text-align:right;">
15
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
3.393989e-11
</td>
<td style="text-align:right;">
8.836799e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0070887" class="uri">GO:0070887</a>
</td>
<td style="text-align:left;">
cellular response to chemical stimulus
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
2128
</td>
<td style="text-align:right;">
55
</td>
<td style="text-align:right;">
53
</td>
<td style="text-align:right;">
7.194361e-11
</td>
<td style="text-align:right;">
6.757138e-05
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0071222" class="uri">GO:0071222</a>
</td>
<td style="text-align:left;">
cellular response to lipopolysaccharide
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
124
</td>
<td style="text-align:right;">
14
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
8.694453e-11
</td>
<td style="text-align:right;">
8.530904e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0071219" class="uri">GO:0071219</a>
</td>
<td style="text-align:left;">
cellular response to molecule of bacterial origin
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
129
</td>
<td style="text-align:right;">
14
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.845261e-10
</td>
<td style="text-align:right;">
8.671145e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0048018" class="uri">GO:0048018</a>
</td>
<td style="text-align:left;">
receptor ligand activity
</td>
<td style="text-align:left;">
MF
</td>
<td style="text-align:right;">
162
</td>
<td style="text-align:right;">
16
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
5.606544e-10
</td>
<td style="text-align:right;">
3.199877e-02
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:1901700" class="uri">GO:1901700</a>
</td>
<td style="text-align:left;">
response to oxygen-containing compound
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
1067
</td>
<td style="text-align:right;">
36
</td>
<td style="text-align:right;">
36
</td>
<td style="text-align:right;">
6.935054e-10
</td>
<td style="text-align:right;">
3.249947e-06
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0009617" class="uri">GO:0009617</a>
</td>
<td style="text-align:left;">
response to bacterium
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
315
</td>
<td style="text-align:right;">
20
</td>
<td style="text-align:right;">
10
</td>
<td style="text-align:right;">
8.468669e-10
</td>
<td style="text-align:right;">
3.263032e-02
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0032496" class="uri">GO:0032496</a>
</td>
<td style="text-align:left;">
response to lipopolysaccharide
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
197
</td>
<td style="text-align:right;">
16
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
8.721473e-10
</td>
<td style="text-align:right;">
1.983563e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0030546" class="uri">GO:0030546</a>
</td>
<td style="text-align:left;">
signaling receptor activator activity
</td>
<td style="text-align:left;">
MF
</td>
<td style="text-align:right;">
168
</td>
<td style="text-align:right;">
16
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
8.949441e-10
</td>
<td style="text-align:right;">
3.698598e-02
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0062023" class="uri">GO:0062023</a>
</td>
<td style="text-align:left;">
collagen-containing extracellular matrix
</td>
<td style="text-align:left;">
CC
</td>
<td style="text-align:right;">
201
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
19
</td>
<td style="text-align:right;">
2.401369e-01
</td>
<td style="text-align:right;">
1.311800e-09
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0033993" class="uri">GO:0033993</a>
</td>
<td style="text-align:left;">
response to lipid
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
542
</td>
<td style="text-align:right;">
25
</td>
<td style="text-align:right;">
18
</td>
<td style="text-align:right;">
1.392352e-09
</td>
<td style="text-align:right;">
1.685606e-03
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0019221" class="uri">GO:0019221</a>
</td>
<td style="text-align:left;">
cytokine-mediated signaling pathway
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
516
</td>
<td style="text-align:right;">
24
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
2.114068e-09
</td>
<td style="text-align:right;">
6.697433e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0071396" class="uri">GO:0071396</a>
</td>
<td style="text-align:left;">
cellular response to lipid
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
362
</td>
<td style="text-align:right;">
20
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
2.136443e-09
</td>
<td style="text-align:right;">
9.091733e-02
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0002237" class="uri">GO:0002237</a>
</td>
<td style="text-align:left;">
response to molecule of bacterial origin
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
207
</td>
<td style="text-align:right;">
16
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
2.296220e-09
</td>
<td style="text-align:right;">
1.143807e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0071345" class="uri">GO:0071345</a>
</td>
<td style="text-align:left;">
cellular response to cytokine stimulus
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
731
</td>
<td style="text-align:right;">
28
</td>
<td style="text-align:right;">
11
</td>
<td style="text-align:right;">
4.987946e-09
</td>
<td style="text-align:right;">
5.385755e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0005788" class="uri">GO:0005788</a>
</td>
<td style="text-align:left;">
endoplasmic reticulum lumen
</td>
<td style="text-align:left;">
CC
</td>
<td style="text-align:right;">
196
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
17
</td>
<td style="text-align:right;">
5.944542e-03
</td>
<td style="text-align:right;">
6.331421e-09
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0034097" class="uri">GO:0034097</a>
</td>
<td style="text-align:left;">
response to cytokine
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
796
</td>
<td style="text-align:right;">
29
</td>
<td style="text-align:right;">
13
</td>
<td style="text-align:right;">
8.133789e-09
</td>
<td style="text-align:right;">
4.177809e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0007166" class="uri">GO:0007166</a>
</td>
<td style="text-align:left;">
cell surface receptor signaling pathway
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
1896
</td>
<td style="text-align:right;">
48
</td>
<td style="text-align:right;">
45
</td>
<td style="text-align:right;">
1.529203e-08
</td>
<td style="text-align:right;">
1.525263e-03
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0016021" class="uri">GO:0016021</a>
</td>
<td style="text-align:left;">
integral component of membrane
</td>
<td style="text-align:left;">
CC
</td>
<td style="text-align:right;">
2641
</td>
<td style="text-align:right;">
30
</td>
<td style="text-align:right;">
77
</td>
<td style="text-align:right;">
6.344215e-01
</td>
<td style="text-align:right;">
1.984126e-08
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0005102" class="uri">GO:0005102</a>
</td>
<td style="text-align:left;">
signaling receptor binding
</td>
<td style="text-align:left;">
MF
</td>
<td style="text-align:right;">
861
</td>
<td style="text-align:right;">
30
</td>
<td style="text-align:right;">
30
</td>
<td style="text-align:right;">
3.360793e-08
</td>
<td style="text-align:right;">
3.065393e-05
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0016477" class="uri">GO:0016477</a>
</td>
<td style="text-align:left;">
cell migration
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
923
</td>
<td style="text-align:right;">
30
</td>
<td style="text-align:right;">
28
</td>
<td style="text-align:right;">
8.525583e-08
</td>
<td style="text-align:right;">
3.737645e-04
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0019730" class="uri">GO:0019730</a>
</td>
<td style="text-align:left;">
antimicrobial humoral response
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
34
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1.647947e-07
</td>
<td style="text-align:right;">
1.985506e-02
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0005783" class="uri">GO:0005783</a>
</td>
<td style="text-align:left;">
endoplasmic reticulum
</td>
<td style="text-align:left;">
CC
</td>
<td style="text-align:right;">
1384
</td>
<td style="text-align:right;">
12
</td>
<td style="text-align:right;">
45
</td>
<td style="text-align:right;">
8.251955e-01
</td>
<td style="text-align:right;">
2.345918e-07
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0061844" class="uri">GO:0061844</a>
</td>
<td style="text-align:left;">
antimicrobial humoral immune response mediated by antimicrobial peptide
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
14
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
4.348766e-07
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0006952" class="uri">GO:0006952</a>
</td>
<td style="text-align:left;">
defense response
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:right;">
925
</td>
<td style="text-align:right;">
29
</td>
<td style="text-align:right;">
21
</td>
<td style="text-align:right;">
5.246203e-07
</td>
<td style="text-align:right;">
5.494272e-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="enriched-go-terms" class="section level3">
<h3>Enriched GO terms</h3>
<pre class="r"><code>top_go &lt;- list()
for (i in 1:ncol(contrast)) {
  # at each iteration, let x = name of each contrast group
  x &lt;- comparison_group[i, ]

  top_go[[x]] &lt;- goana_treat_sig[[x]][1:7,]
}
# combine all list from big merged_go into one
merged_go &lt;- as.data.frame(do.call(rbind, lapply(top_go, as.data.frame))) %&gt;% rownames_to_column(&quot;group&quot;)

# remove KEGG ID from group names
merged_go$group &lt;- gsub(pattern = &quot;\\..*&quot;, &quot;&quot;, merged_go$group)

# direction column
merged_go &lt;- merged_go %&gt;% dplyr::mutate(direction = case_when
(
  Up &gt; Down ~ &quot;Up-\nregulated\n&quot;,
  Up &lt; Down ~ &quot;Down-\nregulated\n&quot;,
  TRUE ~ &quot;Mutual&quot;
))

# geneRatio column 
merged_go &lt;- merged_go %&gt;% dplyr::mutate(geneRatio = case_when
(
  Up &gt; Down &amp; group == &quot;UT vs CKI&quot; ~ Up / 215,
  Up &gt; Down &amp; group == &quot;CKI-Omt vs CKI&quot; ~ Up / 689,
  Up &gt; Down &amp; group == &quot;CKI-Mac vs CKI&quot; ~ Up / 353,
  Up &gt; Down &amp; group == &quot;CKI-Nme vs CKI&quot; ~ Up / 382,
  Up &gt; Down &amp; group == &quot;CKI-Tri vs CKI&quot; ~ Up / 358,
  Up &lt; Down &amp; group == &quot;UT vs CKI&quot; ~ Down / 215,
  Up &lt; Down &amp; group == &quot;CKI-Omt vs CKI&quot; ~ Down / 689,
  Up &lt; Down &amp; group == &quot;CKI-Mac vs CKI&quot; ~ Down / 353,
  Up &lt; Down &amp; group == &quot;CKI-Nme vs CKI&quot; ~ Down / 382,
  Up &lt; Down &amp; group == &quot;CKI-Tri vs CKI&quot; ~ Down / 358,
))

# pvalue column
merged_go &lt;- merged_go %&gt;% dplyr::mutate(pValue = case_when
(
  Up &gt; Down ~ P.Up,
  Up &lt; Down ~ P.Down,
  TRUE ~ P.Up
))

merged_go &lt;- merged_go %&gt;% filter(pValue &lt;= 0.05)

# Transform the column &#39;Description&#39; into factors
merged_go$Term &lt;- as.factor(merged_go$Term)

# Transform FDR values by -log10(&#39;FDR values&#39;)
merged_go$&quot;|log10(FDR)|&quot; &lt;- -(log10(merged_go$pValue))

# Change factor order
merged_go$group &lt;- factor(merged_go$group, levels = c(&quot;UT vs CKI&quot;, &quot;CKI-Omt vs CKI&quot;, &quot;CKI-Mac vs CKI&quot;, &quot;CKI-Nme vs CKI&quot;, &quot;CKI-Tri vs CKI&quot;))

# merged_go$Term &lt;- factor(merged_go, levels = rev(levels(merged_go$Term)))

group.labs &lt;- merged_go$group

ggplot(merged_go, aes(x = Term, y = group)) +
  geom_point(data = merged_go, aes(x = Term, y = group, size = geneRatio, colour = `|log10(FDR)|`, shape = direction), alpha = .9) +
  # scale_y_discrete(labels =group.labs)+
  scale_shape_manual(values = c(15, 16, 17)) +
  scale_color_gradient(low = &quot;dodgerblue3&quot;, high = &quot;firebrick3&quot;, limits = c(0, NA)) +
  coord_flip() +
  theme_bw() +
  theme(
    axis.ticks.length = unit(-0.1, &quot;cm&quot;),
    axis.text.x = element_text(margin = margin(5, 5, 0, 5, &quot;pt&quot;), angle = 45, hjust = 1),
    axis.text.y = element_text(margin = margin(5, 5, 5, 5, &quot;pt&quot;)),
    axis.text = element_text(color = &quot;black&quot;),
    axis.title.x = element_blank()
  ) +
  xlab(label = &quot;&quot;) +
  labs(color = expression(&quot;-log&quot;[10] * &quot;FDR&quot;), size = &quot;Gene Ratio&quot;, shape = &quot;Expression&quot;)</code></pre>
<p><img src="figure/go.Rmd/dotPlot-1.png" width="768" style="display: block; margin: auto;" /></p>
<pre class="r"><code>ggsave(filename = &quot;dot_goanna_treat.svg&quot;, plot = last_plot(), path = here::here(&quot;2_plots/3_go/&quot;))</code></pre>
</div>
</div>
<div id="export-data" class="section level1">
<h1>Export Data</h1>
<pre class="r"><code>writexl::write_xlsx(x = goana_treat, path = here::here(&quot;3_output/goana_treat.xlsx&quot;))
# writexl::write_xlsx(x = enrichGO_treat, path = here::here(&quot;3_output/enrichGO_treat.xlsx&quot;))</code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.1.0 (2021-05-18)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19043)

Matrix products: default

locale:
[1] LC_COLLATE=English_Australia.1252  LC_CTYPE=English_Australia.1252   
[3] LC_MONETARY=English_Australia.1252 LC_NUMERIC=C                      
[5] LC_TIME=English_Australia.1252    

attached base packages:
 [1] parallel  stats4    grid      stats     graphics  grDevices utils    
 [8] datasets  methods   base     

other attached packages:
 [1] biomaRt_2.48.3        org.Hs.eg.db_3.13.0   AnnotationDbi_1.54.1 
 [4] IRanges_2.26.0        S4Vectors_0.30.0      Biobase_2.52.0       
 [7] BiocGenerics_0.38.0   clusterProfiler_4.0.5 Glimma_2.2.0         
[10] edgeR_3.34.0          limma_3.48.1          ggrepel_0.9.1        
[13] ggbiplot_0.55         scales_1.1.1          plyr_1.8.6           
[16] ggpubr_0.4.0          RColorBrewer_1.1-2    pheatmap_1.0.12      
[19] cowplot_1.1.1         pander_0.6.4          kableExtra_1.3.4     
[22] forcats_0.5.1         stringr_1.4.0         purrr_0.3.4          
[25] tidyr_1.1.3           ggplot2_3.3.5         tidyverse_1.3.1      
[28] reshape2_1.4.4        tibble_3.1.2          readr_2.0.0          
[31] magrittr_2.0.1        dplyr_1.0.7          

loaded via a namespace (and not attached):
  [1] utf8_1.2.1                  tidyselect_1.1.1           
  [3] RSQLite_2.2.7               htmlwidgets_1.5.4          
  [5] BiocParallel_1.26.1         scatterpie_0.1.7           
  [7] munsell_0.5.0               statmod_1.4.36             
  [9] withr_2.4.2                 colorspace_2.0-2           
 [11] GOSemSim_2.18.1             filelock_1.0.2             
 [13] highr_0.9                   knitr_1.36                 
 [15] rstudioapi_0.13             ggsignif_0.6.3             
 [17] DOSE_3.18.3                 MatrixGenerics_1.4.3       
 [19] labeling_0.4.2              git2r_0.28.0               
 [21] GenomeInfoDbData_1.2.6      polyclip_1.10-0            
 [23] bit64_4.0.5                 farver_2.1.0               
 [25] rprojroot_2.0.2             downloader_0.4             
 [27] treeio_1.16.2               vctrs_0.3.8                
 [29] generics_0.1.0              xfun_0.24                  
 [31] BiocFileCache_2.0.0         R6_2.5.1                   
 [33] GenomeInfoDb_1.28.4         graphlayouts_0.7.1         
 [35] locfit_1.5-9.4              gridGraphics_0.5-1         
 [37] bitops_1.0-7                cachem_1.0.5               
 [39] fgsea_1.18.0                DelayedArray_0.18.0        
 [41] assertthat_0.2.1            promises_1.2.0.1           
 [43] ggraph_2.0.5                enrichplot_1.12.3          
 [45] gtable_0.3.0                tidygraph_1.2.0            
 [47] workflowr_1.6.2             rlang_0.4.11               
 [49] genefilter_1.74.0           systemfonts_1.0.2          
 [51] splines_4.1.0               lazyeval_0.2.2             
 [53] rstatix_0.7.0               broom_0.7.9                
 [55] yaml_2.2.1                  abind_1.4-5                
 [57] modelr_0.1.8                backports_1.2.1            
 [59] httpuv_1.6.1                qvalue_2.24.0              
 [61] tools_4.1.0                 ggplotify_0.1.0            
 [63] ellipsis_0.3.2              jquerylib_0.1.4            
 [65] Rcpp_1.0.6                  progress_1.2.2             
 [67] zlibbioc_1.38.0             BiasedUrn_1.07             
 [69] RCurl_1.98-1.3              prettyunits_1.1.1          
 [71] viridis_0.6.2               SummarizedExperiment_1.22.0
 [73] haven_2.4.3                 fs_1.5.0                   
 [75] here_1.0.1                  data.table_1.14.0          
 [77] DO.db_2.9                   openxlsx_4.2.4             
 [79] reprex_2.0.1                whisker_0.4                
 [81] matrixStats_0.60.1          patchwork_1.1.1            
 [83] hms_1.1.1                   evaluate_0.14              
 [85] xtable_1.8-4                XML_3.99-0.6               
 [87] rio_0.5.27                  readxl_1.3.1               
 [89] gridExtra_2.3               compiler_4.1.0             
 [91] shadowtext_0.0.9            writexl_1.4.0              
 [93] crayon_1.4.1                htmltools_0.5.2            
 [95] ggfun_0.0.4                 later_1.2.0                
 [97] tzdb_0.1.2                  aplot_0.1.1                
 [99] geneplotter_1.70.0          lubridate_1.7.10           
[101] DBI_1.1.1                   tweenr_1.0.2               
[103] dbplyr_2.1.1                rappdirs_0.3.3             
[105] MASS_7.3-54                 Matrix_1.3-3               
[107] car_3.0-11                  cli_3.0.1                  
[109] igraph_1.2.6                GenomicRanges_1.44.0       
[111] pkgconfig_2.0.3             foreign_0.8-81             
[113] xml2_1.3.2                  ggtree_3.0.4               
[115] svglite_2.0.0               annotate_1.70.0            
[117] bslib_0.3.1                 webshot_0.5.2              
[119] XVector_0.32.0              rvest_1.0.2                
[121] yulab.utils_0.0.4           digest_0.6.27              
[123] Biostrings_2.60.1           rmarkdown_2.11             
[125] cellranger_1.1.0            fastmatch_1.1-3            
[127] tidytree_0.3.5              curl_4.3.2                 
[129] nlme_3.1-152                lifecycle_1.0.1            
[131] jsonlite_1.7.2              carData_3.0-4              
[133] viridisLite_0.4.0           fansi_0.5.0                
[135] pillar_1.6.4                lattice_0.20-44            
[137] KEGGREST_1.32.0             fastmap_1.1.0              
[139] httr_1.4.2                  survival_3.2-11            
[141] GO.db_3.13.0                glue_1.4.2                 
[143] zip_2.2.0                   png_0.1-7                  
[145] bit_4.0.4                   ggforce_0.3.3              
[147] stringi_1.6.2               sass_0.4.0                 
[149] blob_1.2.2                  DESeq2_1.32.0              
[151] memoise_2.0.0               ape_5.5                    </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
