<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Ha Tran" />


<title>KEGG Analysis</title>

<script src="site_libs/header-attrs-2.11/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/lightable-0.0.1/lightable.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">CKI_DGE</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Setting Up
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="obtainData.html">Obtaining Data</a>
    </li>
    <li>
      <a href="setUp.html">QC and Pre-processing</a>
    </li>
  </ul>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Analysis
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="dge.html">DE Analysis</a>
    </li>
    <li>
      <a href="go.html">GO Term Enrichment Analysis</a>
    </li>
    <li>
      <a href="kegg.html">KEGG Pathway Enrichment Analysis</a>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">KEGG Analysis</h1>
<h4 class="author">Ha Tran</h4>
<h4 class="date">19/10/2021</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2021-11-13
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 5 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 2
</p>
<p>
<strong>Knit directory:</strong> <code>01_CKI_DGE/1_analysis/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges" class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown file has unstaged changes. To know which version of the R Markdown file created these results, you’ll want to first commit it to the Git repo. If you’re still working on the analysis, you can ignore this warning. When you’re finished, you can run <code>wflow_publish</code> to commit the R Markdown file and build the HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongobjectspresent"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>Environment:</strong> objects present </a>
</p>
</div>
<div id="strongEnvironmentstrongobjectspresent" class="panel-collapse collapse">
<div class="panel-body">
<p>
The global environment had objects present when the code in the R Markdown file was run. These objects can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment. Use <code>wflow_publish</code> or <code>wflow_build</code> to ensure that the code is always run in an empty environment.
</p>
<p>
The following objects were defined in the global environment when these results were created:
</p>
<table class="table table-condensed table-hover">
<thead>
<tr>
<th style="text-align:left;">
Name
</th>
<th style="text-align:left;">
Class
</th>
<th style="text-align:left;">
Size
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
anno
</td>
<td style="text-align:left;">
data.frame
</td>
<td style="text-align:left;">
2.3 Kb
</td>
</tr>
<tr>
<td style="text-align:left;">
anno_colour
</td>
<td style="text-align:left;">
character
</td>
<td style="text-align:left;">
624 bytes
</td>
</tr>
<tr>
<td style="text-align:left;">
anno_colours
</td>
<td style="text-align:left;">
list
</td>
<td style="text-align:left;">
912 bytes
</td>
</tr>
<tr>
<td style="text-align:left;">
common
</td>
<td style="text-align:left;">
data.frame
</td>
<td style="text-align:left;">
310 Kb
</td>
</tr>
<tr>
<td style="text-align:left;">
comparison_group
</td>
<td style="text-align:left;">
data.frame
</td>
<td style="text-align:left;">
1.1 Kb
</td>
</tr>
<tr>
<td style="text-align:left;">
contrast
</td>
<td style="text-align:left;">
matrix;array
</td>
<td style="text-align:left;">
1.7 Kb
</td>
</tr>
<tr>
<td style="text-align:left;">
designMatrix
</td>
<td style="text-align:left;">
matrix;array
</td>
<td style="text-align:left;">
3.5 Kb
</td>
</tr>
<tr>
<td style="text-align:left;">
dge
</td>
<td style="text-align:left;">
DGEList
</td>
<td style="text-align:left;">
12 Mb
</td>
</tr>
<tr>
<td style="text-align:left;">
goana_treat
</td>
<td style="text-align:left;">
list
</td>
<td style="text-align:left;">
22.2 Mb
</td>
</tr>
<tr>
<td style="text-align:left;">
goana_treat_sig
</td>
<td style="text-align:left;">
list
</td>
<td style="text-align:left;">
546 Kb
</td>
</tr>
<tr>
<td style="text-align:left;">
goSummaries
</td>
<td style="text-align:left;">
tbl_df;tbl;data.frame
</td>
<td style="text-align:left;">
13.4 Mb
</td>
</tr>
<tr>
<td style="text-align:left;">
group.labs
</td>
<td style="text-align:left;">
factor
</td>
<td style="text-align:left;">
944 bytes
</td>
</tr>
<tr>
<td style="text-align:left;">
i
</td>
<td style="text-align:left;">
integer
</td>
<td style="text-align:left;">
56 bytes
</td>
</tr>
<tr>
<td style="text-align:left;">
logCPM
</td>
<td style="text-align:left;">
matrix;array
</td>
<td style="text-align:left;">
8.4 Kb
</td>
</tr>
<tr>
<td style="text-align:left;">
MAplot
</td>
<td style="text-align:left;">
list
</td>
<td style="text-align:left;">
48 bytes
</td>
</tr>
<tr>
<td style="text-align:left;">
merged_go
</td>
<td style="text-align:left;">
data.frame
</td>
<td style="text-align:left;">
18 Kb
</td>
</tr>
<tr>
<td style="text-align:left;">
minPath
</td>
<td style="text-align:left;">
numeric
</td>
<td style="text-align:left;">
56 bytes
</td>
</tr>
<tr>
<td style="text-align:left;">
my_palette
</td>
<td style="text-align:left;">
character
</td>
<td style="text-align:left;">
12.6 Kb
</td>
</tr>
<tr>
<td style="text-align:left;">
NB_dispersion
</td>
<td style="text-align:left;">
DGEList
</td>
<td style="text-align:left;">
12.5 Mb
</td>
</tr>
<tr>
<td style="text-align:left;">
qlf
</td>
<td style="text-align:left;">
list
</td>
<td style="text-align:left;">
101 Mb
</td>
</tr>
<tr>
<td style="text-align:left;">
qlf_all
</td>
<td style="text-align:left;">
list
</td>
<td style="text-align:left;">
49.2 Mb
</td>
</tr>
<tr>
<td style="text-align:left;">
qlf_decideTest
</td>
<td style="text-align:left;">
list
</td>
<td style="text-align:left;">
10.8 Kb
</td>
</tr>
<tr>
<td style="text-align:left;">
qlf_dispersion
</td>
<td style="text-align:left;">
DGEGLM
</td>
<td style="text-align:left;">
20.6 Mb
</td>
</tr>
<tr>
<td style="text-align:left;">
qlf_sig
</td>
<td style="text-align:left;">
list
</td>
<td style="text-align:left;">
37.5 Mb
</td>
</tr>
<tr>
<td style="text-align:left;">
top_go
</td>
<td style="text-align:left;">
list
</td>
<td style="text-align:left;">
30.2 Kb
</td>
</tr>
<tr>
<td style="text-align:left;">
top_unique
</td>
<td style="text-align:left;">
list
</td>
<td style="text-align:left;">
256.4 Kb
</td>
</tr>
<tr>
<td style="text-align:left;">
treat
</td>
<td style="text-align:left;">
list
</td>
<td style="text-align:left;">
96.1 Mb
</td>
</tr>
<tr>
<td style="text-align:left;">
treat_all
</td>
<td style="text-align:left;">
list
</td>
<td style="text-align:left;">
49.2 Mb
</td>
</tr>
<tr>
<td style="text-align:left;">
treat_decideTest
</td>
<td style="text-align:left;">
list
</td>
<td style="text-align:left;">
10.8 Kb
</td>
</tr>
<tr>
<td style="text-align:left;">
treat_sig
</td>
<td style="text-align:left;">
list
</td>
<td style="text-align:left;">
1.7 Mb
</td>
</tr>
<tr>
<td style="text-align:left;">
unique
</td>
<td style="text-align:left;">
data.frame
</td>
<td style="text-align:left;">
4.6 Kb
</td>
</tr>
<tr>
<td style="text-align:left;">
x
</td>
<td style="text-align:left;">
character
</td>
<td style="text-align:left;">
120 bytes
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed12345code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(12345)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed12345code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(12345)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomtranmanhha13501CKIDGEtree848e971e9ccbf102898eeb089076953d68a0817ftargetblank848e971a"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/tranmanhha135/01_CKI_DGE/tree/848e971e9ccbf102898eeb089076953d68a0817f" target="_blank">848e971</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomtranmanhha13501CKIDGEtree848e971e9ccbf102898eeb089076953d68a0817ftargetblank848e971a" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/tranmanhha135/01_CKI_DGE/tree/848e971e9ccbf102898eeb089076953d68a0817f" target="_blank">848e971</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/

Untracked files:
    Untracked:  0_data/rds_object/qlf_all.rds
    Untracked:  0_data/rds_object/qlf_sig.rds
    Untracked:  0_data/rds_object/treat_all.rds
    Untracked:  0_data/rds_object/treat_sig.rds
    Untracked:  1_analysis/about.html
    Untracked:  1_analysis/dge.html
    Untracked:  1_analysis/figure/dge.Rmd/NBDispersion-1.png
    Untracked:  1_analysis/figure/dge.Rmd/coolmap-1.png
    Untracked:  1_analysis/figure/dge.Rmd/heatmap_common-1.png
    Untracked:  1_analysis/figure/dge.Rmd/heatmap_unique-1.png
    Untracked:  1_analysis/figure/dge.Rmd/qlf_dispersion-1.png
    Untracked:  1_analysis/figure/dge.Rmd/volcano_treat-1.png
    Untracked:  1_analysis/figure/dge.Rmd/volcano_treat-2.png
    Untracked:  1_analysis/figure/dge.Rmd/volcano_treat-3.png
    Untracked:  1_analysis/figure/dge.Rmd/volcano_treat-4.png
    Untracked:  1_analysis/figure/dge.Rmd/volcano_treat-5.png
    Untracked:  1_analysis/figure/go.Rmd/
    Untracked:  1_analysis/figure/kegg.Rmd/completePathway-1.png
    Untracked:  1_analysis/figure/kegg.Rmd/dotPlot-1.png
    Untracked:  1_analysis/figure/kegg.Rmd/partialPathway, -1.png
    Untracked:  1_analysis/figure/kegg.Rmd/partialPathway-1.png
    Untracked:  1_analysis/figure/setUp.Rmd/afterFiltering-2.png
    Untracked:  1_analysis/go.html
    Untracked:  1_analysis/index.html
    Untracked:  1_analysis/site_libs/
    Untracked:  2_plots/1_setup/after_normalisation.svg
    Untracked:  2_plots/2_dge/MA_treat_CKI-Mac vs CKI.svg
    Untracked:  2_plots/2_dge/MA_treat_CKI-Nme vs CKI.svg
    Untracked:  2_plots/2_dge/MA_treat_CKI-Omt vs CKI.svg
    Untracked:  2_plots/2_dge/MA_treat_CKI-Tri vs CKI.svg
    Untracked:  2_plots/2_dge/MA_treat_UT vs CKI.svg
    Untracked:  2_plots/2_dge/NB_dispersion.svg
    Untracked:  2_plots/2_dge/heatmap_logCPM_common.svg
    Untracked:  2_plots/2_dge/heatmap_logCPM_unique.svg
    Untracked:  2_plots/2_dge/pValue_treat_CKI-Mac vs CKI.svg
    Untracked:  2_plots/2_dge/pValue_treat_CKI-Nme vs CKI.svg
    Untracked:  2_plots/2_dge/pValue_treat_CKI-Omt vs CKI.svg
    Untracked:  2_plots/2_dge/pValue_treat_CKI-Tri vs CKI.svg
    Untracked:  2_plots/2_dge/pValue_treat_UT vs CKI.svg
    Untracked:  2_plots/2_dge/volcano_treat_CKI-Mac vs CKI.svg
    Untracked:  2_plots/2_dge/volcano_treat_CKI-Nme vs CKI.svg
    Untracked:  2_plots/2_dge/volcano_treat_CKI-Omt vs CKI.svg
    Untracked:  2_plots/2_dge/volcano_treat_CKI-Tri vs CKI.svg
    Untracked:  2_plots/2_dge/volcano_treat_UT vs CKI.svg
    Untracked:  2_plots/3_go/
    Untracked:  2_plots/4_kegg/
    Untracked:  3_output/enrichKEGG_treat_sig.xlsx
    Untracked:  3_output/goana_treat.xlsx
    Untracked:  3_output/kegga_treat_sig.xlsx
    Untracked:  3_output/qlf_decideTest.xlsx
    Untracked:  3_output/qlf_sig.xlsx
    Untracked:  3_output/treat_decideTest.xlsx
    Untracked:  3_output/treat_sig.xlsx
    Untracked:  Meeting Notes.pdf

Unstaged changes:
    Modified:   0_data/rds_object/dge.rds
    Deleted:    0_data/rds_object/dge_treat_unfiltered.rds
    Deleted:    0_data/rds_object/enrichGO_treat.rds
    Deleted:    0_data/rds_object/enrichKEGG_treat_filtered.rds
    Modified:   0_data/rds_object/qlf.rds
    Deleted:    0_data/rds_object/qlf_filtered.rds
    Modified:   0_data/rds_object/treat.rds
    Deleted:    0_data/rds_object/treat_filtered.rds
    Deleted:    0_data/rds_object/treat_unfiltered.rds
    Modified:   1_analysis/dge.Rmd
    Modified:   1_analysis/figure/dge.Rmd/MAplotCustom_treat-1.png
    Modified:   1_analysis/figure/dge.Rmd/MAplotCustom_treat-2.png
    Modified:   1_analysis/figure/dge.Rmd/MAplotCustom_treat-3.png
    Modified:   1_analysis/figure/dge.Rmd/MAplotCustom_treat-4.png
    Modified:   1_analysis/figure/dge.Rmd/MAplotCustom_treat-5.png
    Modified:   1_analysis/figure/dge.Rmd/pValueHistogram_treat-1.png
    Modified:   1_analysis/figure/dge.Rmd/pValueHistogram_treat-2.png
    Modified:   1_analysis/figure/dge.Rmd/pValueHistogram_treat-3.png
    Modified:   1_analysis/figure/dge.Rmd/pValueHistogram_treat-4.png
    Modified:   1_analysis/figure/dge.Rmd/pValueHistogram_treat-5.png
    Modified:   1_analysis/figure/setUp.Rmd/MDSplot-1.png
    Modified:   1_analysis/figure/setUp.Rmd/afterFiltering-1.png
    Modified:   1_analysis/figure/setUp.Rmd/libraryChecks-1.png
    Modified:   1_analysis/figure/setUp.Rmd/pcaSampleGroup-1.png
    Modified:   1_analysis/figure/setUp.Rmd/pcaSampleType-1.png
    Modified:   1_analysis/figure/setUp.Rmd/visualisingNormalisation-1.png
    Modified:   1_analysis/go.Rmd
    Modified:   1_analysis/kegg.Rmd
    Modified:   1_analysis/setUp.Rmd
    Modified:   2_plots/1_setup/counts_before_and_after_filtering.svg
    Modified:   2_plots/1_setup/library_size.svg
    Modified:   2_plots/1_setup/sample_group_PCA_plot.svg
    Modified:   2_plots/1_setup/sample_type_PCA_plot.svg
    Deleted:    2_plots/2_dge/MA_plot_treat_CKI-Mac vs CKI.svg
    Deleted:    2_plots/2_dge/MA_plot_treat_CKI-Nme vs CKI.svg
    Deleted:    2_plots/2_dge/MA_plot_treat_CKI-Omt vs CKI.svg
    Deleted:    2_plots/2_dge/MA_plot_treat_CKI-Tri vs CKI.svg
    Deleted:    2_plots/2_dge/MA_plot_treat_UT vs CKI.svg
    Deleted:    2_plots/2_dge/estimate_dispersion.svg
    Modified:   2_plots/2_dge/qlf_dispersion.svg
    Deleted:    2_plots/2_dge/qlf_pValue_histogram_CKI-Mac vs CKI.svg
    Deleted:    2_plots/2_dge/qlf_pValue_histogram_CKI-Nme vs CKI.svg
    Deleted:    2_plots/2_dge/qlf_pValue_histogram_CKI-Omt vs CKI.svg
    Deleted:    2_plots/2_dge/qlf_pValue_histogram_CKI-Tri vs CKI.svg
    Deleted:    2_plots/2_dge/qlf_pValue_histogram_UT vs CKI.svg
    Deleted:    2_plots/2_dge/treat_pValue_histogram_CKI-Mac vs CKI.svg
    Deleted:    2_plots/2_dge/treat_pValue_histogram_CKI-Nme vs CKI.svg
    Deleted:    2_plots/2_dge/treat_pValue_histogram_CKI-Omt vs CKI.svg
    Deleted:    2_plots/2_dge/treat_pValue_histogram_CKI-Tri vs CKI.svg
    Deleted:    2_plots/2_dge/treat_pValue_histogram_UT vs CKI.svg
    Deleted:    2_plots/2_dge/volcano_plot_treat_CKI-Mac vs CKI.svg
    Deleted:    2_plots/2_dge/volcano_plot_treat_CKI-Nme vs CKI.svg
    Deleted:    2_plots/2_dge/volcano_plot_treat_CKI-Omt vs CKI.svg
    Deleted:    2_plots/2_dge/volcano_plot_treat_CKI-Tri vs CKI.svg
    Deleted:    2_plots/2_dge/volcano_plot_treat_UT vs CKI.svg
    Deleted:    3_output/enrichGO_treat.xlsx
    Deleted:    3_output/enrichKEGG_treat_unfiltered.xlsx
    Deleted:    3_output/qlf_filtered.xlsx
    Deleted:    3_output/treat_filtered.xlsx

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>1_analysis/kegg.Rmd</code>) and HTML (<code>docs/kegg.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/01_CKI_DGE/blob/848e971e9ccbf102898eeb089076953d68a0817f/1_analysis/kegg.Rmd" target="_blank">848e971</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-11-08
</td>
<td>
adjust plot size, added dot plot for GO and KEGG, added heatmap for DE genes, minor cosmetic changes
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/01_CKI_DGE/848e971e9ccbf102898eeb089076953d68a0817f/docs/kegg.html" target="_blank">848e971</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-11-08
</td>
<td>
adjust plot size, added dot plot for GO and KEGG, added heatmap for DE genes, minor cosmetic changes
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/01_CKI_DGE/blob/1a47b4569a1da246294ec62c8e47e2f6e2638e10/1_analysis/kegg.Rmd" target="_blank">1a47b45</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-31
</td>
<td>
changed heatmap size
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/01_CKI_DGE/1a47b4569a1da246294ec62c8e47e2f6e2638e10/docs/kegg.html" target="_blank">1a47b45</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-31
</td>
<td>
changed heatmap size
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/01_CKI_DGE/blob/70049b22b9a79841a2368454293f8825f08980dd/1_analysis/kegg.Rmd" target="_blank">70049b2</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-31
</td>
<td>
Remove old GO and KEGG code, adjusted directory paths, updated KEGG, minor cosmetic changes
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/01_CKI_DGE/70049b22b9a79841a2368454293f8825f08980dd/docs/kegg.html" target="_blank">70049b2</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-31
</td>
<td>
Remove old GO and KEGG code, adjusted directory paths, updated KEGG, minor cosmetic changes
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/01_CKI_DGE/blob/6823b3c881eddd07176bce86153e0543293d7f02/1_analysis/kegg.Rmd" target="_blank">6823b3c</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-30
</td>
<td>
Add minor cosmetic changes
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/01_CKI_DGE/6823b3c881eddd07176bce86153e0543293d7f02/docs/kegg.html" target="_blank">6823b3c</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-30
</td>
<td>
Add minor cosmetic changes
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/01_CKI_DGE/blob/7a6571ba908cf21e984811a75b8123b1e28a4d69/1_analysis/kegg.Rmd" target="_blank">7a6571b</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-30
</td>
<td>
minor changes
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/01_CKI_DGE/7a6571ba908cf21e984811a75b8123b1e28a4d69/docs/kegg.html" target="_blank">7a6571b</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-30
</td>
<td>
minor changes
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/01_CKI_DGE/f5f4f74bfd77b68627ec8414b040c66a60ed7344/docs/kegg.html" target="_blank">f5f4f74</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-30
</td>
<td>
please please please
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/01_CKI_DGE/blob/a6d8f8a05c71e8da432f180ed252874e35ca0ae1/1_analysis/kegg.Rmd" target="_blank">a6d8f8a</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-30
</td>
<td>
what is happening
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/01_CKI_DGE/a6d8f8a05c71e8da432f180ed252874e35ca0ae1/docs/kegg.html" target="_blank">a6d8f8a</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-30
</td>
<td>
what is happening
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/01_CKI_DGE/blob/d769a7cbf44fe4841bdf220ed214b5f73c00b47a/1_analysis/kegg.Rmd" target="_blank">d769a7c</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-30
</td>
<td>
Upload GO and KEGG analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/01_CKI_DGE/d769a7cbf44fe4841bdf220ed214b5f73c00b47a/docs/kegg.html" target="_blank">d769a7c</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-30
</td>
<td>
Upload GO and KEGG analysis
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="data-setup" class="section level1">
<h1>Data Setup</h1>
<div id="load-library" class="section level3">
<h3>Load Library</h3>
<pre class="r"><code># working with data
library(dplyr)
library(magrittr)
library(readr)
library(tibble)
library(reshape2)
library(tidyverse)
library(KEGGREST)

# Visualisation:
library(kableExtra)
library(ggplot2)
library(grid)
library(pander)
library(cowplot)
library(pheatmap)

# Custom ggplot 
library(ggbiplot)
library(ggrepel)

# Bioconductor packages:
library(edgeR)
library(limma)
library(Glimma)
library(clusterProfiler)
library(org.Hs.eg.db)
library(enrichplot)
library(biomaRt)

# Set ggplot theme
theme_set(theme_minimal())</code></pre>
</div>
<div id="import-rds-data" class="section level3">
<h3>Import RDS Data</h3>
<p>DGElist object containing the raw feature count, sample metadata, and gene metadata, created in the Set Up stage.</p>
<pre class="r"><code># load DGElist previously created in the set up
dge &lt;- readRDS(here::here(&quot;0_data/rds_object/dge.rds&quot;))
designMatrix &lt;- readRDS(here::here(&quot;0_data/rds_object/designMatrix.rds&quot;))
contrast &lt;- readRDS(here::here(&quot;0_data/rds_object/contrastMatrix.rds&quot;))
treat &lt;- readRDS(here::here(&quot;0_data/rds_object/treat.rds&quot;))
treat_all &lt;- readRDS(here::here(&quot;0_data/rds_object/treat_all.rds&quot;))
treat_sig &lt;- readRDS(here::here(&quot;0_data/rds_object/treat_sig.rds&quot;))</code></pre>
</div>
</div>
<div id="kegg-enrichment-analysis" class="section level1">
<h1>KEGG enrichment analysis</h1>
<div id="using-the-clusterprofilerenrichkegg-package" class="section level3 tabset">
<h3 class="tabset">Using the <code>clusterProfiler::enrichKEGG</code> package</h3>
<p>KEGG pathway enrichment analysis is performed with GLM:QLF + TREAT.</p>
<p>The top 30 most significant KEGG terms are displayed in relation to their respective comparison. KEGG terms with <code>qvalue &lt;= 0.2</code> are considered as significant</p>
<pre class="r"><code>comparison_group=colnames(contrast) %&gt;% as.data.frame()
# Create list object
enrichKEGG_treat=list()
enrichKEGG_treat_sig=list()

for (i in 1:ncol(contrast)){
  #at each iteration, let x = name of each contrast group
  x=comparison_group[i,]
  
  #populate the enrichGo list with all KEGG terms
  KEGGresults &lt;- clusterProfiler::enrichKEGG(gene = treat_sig[[x]]$entrezid,
                        organism = &quot;hsa&quot;,
                        pvalueCutoff = 0.05,
                        pAdjustMethod = &quot;fdr&quot;) 
  
  enrichKEGG_treat[[x]] &lt;- KEGGresults@result %&gt;% as.data.frame()
 
  #filter only significant KEGG terms
  enrichKEGG_treat_sig[[x]] &lt;- enrichKEGG_treat[[x]] %&gt;% dplyr::filter(qvalue &lt;= 0.2)
} </code></pre>
</div>
<div id="edgerkegga-package" class="section level3 tabset">
<h3 class="tabset"><code>edgeR::kegga</code> package</h3>
<p>Through edgeR, the proportion of upregulation and downregulated genes enriching the KEGG pathway can be determined</p>
<pre class="r"><code>comparison_group=colnames(contrast) %&gt;% as.data.frame()
# Create list object
kegga_treat=list()
kegga_treat_sig=list()
# kegga_treat_sig=list()

for (i in 1:ncol(contrast)){
  #at each iteration, let x = name of each contrast group
  x=comparison_group[i,]
  
  #populate the enrichGo list with all KEGG terms
  kegga_treat[[x]] &lt;- edgeR::kegga.DGELRT(de = treat[[x]],
                                    geneid = treat[[x]]$genes$entrezid,
                                    species = &quot;Hs&quot;,
                                    FDR = 0.05, 
                                    trend = F) 
  
  kegga_treat_sig[[x]] &lt;- topKEGG(results = kegga_treat[[x]], number = Inf)
}</code></pre>
<div id="ut-vs-cki" class="section level4">
<h4>UT vs CKI</h4>
<pre class="r"><code>kegga_treat_sig[[&quot;UT vs CKI&quot;]][1:30,] %&gt;% as.data.frame() %&gt;% kable(digits = Inf) %&gt;% kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;% scroll_box(height=&quot;600px&quot;)</code></pre>
<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Pathway
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
N
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Up
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Down
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
P.Up
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
P.Down
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
path:hsa00140
</td>
<td style="text-align:left;">
Steroid hormone biosynthesis
</td>
<td style="text-align:right;">
19
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
1.000000e+00
</td>
<td style="text-align:right;">
8.170340e-09
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa00980
</td>
<td style="text-align:left;">
Metabolism of xenobiotics by cytochrome P450
</td>
<td style="text-align:right;">
25
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
1.000000e+00
</td>
<td style="text-align:right;">
1.884032e-06
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05146
</td>
<td style="text-align:left;">
Amoebiasis
</td>
<td style="text-align:right;">
61
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
7.503240e-06
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05204
</td>
<td style="text-align:left;">
Chemical carcinogenesis - DNA adducts
</td>
<td style="text-align:right;">
21
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
1.000000e+00
</td>
<td style="text-align:right;">
2.653389e-05
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05323
</td>
<td style="text-align:left;">
Rheumatoid arthritis
</td>
<td style="text-align:right;">
59
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
9.270535e-05
</td>
<td style="text-align:right;">
3.967717e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04061
</td>
<td style="text-align:left;">
Viral protein interaction with cytokine and cytokine receptor
</td>
<td style="text-align:right;">
32
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1.051605e-04
</td>
<td style="text-align:right;">
3.014869e-02
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04657
</td>
<td style="text-align:left;">
IL-17 signaling pathway
</td>
<td style="text-align:right;">
64
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.369199e-04
</td>
<td style="text-align:right;">
4.221298e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05417
</td>
<td style="text-align:left;">
Lipid and atherosclerosis
</td>
<td style="text-align:right;">
154
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1.952556e-04
</td>
<td style="text-align:right;">
3.780755e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04933
</td>
<td style="text-align:left;">
AGE-RAGE signaling pathway in diabetic complications
</td>
<td style="text-align:right;">
80
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
3.913846e-04
</td>
<td style="text-align:right;">
4.963894e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04621
</td>
<td style="text-align:left;">
NOD-like receptor signaling pathway
</td>
<td style="text-align:right;">
129
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
5.026596e-04
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05134
</td>
<td style="text-align:left;">
Legionellosis
</td>
<td style="text-align:right;">
50
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
6.050657e-04
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05144
</td>
<td style="text-align:left;">
Malaria
</td>
<td style="text-align:right;">
24
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
8.230637e-04
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05167
</td>
<td style="text-align:left;">
Kaposi sarcoma-associated herpesvirus infection
</td>
<td style="text-align:right;">
149
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
1.070941e-03
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04913
</td>
<td style="text-align:left;">
Ovarian steroidogenesis
</td>
<td style="text-align:right;">
26
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1.000000e+00
</td>
<td style="text-align:right;">
1.351010e-03
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa00380
</td>
<td style="text-align:left;">
Tryptophan metabolism
</td>
<td style="text-align:right;">
26
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1.841189e-01
</td>
<td style="text-align:right;">
1.351010e-03
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04060
</td>
<td style="text-align:left;">
Cytokine-cytokine receptor interaction
</td>
<td style="text-align:right;">
115
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
2.017760e-03
</td>
<td style="text-align:right;">
7.478291e-02
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04110
</td>
<td style="text-align:left;">
Cell cycle
</td>
<td style="text-align:right;">
121
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
2.519758e-03
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05160
</td>
<td style="text-align:left;">
Hepatitis C
</td>
<td style="text-align:right;">
121
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
2.519758e-03
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05164
</td>
<td style="text-align:left;">
Influenza A
</td>
<td style="text-align:right;">
126
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
3.002825e-03
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04218
</td>
<td style="text-align:left;">
Cellular senescence
</td>
<td style="text-align:right;">
141
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
4.852209e-03
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa00340
</td>
<td style="text-align:left;">
Histidine metabolism
</td>
<td style="text-align:right;">
13
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1.000000e+00
</td>
<td style="text-align:right;">
5.264426e-03
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04512
</td>
<td style="text-align:left;">
ECM-receptor interaction
</td>
<td style="text-align:right;">
48
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
6.146326e-03
</td>
<td style="text-align:right;">
3.370385e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04371
</td>
<td style="text-align:left;">
Apelin signaling pathway
</td>
<td style="text-align:right;">
89
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
1.000000e+00
</td>
<td style="text-align:right;">
6.956942e-03
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa00010
</td>
<td style="text-align:left;">
Glycolysis / Gluconeogenesis
</td>
<td style="text-align:right;">
46
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1.000000e+00
</td>
<td style="text-align:right;">
6.974996e-03
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04114
</td>
<td style="text-align:left;">
Oocyte meiosis
</td>
<td style="text-align:right;">
99
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
7.403498e-03
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04936
</td>
<td style="text-align:left;">
Alcoholic liver disease
</td>
<td style="text-align:right;">
104
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
8.781487e-03
</td>
<td style="text-align:right;">
5.904131e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05162
</td>
<td style="text-align:left;">
Measles
</td>
<td style="text-align:right;">
108
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
9.997326e-03
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05171
</td>
<td style="text-align:left;">
Coronavirus disease - COVID-19
</td>
<td style="text-align:right;">
168
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.000957e-02
</td>
<td style="text-align:right;">
7.644099e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04068
</td>
<td style="text-align:left;">
FoxO signaling pathway
</td>
<td style="text-align:right;">
109
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
1.031748e-02
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05120
</td>
<td style="text-align:left;">
Epithelial cell signaling in Helicobacter pylori infection
</td>
<td style="text-align:right;">
59
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
1.086421e-02
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="cki-mac-vs-cki" class="section level4">
<h4>CKI-Mac vs CKI</h4>
<pre class="r"><code>kegga_treat_sig[[&quot;CKI-Mac vs CKI&quot;]][1:30,] %&gt;% as.data.frame() %&gt;% kable(digits = Inf) %&gt;% kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;% scroll_box(height=&quot;600px&quot;)</code></pre>
<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Pathway
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
N
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Up
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Down
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
P.Up
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
P.Down
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
path:hsa04657
</td>
<td style="text-align:left;">
IL-17 signaling pathway
</td>
<td style="text-align:right;">
64
</td>
<td style="text-align:right;">
14
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
1.798677e-13
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04060
</td>
<td style="text-align:left;">
Cytokine-cytokine receptor interaction
</td>
<td style="text-align:right;">
115
</td>
<td style="text-align:right;">
16
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
5.183302e-12
</td>
<td style="text-align:right;">
4.519540e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05323
</td>
<td style="text-align:left;">
Rheumatoid arthritis
</td>
<td style="text-align:right;">
59
</td>
<td style="text-align:right;">
11
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
4.796450e-10
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04668
</td>
<td style="text-align:left;">
TNF signaling pathway
</td>
<td style="text-align:right;">
94
</td>
<td style="text-align:right;">
12
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
7.127366e-09
</td>
<td style="text-align:right;">
7.158749e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04061
</td>
<td style="text-align:left;">
Viral protein interaction with cytokine and cytokine receptor
</td>
<td style="text-align:right;">
32
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.021030e-08
</td>
<td style="text-align:right;">
3.477287e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05417
</td>
<td style="text-align:left;">
Lipid and atherosclerosis
</td>
<td style="text-align:right;">
154
</td>
<td style="text-align:right;">
14
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
3.309226e-08
</td>
<td style="text-align:right;">
6.082960e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05146
</td>
<td style="text-align:left;">
Amoebiasis
</td>
<td style="text-align:right;">
61
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1.608522e-07
</td>
<td style="text-align:right;">
1.934733e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05202
</td>
<td style="text-align:left;">
Transcriptional misregulation in cancer
</td>
<td style="text-align:right;">
122
</td>
<td style="text-align:right;">
11
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1.119545e-06
</td>
<td style="text-align:right;">
4.825713e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04080
</td>
<td style="text-align:left;">
Neuroactive ligand-receptor interaction
</td>
<td style="text-align:right;">
76
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
1.000000e+00
</td>
<td style="text-align:right;">
7.023903e-06
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04064
</td>
<td style="text-align:left;">
NF-kappa B signaling pathway
</td>
<td style="text-align:right;">
73
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
7.993704e-06
</td>
<td style="text-align:right;">
2.519953e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04621
</td>
<td style="text-align:left;">
NOD-like receptor signaling pathway
</td>
<td style="text-align:right;">
129
</td>
<td style="text-align:right;">
10
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
1.349574e-05
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04640
</td>
<td style="text-align:left;">
Hematopoietic cell lineage
</td>
<td style="text-align:right;">
41
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
2.111429e-05
</td>
<td style="text-align:right;">
4.217049e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05171
</td>
<td style="text-align:left;">
Coronavirus disease - COVID-19
</td>
<td style="text-align:right;">
168
</td>
<td style="text-align:right;">
11
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
2.465219e-05
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05134
</td>
<td style="text-align:left;">
Legionellosis
</td>
<td style="text-align:right;">
50
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
6.721155e-05
</td>
<td style="text-align:right;">
4.873363e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05144
</td>
<td style="text-align:left;">
Malaria
</td>
<td style="text-align:right;">
24
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
3.211999e-04
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04151
</td>
<td style="text-align:left;">
PI3K-Akt signaling pathway
</td>
<td style="text-align:right;">
225
</td>
<td style="text-align:right;">
11
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
3.354109e-04
</td>
<td style="text-align:right;">
7.881236e-02
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04630
</td>
<td style="text-align:left;">
JAK-STAT signaling pathway
</td>
<td style="text-align:right;">
96
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
3.995657e-04
</td>
<td style="text-align:right;">
1.346765e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04933
</td>
<td style="text-align:left;">
AGE-RAGE signaling pathway in diabetic complications
</td>
<td style="text-align:right;">
80
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
8.982439e-04
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05133
</td>
<td style="text-align:left;">
Pertussis
</td>
<td style="text-align:right;">
54
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
9.373373e-04
</td>
<td style="text-align:right;">
5.140760e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05167
</td>
<td style="text-align:left;">
Kaposi sarcoma-associated herpesvirus infection
</td>
<td style="text-align:right;">
149
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1.200180e-03
</td>
<td style="text-align:right;">
5.902619e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05219
</td>
<td style="text-align:left;">
Bladder cancer
</td>
<td style="text-align:right;">
37
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
4.080434e-01
</td>
<td style="text-align:right;">
1.395048e-03
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05120
</td>
<td style="text-align:left;">
Epithelial cell signaling in Helicobacter pylori infection
</td>
<td style="text-align:right;">
59
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
1.402484e-03
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05332
</td>
<td style="text-align:left;">
Graft-versus-host disease
</td>
<td style="text-align:right;">
17
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
1.603850e-03
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa00532
</td>
<td style="text-align:left;">
Glycosaminoglycan biosynthesis - chondroitin sulfate / dermatan sulfate
</td>
<td style="text-align:right;">
18
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1.000000e+00
</td>
<td style="text-align:right;">
1.609430e-03
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05224
</td>
<td style="text-align:left;">
Breast cancer
</td>
<td style="text-align:right;">
102
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
1.000000e+00
</td>
<td style="text-align:right;">
2.331468e-03
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04932
</td>
<td style="text-align:left;">
Non-alcoholic fatty liver disease
</td>
<td style="text-align:right;">
135
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
2.941332e-03
</td>
<td style="text-align:right;">
8.363805e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05142
</td>
<td style="text-align:left;">
Chagas disease
</td>
<td style="text-align:right;">
71
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
3.186862e-03
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04936
</td>
<td style="text-align:left;">
Alcoholic liver disease
</td>
<td style="text-align:right;">
104
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
3.441766e-03
</td>
<td style="text-align:right;">
4.018733e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04512
</td>
<td style="text-align:left;">
ECM-receptor interaction
</td>
<td style="text-align:right;">
48
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
2.972377e-02
</td>
<td style="text-align:right;">
3.672074e-03
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05140
</td>
<td style="text-align:left;">
Leishmaniasis
</td>
<td style="text-align:right;">
50
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
5.243869e-03
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="cki-nme-vs-cki" class="section level4">
<h4>CKI-Nme vs CKI</h4>
<pre class="r"><code>kegga_treat_sig[[&quot;CKI-Nme vs CKI&quot;]][1:30,] %&gt;% as.data.frame() %&gt;% kable(digits = Inf) %&gt;% kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;% scroll_box(height=&quot;600px&quot;)</code></pre>
<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Pathway
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
N
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Up
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Down
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
P.Up
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
P.Down
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
path:hsa04657
</td>
<td style="text-align:left;">
IL-17 signaling pathway
</td>
<td style="text-align:right;">
64
</td>
<td style="text-align:right;">
13
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.069904e-13
</td>
<td style="text-align:right;">
7.038702e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05323
</td>
<td style="text-align:left;">
Rheumatoid arthritis
</td>
<td style="text-align:right;">
59
</td>
<td style="text-align:right;">
11
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
2.453107e-11
</td>
<td style="text-align:right;">
3.045174e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05417
</td>
<td style="text-align:left;">
Lipid and atherosclerosis
</td>
<td style="text-align:right;">
154
</td>
<td style="text-align:right;">
14
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
9.299453e-10
</td>
<td style="text-align:right;">
3.284634e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04061
</td>
<td style="text-align:left;">
Viral protein interaction with cytokine and cytokine receptor
</td>
<td style="text-align:right;">
32
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
3.640487e-08
</td>
<td style="text-align:right;">
2.174725e-02
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04060
</td>
<td style="text-align:left;">
Cytokine-cytokine receptor interaction
</td>
<td style="text-align:right;">
115
</td>
<td style="text-align:right;">
11
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
3.760749e-08
</td>
<td style="text-align:right;">
6.558289e-02
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05146
</td>
<td style="text-align:left;">
Amoebiasis
</td>
<td style="text-align:right;">
61
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
2.504620e-07
</td>
<td style="text-align:right;">
3.184025e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04668
</td>
<td style="text-align:left;">
TNF signaling pathway
</td>
<td style="text-align:right;">
94
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
6.796951e-07
</td>
<td style="text-align:right;">
2.594398e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05134
</td>
<td style="text-align:left;">
Legionellosis
</td>
<td style="text-align:right;">
50
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
9.208567e-07
</td>
<td style="text-align:right;">
6.133388e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05202
</td>
<td style="text-align:left;">
Transcriptional misregulation in cancer
</td>
<td style="text-align:right;">
122
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
6.060349e-06
</td>
<td style="text-align:right;">
1.969396e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04621
</td>
<td style="text-align:left;">
NOD-like receptor signaling pathway
</td>
<td style="text-align:right;">
129
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
7.064271e-05
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04080
</td>
<td style="text-align:left;">
Neuroactive ligand-receptor interaction
</td>
<td style="text-align:right;">
76
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
5.588656e-01
</td>
<td style="text-align:right;">
8.644346e-05
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05144
</td>
<td style="text-align:left;">
Malaria
</td>
<td style="text-align:right;">
24
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
1.118790e-04
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04064
</td>
<td style="text-align:left;">
NF-kappa B signaling pathway
</td>
<td style="text-align:right;">
73
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1.254328e-04
</td>
<td style="text-align:right;">
3.998752e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04015
</td>
<td style="text-align:left;">
Rap1 signaling pathway
</td>
<td style="text-align:right;">
142
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
10
</td>
<td style="text-align:right;">
7.841638e-01
</td>
<td style="text-align:right;">
3.445897e-04
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05120
</td>
<td style="text-align:left;">
Epithelial cell signaling in Helicobacter pylori infection
</td>
<td style="text-align:right;">
59
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
4.060664e-04
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05332
</td>
<td style="text-align:left;">
Graft-versus-host disease
</td>
<td style="text-align:right;">
17
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
7.254868e-04
</td>
<td style="text-align:right;">
2.757636e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04936
</td>
<td style="text-align:left;">
Alcoholic liver disease
</td>
<td style="text-align:right;">
104
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
8.540500e-04
</td>
<td style="text-align:right;">
5.849777e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04640
</td>
<td style="text-align:left;">
Hematopoietic cell lineage
</td>
<td style="text-align:right;">
41
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
9.266478e-04
</td>
<td style="text-align:right;">
4.136236e-02
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05167
</td>
<td style="text-align:left;">
Kaposi sarcoma-associated herpesvirus infection
</td>
<td style="text-align:right;">
149
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
1.078795e-03
</td>
<td style="text-align:right;">
3.071872e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05200
</td>
<td style="text-align:left;">
Pathways in cancer
</td>
<td style="text-align:right;">
368
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
16
</td>
<td style="text-align:right;">
4.356827e-02
</td>
<td style="text-align:right;">
1.601716e-03
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05020
</td>
<td style="text-align:left;">
Prion disease
</td>
<td style="text-align:right;">
207
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1.689907e-03
</td>
<td style="text-align:right;">
9.040065e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05140
</td>
<td style="text-align:left;">
Leishmaniasis
</td>
<td style="text-align:right;">
50
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1.957010e-03
</td>
<td style="text-align:right;">
2.416715e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05152
</td>
<td style="text-align:left;">
Tuberculosis
</td>
<td style="text-align:right;">
123
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
2.029032e-03
</td>
<td style="text-align:right;">
4.080082e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05171
</td>
<td style="text-align:left;">
Coronavirus disease - COVID-19
</td>
<td style="text-align:right;">
168
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
2.150487e-03
</td>
<td style="text-align:right;">
9.595671e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04915
</td>
<td style="text-align:left;">
Estrogen signaling pathway
</td>
<td style="text-align:right;">
87
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
2.362883e-03
</td>
<td style="text-align:right;">
4.887625e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05133
</td>
<td style="text-align:left;">
Pertussis
</td>
<td style="text-align:right;">
54
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
2.600630e-03
</td>
<td style="text-align:right;">
2.696231e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04151
</td>
<td style="text-align:left;">
PI3K-Akt signaling pathway
</td>
<td style="text-align:right;">
225
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
11
</td>
<td style="text-align:right;">
2.839250e-03
</td>
<td style="text-align:right;">
3.528335e-03
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa01523
</td>
<td style="text-align:left;">
Antifolate resistance
</td>
<td style="text-align:right;">
27
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
2.886730e-03
</td>
<td style="text-align:right;">
4.010830e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa00100
</td>
<td style="text-align:left;">
Steroid biosynthesis
</td>
<td style="text-align:right;">
16
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1.000000e+00
</td>
<td style="text-align:right;">
3.059437e-03
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05224
</td>
<td style="text-align:left;">
Breast cancer
</td>
<td style="text-align:right;">
102
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
1.000000e+00
</td>
<td style="text-align:right;">
3.075748e-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="cki-omt-vs-cki" class="section level4">
<h4>CKI-Omt vs CKI</h4>
<pre class="r"><code>kegga_treat_sig[[&quot;CKI-Omt vs CKI&quot;]][1:30,] %&gt;% as.data.frame() %&gt;% kable(digits = Inf) %&gt;% kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;% scroll_box(height=&quot;600px&quot;)</code></pre>
<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Pathway
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
N
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Up
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Down
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
P.Up
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
P.Down
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
path:hsa04110
</td>
<td style="text-align:left;">
Cell cycle
</td>
<td style="text-align:right;">
121
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
23
</td>
<td style="text-align:right;">
4.135760e-01
</td>
<td style="text-align:right;">
1.149497e-11
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04060
</td>
<td style="text-align:left;">
Cytokine-cytokine receptor interaction
</td>
<td style="text-align:right;">
115
</td>
<td style="text-align:right;">
17
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
6.437442e-11
</td>
<td style="text-align:right;">
9.032857e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04657
</td>
<td style="text-align:left;">
IL-17 signaling pathway
</td>
<td style="text-align:right;">
64
</td>
<td style="text-align:right;">
13
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
2.006953e-10
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05323
</td>
<td style="text-align:left;">
Rheumatoid arthritis
</td>
<td style="text-align:right;">
59
</td>
<td style="text-align:right;">
11
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.340250e-08
</td>
<td style="text-align:right;">
8.678446e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04061
</td>
<td style="text-align:left;">
Viral protein interaction with cytokine and cytokine receptor
</td>
<td style="text-align:right;">
32
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.193469e-07
</td>
<td style="text-align:right;">
6.659379e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04668
</td>
<td style="text-align:left;">
TNF signaling pathway
</td>
<td style="text-align:right;">
94
</td>
<td style="text-align:right;">
12
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
2.353460e-07
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04640
</td>
<td style="text-align:left;">
Hematopoietic cell lineage
</td>
<td style="text-align:right;">
41
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
9.333811e-07
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05146
</td>
<td style="text-align:left;">
Amoebiasis
</td>
<td style="text-align:right;">
61
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
2.284843e-06
</td>
<td style="text-align:right;">
8.766280e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa03460
</td>
<td style="text-align:left;">
Fanconi anemia pathway
</td>
<td style="text-align:right;">
48
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
10
</td>
<td style="text-align:right;">
1.000000e+00
</td>
<td style="text-align:right;">
3.468817e-06
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05202
</td>
<td style="text-align:left;">
Transcriptional misregulation in cancer
</td>
<td style="text-align:right;">
122
</td>
<td style="text-align:right;">
12
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
3.975183e-06
</td>
<td style="text-align:right;">
7.831845e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04630
</td>
<td style="text-align:left;">
JAK-STAT signaling pathway
</td>
<td style="text-align:right;">
96
</td>
<td style="text-align:right;">
10
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1.546535e-05
</td>
<td style="text-align:right;">
6.315429e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa03440
</td>
<td style="text-align:left;">
Homologous recombination
</td>
<td style="text-align:right;">
39
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
1.000000e+00
</td>
<td style="text-align:right;">
3.773112e-05
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05417
</td>
<td style="text-align:left;">
Lipid and atherosclerosis
</td>
<td style="text-align:right;">
154
</td>
<td style="text-align:right;">
12
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
4.259294e-05
</td>
<td style="text-align:right;">
9.950238e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04914
</td>
<td style="text-align:left;">
Progesterone-mediated oocyte maturation
</td>
<td style="text-align:right;">
77
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
11
</td>
<td style="text-align:right;">
1.000000e+00
</td>
<td style="text-align:right;">
4.954075e-05
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04064
</td>
<td style="text-align:left;">
NF-kappa B signaling pathway
</td>
<td style="text-align:right;">
73
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
7.724718e-05
</td>
<td style="text-align:right;">
9.183598e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04114
</td>
<td style="text-align:left;">
Oocyte meiosis
</td>
<td style="text-align:right;">
99
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
12
</td>
<td style="text-align:right;">
8.554284e-01
</td>
<td style="text-align:right;">
1.174922e-04
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa00260
</td>
<td style="text-align:left;">
Glycine, serine and threonine metabolism
</td>
<td style="text-align:right;">
26
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1.203799e-04
</td>
<td style="text-align:right;">
5.545767e-02
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04080
</td>
<td style="text-align:left;">
Neuroactive ligand-receptor interaction
</td>
<td style="text-align:right;">
76
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
10
</td>
<td style="text-align:right;">
7.731078e-01
</td>
<td style="text-align:right;">
2.186150e-04
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05134
</td>
<td style="text-align:left;">
Legionellosis
</td>
<td style="text-align:right;">
50
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
3.759009e-04
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04115
</td>
<td style="text-align:left;">
p53 signaling pathway
</td>
<td style="text-align:right;">
67
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
4.011926e-02
</td>
<td style="text-align:right;">
3.839455e-04
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa03030
</td>
<td style="text-align:left;">
DNA replication
</td>
<td style="text-align:right;">
35
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
1.000000e+00
</td>
<td style="text-align:right;">
9.910943e-04
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05144
</td>
<td style="text-align:left;">
Malaria
</td>
<td style="text-align:right;">
24
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
1.054800e-03
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05167
</td>
<td style="text-align:left;">
Kaposi sarcoma-associated herpesvirus infection
</td>
<td style="text-align:right;">
149
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
2.375477e-03
</td>
<td style="text-align:right;">
9.630064e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05143
</td>
<td style="text-align:left;">
African trypanosomiasis
</td>
<td style="text-align:right;">
15
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
2.708445e-03
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04621
</td>
<td style="text-align:left;">
NOD-like receptor signaling pathway
</td>
<td style="text-align:right;">
129
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
3.488666e-03
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04923
</td>
<td style="text-align:left;">
Regulation of lipolysis in adipocytes
</td>
<td style="text-align:right;">
33
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
3.547179e-03
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05332
</td>
<td style="text-align:left;">
Graft-versus-host disease
</td>
<td style="text-align:right;">
17
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
3.934360e-03
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa01230
</td>
<td style="text-align:left;">
Biosynthesis of amino acids
</td>
<td style="text-align:right;">
57
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
4.722445e-03
</td>
<td style="text-align:right;">
3.005321e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05162
</td>
<td style="text-align:left;">
Measles
</td>
<td style="text-align:right;">
108
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
4.848407e-03
</td>
<td style="text-align:right;">
7.091949e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05171
</td>
<td style="text-align:left;">
Coronavirus disease - COVID-19
</td>
<td style="text-align:right;">
168
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
5.261665e-03
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="cki-tri-vs-cki" class="section level4">
<h4>CKI-Tri vs CKI</h4>
<pre class="r"><code>kegga_treat_sig[[&quot;CKI-Tri vs CKI&quot;]][1:30,] %&gt;% as.data.frame() %&gt;% kable(digits = Inf ) %&gt;% kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;% scroll_box(height=&quot;600px&quot;)</code></pre>
<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Pathway
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
N
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Up
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Down
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
P.Up
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
P.Down
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
path:hsa04657
</td>
<td style="text-align:left;">
IL-17 signaling pathway
</td>
<td style="text-align:right;">
64
</td>
<td style="text-align:right;">
13
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
3.616725e-13
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04060
</td>
<td style="text-align:left;">
Cytokine-cytokine receptor interaction
</td>
<td style="text-align:right;">
115
</td>
<td style="text-align:right;">
15
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
4.608758e-12
</td>
<td style="text-align:right;">
5.513003e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05323
</td>
<td style="text-align:left;">
Rheumatoid arthritis
</td>
<td style="text-align:right;">
59
</td>
<td style="text-align:right;">
11
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
6.791268e-11
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04061
</td>
<td style="text-align:left;">
Viral protein interaction with cytokine and cytokine receptor
</td>
<td style="text-align:right;">
32
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
2.438238e-09
</td>
<td style="text-align:right;">
4.031305e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05417
</td>
<td style="text-align:left;">
Lipid and atherosclerosis
</td>
<td style="text-align:right;">
154
</td>
<td style="text-align:right;">
14
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
3.191468e-09
</td>
<td style="text-align:right;">
4.477029e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05146
</td>
<td style="text-align:left;">
Amoebiasis
</td>
<td style="text-align:right;">
61
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
3.369301e-08
</td>
<td style="text-align:right;">
2.547865e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04668
</td>
<td style="text-align:left;">
TNF signaling pathway
</td>
<td style="text-align:right;">
94
</td>
<td style="text-align:right;">
10
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1.402841e-07
</td>
<td style="text-align:right;">
4.447782e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05202
</td>
<td style="text-align:left;">
Transcriptional misregulation in cancer
</td>
<td style="text-align:right;">
122
</td>
<td style="text-align:right;">
10
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1.617675e-06
</td>
<td style="text-align:right;">
3.093831e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05134
</td>
<td style="text-align:left;">
Legionellosis
</td>
<td style="text-align:right;">
50
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.725302e-06
</td>
<td style="text-align:right;">
5.537732e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04640
</td>
<td style="text-align:left;">
Hematopoietic cell lineage
</td>
<td style="text-align:right;">
41
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
7.505255e-06
</td>
<td style="text-align:right;">
4.838920e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04080
</td>
<td style="text-align:left;">
Neuroactive ligand-receptor interaction
</td>
<td style="text-align:right;">
76
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
1.000000e+00
</td>
<td style="text-align:right;">
2.744942e-05
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04630
</td>
<td style="text-align:left;">
JAK-STAT signaling pathway
</td>
<td style="text-align:right;">
96
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1.314334e-04
</td>
<td style="text-align:right;">
1.982536e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04621
</td>
<td style="text-align:left;">
NOD-like receptor signaling pathway
</td>
<td style="text-align:right;">
129
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
1.357830e-04
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05144
</td>
<td style="text-align:left;">
Malaria
</td>
<td style="text-align:right;">
24
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
1.604193e-04
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04064
</td>
<td style="text-align:left;">
NF-kappa B signaling pathway
</td>
<td style="text-align:right;">
73
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
2.088869e-04
</td>
<td style="text-align:right;">
3.258244e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04218
</td>
<td style="text-align:left;">
Cellular senescence
</td>
<td style="text-align:right;">
141
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
2.506383e-04
</td>
<td style="text-align:right;">
7.561901e-02
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04151
</td>
<td style="text-align:left;">
PI3K-Akt signaling pathway
</td>
<td style="text-align:right;">
225
</td>
<td style="text-align:right;">
10
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
3.152155e-04
</td>
<td style="text-align:right;">
6.946354e-02
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05167
</td>
<td style="text-align:left;">
Kaposi sarcoma-associated herpesvirus infection
</td>
<td style="text-align:right;">
149
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
3.643350e-04
</td>
<td style="text-align:right;">
4.266085e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05120
</td>
<td style="text-align:left;">
Epithelial cell signaling in Helicobacter pylori infection
</td>
<td style="text-align:right;">
59
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
6.219373e-04
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05171
</td>
<td style="text-align:left;">
Coronavirus disease - COVID-19
</td>
<td style="text-align:right;">
168
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
8.075663e-04
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05332
</td>
<td style="text-align:left;">
Graft-versus-host disease
</td>
<td style="text-align:right;">
17
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
9.511499e-04
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa00340
</td>
<td style="text-align:left;">
Histidine metabolism
</td>
<td style="text-align:right;">
13
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1.000000e+00
</td>
<td style="text-align:right;">
1.020990e-03
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04010
</td>
<td style="text-align:left;">
MAPK signaling pathway
</td>
<td style="text-align:right;">
217
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
1.030437e-03
</td>
<td style="text-align:right;">
4.579362e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04936
</td>
<td style="text-align:left;">
Alcoholic liver disease
</td>
<td style="text-align:right;">
104
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1.384170e-03
</td>
<td style="text-align:right;">
2.315744e-01
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05142
</td>
<td style="text-align:left;">
Chagas disease
</td>
<td style="text-align:right;">
71
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
1.445664e-03
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa04933
</td>
<td style="text-align:left;">
AGE-RAGE signaling pathway in diabetic complications
</td>
<td style="text-align:right;">
80
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
2.454243e-03
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05140
</td>
<td style="text-align:left;">
Leishmaniasis
</td>
<td style="text-align:right;">
50
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
2.746668e-03
</td>
<td style="text-align:right;">
1.000000e+00
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05219
</td>
<td style="text-align:left;">
Bladder cancer
</td>
<td style="text-align:right;">
37
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
3.539763e-01
</td>
<td style="text-align:right;">
2.764455e-03
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05165
</td>
<td style="text-align:left;">
Human papillomavirus infection
</td>
<td style="text-align:right;">
258
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
11
</td>
<td style="text-align:right;">
3.227072e-02
</td>
<td style="text-align:right;">
2.906586e-03
</td>
</tr>
<tr>
<td style="text-align:left;">
path:hsa05020
</td>
<td style="text-align:left;">
Prion disease
</td>
<td style="text-align:right;">
207
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
3.027138e-03
</td>
<td style="text-align:right;">
8.468534e-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="visualisation" class="section level1">
<h1>Visualisation</h1>
<pre class="r"><code>top_kegg=list()
for (i in 1:ncol(contrast)){
  #at each iteration, let x = name of each contrast group
  x=comparison_group[i,]

top_kegg[[x]] &lt;- kegga_treat_sig[[x]][1:15,]
}
#combine all list from big merged_kegg into one
merged_kegg &lt;- as.data.frame(do.call(rbind, lapply(top_kegg, as.data.frame))) %&gt;% rownames_to_column(&quot;group&quot;)

#remove KEGG ID from group names
merged_kegg$group &lt;- gsub(pattern = &quot;\\..*&quot;, &quot;&quot;, merged_kegg$group)

#direction column
merged_kegg &lt;- merged_kegg %&gt;% dplyr::mutate(direction = case_when
                         (Up &gt; Down ~ &quot;Up-\nregulated\n&quot;,
                          Up &lt; Down ~ &quot;Down-\nregulated\n&quot;,
                          TRUE ~ &quot;Mutual&quot;))

#geneRatio column
merged_kegg &lt;- merged_kegg %&gt;% dplyr::mutate(geneRatio = case_when
                                     (Up &gt; Down &amp; group == &quot;UT vs CKI&quot; ~ Up/215,
                                       Up &gt; Down &amp; group == &quot;CKI-Omt vs CKI&quot; ~ Up/689,
                                        Up &gt; Down &amp; group == &quot;CKI-Mac vs CKI&quot; ~ Up/353,
                                        Up &gt; Down &amp; group == &quot;CKI-Nme vs CKI&quot; ~ Up/382,
                                        Up &gt; Down &amp; group == &quot;CKI-Tri vs CKI&quot; ~ Up/358,
                                      Up &lt; Down &amp; group == &quot;UT vs CKI&quot; ~ Down/215,
                                       Up &lt; Down &amp; group == &quot;CKI-Omt vs CKI&quot; ~ Down/689,
                                        Up &lt; Down &amp; group == &quot;CKI-Mac vs CKI&quot; ~ Down/353,
                                        Up &lt; Down &amp; group == &quot;CKI-Nme vs CKI&quot; ~ Down/382,
                                        Up &lt; Down &amp; group == &quot;CKI-Tri vs CKI&quot; ~ Down/358,
                                       ))

#pvalue colum
merged_kegg &lt;- merged_kegg %&gt;% dplyr::mutate(pValue = case_when
                         (Up &gt; Down ~ P.Up,
                          Up &lt; Down ~ P.Down,
                          TRUE ~ P.Up))

merged_kegg &lt;- merged_kegg %&gt;% filter(pValue &lt;= 0.05)

# Transform the column &#39;Description&#39; into factors
merged_kegg$Pathway &lt;- as.factor(merged_kegg$Pathway)

# Transform FDR values by -log10(&#39;FDR values&#39;)
merged_kegg$&#39;|log10(FDR)|&#39; &lt;- -(log10(merged_kegg$pValue))

# Change factor order
merged_kegg$group &lt;- factor(merged_kegg$group, levels = c(&quot;UT vs CKI&quot;, &quot;CKI-Omt vs CKI&quot;, &quot;CKI-Mac vs CKI&quot;, &quot;CKI-Nme vs CKI&quot;, &quot;CKI-Tri vs CKI&quot;))

merged_kegg$Pathway &lt;- factor(merged_kegg$Pathway, levels = rev(levels(merged_kegg$Pathway)))

group.labs &lt;- merged_kegg$group

ggplot(merged_kegg, aes(x = Pathway, y = group)) +
  geom_point(data=merged_kegg,aes(x=Pathway, y=group, size = geneRatio, colour = `|log10(FDR)|`, shape = direction), alpha=.9)+
  # scale_y_discrete(labels =group.labs)+
  scale_shape_manual(values = c(15,16,17))+
  scale_color_gradient(low = &quot;dodgerblue3&quot;, high = &quot;firebrick3&quot;, limits=c(0, NA))+
  coord_flip()+
  theme_bw()+
  theme(axis.ticks.length=unit(-0.1, &quot;cm&quot;),
        axis.text.x = element_text(margin=margin(5,5,0,5,&quot;pt&quot;),angle = 45, hjust = 1),
        axis.text.y = element_text(margin=margin(5,5,5,5,&quot;pt&quot;)),
        axis.text = element_text(color = &quot;black&quot;),
        axis.title.x=element_blank())+
  xlab(label = &quot;&quot;)+
  labs(color=expression(&quot;-log&quot;[10]*&quot;FDR&quot;), size=&quot;Gene Ratio&quot;, shape = &quot;Expression&quot;)</code></pre>
<p><img src="figure/kegg.Rmd/dotPlot-1.png" width="768" style="display: block; margin: auto;" /></p>
<pre class="r"><code>  # theme(legend.position = &quot;bottom&quot;)
ggsave(filename = &quot;kegga_treat.svg&quot;, plot = last_plot(), path = here::here(&quot;2_plots/4_kegg/&quot;))</code></pre>
<pre class="r"><code>#kegg pathway of interest
kegg_id &lt;- &quot;hsa04110&quot;
kegg_pathway &lt;- KEGGREST::keggGet(kegg_id)
kegg_pathway_name &lt;- kegg_pathway[[1]]$NAME</code></pre>
<div id="heatmap-of-cell-cycle---homo-sapiens-human" class="section level2 tabset">
<h2 class="tabset">Heatmap of Cell cycle - Homo sapiens (human)</h2>
<div id="partial" class="section level3">
<h3>Partial</h3>
<p>The partial heatmap illustrates the logFC of significant DE genes within the Cell cycle - Homo sapiens (human) pathway.</p>
<pre class="r"><code># kegg_id &lt;- &quot;hsa04657&quot;
# kegg_pathway &lt;- KEGGREST::keggGet(kegg_id)
# kegg_pathway_name &lt;- kegg_pathway[[1]]$NAME
entrezID=list()
matrix_all=list()
for (i in 1:ncol(contrast)){
  #at each iteration, let x = name of each contrast group
  x=comparison_group[i,]
  
  entrezID[[x]] &lt;- kegg_pathway[[1]]$GENE %&gt;% as.data.frame()
  entrezID[[x]] &lt;- entrezID[[x]][seq(1, nrow(entrezID[[x]]), 2),] %&gt;% as.data.frame()
  colnames(entrezID[[x]]) &lt;- &quot;entrezid&quot;
  
  match &lt;- treat_sig[[x]]$entrezid %in% entrezID[[x]]$entrezid
  matrix_all[[x]] &lt;- treat_sig[[x]][match, c(&quot;gene_name&quot;, &quot;logFC&quot;)]
}

big_matrix &lt;- as.data.frame(do.call(rbind, lapply(matrix_all, as.data.frame))) %&gt;% rownames_to_column(&quot;group&quot;) 
big_matrix$group &lt;- gsub(pattern = &quot;\\..*&quot;, &quot;&quot;, big_matrix$group)

big_matrix &lt;- pivot_wider(big_matrix, names_from = group, values_from = logFC) %&gt;% column_to_rownames(&quot;gene_name&quot;)
# big_matrix[is.na(big_matrix)]=-0.7305
summary(big_matrix)</code></pre>
<pre><code> CKI-Mac vs CKI   CKI-Nme vs CKI   CKI-Omt vs CKI   CKI-Tri vs CKI  
 Min.   :-2.762   Min.   :-3.720   Min.   :-4.734   Min.   :-3.899  
 1st Qu.:-1.899   1st Qu.:-2.129   1st Qu.:-3.202   1st Qu.:-2.058  
 Median :-1.891   Median :-2.035   Median :-2.492   Median :-1.924  
 Mean   :-1.205   Mean   :-1.643   Mean   :-2.183   Mean   :-1.638  
 3rd Qu.:-1.661   3rd Qu.:-1.921   3rd Qu.:-1.979   3rd Qu.:-1.752  
 Max.   : 2.190   Max.   : 1.973   Max.   : 3.273   Max.   : 2.302  
 NA&#39;s   :25       NA&#39;s   :24       NA&#39;s   :4        NA&#39;s   :22      
   UT vs CKI    
 Min.   :1.921  
 1st Qu.:1.934  
 Median :2.150  
 Mean   :2.150  
 3rd Qu.:2.226  
 Max.   :2.519  
 NA&#39;s   :25     </code></pre>
<pre class="r"><code>## this was a waste of time
# huge_matrix &lt;- as.data.frame(do.call(rbind, lapply(treat_all, as.data.frame))) %&gt;% rownames_to_column(&quot;group&quot;) %&gt;% dplyr::select(c(&quot;group&quot;, &quot;gene&quot;, &quot;logFC&quot;))
# huge_matrix$group &lt;- gsub(pattern = &quot;\\..*&quot;, &quot;&quot;, huge_matrix$group)
# huge_matrix &lt;- pivot_wider(huge_matrix, names_from = group, values_from = logFC) %&gt;% column_to_rownames(&quot;gene&quot;)
# huge_matrix &lt;- huge_matrix[rownames(big_matrix),]

  my_palette &lt;- colorRampPalette(c(
  rgb(32,121,226, maxColorValue = 255),
  # rgb(144,203,180, maxColorValue = 255), 
  rgb(255,255,255, maxColorValue = 255), 
  # rgb(251,192,52, maxColorValue = 255), 
  rgb(226,46,45, maxColorValue = 255)))(n = 201)
 
p &lt;- pheatmap(mat = big_matrix,
         cluster_cols = F,
         cluster_rows = F,
         clustering_distance_rows = &quot;euclidean&quot;,
         treeheight_row = 30,
         treeheight_col = 20,
         cutree_rows = 4,
         cutree_cols = 3,
         na_col = &quot;white&quot;,
         main = paste0(&quot;Heatmap of &quot;, kegg_pathway_name,&quot;\n&quot;),
         angle_col =90,
         
         # scale = &quot;row&quot;,
         legend_breaks = c(seq(-8, 8, by = 1), max(big_matrix, na.rm = T)),
         legend_labels = c(seq(-8, 8, by = 1), &quot;logCPM\n&quot;),
         # border_color = F,
         border_color = &quot;gray90&quot;,
         color = my_palette,
         )</code></pre>
<p><img src="figure/kegg.Rmd/partialPathway,%20-1.png" width="768" style="display: block; margin: auto;" /></p>
<pre class="r"><code>invisible(dev.print(device = svg, here::here(&quot;2_plots/4_kegg/heatmap_partial.svg&quot;)))</code></pre>
<pre class="r"><code>entrezID=list()
matrix=list()
for (i in 1:ncol(contrast)){
  #at each iteration, let x = name of each contrast group
  x=comparison_group[i,]
  
  entrezID[[x]] &lt;- enrichKEGG_treat_sig[[x]][,c(&quot;ID&quot;, &quot;geneID&quot;)] %&gt;% as.data.frame()
  entrezID[[x]] &lt;- entrezID[[x]][kegg_id,&quot;geneID&quot;] %&gt;% as.data.frame()
  entrezID[[x]] &lt;- separate_rows(entrezID[[x]],.,sep = &quot;/&quot;)
  colnames(entrezID[[x]]) &lt;- &quot;entrezid&quot;
  
  match &lt;- treat_sig[[x]]$entrezid %in% entrezID[[x]]$entrezid
  matrix[[x]] &lt;- treat_sig[[x]][match, c(&quot;gene_name&quot;, &quot;logFC&quot;)]
}

matrix_merged &lt;- 
  matrix$`UT vs CKI` %&gt;% 
  left_join(matrix$`CKI-Mac vs CKI`, by = &quot;gene_name&quot;) %&gt;% 
  left_join(matrix$`CKI-Nme vs CKI`, by = &quot;gene_name&quot;) %&gt;% 
  left_join(matrix$`CKI-Omt vs CKI`, by = &quot;gene_name&quot;) %&gt;% 
  left_join(matrix$`CKI-Tri vs CKI`, by = &quot;gene_name&quot;) %&gt;% 
  column_to_rownames(&quot;gene_name&quot;)

colnames(matrix_merged) &lt;- c(&quot;UT\nvs\nCKI&quot;, &quot;CKI-Mac\nvs\nCKI&quot;, &quot;CKI-Nme\nvs\nCKI&quot;, &quot;CKI-Omt\nvs\nCKI&quot;, &quot;CKI-Tri\nvs\nCKI&quot;)
my_palette &lt;- colorRampPalette(c(
  rgb(32,121,226, maxColorValue = 255),
  # rgb(144,203,180, maxColorValue = 255), 
  rgb(254,248,239, maxColorValue = 255), 
  # rgb(251,192,52, maxColorValue = 255), 
  rgb(226,46,45, maxColorValue = 255)))(n = 201)
pheatmap(mat = matrix_merged,
         clustering_distance_rows = &quot;euclidean&quot;,
         treeheight_row = 70,
         legend_labels = &quot;logFC&quot;,
         main = paste0(&quot;Heatmap of &quot;, kegg_pathway_name),
         # fontsize = 14,
         # fontsize_row = 12,
         # fontsize_number = 10,
         # fontsize_col = 18,
         angle_col =0,
         border_color = &quot;gray70&quot;,
         color = my_palette)
invisible(dev.print(svg, here::here(&quot;2_plots/4_kegg/heatmap_partial.svg&quot;)))</code></pre>
</div>
<div id="complete-pathway" class="section level3">
<h3>Complete Pathway</h3>
<p>This heatmap illustrates the logfc of all the genes within the Cell cycle - Homo sapiens (human), regardless of their statistical significance. Th</p>
<pre class="r"><code>entrezID=list()
matrix_all=list()
for (i in 1:ncol(contrast)){
  #at each iteration, let x = name of each contrast group
  x=comparison_group[i,]
  
  entrezID[[x]] &lt;- kegg_pathway[[1]]$GENE %&gt;% as.data.frame()
  entrezID[[x]] &lt;- entrezID[[x]][seq(1, nrow(entrezID[[x]]), 2),] %&gt;% as.data.frame()
  colnames(entrezID[[x]]) &lt;- &quot;entrezid&quot;
  
  match &lt;- treat_all[[x]]$entrezid %in% entrezID[[x]]$entrezid
  matrix_all[[x]] &lt;- treat_all[[x]][match, c(&quot;gene_name&quot;, &quot;logFC&quot;)]
}
matrix_all_merged &lt;- matrix_all$`UT vs CKI` %&gt;% 
  left_join(matrix_all$`CKI-Mac vs CKI`, by = &quot;gene_name&quot;) %&gt;% 
  left_join(matrix_all$`CKI-Nme vs CKI`, by = &quot;gene_name&quot;) %&gt;% 
  left_join(matrix_all$`CKI-Omt vs CKI`, by = &quot;gene_name&quot;) %&gt;% 
  left_join(matrix_all$`CKI-Tri vs CKI`, by = &quot;gene_name&quot;) %&gt;% 
  column_to_rownames(&quot;gene_name&quot;)
colnames(matrix_all_merged) &lt;- c(&quot;UT vs CKI&quot;, &quot;CKI-Mac vs CKI&quot;, &quot;CKI-Nme vs CKI&quot;, &quot;CKI-Omt vs CKI&quot;, &quot;CKI-Tri vs CKI&quot;)

summary(matrix_all_merged)</code></pre>
<pre><code>   UT vs CKI       CKI-Mac vs CKI    CKI-Nme vs CKI    CKI-Omt vs CKI   
 Min.   :-1.1178   Min.   :-2.7624   Min.   :-3.7203   Min.   :-4.7338  
 1st Qu.: 0.0725   1st Qu.:-0.8730   1st Qu.:-0.7379   1st Qu.:-1.2907  
 Median : 0.2723   Median :-0.1407   Median :-0.1544   Median :-0.2154  
 Mean   : 0.3832   Mean   :-0.2785   Mean   :-0.2538   Mean   :-0.5285  
 3rd Qu.: 0.7418   3rd Qu.: 0.2129   3rd Qu.: 0.2733   3rd Qu.: 0.3363  
 Max.   : 2.5192   Max.   : 2.1897   Max.   : 1.9727   Max.   : 3.2729  
 CKI-Tri vs CKI   
 Min.   :-3.8990  
 1st Qu.:-0.9164  
 Median :-0.1483  
 Mean   :-0.3246  
 3rd Qu.: 0.2606  
 Max.   : 2.3024  </code></pre>
<pre class="r"><code>p &lt;- pheatmap(mat = matrix_all_merged,
         cluster_cols = T,
         clustering_distance_rows = &quot;euclidean&quot;,
         treeheight_row = 60,
         treeheight_col = 20,
         cutree_rows = 4,
         cutree_cols = 3,
         
         main = paste0(&quot;Heatmap of &quot;, kegg_pathway_name),
         
         scale = &quot;row&quot;,
         legend = T,
         legend_breaks = c(seq(-3, 3, by = .5), 1.7),
         legend_labels = c(seq(-3, 3, by = .5), &quot;Z-Score\n\n&quot;),
         # border_color = F,
         border_color = &quot;gray70&quot;,
         color = my_palette,
         )</code></pre>
<p><img src="figure/kegg.Rmd/completePathway-1.png" width="768" style="display: block; margin: auto;" /></p>
<pre class="r"><code># ggsave(filename = &quot;heatmap_all.svg&quot;,plot = last_plot(), path = here::here(&quot;2_plots/4_kegg/&quot;))
invisible(dev.print(width = 8.8, height = 22, device = svg, here::here(&quot;2_plots/4_kegg/heatmap_complete.svg&quot;)))</code></pre>
</div>
</div>
</div>
<div id="export-data" class="section level1">
<h1>Export Data</h1>
<pre class="r"><code>writexl::write_xlsx(x = enrichKEGG_treat, path = here::here(&quot;3_output/enrichKEGG_treat_sig.xlsx&quot;))
writexl::write_xlsx(x = kegga_treat, path = here::here(&quot;3_output/kegga_treat_sig.xlsx&quot;))</code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.1.0 (2021-05-18)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19043)

Matrix products: default

locale:
[1] LC_COLLATE=English_Australia.1252  LC_CTYPE=English_Australia.1252   
[3] LC_MONETARY=English_Australia.1252 LC_NUMERIC=C                      
[5] LC_TIME=English_Australia.1252    

attached base packages:
 [1] parallel  stats4    grid      stats     graphics  grDevices utils    
 [8] datasets  methods   base     

other attached packages:
 [1] enrichplot_1.12.3     KEGGREST_1.32.0       biomaRt_2.48.3       
 [4] org.Hs.eg.db_3.13.0   AnnotationDbi_1.54.1  IRanges_2.26.0       
 [7] S4Vectors_0.30.0      Biobase_2.52.0        BiocGenerics_0.38.0  
[10] clusterProfiler_4.0.5 Glimma_2.2.0          edgeR_3.34.0         
[13] limma_3.48.1          ggrepel_0.9.1         ggbiplot_0.55        
[16] scales_1.1.1          plyr_1.8.6            ggpubr_0.4.0         
[19] RColorBrewer_1.1-2    pheatmap_1.0.12       cowplot_1.1.1        
[22] pander_0.6.4          kableExtra_1.3.4      forcats_0.5.1        
[25] stringr_1.4.0         purrr_0.3.4           tidyr_1.1.3          
[28] ggplot2_3.3.5         tidyverse_1.3.1       reshape2_1.4.4       
[31] tibble_3.1.2          readr_2.0.0           magrittr_2.0.1       
[34] dplyr_1.0.7          

loaded via a namespace (and not attached):
  [1] utf8_1.2.1                  tidyselect_1.1.1           
  [3] RSQLite_2.2.7               htmlwidgets_1.5.4          
  [5] BiocParallel_1.26.1         scatterpie_0.1.7           
  [7] munsell_0.5.0               statmod_1.4.36             
  [9] withr_2.4.2                 colorspace_2.0-2           
 [11] GOSemSim_2.18.1             filelock_1.0.2             
 [13] highr_0.9                   knitr_1.36                 
 [15] rstudioapi_0.13             ggsignif_0.6.3             
 [17] DOSE_3.18.3                 MatrixGenerics_1.4.3       
 [19] labeling_0.4.2              git2r_0.28.0               
 [21] GenomeInfoDbData_1.2.6      polyclip_1.10-0            
 [23] bit64_4.0.5                 farver_2.1.0               
 [25] rprojroot_2.0.2             downloader_0.4             
 [27] treeio_1.16.2               vctrs_0.3.8                
 [29] generics_0.1.0              xfun_0.24                  
 [31] BiocFileCache_2.0.0         R6_2.5.1                   
 [33] GenomeInfoDb_1.28.4         graphlayouts_0.7.1         
 [35] locfit_1.5-9.4              gridGraphics_0.5-1         
 [37] bitops_1.0-7                cachem_1.0.5               
 [39] fgsea_1.18.0                DelayedArray_0.18.0        
 [41] assertthat_0.2.1            promises_1.2.0.1           
 [43] ggraph_2.0.5                gtable_0.3.0               
 [45] tidygraph_1.2.0             workflowr_1.6.2            
 [47] rlang_0.4.11                genefilter_1.74.0          
 [49] systemfonts_1.0.2           splines_4.1.0              
 [51] lazyeval_0.2.2              rstatix_0.7.0              
 [53] broom_0.7.9                 yaml_2.2.1                 
 [55] abind_1.4-5                 modelr_0.1.8               
 [57] backports_1.2.1             httpuv_1.6.1               
 [59] qvalue_2.24.0               tools_4.1.0                
 [61] ggplotify_0.1.0             ellipsis_0.3.2             
 [63] jquerylib_0.1.4             Rcpp_1.0.6                 
 [65] progress_1.2.2              zlibbioc_1.38.0            
 [67] BiasedUrn_1.07              RCurl_1.98-1.3             
 [69] prettyunits_1.1.1           viridis_0.6.2              
 [71] SummarizedExperiment_1.22.0 haven_2.4.3                
 [73] fs_1.5.0                    here_1.0.1                 
 [75] data.table_1.14.0           DO.db_2.9                  
 [77] openxlsx_4.2.4              reprex_2.0.1               
 [79] whisker_0.4                 matrixStats_0.60.1         
 [81] patchwork_1.1.1             hms_1.1.1                  
 [83] evaluate_0.14               xtable_1.8-4               
 [85] XML_3.99-0.6                rio_0.5.27                 
 [87] readxl_1.3.1                gridExtra_2.3              
 [89] compiler_4.1.0              shadowtext_0.0.9           
 [91] writexl_1.4.0               crayon_1.4.1               
 [93] htmltools_0.5.2             ggfun_0.0.4                
 [95] later_1.2.0                 tzdb_0.1.2                 
 [97] aplot_0.1.1                 geneplotter_1.70.0         
 [99] lubridate_1.7.10            DBI_1.1.1                  
[101] tweenr_1.0.2                dbplyr_2.1.1               
[103] rappdirs_0.3.3              MASS_7.3-54                
[105] Matrix_1.3-3                car_3.0-11                 
[107] cli_3.0.1                   igraph_1.2.6               
[109] GenomicRanges_1.44.0        pkgconfig_2.0.3            
[111] foreign_0.8-81              xml2_1.3.2                 
[113] ggtree_3.0.4                svglite_2.0.0              
[115] annotate_1.70.0             bslib_0.3.1                
[117] webshot_0.5.2               XVector_0.32.0             
[119] rvest_1.0.2                 yulab.utils_0.0.4          
[121] digest_0.6.27               Biostrings_2.60.1          
[123] rmarkdown_2.11              cellranger_1.1.0           
[125] fastmatch_1.1-3             tidytree_0.3.5             
[127] curl_4.3.2                  nlme_3.1-152               
[129] lifecycle_1.0.1             jsonlite_1.7.2             
[131] carData_3.0-4               viridisLite_0.4.0          
[133] fansi_0.5.0                 pillar_1.6.4               
[135] lattice_0.20-44             fastmap_1.1.0              
[137] httr_1.4.2                  survival_3.2-11            
[139] GO.db_3.13.0                glue_1.4.2                 
[141] zip_2.2.0                   png_0.1-7                  
[143] bit_4.0.4                   ggforce_0.3.3              
[145] stringi_1.6.2               sass_0.4.0                 
[147] blob_1.2.2                  DESeq2_1.32.0              
[149] memoise_2.0.0               ape_5.5                    </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
