<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Ha Tran" />


<title>KEGG Analysis</title>

<script src="site_libs/header-attrs-2.11/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/lightable-0.0.1/lightable.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">CKI_DGE</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Setting Up
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="obtainData.html">Obtaining Data</a>
    </li>
    <li>
      <a href="setUp.html">QC and Pre-processing</a>
    </li>
  </ul>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Analysis
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="dge.html">DE Analysis</a>
    </li>
    <li>
      <a href="go.html">GO Term Enrichment Analysis</a>
    </li>
    <li>
      <a href="kegg.html">KEGG Pathway Enrichment Analysis</a>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">KEGG Analysis</h1>
<h4 class="author">Ha Tran</h4>
<h4 class="date">19/10/2021</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2021-11-08
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 5 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 2
</p>
<p>
<strong>Knit directory:</strong> <code>01_CKI_DGE/1_analysis/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges" class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown file has unstaged changes. To know which version of the R Markdown file created these results, you’ll want to first commit it to the Git repo. If you’re still working on the analysis, you can ignore this warning. When you’re finished, you can run <code>wflow_publish</code> to commit the R Markdown file and build the HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongobjectspresent"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>Environment:</strong> objects present </a>
</p>
</div>
<div id="strongEnvironmentstrongobjectspresent" class="panel-collapse collapse">
<div class="panel-body">
<p>
The global environment had objects present when the code in the R Markdown file was run. These objects can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment. Use <code>wflow_publish</code> or <code>wflow_build</code> to ensure that the code is always run in an empty environment.
</p>
<p>
The following objects were defined in the global environment when these results were created:
</p>
<table class="table table-condensed table-hover">
<thead>
<tr>
<th style="text-align:left;">
Name
</th>
<th style="text-align:left;">
Class
</th>
<th style="text-align:left;">
Size
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
anno_colours
</td>
<td style="text-align:left;">
list
</td>
<td style="text-align:left;">
912 bytes
</td>
</tr>
<tr>
<td style="text-align:left;">
big_df
</td>
<td style="text-align:left;">
list
</td>
<td style="text-align:left;">
43.9 Kb
</td>
</tr>
<tr>
<td style="text-align:left;">
comparison_group
</td>
<td style="text-align:left;">
data.frame
</td>
<td style="text-align:left;">
1.1 Kb
</td>
</tr>
<tr>
<td style="text-align:left;">
contrast
</td>
<td style="text-align:left;">
matrix;array
</td>
<td style="text-align:left;">
1.7 Kb
</td>
</tr>
<tr>
<td style="text-align:left;">
designMatrix
</td>
<td style="text-align:left;">
matrix;array
</td>
<td style="text-align:left;">
3.5 Kb
</td>
</tr>
<tr>
<td style="text-align:left;">
df
</td>
<td style="text-align:left;">
data.frame
</td>
<td style="text-align:left;">
9.6 Kb
</td>
</tr>
<tr>
<td style="text-align:left;">
dge
</td>
<td style="text-align:left;">
DGEList
</td>
<td style="text-align:left;">
12 Mb
</td>
</tr>
<tr>
<td style="text-align:left;">
enrichGO_treat
</td>
<td style="text-align:left;">
list
</td>
<td style="text-align:left;">
716.4 Kb
</td>
</tr>
<tr>
<td style="text-align:left;">
estimateDispersion
</td>
<td style="text-align:left;">
DGEList
</td>
<td style="text-align:left;">
12.5 Mb
</td>
</tr>
<tr>
<td style="text-align:left;">
fit_GLM_qlf
</td>
<td style="text-align:left;">
DGEGLM
</td>
<td style="text-align:left;">
20.7 Mb
</td>
</tr>
<tr>
<td style="text-align:left;">
GOresults
</td>
<td style="text-align:left;">
enrichResult
</td>
<td style="text-align:left;">
123.9 Mb
</td>
</tr>
<tr>
<td style="text-align:left;">
goSummaries
</td>
<td style="text-align:left;">
tbl_df;tbl;data.frame
</td>
<td style="text-align:left;">
13.4 Mb
</td>
</tr>
<tr>
<td style="text-align:left;">
group.labs
</td>
<td style="text-align:left;">
factor
</td>
<td style="text-align:left;">
944 bytes
</td>
</tr>
<tr>
<td style="text-align:left;">
i
</td>
<td style="text-align:left;">
integer
</td>
<td style="text-align:left;">
56 bytes
</td>
</tr>
<tr>
<td style="text-align:left;">
logCPM
</td>
<td style="text-align:left;">
matrix;array
</td>
<td style="text-align:left;">
7.8 Kb
</td>
</tr>
<tr>
<td style="text-align:left;">
minPath
</td>
<td style="text-align:left;">
numeric
</td>
<td style="text-align:left;">
56 bytes
</td>
</tr>
<tr>
<td style="text-align:left;">
my_palette
</td>
<td style="text-align:left;">
character
</td>
<td style="text-align:left;">
12.6 Kb
</td>
</tr>
<tr>
<td style="text-align:left;">
qlf
</td>
<td style="text-align:left;">
list
</td>
<td style="text-align:left;">
101.1 Mb
</td>
</tr>
<tr>
<td style="text-align:left;">
qlf_decideTest
</td>
<td style="text-align:left;">
list
</td>
<td style="text-align:left;">
5.6 Kb
</td>
</tr>
<tr>
<td style="text-align:left;">
qlf_filtered
</td>
<td style="text-align:left;">
list
</td>
<td style="text-align:left;">
37.5 Mb
</td>
</tr>
<tr>
<td style="text-align:left;">
qlf_unfiltered
</td>
<td style="text-align:left;">
list
</td>
<td style="text-align:left;">
49.2 Mb
</td>
</tr>
<tr>
<td style="text-align:left;">
sample_type
</td>
<td style="text-align:left;">
character
</td>
<td style="text-align:left;">
624 bytes
</td>
</tr>
<tr>
<td style="text-align:left;">
test
</td>
<td style="text-align:left;">
data.frame
</td>
<td style="text-align:left;">
2.3 Kb
</td>
</tr>
<tr>
<td style="text-align:left;">
test3
</td>
<td style="text-align:left;">
data.frame
</td>
<td style="text-align:left;">
310 Kb
</td>
</tr>
<tr>
<td style="text-align:left;">
treat
</td>
<td style="text-align:left;">
list
</td>
<td style="text-align:left;">
96.2 Mb
</td>
</tr>
<tr>
<td style="text-align:left;">
treat_decideTest
</td>
<td style="text-align:left;">
list
</td>
<td style="text-align:left;">
5.6 Kb
</td>
</tr>
<tr>
<td style="text-align:left;">
treat_filtered
</td>
<td style="text-align:left;">
list
</td>
<td style="text-align:left;">
1.7 Mb
</td>
</tr>
<tr>
<td style="text-align:left;">
treat_unfiltered
</td>
<td style="text-align:left;">
list
</td>
<td style="text-align:left;">
49.2 Mb
</td>
</tr>
<tr>
<td style="text-align:left;">
x
</td>
<td style="text-align:left;">
character
</td>
<td style="text-align:left;">
120 bytes
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed12345code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(12345)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed12345code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(12345)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomtranmanhha13501CKIDGEtree1a47b4569a1da246294ec62c8e47e2f6e2638e10targetblank1a47b45a"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/tranmanhha135/01_CKI_DGE/tree/1a47b4569a1da246294ec62c8e47e2f6e2638e10" target="_blank">1a47b45</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomtranmanhha13501CKIDGEtree1a47b4569a1da246294ec62c8e47e2f6e2638e10targetblank1a47b45a" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/tranmanhha135/01_CKI_DGE/tree/1a47b4569a1da246294ec62c8e47e2f6e2638e10" target="_blank">1a47b45</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/

Untracked files:
    Untracked:  1_analysis/about.html
    Untracked:  1_analysis/dge.html
    Untracked:  1_analysis/figure/dge.Rmd/coolmap-1.png
    Untracked:  1_analysis/figure/dge.Rmd/volcano_treat-1.png
    Untracked:  1_analysis/figure/dge.Rmd/volcano_treat-2.png
    Untracked:  1_analysis/figure/dge.Rmd/volcano_treat-3.png
    Untracked:  1_analysis/figure/dge.Rmd/volcano_treat-4.png
    Untracked:  1_analysis/figure/dge.Rmd/volcano_treat-5.png
    Untracked:  1_analysis/figure/go.Rmd/
    Untracked:  1_analysis/figure/kegg.Rmd/completePathway-1.png
    Untracked:  1_analysis/figure/kegg.Rmd/partialPathway-1.png
    Untracked:  1_analysis/figure/setUp.Rmd/afterFiltering-2.png
    Untracked:  1_analysis/go.html
    Untracked:  1_analysis/index.html
    Untracked:  1_analysis/site_libs/
    Untracked:  2_plots/1_setup/counts_after_filtering.svg
    Untracked:  2_plots/1_setup/counts_before_filtering.svg
    Untracked:  2_plots/2_dge/logCPM_heatmap.svg
    Untracked:  2_plots/3_go/
    Untracked:  2_plots/4_kegg/

Unstaged changes:
    Modified:   0_data/rds_object/dge.rds
    Modified:   0_data/rds_object/enrichGO_treat.rds
    Modified:   0_data/rds_object/enrichKEGG_treat_filtered.rds
    Modified:   0_data/rds_object/qlf.rds
    Modified:   0_data/rds_object/qlf_filtered.rds
    Modified:   0_data/rds_object/treat.rds
    Modified:   0_data/rds_object/treat_filtered.rds
    Modified:   0_data/rds_object/treat_unfiltered.rds
    Modified:   1_analysis/about.Rmd
    Modified:   1_analysis/dge.Rmd
    Modified:   1_analysis/figure/dge.Rmd/MAplotCustom_treat-1.png
    Modified:   1_analysis/figure/dge.Rmd/MAplotCustom_treat-2.png
    Modified:   1_analysis/figure/dge.Rmd/MAplotCustom_treat-3.png
    Modified:   1_analysis/figure/dge.Rmd/MAplotCustom_treat-4.png
    Modified:   1_analysis/figure/dge.Rmd/MAplotCustom_treat-5.png
    Modified:   1_analysis/figure/dge.Rmd/estimateDispersion-1.png
    Modified:   1_analysis/figure/dge.Rmd/fitGLMqlf-1.png
    Modified:   1_analysis/figure/dge.Rmd/pValueHistogram_treat-1.png
    Modified:   1_analysis/figure/dge.Rmd/pValueHistogram_treat-2.png
    Modified:   1_analysis/figure/dge.Rmd/pValueHistogram_treat-3.png
    Modified:   1_analysis/figure/dge.Rmd/pValueHistogram_treat-4.png
    Modified:   1_analysis/figure/dge.Rmd/pValueHistogram_treat-5.png
    Modified:   1_analysis/figure/kegg.Rmd/unnamed-chunk-6-1.png
    Modified:   1_analysis/figure/setUp.Rmd/MDSplot-1.png
    Modified:   1_analysis/figure/setUp.Rmd/afterFiltering-1.png
    Modified:   1_analysis/figure/setUp.Rmd/libraryChecks-1.png
    Modified:   1_analysis/figure/setUp.Rmd/pcaSampleGroup-1.png
    Modified:   1_analysis/figure/setUp.Rmd/pcaSampleType-1.png
    Modified:   1_analysis/figure/setUp.Rmd/visualisingNormalisation-1.png
    Modified:   1_analysis/go.Rmd
    Modified:   1_analysis/kegg.Rmd
    Modified:   1_analysis/obtainData.Rmd
    Modified:   1_analysis/setUp.Rmd
    Modified:   2_plots/1_setup/counts_before_and_after_filtering.svg
    Modified:   2_plots/1_setup/library_size.svg
    Modified:   2_plots/1_setup/sample_group_PCA_plot.svg
    Modified:   2_plots/1_setup/sample_type_PCA_plot.svg
    Modified:   2_plots/2_dge/MA_plot_treat_CKI-Mac vs CKI.svg
    Modified:   2_plots/2_dge/MA_plot_treat_CKI-Nme vs CKI.svg
    Modified:   2_plots/2_dge/MA_plot_treat_CKI-Omt vs CKI.svg
    Modified:   2_plots/2_dge/MA_plot_treat_CKI-Tri vs CKI.svg
    Modified:   2_plots/2_dge/MA_plot_treat_UT vs CKI.svg
    Modified:   2_plots/2_dge/estimate_dispersion.svg
    Modified:   2_plots/2_dge/qlf_dispersion.svg
    Modified:   2_plots/2_dge/qlf_pValue_histogram_CKI-Mac vs CKI.svg
    Modified:   2_plots/2_dge/qlf_pValue_histogram_CKI-Nme vs CKI.svg
    Modified:   2_plots/2_dge/qlf_pValue_histogram_CKI-Omt vs CKI.svg
    Modified:   2_plots/2_dge/qlf_pValue_histogram_CKI-Tri vs CKI.svg
    Modified:   2_plots/2_dge/qlf_pValue_histogram_UT vs CKI.svg
    Modified:   2_plots/2_dge/treat_pValue_histogram_CKI-Mac vs CKI.svg
    Modified:   2_plots/2_dge/treat_pValue_histogram_CKI-Nme vs CKI.svg
    Modified:   2_plots/2_dge/treat_pValue_histogram_CKI-Omt vs CKI.svg
    Modified:   2_plots/2_dge/treat_pValue_histogram_CKI-Tri vs CKI.svg
    Modified:   2_plots/2_dge/treat_pValue_histogram_UT vs CKI.svg
    Modified:   2_plots/2_dge/volcano_plot_treat_CKI-Mac vs CKI.svg
    Modified:   2_plots/2_dge/volcano_plot_treat_CKI-Nme vs CKI.svg
    Modified:   2_plots/2_dge/volcano_plot_treat_CKI-Omt vs CKI.svg
    Modified:   2_plots/2_dge/volcano_plot_treat_CKI-Tri vs CKI.svg
    Modified:   2_plots/2_dge/volcano_plot_treat_UT vs CKI.svg
    Modified:   3_output/enrichGO_treat.xlsx
    Modified:   3_output/enrichKEGG_treat_unfiltered.xlsx
    Modified:   3_output/qlf_filtered.xlsx
    Modified:   3_output/treat_filtered.xlsx

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>1_analysis/kegg.Rmd</code>) and HTML (<code>docs/kegg.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/01_CKI_DGE/blob/1a47b4569a1da246294ec62c8e47e2f6e2638e10/1_analysis/kegg.Rmd" target="_blank">1a47b45</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-31
</td>
<td>
changed heatmap size
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/01_CKI_DGE/1a47b4569a1da246294ec62c8e47e2f6e2638e10/docs/kegg.html" target="_blank">1a47b45</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-31
</td>
<td>
changed heatmap size
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/01_CKI_DGE/blob/70049b22b9a79841a2368454293f8825f08980dd/1_analysis/kegg.Rmd" target="_blank">70049b2</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-31
</td>
<td>
Remove old GO and KEGG code, adjusted directory paths, updated KEGG, minor cosmetic changes
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/01_CKI_DGE/70049b22b9a79841a2368454293f8825f08980dd/docs/kegg.html" target="_blank">70049b2</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-31
</td>
<td>
Remove old GO and KEGG code, adjusted directory paths, updated KEGG, minor cosmetic changes
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/01_CKI_DGE/blob/6823b3c881eddd07176bce86153e0543293d7f02/1_analysis/kegg.Rmd" target="_blank">6823b3c</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-30
</td>
<td>
Add minor cosmetic changes
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/01_CKI_DGE/6823b3c881eddd07176bce86153e0543293d7f02/docs/kegg.html" target="_blank">6823b3c</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-30
</td>
<td>
Add minor cosmetic changes
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/01_CKI_DGE/blob/7a6571ba908cf21e984811a75b8123b1e28a4d69/1_analysis/kegg.Rmd" target="_blank">7a6571b</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-30
</td>
<td>
minor changes
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/01_CKI_DGE/7a6571ba908cf21e984811a75b8123b1e28a4d69/docs/kegg.html" target="_blank">7a6571b</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-30
</td>
<td>
minor changes
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/01_CKI_DGE/f5f4f74bfd77b68627ec8414b040c66a60ed7344/docs/kegg.html" target="_blank">f5f4f74</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-30
</td>
<td>
please please please
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/01_CKI_DGE/blob/a6d8f8a05c71e8da432f180ed252874e35ca0ae1/1_analysis/kegg.Rmd" target="_blank">a6d8f8a</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-30
</td>
<td>
what is happening
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/01_CKI_DGE/a6d8f8a05c71e8da432f180ed252874e35ca0ae1/docs/kegg.html" target="_blank">a6d8f8a</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-30
</td>
<td>
what is happening
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/01_CKI_DGE/blob/d769a7cbf44fe4841bdf220ed214b5f73c00b47a/1_analysis/kegg.Rmd" target="_blank">d769a7c</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-30
</td>
<td>
Upload GO and KEGG analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/01_CKI_DGE/d769a7cbf44fe4841bdf220ed214b5f73c00b47a/docs/kegg.html" target="_blank">d769a7c</a>
</td>
<td>
Ha Tran
</td>
<td>
2021-10-30
</td>
<td>
Upload GO and KEGG analysis
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="data-setup" class="section level1">
<h1>Data Setup</h1>
<div id="load-library" class="section level3">
<h3>Load Library</h3>
<pre class="r"><code># working with data
library(dplyr)
library(magrittr)
library(readr)
library(tibble)
library(reshape2)
library(tidyverse)
library(KEGGREST)

# Visualisation:
library(kableExtra)
library(ggplot2)
library(grid)
library(pander)
library(cowplot)
library(pheatmap)

# Custom ggplot 
library(ggbiplot)
library(ggrepel)

# Bioconductor packages:
library(edgeR)
library(limma)
library(Glimma)
library(clusterProfiler)
library(org.Hs.eg.db)
library(enrichplot)
library(biomaRt)

# Set ggplot theme
theme_set(theme_minimal())

theme_update(
  legend.background = element_rect(fill = &quot;transparent&quot;, colour = NA),
  legend.box.background = element_rect(fill = &quot;transparent&quot;, colour = NA),
  
  panel.background = element_rect(fill = &quot;transparent&quot;,colour = NA),
  plot.background = element_rect(fill = &quot;transparent&quot;, colour = NA),
  
  plot.title = element_text(color = &quot;gray20&quot;, size = 24, angle = 0, hjust = 0, vjust = .5, face = &quot;bold&quot;),
  plot.subtitle = element_text(color = &quot;gray25&quot;, size = 20, angle = 0, hjust = 0, vjust = .5, face = &quot;plain&quot;),
  legend.title = element_text(color = &quot;gray25&quot;, size = 16, angle = 0, hjust = 0, vjust = .5, face = &quot;plain&quot;),
  legend.text = element_text(color = &quot;gray25&quot;, size = 12, angle = 0, hjust = 0, vjust = .5, face = &quot;plain&quot;),
  axis.text.x = element_text(color = &quot;gray30&quot;, size = 12, angle = 0, hjust = .5, vjust = .5, face = &quot;plain&quot;),
  axis.text.y = element_text(color = &quot;gray30&quot;, size = 12, angle = 0, hjust = 1, vjust = 0, face = &quot;plain&quot;),  
  axis.title.x = element_text(color = &quot;gray30&quot;, size = 14, angle = 0, hjust = .5, vjust = 0, face = &quot;plain&quot;),
  axis.title.y = element_text(color = &quot;gray30&quot;, size = 14, angle = 90, hjust = .5, vjust = .5, face = &quot;plain&quot;))</code></pre>
</div>
<div id="import-rds-data" class="section level3">
<h3>Import RDS Data</h3>
<p>DGElist object containing the raw feature count, sample metadata, and gene metadata, created in the Set Up stage.</p>
<pre class="r"><code># load DGElist previously created in the set up
dge &lt;- readRDS(here::here(&quot;0_data/rds_object/dge.rds&quot;))
designMatrix &lt;- readRDS(here::here(&quot;0_data/rds_object/designMatrix.rds&quot;))
contrast &lt;- readRDS(here::here(&quot;0_data/rds_object/contrastMatrix.rds&quot;))

treat &lt;- readRDS(here::here(&quot;0_data/rds_object/treat.rds&quot;))
treat_unfiltered &lt;- readRDS(here::here(&quot;0_data/rds_object/treat_unfiltered.rds&quot;))
treat_filtered &lt;- readRDS(here::here(&quot;0_data/rds_object/treat_filtered.rds&quot;))</code></pre>
</div>
</div>
<div id="kegg-enrichment-analysis" class="section level1">
<h1>KEGG enrichment analysis</h1>
<div id="using-the-clusterprofilerenrichkegg-package" class="section level3 tabset">
<h3 class="tabset">Using the <code>clusterProfiler::enrichKEGG</code> package</h3>
<p>KEGG pathway enrichment analysis is performed with GLM:QLF + TREAT.</p>
<p>The top 30 most significant KEGG terms are displayed in relation to their respective comparison. KEGG terms with <code>qvalue &lt;= 0.2</code> are considered as significant</p>
<pre class="r"><code>comparison_group=colnames(contrast) %&gt;% as.data.frame()

# Create list object
enrichKEGG_treat=list()
enrichKEGG_treat_sig=list()
for (i in 1:ncol(contrast)){
  #at each iteration, let x = name of each contrast group
  x=comparison_group[i,]
  
  #populate the enrichGo list with all KEGG terms
  KEGGresults &lt;- clusterProfiler::enrichKEGG(gene = treat_filtered[[x]]$entrezid,
                        organism = &quot;hsa&quot;,
                        pvalueCutoff = 0.05,
                        pAdjustMethod = &quot;fdr&quot;) 
  
  enrichKEGG_treat[[x]] &lt;- KEGGresults@result %&gt;% as.data.frame()
 
  #filter only significant KEGG terms
  enrichKEGG_treat_sig[[x]] &lt;- enrichKEGG_treat[[x]] %&gt;% dplyr::filter(qvalue &lt;= 0.2)
} 
# save rds object for use in downstream GO and KEGG analysis
# enrichKEGG_treat %&gt;% saveRDS(file = here::here(&quot;0_data/rds_object/enrichKEGG_treat_unfiltered.rds&quot;))
enrichKEGG_treat_sig %&gt;% saveRDS(file = here::here(&quot;0_data/rds_object/enrichKEGG_treat_filtered.rds&quot;))

#save each unfiltered comparison group in the output directory
writexl::write_xlsx(x = enrichKEGG_treat,
                    path = here::here(&quot;3_output/enrichKEGG_treat_unfiltered.xlsx&quot;))</code></pre>
<div id="ut-vs-cki" class="section level4">
<h4>UT vs CKI</h4>
<pre class="r"><code>enrichKEGG_treat_sig[[&quot;UT vs CKI&quot;]][1:12,] %&gt;% as.data.frame() %&gt;% dplyr::select(-c(&quot;ID&quot;)) %&gt;% kable() %&gt;% kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;% scroll_box(height=&quot;600px&quot;)</code></pre>
<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Description
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
GeneRatio
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
BgRatio
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
pvalue
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
p.adjust
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
qvalue
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
geneID
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
hsa00140
</td>
<td style="text-align:left;">
Steroid hormone biosynthesis
</td>
<td style="text-align:left;">
6/95
</td>
<td style="text-align:left;">
61/8105
</td>
<td style="text-align:right;">
0.0000731
</td>
<td style="text-align:right;">
0.0150522
</td>
<td style="text-align:right;">
0.0134600
</td>
<td style="text-align:left;">
1543/1545/8644/1645/3290/1646
</td>
<td style="text-align:right;">
6
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05323
</td>
<td style="text-align:left;">
Rheumatoid arthritis
</td>
<td style="text-align:left;">
6/95
</td>
<td style="text-align:left;">
93/8105
</td>
<td style="text-align:right;">
0.0007427
</td>
<td style="text-align:right;">
0.0354886
</td>
<td style="text-align:right;">
0.0317348
</td>
<td style="text-align:left;">
7042/4312/2919/3569/3576/2920
</td>
<td style="text-align:right;">
6
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04657
</td>
<td style="text-align:left;">
IL-17 signaling pathway
</td>
<td style="text-align:left;">
6/95
</td>
<td style="text-align:left;">
94/8105
</td>
<td style="text-align:right;">
0.0007859
</td>
<td style="text-align:right;">
0.0354886
</td>
<td style="text-align:right;">
0.0317348
</td>
<td style="text-align:left;">
4312/2919/3569/3934/3576/2920
</td>
<td style="text-align:right;">
6
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05417
</td>
<td style="text-align:left;">
Lipid and atherosclerosis
</td>
<td style="text-align:left;">
9/95
</td>
<td style="text-align:left;">
215/8105
</td>
<td style="text-align:right;">
0.0008952
</td>
<td style="text-align:right;">
0.0354886
</td>
<td style="text-align:right;">
0.0317348
</td>
<td style="text-align:left;">
1543/19/4312/2919/3569/7157/9619/3576/2920
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04061
</td>
<td style="text-align:left;">
Viral protein interaction with cytokine and cytokine receptor
</td>
<td style="text-align:left;">
6/95
</td>
<td style="text-align:left;">
100/8105
</td>
<td style="text-align:right;">
0.0010877
</td>
<td style="text-align:right;">
0.0354886
</td>
<td style="text-align:right;">
0.0317348
</td>
<td style="text-align:left;">
2919/3569/3576/2920/7133/8794
</td>
<td style="text-align:right;">
6
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04933
</td>
<td style="text-align:left;">
AGE-RAGE signaling pathway in diabetic complications
</td>
<td style="text-align:left;">
6/95
</td>
<td style="text-align:left;">
100/8105
</td>
<td style="text-align:right;">
0.0010877
</td>
<td style="text-align:right;">
0.0354886
</td>
<td style="text-align:right;">
0.0317348
</td>
<td style="text-align:left;">
7042/3569/7056/2335/3576/1958
</td>
<td style="text-align:right;">
6
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05146
</td>
<td style="text-align:left;">
Amoebiasis
</td>
<td style="text-align:left;">
6/95
</td>
<td style="text-align:left;">
102/8105
</td>
<td style="text-align:right;">
0.0012059
</td>
<td style="text-align:right;">
0.0354886
</td>
<td style="text-align:right;">
0.0317348
</td>
<td style="text-align:left;">
7042/2919/3569/2335/3576/2920
</td>
<td style="text-align:right;">
6
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa00380
</td>
<td style="text-align:left;">
Tryptophan metabolism
</td>
<td style="text-align:left;">
4/95
</td>
<td style="text-align:left;">
42/8105
</td>
<td style="text-align:right;">
0.0014099
</td>
<td style="text-align:right;">
0.0363054
</td>
<td style="text-align:right;">
0.0324652
</td>
<td style="text-align:left;">
1543/1545/51166/224
</td>
<td style="text-align:right;">
4
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa00980
</td>
<td style="text-align:left;">
Metabolism of xenobiotics by cytochrome P450
</td>
<td style="text-align:left;">
5/95
</td>
<td style="text-align:left;">
78/8105
</td>
<td style="text-align:right;">
0.0021400
</td>
<td style="text-align:right;">
0.0489820
</td>
<td style="text-align:right;">
0.0438010
</td>
<td style="text-align:left;">
1543/1545/1645/3290/218
</td>
<td style="text-align:right;">
5
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05134
</td>
<td style="text-align:left;">
Legionellosis
</td>
<td style="text-align:left;">
4/95
</td>
<td style="text-align:left;">
57/8105
</td>
<td style="text-align:right;">
0.0043535
</td>
<td style="text-align:right;">
0.0896814
</td>
<td style="text-align:right;">
0.0801954
</td>
<td style="text-align:left;">
2919/3569/3576/2920
</td>
<td style="text-align:right;">
4
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05204
</td>
<td style="text-align:left;">
Chemical carcinogenesis - DNA adducts
</td>
<td style="text-align:left;">
4/95
</td>
<td style="text-align:left;">
69/8105
</td>
<td style="text-align:right;">
0.0085661
</td>
<td style="text-align:right;">
0.1604196
</td>
<td style="text-align:right;">
0.1434514
</td>
<td style="text-align:left;">
1543/1545/3290/1646
</td>
<td style="text-align:right;">
4
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05219
</td>
<td style="text-align:left;">
Bladder cancer
</td>
<td style="text-align:left;">
3/95
</td>
<td style="text-align:left;">
41/8105
</td>
<td style="text-align:right;">
0.0120540
</td>
<td style="text-align:right;">
0.2069263
</td>
<td style="text-align:right;">
0.1850388
</td>
<td style="text-align:left;">
4312/7157/3576
</td>
<td style="text-align:right;">
3
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="cki-mac-vs-cki" class="section level4">
<h4>CKI-Mac vs CKI</h4>
<pre class="r"><code>enrichKEGG_treat_sig[[&quot;CKI-Mac vs CKI&quot;]][1:30,] %&gt;% as.data.frame() %&gt;% dplyr::select(-c(&quot;ID&quot;)) %&gt;% kable() %&gt;% kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;% scroll_box(height=&quot;600px&quot;)</code></pre>
<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Description
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
GeneRatio
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
BgRatio
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
pvalue
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
p.adjust
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
qvalue
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
geneID
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
hsa04657
</td>
<td style="text-align:left;">
IL-17 signaling pathway
</td>
<td style="text-align:left;">
14/158
</td>
<td style="text-align:left;">
94/8105
</td>
<td style="text-align:right;">
0.0000000
</td>
<td style="text-align:right;">
0.0000007
</td>
<td style="text-align:right;">
0.0000006
</td>
<td style="text-align:left;">
3569/2919/3553/5743/2921/1437/1440/3934/2920/3576/6364/7128/4314/7184
</td>
<td style="text-align:right;">
14
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04668
</td>
<td style="text-align:left;">
TNF signaling pathway
</td>
<td style="text-align:left;">
13/158
</td>
<td style="text-align:left;">
112/8105
</td>
<td style="text-align:right;">
0.0000002
</td>
<td style="text-align:right;">
0.0000293
</td>
<td style="text-align:right;">
0.0000255
</td>
<td style="text-align:left;">
3569/2919/3553/5743/2921/1437/2920/6364/7128/9021/4314/9586/7185
</td>
<td style="text-align:right;">
13
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05323
</td>
<td style="text-align:left;">
Rheumatoid arthritis
</td>
<td style="text-align:left;">
11/158
</td>
<td style="text-align:left;">
93/8105
</td>
<td style="text-align:right;">
0.0000017
</td>
<td style="text-align:right;">
0.0001426
</td>
<td style="text-align:right;">
0.0001242
</td>
<td style="text-align:left;">
3569/2919/3553/2921/1437/3552/2920/3576/6364/245972/4314
</td>
<td style="text-align:right;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05417
</td>
<td style="text-align:left;">
Lipid and atherosclerosis
</td>
<td style="text-align:left;">
16/158
</td>
<td style="text-align:left;">
215/8105
</td>
<td style="text-align:right;">
0.0000040
</td>
<td style="text-align:right;">
0.0002151
</td>
<td style="text-align:right;">
0.0001873
</td>
<td style="text-align:left;">
3569/2919/3553/2921/6648/19/1543/2920/3576/4688/3309/4314/929/1649/7184/114548
</td>
<td style="text-align:right;">
16
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05146
</td>
<td style="text-align:left;">
Amoebiasis
</td>
<td style="text-align:left;">
11/158
</td>
<td style="text-align:left;">
102/8105
</td>
<td style="text-align:right;">
0.0000042
</td>
<td style="text-align:right;">
0.0002151
</td>
<td style="text-align:right;">
0.0001873
</td>
<td style="text-align:left;">
3569/2919/3553/2921/1437/2920/3576/3911/284217/929/2774
</td>
<td style="text-align:right;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04060
</td>
<td style="text-align:left;">
Cytokine-cytokine receptor interaction
</td>
<td style="text-align:left;">
18/158
</td>
<td style="text-align:left;">
295/8105
</td>
<td style="text-align:right;">
0.0000160
</td>
<td style="text-align:right;">
0.0006789
</td>
<td style="text-align:right;">
0.0005914
</td>
<td style="text-align:left;">
3569/2919/3553/2921/1437/3552/1440/9966/2920/3576/6364/652/26525/11009/3570/4803/3624/8794
</td>
<td style="text-align:right;">
18
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04064
</td>
<td style="text-align:left;">
NF-kappa B signaling pathway
</td>
<td style="text-align:left;">
10/158
</td>
<td style="text-align:left;">
104/8105
</td>
<td style="text-align:right;">
0.0000323
</td>
<td style="text-align:right;">
0.0011760
</td>
<td style="text-align:right;">
0.0010243
</td>
<td style="text-align:left;">
2919/3553/5743/2921/2920/3576/29775/7128/929/7185
</td>
<td style="text-align:right;">
10
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05202
</td>
<td style="text-align:left;">
Transcriptional misregulation in cancer
</td>
<td style="text-align:left;">
13/158
</td>
<td style="text-align:left;">
192/8105
</td>
<td style="text-align:right;">
0.0000901
</td>
<td style="text-align:right;">
0.0028724
</td>
<td style="text-align:right;">
0.0025018
</td>
<td style="text-align:left;">
3569/1437/3576/64332/8464/3486/4314/929/1649/894/7185/26471/7490
</td>
<td style="text-align:right;">
13
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05134
</td>
<td style="text-align:left;">
Legionellosis
</td>
<td style="text-align:left;">
7/158
</td>
<td style="text-align:left;">
57/8105
</td>
<td style="text-align:right;">
0.0001093
</td>
<td style="text-align:right;">
0.0030955
</td>
<td style="text-align:right;">
0.0026962
</td>
<td style="text-align:left;">
3569/2919/3553/2921/2920/3576/929
</td>
<td style="text-align:right;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04061
</td>
<td style="text-align:left;">
Viral protein interaction with cytokine and cytokine receptor
</td>
<td style="text-align:left;">
9/158
</td>
<td style="text-align:left;">
100/8105
</td>
<td style="text-align:right;">
0.0001354
</td>
<td style="text-align:right;">
0.0034532
</td>
<td style="text-align:right;">
0.0030077
</td>
<td style="text-align:left;">
3569/2919/2921/2920/3576/6364/11009/3570/8794
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04151
</td>
<td style="text-align:left;">
PI3K-Akt signaling pathway
</td>
<td style="text-align:left;">
17/158
</td>
<td style="text-align:left;">
354/8105
</td>
<td style="text-align:right;">
0.0005122
</td>
<td style="text-align:right;">
0.0118735
</td>
<td style="text-align:right;">
0.0103418
</td>
<td style="text-align:left;">
3569/1440/3911/896/3371/8515/284217/3570/4803/5649/1950/9586/894/2261/7184/6446/3693
</td>
<td style="text-align:right;">
17
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05165
</td>
<td style="text-align:left;">
Human papillomavirus infection
</td>
<td style="text-align:left;">
16/158
</td>
<td style="text-align:left;">
331/8105
</td>
<td style="text-align:right;">
0.0007007
</td>
<td style="text-align:right;">
0.0148897
</td>
<td style="text-align:right;">
0.0129690
</td>
<td style="text-align:left;">
5743/3911/245972/896/3371/8515/284217/7477/4854/5649/1950/9586/894/4242/3693/1869
</td>
<td style="text-align:right;">
16
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05219
</td>
<td style="text-align:left;">
Bladder cancer
</td>
<td style="text-align:left;">
5/158
</td>
<td style="text-align:left;">
41/8105
</td>
<td style="text-align:right;">
0.0011235
</td>
<td style="text-align:right;">
0.0220384
</td>
<td style="text-align:right;">
0.0191955
</td>
<td style="text-align:left;">
3576/1950/1870/2261/1869
</td>
<td style="text-align:right;">
5
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04512
</td>
<td style="text-align:left;">
ECM-receptor interaction
</td>
<td style="text-align:left;">
7/158
</td>
<td style="text-align:left;">
88/8105
</td>
<td style="text-align:right;">
0.0015851
</td>
<td style="text-align:right;">
0.0288719
</td>
<td style="text-align:right;">
0.0251475
</td>
<td style="text-align:left;">
3911/3371/8515/284217/5649/3693/375790
</td>
<td style="text-align:right;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04640
</td>
<td style="text-align:left;">
Hematopoietic cell lineage
</td>
<td style="text-align:left;">
7/158
</td>
<td style="text-align:left;">
99/8105
</td>
<td style="text-align:right;">
0.0031114
</td>
<td style="text-align:right;">
0.0498165
</td>
<td style="text-align:right;">
0.0433903
</td>
<td style="text-align:left;">
3569/3553/1437/3552/1440/3570/929
</td>
<td style="text-align:right;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04621
</td>
<td style="text-align:left;">
NOD-like receptor signaling pathway
</td>
<td style="text-align:left;">
10/158
</td>
<td style="text-align:left;">
184/8105
</td>
<td style="text-align:right;">
0.0031257
</td>
<td style="text-align:right;">
0.0498165
</td>
<td style="text-align:right;">
0.0433903
</td>
<td style="text-align:left;">
3569/2919/3553/2921/2920/3576/7128/114548/837/10135
</td>
<td style="text-align:right;">
10
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04218
</td>
<td style="text-align:left;">
Cellular senescence
</td>
<td style="text-align:left;">
9/158
</td>
<td style="text-align:left;">
156/8105
</td>
<td style="text-align:right;">
0.0033650
</td>
<td style="text-align:right;">
0.0502266
</td>
<td style="text-align:right;">
0.0437474
</td>
<td style="text-align:left;">
3569/3552/3576/5054/896/3486/894/1870/1869
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05133
</td>
<td style="text-align:left;">
Pertussis
</td>
<td style="text-align:left;">
6/158
</td>
<td style="text-align:left;">
76/8105
</td>
<td style="text-align:right;">
0.0035454
</td>
<td style="text-align:right;">
0.0502266
</td>
<td style="text-align:right;">
0.0437474
</td>
<td style="text-align:left;">
3569/3553/3552/3576/929/114548
</td>
<td style="text-align:right;">
6
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04630
</td>
<td style="text-align:left;">
JAK-STAT signaling pathway
</td>
<td style="text-align:left;">
9/158
</td>
<td style="text-align:left;">
162/8105
</td>
<td style="text-align:right;">
0.0043201
</td>
<td style="text-align:right;">
0.0579807
</td>
<td style="text-align:right;">
0.0505013
</td>
<td style="text-align:left;">
3569/1437/1440/11009/896/3570/9021/1950/894
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05167
</td>
<td style="text-align:left;">
Kaposi sarcoma-associated herpesvirus infection
</td>
<td style="text-align:left;">
10/158
</td>
<td style="text-align:left;">
194/8105
</td>
<td style="text-align:right;">
0.0045561
</td>
<td style="text-align:right;">
0.0580906
</td>
<td style="text-align:right;">
0.0505970
</td>
<td style="text-align:left;">
3569/2919/5743/2921/1437/2920/3576/1827/1870/1869
</td>
<td style="text-align:right;">
10
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05171
</td>
<td style="text-align:left;">
Coronavirus disease - COVID-19
</td>
<td style="text-align:left;">
11/158
</td>
<td style="text-align:left;">
232/8105
</td>
<td style="text-align:right;">
0.0055758
</td>
<td style="text-align:right;">
0.0677059
</td>
<td style="text-align:right;">
0.0589719
</td>
<td style="text-align:left;">
3569/3553/1437/1440/3576/200916/3570/629/4314/23521/114548
</td>
<td style="text-align:right;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04936
</td>
<td style="text-align:left;">
Alcoholic liver disease
</td>
<td style="text-align:left;">
8/158
</td>
<td style="text-align:left;">
142/8105
</td>
<td style="text-align:right;">
0.0064614
</td>
<td style="text-align:right;">
0.0720053
</td>
<td style="text-align:right;">
0.0627167
</td>
<td style="text-align:left;">
3569/2919/3553/2921/2920/3576/2194/929
</td>
<td style="text-align:right;">
8
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa00532
</td>
<td style="text-align:left;">
Glycosaminoglycan biosynthesis - chondroitin sulfate / dermatan sulfate
</td>
<td style="text-align:left;">
3/158
</td>
<td style="text-align:left;">
20/8105
</td>
<td style="text-align:right;">
0.0064946
</td>
<td style="text-align:right;">
0.0720053
</td>
<td style="text-align:right;">
0.0627167
</td>
<td style="text-align:left;">
56548/54480/55501
</td>
<td style="text-align:right;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05222
</td>
<td style="text-align:left;">
Small cell lung cancer
</td>
<td style="text-align:left;">
6/158
</td>
<td style="text-align:left;">
92/8105
</td>
<td style="text-align:right;">
0.0089725
</td>
<td style="text-align:right;">
0.0953326
</td>
<td style="text-align:right;">
0.0830348
</td>
<td style="text-align:left;">
5743/3911/284217/1870/7185/1869
</td>
<td style="text-align:right;">
6
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04932
</td>
<td style="text-align:left;">
Non-alcoholic fatty liver disease
</td>
<td style="text-align:left;">
8/158
</td>
<td style="text-align:left;">
155/8105
</td>
<td style="text-align:right;">
0.0107247
</td>
<td style="text-align:right;">
0.1091708
</td>
<td style="text-align:right;">
0.0950879
</td>
<td style="text-align:left;">
3569/3553/3552/3576/3570/9021/1649/125965
</td>
<td style="text-align:right;">
8
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05215
</td>
<td style="text-align:left;">
Prostate cancer
</td>
<td style="text-align:left;">
6/158
</td>
<td style="text-align:left;">
97/8105
</td>
<td style="text-align:right;">
0.0114896
</td>
<td style="text-align:right;">
0.1091708
</td>
<td style="text-align:right;">
0.0950879
</td>
<td style="text-align:left;">
4314/1950/9586/1870/7184/1869
</td>
<td style="text-align:right;">
6
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05120
</td>
<td style="text-align:left;">
Epithelial cell signaling in Helicobacter pylori infection
</td>
<td style="text-align:left;">
5/158
</td>
<td style="text-align:left;">
70/8105
</td>
<td style="text-align:right;">
0.0115593
</td>
<td style="text-align:right;">
0.1091708
</td>
<td style="text-align:right;">
0.0950879
</td>
<td style="text-align:left;">
2919/2921/2920/3576/245972
</td>
<td style="text-align:right;">
5
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04933
</td>
<td style="text-align:left;">
AGE-RAGE signaling pathway in diabetic complications
</td>
<td style="text-align:left;">
6/158
</td>
<td style="text-align:left;">
100/8105
</td>
<td style="text-align:right;">
0.0132195
</td>
<td style="text-align:right;">
0.1203915
</td>
<td style="text-align:right;">
0.1048611
</td>
<td style="text-align:left;">
3569/3553/3552/3576/5054/2152
</td>
<td style="text-align:right;">
6
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05130
</td>
<td style="text-align:left;">
Pathogenic Escherichia coli infection
</td>
<td style="text-align:left;">
9/158
</td>
<td style="text-align:left;">
197/8105
</td>
<td style="text-align:right;">
0.0147467
</td>
<td style="text-align:right;">
0.1296695
</td>
<td style="text-align:right;">
0.1129423
</td>
<td style="text-align:left;">
3569/3553/9076/3576/10381/27128/114548/837/9075
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05144
</td>
<td style="text-align:left;">
Malaria
</td>
<td style="text-align:left;">
4/158
</td>
<td style="text-align:left;">
50/8105
</td>
<td style="text-align:right;">
0.0159762
</td>
<td style="text-align:right;">
0.1357977
</td>
<td style="text-align:right;">
0.1182799
</td>
<td style="text-align:left;">
3569/3553/1440/3576
</td>
<td style="text-align:right;">
4
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="cki-nme-vs-cki" class="section level4">
<h4>CKI-Nme vs CKI</h4>
<pre class="r"><code>enrichKEGG_treat_sig[[&quot;CKI-Nme vs CKI&quot;]][1:30,] %&gt;% as.data.frame() %&gt;% dplyr::select(-c(&quot;ID&quot;)) %&gt;% kable() %&gt;% kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;% scroll_box(height=&quot;600px&quot;)</code></pre>
<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Description
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
GeneRatio
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
BgRatio
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
pvalue
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
p.adjust
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
qvalue
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
geneID
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
hsa04657
</td>
<td style="text-align:left;">
IL-17 signaling pathway
</td>
<td style="text-align:left;">
13/185
</td>
<td style="text-align:left;">
94/8105
</td>
<td style="text-align:right;">
0.0000002
</td>
<td style="text-align:right;">
0.0000470
</td>
<td style="text-align:right;">
0.0000417
</td>
<td style="text-align:left;">
5743/7184/3569/2921/2919/3576/3934/2920/4314/2353/4312/6364/1440
</td>
<td style="text-align:right;">
13
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05323
</td>
<td style="text-align:left;">
Rheumatoid arthritis
</td>
<td style="text-align:left;">
12/185
</td>
<td style="text-align:left;">
93/8105
</td>
<td style="text-align:right;">
0.0000012
</td>
<td style="text-align:right;">
0.0001523
</td>
<td style="text-align:right;">
0.0001352
</td>
<td style="text-align:left;">
3569/2921/2919/3576/3552/2920/245972/4314/2353/4312/3109/6364
</td>
<td style="text-align:right;">
12
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05417
</td>
<td style="text-align:left;">
Lipid and atherosclerosis
</td>
<td style="text-align:left;">
17/185
</td>
<td style="text-align:left;">
215/8105
</td>
<td style="text-align:right;">
0.0000076
</td>
<td style="text-align:right;">
0.0006599
</td>
<td style="text-align:right;">
0.0005859
</td>
<td style="text-align:left;">
1543/7184/19/3569/2921/2919/3576/2920/4688/3309/4314/929/2353/4312/1649/3310/5608
</td>
<td style="text-align:right;">
17
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04668
</td>
<td style="text-align:left;">
TNF signaling pathway
</td>
<td style="text-align:left;">
11/185
</td>
<td style="text-align:left;">
112/8105
</td>
<td style="text-align:right;">
0.0000460
</td>
<td style="text-align:right;">
0.0030130
</td>
<td style="text-align:right;">
0.0026753
</td>
<td style="text-align:left;">
5743/3569/2921/2919/2920/9021/4314/2353/6364/5608/9586
</td>
<td style="text-align:right;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04061
</td>
<td style="text-align:left;">
Viral protein interaction with cytokine and cytokine receptor
</td>
<td style="text-align:left;">
10/185
</td>
<td style="text-align:left;">
100/8105
</td>
<td style="text-align:right;">
0.0000879
</td>
<td style="text-align:right;">
0.0046070
</td>
<td style="text-align:right;">
0.0040906
</td>
<td style="text-align:left;">
3569/2921/2919/3576/2920/11009/8794/1436/6364/58985
</td>
<td style="text-align:right;">
10
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05165
</td>
<td style="text-align:left;">
Human papillomavirus infection
</td>
<td style="text-align:left;">
19/185
</td>
<td style="text-align:left;">
331/8105
</td>
<td style="text-align:right;">
0.0001882
</td>
<td style="text-align:right;">
0.0082199
</td>
<td style="text-align:right;">
0.0072985
</td>
<td style="text-align:left;">
5743/896/7477/1869/3693/3911/245972/8515/3955/4854/284217/3371/1950/5649/4242/5159/9134/894/9586
</td>
<td style="text-align:right;">
19
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05134
</td>
<td style="text-align:left;">
Legionellosis
</td>
<td style="text-align:left;">
7/185
</td>
<td style="text-align:left;">
57/8105
</td>
<td style="text-align:right;">
0.0002904
</td>
<td style="text-align:right;">
0.0098780
</td>
<td style="text-align:right;">
0.0087707
</td>
<td style="text-align:left;">
3569/2921/2919/3576/2920/929/3310
</td>
<td style="text-align:right;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05219
</td>
<td style="text-align:left;">
Bladder cancer
</td>
<td style="text-align:left;">
6/185
</td>
<td style="text-align:left;">
41/8105
</td>
<td style="text-align:right;">
0.0003016
</td>
<td style="text-align:right;">
0.0098780
</td>
<td style="text-align:right;">
0.0087707
</td>
<td style="text-align:left;">
3576/1869/2261/1870/1950/4312
</td>
<td style="text-align:right;">
6
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04151
</td>
<td style="text-align:left;">
PI3K-Akt signaling pathway
</td>
<td style="text-align:left;">
19/185
</td>
<td style="text-align:left;">
354/8105
</td>
<td style="text-align:right;">
0.0004411
</td>
<td style="text-align:right;">
0.0128401
</td>
<td style="text-align:right;">
0.0114008
</td>
<td style="text-align:left;">
7184/3569/896/3693/3911/2261/8515/284217/3371/1950/5649/4804/1436/5159/1440/9134/2264/894/9586
</td>
<td style="text-align:right;">
19
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05146
</td>
<td style="text-align:left;">
Amoebiasis
</td>
<td style="text-align:left;">
9/185
</td>
<td style="text-align:left;">
102/8105
</td>
<td style="text-align:right;">
0.0005104
</td>
<td style="text-align:right;">
0.0133719
</td>
<td style="text-align:right;">
0.0118730
</td>
<td style="text-align:left;">
3569/2921/2919/3576/2920/3911/929/284217/2774
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05202
</td>
<td style="text-align:left;">
Transcriptional misregulation in cancer
</td>
<td style="text-align:left;">
12/185
</td>
<td style="text-align:left;">
192/8105
</td>
<td style="text-align:right;">
0.0014437
</td>
<td style="text-align:right;">
0.0343855
</td>
<td style="text-align:right;">
0.0305311
</td>
<td style="text-align:left;">
3569/3576/8464/4314/929/7490/4804/1649/1436/26471/894/64332
</td>
<td style="text-align:right;">
12
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05215
</td>
<td style="text-align:left;">
Prostate cancer
</td>
<td style="text-align:left;">
8/185
</td>
<td style="text-align:left;">
97/8105
</td>
<td style="text-align:right;">
0.0016191
</td>
<td style="text-align:right;">
0.0353502
</td>
<td style="text-align:right;">
0.0313877
</td>
<td style="text-align:left;">
7184/1869/4314/1870/1950/5159/9134/9586
</td>
<td style="text-align:right;">
8
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04010
</td>
<td style="text-align:left;">
MAPK signaling pathway
</td>
<td style="text-align:left;">
15/185
</td>
<td style="text-align:left;">
294/8105
</td>
<td style="text-align:right;">
0.0029101
</td>
<td style="text-align:right;">
0.0524982
</td>
<td style="text-align:right;">
0.0466136
</td>
<td style="text-align:left;">
3552/2261/929/25780/1950/1850/5801/4804/2353/1649/1436/5159/3310/2264/5608
</td>
<td style="text-align:right;">
15
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04218
</td>
<td style="text-align:left;">
Cellular senescence
</td>
<td style="text-align:left;">
10/185
</td>
<td style="text-align:left;">
156/8105
</td>
<td style="text-align:right;">
0.0029641
</td>
<td style="text-align:right;">
0.0524982
</td>
<td style="text-align:right;">
0.0466136
</td>
<td style="text-align:left;">
3569/896/3576/1869/3552/1870/9134/894/4605/5608
</td>
<td style="text-align:right;">
10
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04060
</td>
<td style="text-align:left;">
Cytokine-cytokine receptor interaction
</td>
<td style="text-align:left;">
15/185
</td>
<td style="text-align:left;">
295/8105
</td>
<td style="text-align:right;">
0.0030056
</td>
<td style="text-align:right;">
0.0524982
</td>
<td style="text-align:right;">
0.0466136
</td>
<td style="text-align:left;">
3569/2921/2919/652/3576/3552/2920/11009/8794/26525/4804/1436/6364/1440/58985
</td>
<td style="text-align:right;">
15
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04512
</td>
<td style="text-align:left;">
ECM-receptor interaction
</td>
<td style="text-align:left;">
7/185
</td>
<td style="text-align:left;">
88/8105
</td>
<td style="text-align:right;">
0.0038563
</td>
<td style="text-align:right;">
0.0631475
</td>
<td style="text-align:right;">
0.0560691
</td>
<td style="text-align:left;">
3693/3911/8515/284217/3371/5649/1605
</td>
<td style="text-align:right;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05167
</td>
<td style="text-align:left;">
Kaposi sarcoma-associated herpesvirus infection
</td>
<td style="text-align:left;">
11/185
</td>
<td style="text-align:left;">
194/8105
</td>
<td style="text-align:right;">
0.0047842
</td>
<td style="text-align:right;">
0.0737329
</td>
<td style="text-align:right;">
0.0654680
</td>
<td style="text-align:left;">
5743/3569/2921/2919/3576/1869/2920/1827/1870/2353/5608
</td>
<td style="text-align:right;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa00051
</td>
<td style="text-align:left;">
Fructose and mannose metabolism
</td>
<td style="text-align:left;">
4/185
</td>
<td style="text-align:left;">
33/8105
</td>
<td style="text-align:right;">
0.0064092
</td>
<td style="text-align:right;">
0.0932895
</td>
<td style="text-align:right;">
0.0828324
</td>
<td style="text-align:left;">
57016/230/3795/80201
</td>
<td style="text-align:right;">
4
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04640
</td>
<td style="text-align:left;">
Hematopoietic cell lineage
</td>
<td style="text-align:left;">
7/185
</td>
<td style="text-align:left;">
99/8105
</td>
<td style="text-align:right;">
0.0073390
</td>
<td style="text-align:right;">
0.1012005
</td>
<td style="text-align:right;">
0.0898566
</td>
<td style="text-align:left;">
3569/3552/929/1436/3109/1440/966
</td>
<td style="text-align:right;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04145
</td>
<td style="text-align:left;">
Phagosome
</td>
<td style="text-align:left;">
9/185
</td>
<td style="text-align:left;">
152/8105
</td>
<td style="text-align:right;">
0.0079055
</td>
<td style="text-align:right;">
0.1035625
</td>
<td style="text-align:right;">
0.0919538
</td>
<td style="text-align:left;">
9902/81035/4688/3693/245972/929/7277/3109/10381
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04015
</td>
<td style="text-align:left;">
Rap1 signaling pathway
</td>
<td style="text-align:left;">
11/185
</td>
<td style="text-align:left;">
210/8105
</td>
<td style="text-align:right;">
0.0085494
</td>
<td style="text-align:right;">
0.1066641
</td>
<td style="text-align:right;">
0.0947078
</td>
<td style="text-align:left;">
9002/2357/3397/2261/25780/1950/4804/1436/5159/2264/5608
</td>
<td style="text-align:right;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04064
</td>
<td style="text-align:left;">
NF-kappa B signaling pathway
</td>
<td style="text-align:left;">
7/185
</td>
<td style="text-align:left;">
104/8105
</td>
<td style="text-align:right;">
0.0095267
</td>
<td style="text-align:right;">
0.1093887
</td>
<td style="text-align:right;">
0.0971269
</td>
<td style="text-align:left;">
5743/2921/2919/3576/2920/29775/929
</td>
<td style="text-align:right;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa00100
</td>
<td style="text-align:left;">
Steroid biosynthesis
</td>
<td style="text-align:left;">
3/185
</td>
<td style="text-align:left;">
20/8105
</td>
<td style="text-align:right;">
0.0100203
</td>
<td style="text-align:right;">
0.1093887
</td>
<td style="text-align:right;">
0.0971269
</td>
<td style="text-align:left;">
7108/10682/1718
</td>
<td style="text-align:right;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa00532
</td>
<td style="text-align:left;">
Glycosaminoglycan biosynthesis - chondroitin sulfate / dermatan sulfate
</td>
<td style="text-align:left;">
3/185
</td>
<td style="text-align:left;">
20/8105
</td>
<td style="text-align:right;">
0.0100203
</td>
<td style="text-align:right;">
0.1093887
</td>
<td style="text-align:right;">
0.0971269
</td>
<td style="text-align:left;">
56548/54480/55790
</td>
<td style="text-align:right;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04630
</td>
<td style="text-align:left;">
JAK-STAT signaling pathway
</td>
<td style="text-align:left;">
9/185
</td>
<td style="text-align:left;">
162/8105
</td>
<td style="text-align:right;">
0.0117406
</td>
<td style="text-align:right;">
0.1230410
</td>
<td style="text-align:right;">
0.1092489
</td>
<td style="text-align:left;">
3569/896/11009/9021/1950/5159/1440/894/58985
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04510
</td>
<td style="text-align:left;">
Focal adhesion
</td>
<td style="text-align:left;">
10/185
</td>
<td style="text-align:left;">
201/8105
</td>
<td style="text-align:right;">
0.0166394
</td>
<td style="text-align:right;">
0.1676743
</td>
<td style="text-align:right;">
0.1488792
</td>
<td style="text-align:left;">
896/3693/3911/8515/284217/3371/1950/5649/5159/894
</td>
<td style="text-align:right;">
10
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05222
</td>
<td style="text-align:left;">
Small cell lung cancer
</td>
<td style="text-align:left;">
6/185
</td>
<td style="text-align:left;">
92/8105
</td>
<td style="text-align:right;">
0.0184119
</td>
<td style="text-align:right;">
0.1786636
</td>
<td style="text-align:right;">
0.1586367
</td>
<td style="text-align:left;">
5743/1869/3911/1870/284217/9134
</td>
<td style="text-align:right;">
6
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05120
</td>
<td style="text-align:left;">
Epithelial cell signaling in Helicobacter pylori infection
</td>
<td style="text-align:left;">
5/185
</td>
<td style="text-align:left;">
70/8105
</td>
<td style="text-align:right;">
0.0215007
</td>
<td style="text-align:right;">
0.1973473
</td>
<td style="text-align:right;">
0.1752260
</td>
<td style="text-align:left;">
2921/2919/3576/2920/245972
</td>
<td style="text-align:right;">
5
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa00514
</td>
<td style="text-align:left;">
Other types of O-glycan biosynthesis
</td>
<td style="text-align:left;">
4/185
</td>
<td style="text-align:left;">
47/8105
</td>
<td style="text-align:right;">
0.0218438
</td>
<td style="text-align:right;">
0.1973473
</td>
<td style="text-align:right;">
0.1752260
</td>
<td style="text-align:left;">
55568/3955/4242/2590
</td>
<td style="text-align:right;">
4
</td>
</tr>
<tr>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
NA
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="cki-omt-vs-cki" class="section level4">
<h4>CKI-Omt vs CKI</h4>
<pre class="r"><code>enrichKEGG_treat_sig[[&quot;CKI-Omt vs CKI&quot;]][1:30,] %&gt;% as.data.frame() %&gt;% dplyr::select(-c(&quot;ID&quot;)) %&gt;% kable() %&gt;% kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;% scroll_box(height=&quot;600px&quot;)</code></pre>
<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Description
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
GeneRatio
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
BgRatio
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
pvalue
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
p.adjust
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
qvalue
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
geneID
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
hsa04110
</td>
<td style="text-align:left;">
Cell cycle
</td>
<td style="text-align:left;">
26/287
</td>
<td style="text-align:left;">
126/8105
</td>
<td style="text-align:right;">
0.0000000
</td>
<td style="text-align:right;">
0.0000000
</td>
<td style="text-align:right;">
0.0000000
</td>
<td style="text-align:left;">
890/9088/699/983/701/9134/8318/894/5347/995/993/7272/9700/4085/990/9133/891/896/1870/5933/4174/1869/4175/4998/1647/1875
</td>
<td style="text-align:right;">
26
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa00260
</td>
<td style="text-align:left;">
Glycine, serine and threonine metabolism
</td>
<td style="text-align:left;">
9/287
</td>
<td style="text-align:left;">
40/8105
</td>
<td style="text-align:right;">
0.0000080
</td>
<td style="text-align:right;">
0.0006333
</td>
<td style="text-align:right;">
0.0005748
</td>
<td style="text-align:left;">
314/8639/23464/26227/4129/29968/113675/1491/1757
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04115
</td>
<td style="text-align:left;">
p53 signaling pathway
</td>
<td style="text-align:left;">
12/287
</td>
<td style="text-align:left;">
73/8105
</td>
<td style="text-align:right;">
0.0000082
</td>
<td style="text-align:right;">
0.0006333
</td>
<td style="text-align:right;">
0.0005748
</td>
<td style="text-align:left;">
6241/983/9134/894/9133/891/896/3732/143686/1647/7161/83667
</td>
<td style="text-align:right;">
12
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04218
</td>
<td style="text-align:left;">
Cellular senescence
</td>
<td style="text-align:left;">
18/287
</td>
<td style="text-align:left;">
156/8105
</td>
<td style="text-align:right;">
0.0000092
</td>
<td style="text-align:right;">
0.0006333
</td>
<td style="text-align:right;">
0.0005748
</td>
<td style="text-align:left;">
3569/890/3552/983/9134/894/4605/993/3576/2113/9133/891/896/1870/5933/1869/1647/1875
</td>
<td style="text-align:right;">
18
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa03460
</td>
<td style="text-align:left;">
Fanconi anemia pathway
</td>
<td style="text-align:left;">
10/287
</td>
<td style="text-align:left;">
54/8105
</td>
<td style="text-align:right;">
0.0000159
</td>
<td style="text-align:right;">
0.0008793
</td>
<td style="text-align:right;">
0.0007982
</td>
<td style="text-align:left;">
672/641/83990/55215/2177/5888/29089/2175/116028/9894
</td>
<td style="text-align:right;">
10
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04657
</td>
<td style="text-align:left;">
IL-17 signaling pathway
</td>
<td style="text-align:left;">
13/287
</td>
<td style="text-align:left;">
94/8105
</td>
<td style="text-align:right;">
0.0000241
</td>
<td style="text-align:right;">
0.0011071
</td>
<td style="text-align:right;">
0.0010049
</td>
<td style="text-align:left;">
3569/3553/2921/1437/3576/2919/7128/6364/5743/2920/3934/1440/4314
</td>
<td style="text-align:right;">
13
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa03440
</td>
<td style="text-align:left;">
Homologous recombination
</td>
<td style="text-align:left;">
8/287
</td>
<td style="text-align:left;">
41/8105
</td>
<td style="text-align:right;">
0.0000773
</td>
<td style="text-align:right;">
0.0030495
</td>
<td style="text-align:right;">
0.0027680
</td>
<td style="text-align:left;">
672/641/83990/8438/7516/5888/25788/580
</td>
<td style="text-align:right;">
8
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05323
</td>
<td style="text-align:left;">
Rheumatoid arthritis
</td>
<td style="text-align:left;">
12/287
</td>
<td style="text-align:left;">
93/8105
</td>
<td style="text-align:right;">
0.0000989
</td>
<td style="text-align:right;">
0.0034121
</td>
<td style="text-align:right;">
0.0030972
</td>
<td style="text-align:left;">
3569/3553/3552/2921/1437/3576/2919/6364/2920/245972/4314/155066
</td>
<td style="text-align:right;">
12
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa03030
</td>
<td style="text-align:left;">
DNA replication
</td>
<td style="text-align:left;">
7/287
</td>
<td style="text-align:left;">
36/8105
</td>
<td style="text-align:right;">
0.0002238
</td>
<td style="text-align:right;">
0.0068642
</td>
<td style="text-align:right;">
0.0062307
</td>
<td style="text-align:left;">
5557/2237/5427/4174/4175/5983/5426
</td>
<td style="text-align:right;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04114
</td>
<td style="text-align:left;">
Oocyte meiosis
</td>
<td style="text-align:left;">
13/287
</td>
<td style="text-align:left;">
131/8105
</td>
<td style="text-align:right;">
0.0007195
</td>
<td style="text-align:right;">
0.0198587
</td>
<td style="text-align:right;">
0.0180258
</td>
<td style="text-align:left;">
9088/699/983/9134/5347/995/151648/9700/4085/9133/891/89869/286151
</td>
<td style="text-align:right;">
13
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04914
</td>
<td style="text-align:left;">
Progesterone-mediated oocyte maturation
</td>
<td style="text-align:left;">
11/287
</td>
<td style="text-align:left;">
102/8105
</td>
<td style="text-align:right;">
0.0009190
</td>
<td style="text-align:right;">
0.0230592
</td>
<td style="text-align:right;">
0.0209309
</td>
<td style="text-align:left;">
890/9088/699/983/5347/995/993/4085/9133/891/3835
</td>
<td style="text-align:right;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04668
</td>
<td style="text-align:left;">
TNF signaling pathway
</td>
<td style="text-align:left;">
11/287
</td>
<td style="text-align:left;">
112/8105
</td>
<td style="text-align:right;">
0.0019811
</td>
<td style="text-align:right;">
0.0455649
</td>
<td style="text-align:right;">
0.0413594
</td>
<td style="text-align:left;">
3569/3553/2921/1437/2919/7128/6364/5743/2920/197259/4314
</td>
<td style="text-align:right;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa00360
</td>
<td style="text-align:left;">
Phenylalanine metabolism
</td>
<td style="text-align:left;">
4/287
</td>
<td style="text-align:left;">
17/8105
</td>
<td style="text-align:right;">
0.0025431
</td>
<td style="text-align:right;">
0.0539917
</td>
<td style="text-align:right;">
0.0490085
</td>
<td style="text-align:left;">
314/8639/4129/218
</td>
<td style="text-align:right;">
4
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05146
</td>
<td style="text-align:left;">
Amoebiasis
</td>
<td style="text-align:left;">
10/287
</td>
<td style="text-align:left;">
102/8105
</td>
<td style="text-align:right;">
0.0031788
</td>
<td style="text-align:right;">
0.0626684
</td>
<td style="text-align:right;">
0.0568844
</td>
<td style="text-align:left;">
3569/3553/284217/2921/1437/3576/2919/2920/3911/2774
</td>
<td style="text-align:right;">
10
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa01523
</td>
<td style="text-align:left;">
Antifolate resistance
</td>
<td style="text-align:left;">
5/287
</td>
<td style="text-align:left;">
31/8105
</td>
<td style="text-align:right;">
0.0042864
</td>
<td style="text-align:right;">
0.0786280
</td>
<td style="text-align:right;">
0.0713709
</td>
<td style="text-align:left;">
3569/3553/1719/7298/2352
</td>
<td style="text-align:right;">
5
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa01230
</td>
<td style="text-align:left;">
Biosynthesis of amino acids
</td>
<td style="text-align:left;">
8/287
</td>
<td style="text-align:left;">
75/8105
</td>
<td style="text-align:right;">
0.0048304
</td>
<td style="text-align:right;">
0.0786280
</td>
<td style="text-align:right;">
0.0713709
</td>
<td style="text-align:left;">
445/230/26227/29968/1373/113675/2027/1491
</td>
<td style="text-align:right;">
8
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04630
</td>
<td style="text-align:left;">
JAK-STAT signaling pathway
</td>
<td style="text-align:left;">
13/287
</td>
<td style="text-align:left;">
162/8105
</td>
<td style="text-align:right;">
0.0048430
</td>
<td style="text-align:right;">
0.0786280
</td>
<td style="text-align:right;">
0.0713709
</td>
<td style="text-align:left;">
3569/1437/894/8835/11009/896/3570/1950/1440/3563/5155/58985/5618
</td>
<td style="text-align:right;">
13
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa00051
</td>
<td style="text-align:left;">
Fructose and mannose metabolism
</td>
<td style="text-align:left;">
5/287
</td>
<td style="text-align:left;">
33/8105
</td>
<td style="text-align:right;">
0.0056523
</td>
<td style="text-align:right;">
0.0866693
</td>
<td style="text-align:right;">
0.0786701
</td>
<td style="text-align:left;">
230/57016/3795/80201/5208
</td>
<td style="text-align:right;">
5
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05202
</td>
<td style="text-align:left;">
Transcriptional misregulation in cancer
</td>
<td style="text-align:left;">
14/287
</td>
<td style="text-align:left;">
192/8105
</td>
<td style="text-align:right;">
0.0081368
</td>
<td style="text-align:right;">
0.1099936
</td>
<td style="text-align:right;">
0.0998416
</td>
<td style="text-align:left;">
3569/890/1437/894/3576/7490/64332/3398/4314/51274/1647/26471/3728/1649
</td>
<td style="text-align:right;">
14
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa00350
</td>
<td style="text-align:left;">
Tyrosine metabolism
</td>
<td style="text-align:left;">
5/287
</td>
<td style="text-align:left;">
36/8105
</td>
<td style="text-align:right;">
0.0082371
</td>
<td style="text-align:right;">
0.1099936
</td>
<td style="text-align:right;">
0.0998416
</td>
<td style="text-align:left;">
314/8639/4129/2954/218
</td>
<td style="text-align:right;">
5
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04060
</td>
<td style="text-align:left;">
Cytokine-cytokine receptor interaction
</td>
<td style="text-align:left;">
19/287
</td>
<td style="text-align:left;">
295/8105
</td>
<td style="text-align:right;">
0.0084360
</td>
<td style="text-align:right;">
0.1099936
</td>
<td style="text-align:right;">
0.0998416
</td>
<td style="text-align:left;">
3569/3553/9966/3552/2921/1437/3576/2919/6364/11009/2920/3570/652/9518/1440/3563/58985/26525/5618
</td>
<td style="text-align:right;">
19
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04061
</td>
<td style="text-align:left;">
Viral protein interaction with cytokine and cytokine receptor
</td>
<td style="text-align:left;">
9/287
</td>
<td style="text-align:left;">
100/8105
</td>
<td style="text-align:right;">
0.0087676
</td>
<td style="text-align:right;">
0.1099936
</td>
<td style="text-align:right;">
0.0998416
</td>
<td style="text-align:left;">
3569/2921/3576/2919/6364/11009/2920/3570/58985
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04064
</td>
<td style="text-align:left;">
NF-kappa B signaling pathway
</td>
<td style="text-align:left;">
9/287
</td>
<td style="text-align:left;">
104/8105
</td>
<td style="text-align:right;">
0.0112091
</td>
<td style="text-align:right;">
0.1345094
</td>
<td style="text-align:right;">
0.1220948
</td>
<td style="text-align:left;">
3553/2921/3576/2919/7128/5743/2920/29775/1647
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05165
</td>
<td style="text-align:left;">
Human papillomavirus infection
</td>
<td style="text-align:left;">
20/287
</td>
<td style="text-align:left;">
331/8105
</td>
<td style="text-align:right;">
0.0135332
</td>
<td style="text-align:right;">
0.1556315
</td>
<td style="text-align:right;">
0.1412673
</td>
<td style="text-align:left;">
890/284217/9134/894/8638/5649/5743/3371/896/5933/3911/245972/8515/1869/1950/3955/7481/7483/155066/8321
</td>
<td style="text-align:right;">
20
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05134
</td>
<td style="text-align:left;">
Legionellosis
</td>
<td style="text-align:left;">
6/287
</td>
<td style="text-align:left;">
57/8105
</td>
<td style="text-align:right;">
0.0149279
</td>
<td style="text-align:right;">
0.1648035
</td>
<td style="text-align:right;">
0.1495928
</td>
<td style="text-align:left;">
3569/3553/2921/3576/2919/2920
</td>
<td style="text-align:right;">
6
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa00240
</td>
<td style="text-align:left;">
Pyrimidine metabolism
</td>
<td style="text-align:left;">
6/287
</td>
<td style="text-align:left;">
58/8105
</td>
<td style="text-align:right;">
0.0161713
</td>
<td style="text-align:right;">
0.1689253
</td>
<td style="text-align:right;">
0.1533342
</td>
<td style="text-align:left;">
6241/7083/7298/978/7378/6240
</td>
<td style="text-align:right;">
6
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05226
</td>
<td style="text-align:left;">
Gastric cancer
</td>
<td style="text-align:left;">
11/287
</td>
<td style="text-align:left;">
149/8105
</td>
<td style="text-align:right;">
0.0165253
</td>
<td style="text-align:right;">
0.1689253
</td>
<td style="text-align:right;">
0.1533342
</td>
<td style="text-align:left;">
4041/9134/1870/1869/1950/7481/1647/7483/399694/8321/3728
</td>
<td style="text-align:right;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04068
</td>
<td style="text-align:left;">
FoxO signaling pathway
</td>
<td style="text-align:left;">
10/287
</td>
<td style="text-align:left;">
131/8105
</td>
<td style="text-align:right;">
0.0176292
</td>
<td style="text-align:right;">
0.1737732
</td>
<td style="text-align:right;">
0.1577347
</td>
<td style="text-align:left;">
3569/10733/894/5347/9133/891/6648/1950/1647/6446
</td>
<td style="text-align:right;">
10
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa01200
</td>
<td style="text-align:left;">
Carbon metabolism
</td>
<td style="text-align:left;">
9/287
</td>
<td style="text-align:left;">
115/8105
</td>
<td style="text-align:right;">
0.0205362
</td>
<td style="text-align:right;">
0.1954476
</td>
<td style="text-align:right;">
0.1774086
</td>
<td style="text-align:left;">
230/26227/29968/80201/84532/1373/113675/2027/9563
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04640
</td>
<td style="text-align:left;">
Hematopoietic cell lineage
</td>
<td style="text-align:left;">
8/287
</td>
<td style="text-align:left;">
99/8105
</td>
<td style="text-align:right;">
0.0237713
</td>
<td style="text-align:right;">
0.2186964
</td>
<td style="text-align:right;">
0.1985116
</td>
<td style="text-align:left;">
3569/3553/3552/1437/3570/1440/1604/3563
</td>
<td style="text-align:right;">
8
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="cki-tri-vs-cki" class="section level4">
<h4>CKI-Tri vs CKI</h4>
<pre class="r"><code>enrichKEGG_treat_sig[[&quot;CKI-Tri vs CKI&quot;]][1:30,] %&gt;% as.data.frame() %&gt;% dplyr::select(-c(&quot;ID&quot;)) %&gt;% kable() %&gt;% kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;% scroll_box(height=&quot;600px&quot;)</code></pre>
<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Description
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
GeneRatio
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
BgRatio
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
pvalue
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
p.adjust
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
qvalue
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
geneID
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
hsa04657
</td>
<td style="text-align:left;">
IL-17 signaling pathway
</td>
<td style="text-align:left;">
13/171
</td>
<td style="text-align:left;">
94/8105
</td>
<td style="text-align:right;">
0.0000001
</td>
<td style="text-align:right;">
0.0000190
</td>
<td style="text-align:right;">
0.0000158
</td>
<td style="text-align:left;">
3569/5743/2921/2919/3553/3576/2920/3934/7184/6364/4314/1437/1440
</td>
<td style="text-align:right;">
13
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05323
</td>
<td style="text-align:left;">
Rheumatoid arthritis
</td>
<td style="text-align:left;">
12/171
</td>
<td style="text-align:left;">
93/8105
</td>
<td style="text-align:right;">
0.0000005
</td>
<td style="text-align:right;">
0.0000667
</td>
<td style="text-align:right;">
0.0000558
</td>
<td style="text-align:left;">
3569/2921/2919/3552/3553/3576/2920/245972/6364/4314/1437/3109
</td>
<td style="text-align:right;">
12
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05417
</td>
<td style="text-align:left;">
Lipid and atherosclerosis
</td>
<td style="text-align:left;">
17/171
</td>
<td style="text-align:left;">
215/8105
</td>
<td style="text-align:right;">
0.0000026
</td>
<td style="text-align:right;">
0.0002299
</td>
<td style="text-align:right;">
0.0001922
</td>
<td style="text-align:left;">
3569/1543/2921/2919/19/3553/3576/2920/7184/4688/4314/6648/3309/1649/929/3310/5608
</td>
<td style="text-align:right;">
17
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04668
</td>
<td style="text-align:left;">
TNF signaling pathway
</td>
<td style="text-align:left;">
12/171
</td>
<td style="text-align:left;">
112/8105
</td>
<td style="text-align:right;">
0.0000037
</td>
<td style="text-align:right;">
0.0002500
</td>
<td style="text-align:right;">
0.0002089
</td>
<td style="text-align:left;">
3569/5743/2921/2919/3553/2920/6364/4314/1437/9586/9021/5608
</td>
<td style="text-align:right;">
12
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05146
</td>
<td style="text-align:left;">
Amoebiasis
</td>
<td style="text-align:left;">
11/171
</td>
<td style="text-align:left;">
102/8105
</td>
<td style="text-align:right;">
0.0000090
</td>
<td style="text-align:right;">
0.0004825
</td>
<td style="text-align:right;">
0.0004033
</td>
<td style="text-align:left;">
3569/2921/2919/3553/3576/2920/3911/284217/1437/2774/929
</td>
<td style="text-align:right;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05134
</td>
<td style="text-align:left;">
Legionellosis
</td>
<td style="text-align:left;">
8/171
</td>
<td style="text-align:left;">
57/8105
</td>
<td style="text-align:right;">
0.0000228
</td>
<td style="text-align:right;">
0.0010156
</td>
<td style="text-align:right;">
0.0008488
</td>
<td style="text-align:left;">
3569/2921/2919/3553/3576/2920/929/3310
</td>
<td style="text-align:right;">
8
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04218
</td>
<td style="text-align:left;">
Cellular senescence
</td>
<td style="text-align:left;">
12/171
</td>
<td style="text-align:left;">
156/8105
</td>
<td style="text-align:right;">
0.0001067
</td>
<td style="text-align:right;">
0.0040686
</td>
<td style="text-align:right;">
0.0034005
</td>
<td style="text-align:left;">
3569/3552/3576/1869/896/1870/894/2113/677/9134/3486/5608
</td>
<td style="text-align:right;">
12
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04060
</td>
<td style="text-align:left;">
Cytokine-cytokine receptor interaction
</td>
<td style="text-align:left;">
17/171
</td>
<td style="text-align:left;">
295/8105
</td>
<td style="text-align:right;">
0.0001544
</td>
<td style="text-align:right;">
0.0051542
</td>
<td style="text-align:right;">
0.0043079
</td>
<td style="text-align:left;">
3569/2921/2919/3552/3553/3576/2920/652/9966/11009/6364/1437/26525/1440/8794/3570/4803
</td>
<td style="text-align:right;">
17
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05202
</td>
<td style="text-align:left;">
Transcriptional misregulation in cancer
</td>
<td style="text-align:left;">
13/171
</td>
<td style="text-align:left;">
192/8105
</td>
<td style="text-align:right;">
0.0002002
</td>
<td style="text-align:right;">
0.0054346
</td>
<td style="text-align:right;">
0.0045423
</td>
<td style="text-align:left;">
3569/3576/8464/4314/1437/894/64332/7490/1649/929/3486/1031/26471
</td>
<td style="text-align:right;">
13
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05165
</td>
<td style="text-align:left;">
Human papillomavirus infection
</td>
<td style="text-align:left;">
18/171
</td>
<td style="text-align:left;">
331/8105
</td>
<td style="text-align:right;">
0.0002035
</td>
<td style="text-align:right;">
0.0054346
</td>
<td style="text-align:right;">
0.0045423
</td>
<td style="text-align:left;">
5743/1869/8515/896/245972/3693/3911/284217/3371/1950/7477/9586/894/5649/3955/9134/4242/4854
</td>
<td style="text-align:right;">
18
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04061
</td>
<td style="text-align:left;">
Viral protein interaction with cytokine and cytokine receptor
</td>
<td style="text-align:left;">
9/171
</td>
<td style="text-align:left;">
100/8105
</td>
<td style="text-align:right;">
0.0002461
</td>
<td style="text-align:right;">
0.0059747
</td>
<td style="text-align:right;">
0.0049936
</td>
<td style="text-align:left;">
3569/2921/2919/3576/2920/11009/6364/8794/3570
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04640
</td>
<td style="text-align:left;">
Hematopoietic cell lineage
</td>
<td style="text-align:left;">
8/171
</td>
<td style="text-align:left;">
99/8105
</td>
<td style="text-align:right;">
0.0011163
</td>
<td style="text-align:right;">
0.0248374
</td>
<td style="text-align:right;">
0.0207590
</td>
<td style="text-align:left;">
3569/3552/3553/1437/1440/3570/929/3109
</td>
<td style="text-align:right;">
8
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04151
</td>
<td style="text-align:left;">
PI3K-Akt signaling pathway
</td>
<td style="text-align:left;">
17/171
</td>
<td style="text-align:left;">
354/8105
</td>
<td style="text-align:right;">
0.0012566
</td>
<td style="text-align:right;">
0.0258084
</td>
<td style="text-align:right;">
0.0215706
</td>
<td style="text-align:left;">
3569/7184/8515/896/3693/3911/284217/3371/1950/9586/894/1440/5649/2261/3570/4803/9134
</td>
<td style="text-align:right;">
17
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04064
</td>
<td style="text-align:left;">
NF-kappa B signaling pathway
</td>
<td style="text-align:left;">
8/171
</td>
<td style="text-align:left;">
104/8105
</td>
<td style="text-align:right;">
0.0015365
</td>
<td style="text-align:right;">
0.0284388
</td>
<td style="text-align:right;">
0.0237691
</td>
<td style="text-align:left;">
5743/2921/2919/3553/3576/2920/29775/929
</td>
<td style="text-align:right;">
8
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05219
</td>
<td style="text-align:left;">
Bladder cancer
</td>
<td style="text-align:left;">
5/171
</td>
<td style="text-align:left;">
41/8105
</td>
<td style="text-align:right;">
0.0015977
</td>
<td style="text-align:right;">
0.0284388
</td>
<td style="text-align:right;">
0.0237691
</td>
<td style="text-align:left;">
3576/1869/1870/1950/2261
</td>
<td style="text-align:right;">
5
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04630
</td>
<td style="text-align:left;">
JAK-STAT signaling pathway
</td>
<td style="text-align:left;">
10/171
</td>
<td style="text-align:left;">
162/8105
</td>
<td style="text-align:right;">
0.0021997
</td>
<td style="text-align:right;">
0.0367078
</td>
<td style="text-align:right;">
0.0306803
</td>
<td style="text-align:left;">
3569/11009/896/1950/1437/894/9021/1440/3570/8835
</td>
<td style="text-align:right;">
10
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04512
</td>
<td style="text-align:left;">
ECM-receptor interaction
</td>
<td style="text-align:left;">
7/171
</td>
<td style="text-align:left;">
88/8105
</td>
<td style="text-align:right;">
0.0024874
</td>
<td style="text-align:right;">
0.0388734
</td>
<td style="text-align:right;">
0.0324903
</td>
<td style="text-align:left;">
8515/3693/3911/284217/3371/5649/1605
</td>
<td style="text-align:right;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05167
</td>
<td style="text-align:left;">
Kaposi sarcoma-associated herpesvirus infection
</td>
<td style="text-align:left;">
11/171
</td>
<td style="text-align:left;">
194/8105
</td>
<td style="text-align:right;">
0.0026207
</td>
<td style="text-align:right;">
0.0388734
</td>
<td style="text-align:right;">
0.0324903
</td>
<td style="text-align:left;">
3569/5743/2921/2919/3576/2920/1869/1827/1870/1437/5608
</td>
<td style="text-align:right;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04936
</td>
<td style="text-align:left;">
Alcoholic liver disease
</td>
<td style="text-align:left;">
9/171
</td>
<td style="text-align:left;">
142/8105
</td>
<td style="text-align:right;">
0.0030414
</td>
<td style="text-align:right;">
0.0427390
</td>
<td style="text-align:right;">
0.0357212
</td>
<td style="text-align:left;">
3569/2921/2919/3553/3576/2920/929/5608/224
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa01523
</td>
<td style="text-align:left;">
Antifolate resistance
</td>
<td style="text-align:left;">
4/171
</td>
<td style="text-align:left;">
31/8105
</td>
<td style="text-align:right;">
0.0038581
</td>
<td style="text-align:right;">
0.0515052
</td>
<td style="text-align:right;">
0.0430479
</td>
<td style="text-align:left;">
3569/3553/2352/2348
</td>
<td style="text-align:right;">
4
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05215
</td>
<td style="text-align:left;">
Prostate cancer
</td>
<td style="text-align:left;">
7/171
</td>
<td style="text-align:left;">
97/8105
</td>
<td style="text-align:right;">
0.0042985
</td>
<td style="text-align:right;">
0.0546526
</td>
<td style="text-align:right;">
0.0456785
</td>
<td style="text-align:left;">
7184/1869/1870/1950/4314/9586/9134
</td>
<td style="text-align:right;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04145
</td>
<td style="text-align:left;">
Phagosome
</td>
<td style="text-align:left;">
9/171
</td>
<td style="text-align:left;">
152/8105
</td>
<td style="text-align:right;">
0.0047753
</td>
<td style="text-align:right;">
0.0563730
</td>
<td style="text-align:right;">
0.0471164
</td>
<td style="text-align:left;">
81035/4688/245972/3693/9902/929/10381/7277/3109
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa00051
</td>
<td style="text-align:left;">
Fructose and mannose metabolism
</td>
<td style="text-align:left;">
4/171
</td>
<td style="text-align:left;">
33/8105
</td>
<td style="text-align:right;">
0.0048561
</td>
<td style="text-align:right;">
0.0563730
</td>
<td style="text-align:right;">
0.0471164
</td>
<td style="text-align:left;">
57016/230/5208/3795
</td>
<td style="text-align:right;">
4
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04110
</td>
<td style="text-align:left;">
Cell cycle
</td>
<td style="text-align:left;">
8/171
</td>
<td style="text-align:left;">
126/8105
</td>
<td style="text-align:right;">
0.0050713
</td>
<td style="text-align:right;">
0.0564177
</td>
<td style="text-align:right;">
0.0471537
</td>
<td style="text-align:left;">
1869/896/1870/894/4998/9134/1031/5347
</td>
<td style="text-align:right;">
8
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05166
</td>
<td style="text-align:left;">
Human T-cell leukemia virus 1 infection
</td>
<td style="text-align:left;">
11/171
</td>
<td style="text-align:left;">
222/8105
</td>
<td style="text-align:right;">
0.0072256
</td>
<td style="text-align:right;">
0.0771692
</td>
<td style="text-align:right;">
0.0644978
</td>
<td style="text-align:left;">
3569/1869/896/1870/1437/9586/894/2113/9134/1031/3109
</td>
<td style="text-align:right;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa04010
</td>
<td style="text-align:left;">
MAPK signaling pathway
</td>
<td style="text-align:left;">
13/171
</td>
<td style="text-align:left;">
294/8105
</td>
<td style="text-align:right;">
0.0091875
</td>
<td style="text-align:right;">
0.0943487
</td>
<td style="text-align:right;">
0.0788564
</td>
<td style="text-align:left;">
3552/3553/1850/1950/25780/2261/4803/5801/1649/929/3310/5608/785
</td>
<td style="text-align:right;">
13
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa00340
</td>
<td style="text-align:left;">
Histidine metabolism
</td>
<td style="text-align:left;">
3/171
</td>
<td style="text-align:left;">
22/8105
</td>
<td style="text-align:right;">
0.0105820
</td>
<td style="text-align:right;">
0.1046441
</td>
<td style="text-align:right;">
0.0874613
</td>
<td style="text-align:left;">
218/4129/224
</td>
<td style="text-align:right;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05332
</td>
<td style="text-align:left;">
Graft-versus-host disease
</td>
<td style="text-align:left;">
4/171
</td>
<td style="text-align:left;">
42/8105
</td>
<td style="text-align:right;">
0.0114734
</td>
<td style="text-align:right;">
0.1075446
</td>
<td style="text-align:right;">
0.0898855
</td>
<td style="text-align:left;">
3569/3552/3553/3109
</td>
<td style="text-align:right;">
4
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05410
</td>
<td style="text-align:left;">
Hypertrophic cardiomyopathy
</td>
<td style="text-align:left;">
6/171
</td>
<td style="text-align:left;">
90/8105
</td>
<td style="text-align:right;">
0.0116809
</td>
<td style="text-align:right;">
0.1075446
</td>
<td style="text-align:right;">
0.0898855
</td>
<td style="text-align:left;">
3569/8515/3693/284217/1605/785
</td>
<td style="text-align:right;">
6
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa05222
</td>
<td style="text-align:left;">
Small cell lung cancer
</td>
<td style="text-align:left;">
6/171
</td>
<td style="text-align:left;">
92/8105
</td>
<td style="text-align:right;">
0.0129274
</td>
<td style="text-align:right;">
0.1150542
</td>
<td style="text-align:right;">
0.0961620
</td>
<td style="text-align:left;">
5743/1869/3911/284217/1870/9134
</td>
<td style="text-align:right;">
6
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="visualisation" class="section level1">
<h1>Visualisation</h1>
<pre class="r"><code>big_df=list()
for (i in 1:ncol(contrast)){
  #at each iteration, let x = name of each contrast group
  x=comparison_group[i,]
  
big_df[[x]] &lt;- enrichKEGG_treat[[x]][1:10,]
}

#combine all list from big df into one
df &lt;- as.data.frame(do.call(rbind, lapply(big_df, as.data.frame))) %&gt;% rownames_to_column(&quot;group&quot;) %&gt;% dplyr::select(-c(&quot;BgRatio&quot;, &quot;pvalue&quot;, &quot;geneID&quot;)) 

df &lt;- df %&gt;% separate(col = GeneRatio, into = c(&quot;numerator&quot;, &quot;denomenator&quot;), sep = &quot;/&quot;)
df$numerator &lt;- as.numeric(df$numerator)
df$denomenator &lt;- as.numeric(df$denomenator)
df$&#39;geneRatio&#39; &lt;- df$numerator / df$denomenator
df &lt;- df %&gt;% filter(p.adjust &lt;= 0.05)


#remove GO ID from group names
df$group &lt;- gsub(pattern = &quot;\\.hsa[0-9]*&quot;, &quot;&quot;, df$group)

# Transform the column &#39;Description&#39; into factors
df$Description &lt;- as.factor(df$Description)

# Transform FDR values by -log10(&#39;FDR values&#39;)
df$&#39;|log10(FDR)|&#39; &lt;- -(log10(df$p.adjust))

# Change factor order
df$group &lt;- factor(df$group, levels = c(&quot;CKI-Mac vs CKI&quot;, &quot;CKI-Nme vs CKI&quot;, &quot;CKI-Omt vs CKI&quot;, &quot;CKI-Tri vs CKI&quot;, &quot;UT vs CKI&quot;))
df$Description &lt;- factor(df$Description, levels = rev(levels(df$Description)))

group.labs &lt;- df$group

ggplot(df, aes(x = Description, y = group)) +
  geom_point(data=df,aes(x=Description, y=group, size = geneRatio, colour = `|log10(FDR)|`), alpha=.9)+
  # scale_y_discrete(labels =group.labs)+
  scale_color_gradient(low = &quot;dodgerblue3&quot;, high = &quot;firebrick3&quot;, limits=c(0, NA))+
  coord_flip()+
  theme_bw()+
  theme(axis.ticks.length=unit(-0.1, &quot;cm&quot;),
        axis.text.x = element_text(margin=margin(5,5,0,5,&quot;pt&quot;),angle = 45, hjust = 1),
        axis.text.y = element_text(margin=margin(5,5,5,5,&quot;pt&quot;)),
        axis.text = element_text(color = &quot;black&quot;),
        axis.title.x=element_blank())+
  xlab(label = &quot;&quot;)+
  labs(color=expression(&quot;-log&quot;[10]*&quot;FDR&quot;), size=&quot;Gene Ratio&quot;)</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-6-1.png" width="768" style="display: block; margin: auto;" /></p>
<pre class="r"><code>ggsave(filename = &quot;enrichedKEGG_treat.svg&quot;, width = 7, height = 6.5, plot = last_plot(), path = here::here(&quot;2_plots/4_kegg/&quot;))</code></pre>
<pre class="r"><code>kegg_id &lt;- &quot;hsa04657&quot;
kegg_pathway &lt;- KEGGREST::keggGet(kegg_id)
kegg_pathway_name &lt;- kegg_pathway[[1]]$NAME</code></pre>
<div id="heatmap-of-il-17-signaling-pathway---homo-sapiens-human" class="section level2">
<h2>Heatmap of IL-17 signaling pathway - Homo sapiens (human)</h2>
<div id="partial" class="section level3">
<h3>Partial</h3>
<pre class="r"><code># kegg_id &lt;- &quot;hsa04657&quot;
# kegg_pathway &lt;- KEGGREST::keggGet(kegg_id)
# kegg_pathway_name &lt;- kegg_pathway[[1]]$NAME

entrezID=list()
matrix=list()

for (i in 1:ncol(contrast)){
  #at each iteration, let x = name of each contrast group
  x=comparison_group[i,]
  
  entrezID[[x]] &lt;- enrichKEGG_treat_sig[[x]][,c(&quot;ID&quot;, &quot;geneID&quot;)] %&gt;% as.data.frame()
  entrezID[[x]] &lt;- entrezID[[x]][kegg_id,&quot;geneID&quot;] %&gt;% as.data.frame()
  entrezID[[x]] &lt;- separate_rows(entrezID[[x]],.,sep = &quot;/&quot;)
  colnames(entrezID[[x]]) &lt;- &quot;entrezid&quot;
  
  match &lt;- treat_filtered[[x]]$entrezid %in% entrezID[[x]]$entrezid

  matrix[[x]] &lt;- treat_filtered[[x]][match, c(&quot;gene_name&quot;, &quot;logFC&quot;)]
}

matrix_merged &lt;- matrix$`UT vs CKI` %&gt;% 
  left_join(matrix$`CKI-Mac vs CKI`, by = &quot;gene_name&quot;) %&gt;% 
  left_join(matrix$`CKI-Nme vs CKI`, by = &quot;gene_name&quot;) %&gt;% 
  left_join(matrix$`CKI-Omt vs CKI`, by = &quot;gene_name&quot;) %&gt;% 
  left_join(matrix$`CKI-Tri vs CKI`, by = &quot;gene_name&quot;) %&gt;% 
  column_to_rownames(&quot;gene_name&quot;)

colnames(matrix_merged) &lt;- c(&quot;UT\nvs\nCKI&quot;, &quot;CKI-Mac\nvs\nCKI&quot;, &quot;CKI-Nme\nvs\nCKI&quot;, &quot;CKI-Omt\nvs\nCKI&quot;, &quot;CKI-Tri\nvs\nCKI&quot;)


my_palette &lt;- colorRampPalette(c(
  rgb(32,121,226, maxColorValue = 255),
  # rgb(144,203,180, maxColorValue = 255), 
  rgb(254,238,229, maxColorValue = 255), 
  # rgb(251,192,52, maxColorValue = 255), 
  rgb(226,46,45, maxColorValue = 255)))(n = 201)


pheatmap(mat = matrix_merged,
         clustering_distance_rows = &quot;euclidean&quot;,
         treeheight_row = 70,
         legend_labels = &quot;logFC&quot;,
         main = paste0(&quot;Heatmap of &quot;, kegg_pathway_name),
         # fontsize = 14,
         # fontsize_row = 12,
         # fontsize_number = 10,
         # fontsize_col = 18,
         angle_col =0,
         border_color = &quot;gray70&quot;,
         color = my_palette)</code></pre>
<p><img src="figure/kegg.Rmd/partialPathway-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>invisible(dev.print(svg, here::here(&quot;2_plots/4_kegg/heatmap_partial.svg&quot;)))</code></pre>
</div>
<div id="complete" class="section level3">
<h3>Complete</h3>
<pre class="r"><code># kegg_id &lt;- &quot;hsa04657&quot;
# kegg_pathway &lt;- KEGGREST::keggGet(kegg_id)
# kegg_pathway_name &lt;- kegg_pathway[[1]]$NAME

entrezID=list()
matrix_all=list()

for (i in 1:ncol(contrast)){
  #at each iteration, let x = name of each contrast group
  x=comparison_group[i,]
  
  entrezID[[x]] &lt;- kegg_pathway[[1]]$GENE %&gt;% as.data.frame()
  entrezID[[x]] &lt;- entrezID[[x]][seq(1, nrow(entrezID[[x]]), 2),] %&gt;% as.data.frame()

  colnames(entrezID[[x]]) &lt;- &quot;entrezid&quot;
  
  match &lt;- treat_unfiltered[[x]]$entrezid %in% entrezID[[x]]$entrezid

  matrix_all[[x]] &lt;- treat_unfiltered[[x]][match, c(&quot;gene_name&quot;, &quot;logFC&quot;)]
}

matrix_all_merged &lt;- matrix_all$`UT vs CKI` %&gt;% 
  left_join(matrix_all$`CKI-Mac vs CKI`, by = &quot;gene_name&quot;) %&gt;% 
  left_join(matrix_all$`CKI-Nme vs CKI`, by = &quot;gene_name&quot;) %&gt;% 
  left_join(matrix_all$`CKI-Omt vs CKI`, by = &quot;gene_name&quot;) %&gt;% 
  left_join(matrix_all$`CKI-Tri vs CKI`, by = &quot;gene_name&quot;) %&gt;% 
  column_to_rownames(&quot;gene_name&quot;)

colnames(matrix_all_merged) &lt;- c(&quot;UT vs CKI&quot;, &quot;CKI-Mac vs CKI&quot;, &quot;CKI-Nme vs CKI&quot;, &quot;CKI-Omt vs CKI&quot;, &quot;CKI-Tri vs CKI&quot;)

p &lt;- pheatmap(mat = matrix_all_merged,
         cluster_cols = F,
         clustering_distance_rows = &quot;euclidean&quot;,
         treeheight_row = 70,
         treeheight_col = 50,
         cutree_rows = 4,
         
         # legend_labels = &quot;logFC&quot;,
         main = paste0(&quot;Heatmap of &quot;, kegg_pathway_name),
         # cellwidth = 65,
         # cellheight = 25,
         
         # fontsize = 14,
         # fontsize_row = 12,
         # fontsize_number = 10,
         # fontsize_col = 18,
         angle_col =90,
         
         scale = &quot;row&quot;,
         # legend_breaks = c(seq(-2, 8, by = 2), max(matrix_all_merged)),
         # legend_labels = c(seq(-2, 8, by = 2), &quot;logCPM\n&quot;),
         # border_color = F,
         border_color = &quot;gray70&quot;,
         color = my_palette,
         )</code></pre>
<p><img src="figure/kegg.Rmd/completePathway-1.png" width="768" style="display: block; margin: auto;" /></p>
<pre class="r"><code># ggsave(filename = &quot;heatmap_all.svg&quot;,plot = last_plot(), path = here::here(&quot;2_plots/4_kegg/&quot;))
invisible(dev.print(width = 8.5, height = 18, device = svg, here::here(&quot;2_plots/4_kegg/heatmap_complete.svg&quot;)))</code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.1.0 (2021-05-18)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19043)

Matrix products: default

locale:
[1] LC_COLLATE=English_Australia.1252  LC_CTYPE=English_Australia.1252   
[3] LC_MONETARY=English_Australia.1252 LC_NUMERIC=C                      
[5] LC_TIME=English_Australia.1252    

attached base packages:
 [1] parallel  stats4    grid      stats     graphics  grDevices utils    
 [8] datasets  methods   base     

other attached packages:
 [1] enrichplot_1.12.3     KEGGREST_1.32.0       biomaRt_2.48.3       
 [4] org.Hs.eg.db_3.13.0   AnnotationDbi_1.54.1  IRanges_2.26.0       
 [7] S4Vectors_0.30.0      Biobase_2.52.0        BiocGenerics_0.38.0  
[10] clusterProfiler_4.0.5 Glimma_2.2.0          edgeR_3.34.0         
[13] limma_3.48.1          ggrepel_0.9.1         ggbiplot_0.55        
[16] scales_1.1.1          plyr_1.8.6            ggpubr_0.4.0         
[19] RColorBrewer_1.1-2    pheatmap_1.0.12       cowplot_1.1.1        
[22] pander_0.6.4          kableExtra_1.3.4      forcats_0.5.1        
[25] stringr_1.4.0         purrr_0.3.4           tidyr_1.1.3          
[28] ggplot2_3.3.5         tidyverse_1.3.1       reshape2_1.4.4       
[31] tibble_3.1.2          readr_2.0.0           magrittr_2.0.1       
[34] dplyr_1.0.7          

loaded via a namespace (and not attached):
  [1] utf8_1.2.1                  tidyselect_1.1.1           
  [3] RSQLite_2.2.7               htmlwidgets_1.5.4          
  [5] BiocParallel_1.26.1         scatterpie_0.1.7           
  [7] munsell_0.5.0               statmod_1.4.36             
  [9] withr_2.4.2                 colorspace_2.0-2           
 [11] GOSemSim_2.18.1             filelock_1.0.2             
 [13] highr_0.9                   knitr_1.36                 
 [15] rstudioapi_0.13             ggsignif_0.6.3             
 [17] DOSE_3.18.3                 MatrixGenerics_1.4.3       
 [19] labeling_0.4.2              git2r_0.28.0               
 [21] GenomeInfoDbData_1.2.6      polyclip_1.10-0            
 [23] bit64_4.0.5                 farver_2.1.0               
 [25] rprojroot_2.0.2             downloader_0.4             
 [27] treeio_1.16.2               vctrs_0.3.8                
 [29] generics_0.1.0              xfun_0.24                  
 [31] BiocFileCache_2.0.0         R6_2.5.1                   
 [33] GenomeInfoDb_1.28.4         graphlayouts_0.7.1         
 [35] locfit_1.5-9.4              gridGraphics_0.5-1         
 [37] bitops_1.0-7                cachem_1.0.5               
 [39] fgsea_1.18.0                DelayedArray_0.18.0        
 [41] assertthat_0.2.1            promises_1.2.0.1           
 [43] ggraph_2.0.5                gtable_0.3.0               
 [45] tidygraph_1.2.0             workflowr_1.6.2            
 [47] rlang_0.4.11                genefilter_1.74.0          
 [49] systemfonts_1.0.2           splines_4.1.0              
 [51] lazyeval_0.2.2              rstatix_0.7.0              
 [53] broom_0.7.9                 yaml_2.2.1                 
 [55] abind_1.4-5                 modelr_0.1.8               
 [57] backports_1.2.1             httpuv_1.6.1               
 [59] qvalue_2.24.0               tools_4.1.0                
 [61] ggplotify_0.1.0             ellipsis_0.3.2             
 [63] jquerylib_0.1.4             Rcpp_1.0.6                 
 [65] progress_1.2.2              zlibbioc_1.38.0            
 [67] RCurl_1.98-1.3              prettyunits_1.1.1          
 [69] viridis_0.6.2               SummarizedExperiment_1.22.0
 [71] haven_2.4.3                 fs_1.5.0                   
 [73] here_1.0.1                  data.table_1.14.0          
 [75] DO.db_2.9                   openxlsx_4.2.4             
 [77] reprex_2.0.1                whisker_0.4                
 [79] matrixStats_0.60.1          patchwork_1.1.1            
 [81] hms_1.1.1                   evaluate_0.14              
 [83] xtable_1.8-4                XML_3.99-0.6               
 [85] rio_0.5.27                  readxl_1.3.1               
 [87] gridExtra_2.3               compiler_4.1.0             
 [89] shadowtext_0.0.9            writexl_1.4.0              
 [91] crayon_1.4.1                htmltools_0.5.2            
 [93] ggfun_0.0.4                 later_1.2.0                
 [95] tzdb_0.1.2                  aplot_0.1.1                
 [97] geneplotter_1.70.0          lubridate_1.7.10           
 [99] DBI_1.1.1                   tweenr_1.0.2               
[101] dbplyr_2.1.1                rappdirs_0.3.3             
[103] MASS_7.3-54                 Matrix_1.3-3               
[105] car_3.0-11                  cli_3.0.1                  
[107] igraph_1.2.6                GenomicRanges_1.44.0       
[109] pkgconfig_2.0.3             foreign_0.8-81             
[111] xml2_1.3.2                  ggtree_3.0.4               
[113] svglite_2.0.0               annotate_1.70.0            
[115] bslib_0.3.1                 webshot_0.5.2              
[117] XVector_0.32.0              rvest_1.0.2                
[119] yulab.utils_0.0.4           digest_0.6.27              
[121] Biostrings_2.60.1           rmarkdown_2.11             
[123] cellranger_1.1.0            fastmatch_1.1-3            
[125] tidytree_0.3.5              curl_4.3.2                 
[127] nlme_3.1-152                lifecycle_1.0.1            
[129] jsonlite_1.7.2              carData_3.0-4              
[131] viridisLite_0.4.0           fansi_0.5.0                
[133] pillar_1.6.4                lattice_0.20-44            
[135] fastmap_1.1.0               httr_1.4.2                 
[137] survival_3.2-11             GO.db_3.13.0               
[139] glue_1.4.2                  zip_2.2.0                  
[141] png_0.1-7                   bit_4.0.4                  
[143] ggforce_0.3.3               stringi_1.6.2              
[145] sass_0.4.0                  blob_1.2.2                 
[147] DESeq2_1.32.0               memoise_2.0.0              
[149] ape_5.5                    </code></pre>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
